
<!DOCTYPE html>
<!--[if lt IE 8]>      <html class="no-js nav theme-teal lt-ie10 lt-ie9 lt-ie8  " lang="en-us"> <![endif]-->
<!--[if IE 8]>         <html class="no-js nav theme-teal lt-ie10 lt-ie9  " lang="en-us"> <![endif]-->
<!--[if IE 9]>         <html class="no-js nav theme-teal lt-ie10  " lang="en-us"> <![endif]-->
<!--[if !IE]><!-->     <html class="no-js nav theme-teal  " lang="en-us" > <!--<![endif]-->	
    <head>
        <meta charset="utf-8" />
                		<title>Surveillance Manual | Lab Support | Vaccine Preventable Diseases | CDC</title>
        		
        <meta name="description" content="Vaccines: Laboratory Support for Surveillance of VPDs - Manual for The Surveillance of Vaccine-Preventable Diseases (VPDs) (Chapter 22)" />        <meta name="keywords" content="manual, surveillance manual, surveillance of VPD, vaccine preventable disease surveillance, disease surveillance, lab support for surveillance, laboratory support for surveillance, lab support, Guidelines for Specimen Collection, lab testing" />		<meta name="viewport" content="width=device-width, initial-scale=1" />
		        <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../../TemplatePackage/3.0/images/cdc-touch-icon-144x144.png" />
        <link rel="apple-touch-icon-precomposed" sizes="114x114" href="../../../TemplatePackage/3.0/images/cdc-touch-icon-114x114.png" />
        <link rel="apple-touch-icon-precomposed" sizes="72x72" href="../../../TemplatePackage/3.0/images/cdc-touch-icon-72x72.png" />
        <link rel="apple-touch-icon-precomposed" href="../../../TemplatePackage/3.0/images/cdc-touch-icon-57x57.png" />
        <link rel="shortcut icon" href="../../../TemplatePackage/3.0/images/favicon.ico" />	
		
		<meta name="robots" content="index,follow" />
		<meta property="cdc:template_version" content="3.0" />
		<meta property="cdc:content-id" content="810106"/>
		<meta property="cdc:revision-id" content="1"/>
				<link rel="canonical" href="http://www.cdc.gov/vaccines/pubs/surv-manual/chpt22-lab-support.html"/>
				<!-- Internal Release Version # 3.6.4 -->
                			<link rel="stylesheet" href="../../../TemplatePackage/3.0/css/lib/bootstrap24.css" />
	<link rel="stylesheet" href="../../../TemplatePackage/3.0/css/app.css@3.2.1.7.css" />
	<!--[if gt IE 9]><!--><!--<![endif]-->
	<!--[if IE 7]>
		<link rel="stylesheet" href="/TemplatePackage/3.0/css/lib/font-awesome-ie7.css" />
	<![endif]-->
	<!--[if lt IE 9]>
		<link rel="stylesheet" href="/TemplatePackage/3.0/css/ie.css" />
	<![endif]-->
	<link rel="stylesheet" media="print" href="../../../TemplatePackage/3.0/css/print.css@3.2.1.7.css" />
	<script src="../../../TemplatePackage/3.0/js/libs/modernizr-latest.js"></script>
	<script>
		if(!Date.now){Date.now=function now(){return new Date().getTime()}}
		var page_timing = page_timing || {};
		page_timing.page_start = Date.now();
		function $$(e){if(window.$){e()}else{setTimeout(function(){$$(e)},50)}}
		// (function(){function e(e,t){var n=e.length;while(n--){if(e[n].indexOf(t)>-1){return e[n]}}return false}var t=e(document.getElementsByTagName("html")[0].className.split(" "),"theme-");if(t){var n=document.createElement("link");n.rel="stylesheet";n.type="text/css";n.media="screen";n.href="/TemplatePackage/3.0/css/themes/"+t+".css";document.getElementsByTagName("head")[0].appendChild(n)}})();

		var settings = {}, addthis = null;
	</script>

	<meta name="application-name" content="CDC.gov"/>
	<meta property="cdc:template_version" content="3.2.1.7" />
	<meta name="msapplication-TileColor" content="#0061aa" />
	<meta name="msapplication-square70x70logo" content="/TemplatePackage/3.0/images/Win8_tile_70x70.png" />
	<meta name="msapplication-square150x150logo" content="/TemplatePackage/3.0/images/Win8_tile_150x150.png" />
	<meta name="msapplication-wide310x150logo" content="/TemplatePackage/3.0/images/Win8_tile_310x150.png" />
	<meta name="msapplication-square310x310logo" content="/TemplatePackage/3.0/images/Win8_tile_310x310.png" />
	<meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">		        


      <style type="text/css">@import "../../~inc~/local.css";</style>
  


        <!-- OLD TITLE BEFORE 4/7/2016 <title>CDC - Lab Support for Surveillance of VPDs - Chapter 22 - Vaccine Preventable Diseases (VPDs)</title> -->
<!-- OLD TITLE BEFORE "070711" <title>Vaccines: Pubs/SurvManual/Lab Support Chapter 22</title> -->
<!-- OLD URL BEFORE "070711" http://www.cdc.gov/vaccines/pubs/surv-manual/chpt22-lab-support.htm -->
<!-- OLD CHANNEL NAME remained same at "070711" Vaccines -->
<!-- channel="Vaccines" same channel as of "070711" -->
<!-- Level 1="NCIRD" new level 1 as of "070711" -->
<!-- Level 2="Surveillance" new level 2 as of "070711" -->
<style type="text/css" media="all">
    .cell-shade-teal { background-color: #DBF0EA; }
</style>        		    </head>
    <body>
				



				<div id="skipmenu">
	<a href="chpt22-lab-support.html#searchTarget" class="skippy">Skip directly to search</a>
	<a href="chpt22-lab-support.html#azTab" class="skippy">Skip directly to A to Z list</a>
	<a href="chpt22-lab-support.html#nav-container" class="skippy">Skip directly to navigation</a>
	<a href="chpt22-lab-support.html#share-bar" class="skippy">Skip directly to page options</a>
	<a href="chpt22-lab-support.html#contentArea" class="skippy">Skip directly to site content</a>
</div>		    	<div id="wrapper">
    		
<!-- references to this removed intentionally -->
    		<header id="header">
    			<div class="container">
                    <div class="row">
                    	<div id="logo" class="span14">
                    		<!--iPad touch response and 508 voice reader issue currently under investigation -->
<a href="../../../index.html" title="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People">
	<img src="../../../TemplatePackage/3.0/images/masthead_subpage.svg" alt="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People" class="hidden-one" />
	<img src="../../../TemplatePackage/3.0/images/masthead_1-col.svg" alt="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People" class="visible-one" />
</a>

<!-- IOS9 iPad bug work around - make sure screen readers have a readable version of the logo to read
<a class="tp-sr-only" href="https://www.cdc.gov">Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People</a>
 -->
                    	</div>
                    	<!-- search -->
                    	<div id="searchArea" class="span10 hidden-one hidden-two">							
							
<div class="hidden"><a href="chpt22-lab-support.html#" name="searchTarget">Start of Search Controls</a></div>
<form class="searchForm" method="get" action="https://search.cdc.gov/search">
	<fieldset class="search-form-wrapper ">
		<legend>Search Form Controls</legend>
		<div class="search-form-checkbox">
			<label class="subset"><input type="checkbox" name="subset" checked="checked" value="">Search The CDC</label>
		</div>
		<div class="search-input-wrapper">
			<div class="search-buttons">
				<button class="btn-clear"><span class="sprite-14-close">Cancel</span></button>
				<button class="btn-submit"><span class="sprite-14-search-black">Submit</span></button>
				<input type="submit" value="Submit" style="display:none; visibility:hidden;" /> 
			</div>
			<div class="search-input">
				<label><input type="text" placeholder="SEARCH" name="query" /><span class="hidden">Search The CDC</span></label>
			</div>
		</div>
		<input type="hidden" name="utf8" value="&#x2713;" />
					<input type="hidden" name="affiliate" value="cdc-main" />
			</fieldset>
</form>							<div class="a2z-button pull-right hidden-one hidden-two"><a href="chpt22-lab-support.html#" id="azTab">CDC A-Z Index <span class="icon-angle-down"></span></a></div>
                    	</div>
                    </div>
                    	<div class="hidden-three hidden-four" id="mobile-menu">
		<ul>
			<li id="vp1-menu-button" class="border-bottom-white"><a href="chpt22-lab-support.html#" id="menu-nav"><span class="sprite-16-menu"></span>MENU</a></li>
			<li id="vp1-a2z-button" class="border-bottom-white"><a href="chpt22-lab-support.html#" id="menu-az">CDC A-Z</a></li>
			<li id="vp1-search-button" class="border-bottom-white"><a href="chpt22-lab-support.html#" id="menu-search"><span class="sprite-16-search-white"></span>SEARCH</a></li>
		</ul>
	</div>
<script>if (typeof page_timing !== 'undefined') { page_timing.menu_load = Date.now(); }</script>
                    <div class="a2z-bar hide">
                    	<div>
	<ul>
		<li><a href="../../../az/a.html">A</a></li>
		<li><a href="../../../az/b.html">B</a></li>
		<li><a href="../../../az/c.html">C</a></li>
		<li><a href="../../../az/d.html">D</a></li>
		<li><a href="../../../az/e.html">E</a></li>
		<li><a href="../../../az/f.html">F</a></li>
		<li><a href="../../../az/g.html">G</a></li>
		<li><a href="../../../az/h.html">H</a></li>
		<li><a href="../../../az/i.html">I</a></li>
		<li><a href="../../../az/j.html">J</a></li>
		<li><a href="../../../az/k.html">K</a></li>
		<li><a href="../../../az/l.html">L</a></li>
		<li><a href="../../../az/m.html">M</a></li>
		<li><a href="../../../az/n.html">N</a></li>
		<li><a href="../../../az/o.html">O</a></li>
		<li><a href="../../../az/p.html">P</a></li>
		<li><a href="../../../az/q.html">Q</a></li>
		<li><a href="../../../az/r.html">R</a></li>
		<li><a href="../../../az/s.html">S</a></li>
		<li><a href="../../../az/t.html">T</a></li>
		<li><a href="../../../az/u.html">U</a></li>
		<li><a href="../../../az/v.html">V</a></li>
		<li><a href="../../../az/w.html">W</a></li>
		<li><a href="../../../az/x.html">X</a></li>
		<li><a href="../../../az/y.html">Y</a></li>
		<li><a href="../../../az/z.html">Z</a></li>
		<li><a href="../../../az/0.html">#</a></li>
	</ul>
</div>
<script>if (typeof page_timing !== 'undefined') { page_timing.a2z_load = Date.now(); }</script>                    </div>
                    <div id="searchArea-two-column" class="span24 searchbar hide hidden-three hidden-four">
						  
<div class="hidden"><a href="chpt22-lab-support.html#" name="searchTarget">Start of Search Controls</a></div>
<form class="searchForm" method="get" action="https://search.cdc.gov/search">
	<fieldset class="search-form-wrapper ">
		<legend>Search Form Controls</legend>
		<div class="search-form-checkbox">
			<label class="subset"><input type="checkbox" name="subset" checked="checked" value="">Search The CDC</label>
		</div>
		<div class="search-input-wrapper">
			<div class="search-buttons">
				<button class="btn-clear"><span class="sprite-14-close">Cancel</span></button>
				<button class="btn-submit"><span class="sprite-14-search-black">Submit</span></button>
				<input type="submit" value="Submit" style="display:none; visibility:hidden;" /> 
			</div>
			<div class="search-input">
				<label><input type="text" placeholder="SEARCH" name="query" /><span class="hidden">Search The CDC</span></label>
			</div>
		</div>
		<input type="hidden" name="utf8" value="&#x2713;" />
					<input type="hidden" name="affiliate" value="cdc-main" />
			</fieldset>
</form>					</div>
                    <!--
	See the noscript include for modifying the emergency message. DO NOT add it here.
-->
<script>if (typeof page_timing !== 'undefined') { page_timing.emergency_load = Date.now(); }</script>
    				<div class="titlebar">
    					<h2>


<a href="index.html">Manual for the Surveillance of Vaccine-Preventable Diseases</a>


</h2>
    				</div>
    				<!-- emergency -->
<div id="emergencyContainer">
	<!--
	<div class="alert alert-error en">
		<b>Attention!</b> Uncomment this section and place the emergency message here.
	</div>
	<div class="alert alert-error esp">
		<b>&#161;Atenci&#243;n!</b> Elimine el comentario de esta secci&#243;n y colocar el mensaje de emergencia aqu&#237;.
	</div>	
	-->
</div>

<!-- no script -->
<noscript>
	<div class="alert alert-block"> Note: Javascript is disabled or is not supported by your browser. For this reason, some items on this page will be unavailable. For more information about this message, please visit this page: <a href="../../../other/about_cdcgov.html">About CDC.gov</a>.</div>
</noscript>
<script>if (typeof page_timing !== 'undefined') { page_timing.noscript_load = Date.now(); }</script>
     			</div>
    		</header>
    		<div class="container">				
                <div id="content">
					<div class="row">
        				<!-- left nav -->
        				<div id="nav-container" class="hidden-one hidden-two span5">
    						<nav id="left" class="hidden-one hidden-two">
    							



<h3><a href="index.html">Surveillance Manual Home</a></h3>

<ul id="nav-primary">
<li><a href="front-portion.html">Front Matter</a></li>

<li><a href="chapters.html">Chapters</a>
<ul>
<li><a href="chpt01-dip.html">Chapter 1: Diphtheria</a></li>

<li><a href="chpt02-hib.html">Chapter 2: Haemophilus influenzae type b</a></li>

<li><a href="chpt03-hepa.html">Chapter 3: Hepatitis A</a></li>

<li><a href="chpt04-hepb.html">Chapter 4: Hepatitis B</a></li>

<li><a href="chpt05-hpv.html">Chapter 5: HPV</a></li>

<li><a href="chpt06-influenza.html">Chapter 6: Influenza</a></li>

<li><a href="chpt07-measles.html">Chapter 7: Measles</a></li>

<li><a href="chpt08-mening.html">Chapter 8: Meningococcal</a></li>

<li><a href="chpt09-mumps.html">Chapter 9: Mumps</a></li>

<li><a href="chpt10-pertussis.html">Chapter 10: Pertussis</a></li>

<li><a href="chpt11-pneumo.html">Chapter 11: Pneumococcal</a></li>

<li><a href="chpt12-polio.html">Chapter 12: Polio</a></li>

<li><a href="chpt13-rotavirus.html">Chapter 13: Rotavirus</a></li>

<li><a href="chpt14-rubella.html">Chapter 14: Rubella</a></li>

<li><a href="chpt15-crs.html">Chapter 15: Congenital Rubella Syndrome</a></li>

<li><a href="chpt16-tetanus.html">Chapter 16: Tetanus</a></li>

<li><a href="chpt17-varicella.html">Chapter 17: Varicella</a></li>

<li><a href="chpt18-surv-indicators.html">Chapter 18: Surveillance Indicators</a></li>

<li><a href="chpt19-enhancing-surv.html">Chapter 19: Enhancing Surveillance</a></li>

<li><a href="chpt20-analysis-surv.html">Chapter 20: Analysis of Surveillance Data</a></li>

<li><a href="chpt21-surv-adverse-events.html">Chapter 21: Surveillance for Adverse Events Following Vaccination</a></li>

<li><a href="chpt22-lab-support.html">Chapter 22: Laboratory Support for the Surveillance of Vaccine-Preventable Diseases</a></li>

<li><a href="chpt23-natl-surv-vpd.html">Chapter 23: National Surveillance of Vaccine-Preventable Diseases</a></li>

<li><a href="chpt24-state-specific-surv.html">Chapter 24: State-Specific Surveillance of Vaccine-Preventable Diseases</a></li>
</ul>
</li>

<li><a href="appendix.html">Appendices</a></li>
</ul>


    						</nav>
    						<div class="nav-related">
							


<div class="module-typeG emailupdates">
<span class="sprite-24-govd-icon"><!-- span --></span> <h4>Get Email Updates</h4>
<p>To receive email updates about this page, enter your email address:</p>
<form action="https://public.govdelivery.com/accounts/USCDC/subscribers/qualify" name="govdelivery">
<fieldset>
<input name="topic_id" type="hidden" value="USCDC_11_40">
<label>
<span class="hidden">Enter Email Address</span>
<input class="email" name="email" type="text" value="">
</label>
<a class="explain" href="../../../emailupdates/index.html">What's this?</a>
<a class="button" href="chpt22-lab-support.html#;">Submit</a>

<noscript>&lt;a class="explain" href="https://www.cdc.gov/emailupdates/"&gt;What's this?&lt;/a&gt; &lt;label&gt;&lt;span class="hidden"&gt;Submit Button&lt;/span&gt; &lt;input class="button submit" name="commit" type="submit" value="Submit" /&gt;&lt;/label&gt;</noscript>
</fieldset>
</form>
</div>

    						



	    <div class="module-typeH">
		<h3>Related Links</h3>
		<div>
																	<ul class="list-block">																														<li>					
<a href="../../index.html">Vaccines &amp; Immunizations</a>					</li>										</ul>														</div>
	</div>
	  



							



							</div>    						    				
        				</div>
        				<div id="featureArea">
    						<div class="span19">
    							



    						</div>
        				</div>						
    					<div class="breadcrumbs hidden-one span19">
    						


<div class="bread-crumb hidden-one"></div>

    					</div>
						<div class="span19 pagetitle">	
    						    							<div class='syndicate'>
    						    						    							<div class='mSyndicate'>
    																					<h1>Chapter 22: Laboratory Support for Surveillance of Vaccine-Preventable Diseases</h1>
															    						    							</div>
    						    						    							</div>
    												</div>
						<div id="share-bar" class="span19">		
    						<div class="socialmediabar">
								<div class="row">
            						<div class="hidden-one pull-right">
        								            							            							<div class="pageoptionsContainer formatOptions formatOptionsOff"><span id="po-format-label" class="select-label">Format:</span><div id="po-format" class="format pageoptions pull-right"><div class="select-box"><a href="chpt22-lab-support.html#">Select one</a><span class="icon-angle-down pull-right poarrow"></span></div><ul class="drop-down" aria-labelledby="po-format-label"><li>
        			    	<a class="noDecoration" href="https://www.cdc.gov/vaccines/pubs/surv-manual/chpt22-lab-support.pdf" target="_blank"><span class="sprite-16-pdf"></span> PDF [806 KB]</a>
      </li></ul></div></div>            						</div>	
    								<div class="span8">
    									<div id="socialMediaShareContainer">
	<div class="share_container noLinking">
	 	<a class="share_button_facebook" title="Facebook" href="chpt22-lab-support.html#"><span class="tp-sr-only">Recommend on Facebook</span><span id="share_facebook" class="facebookRecommend"> </span></a>
	 	<a class="share_button_twitter" title="Twitter" href="chpt22-lab-support.html#"><span class="tp-sr-only">Tweet</span><span id="share_twitter" class="twitterTweet"> </span></a>
		<a class="share_button" href="chpt22-lab-support.html#socialMediaShareContainer" ><img src="../../../TemplatePackage/3.0/images/social/addthis-32.png" alt="Share" /><span class="shareButtonEn tp-sr-only" aria-label="Share">Share</span><span class="shareButtonEs tp-sr-only" aria-label="Compartir">Compartir</span></a>
	</div>
</div>

<!-- target anchor for return to top feature in VP1 -->
<a id="content-start" tabindex="-1"></a>
    								</div>
								</div>
    						</div>
    					</div>	
    					<!-- body -->
    					<div id="body" class="span19">
    						<div id="contentArea" >								
        						        														
			<div class="span6 pull-right">
    		<div id="onthispage2" class="module-typeF">
				    			<h4>On this Page</h4>				    			<ul class="list-bullet">
    				    				<li><a href="chpt22-lab-support.html#surv-vpd">Surveillance of Vaccine-Preventable Diseases</a></li>    				    				<li><a href="chpt22-lab-support.html#lab-test">General Guidelines for Specimen Collection and Laboratory Testing</a></li>    				    				<li><a href="chpt22-lab-support.html#specimen">Disease-specific Guidelines for Specimen Collection and Laboratory Testing</a></li>    				    			</ul>
    		</div>
		</div>
	  								        						                                                                                            					            						<div class='syndicate'>
            					            					            						<div class='mSyndicate'>
            					            					        							
<h3>Manual for the Surveillance of Vaccine-Preventable Diseases</h3>

<p><a href="https://www.cdc.gov/vaccines/pubs/surv-manual/chpt22-lab-support.pdf">Printer friendly version [28 pages]</a></p>

<p><span class="bold">Authors:</span> Sandra W. Roush, MT, MPH; Bernard Beall, PhD; Pam Cassiday, MS; Jon Gentsch, PhD; Joe Icenogle, PhD; Leonard Mayer, PhD; Steven M. Oberste, PhD; Daniel C. Payne, PhD, MSPH; Paul Rota, PhD; D. Scott Schmid, PhD; Michael Shaw, PhD; Maria Lucia Tondella, PhD; Annemarie Wasley, ScD</p>

<h2><a class="onThisPageAnchor" id="surv-vpd" name="surv-vpd" title="Surveillance of Vaccine-Preventable Diseases"> </a>Surveillance of Vaccine-Preventable Diseases</h2>

<p>Surveillance for vaccine-preventable diseases (VPDs) requires the close collaboration of clinicians, public health professionals, and laboratorians. Public health surveillance relies on both clinical and laboratory reports of VPDs; therefore, appropriate specimen collection, transport, and laboratory testing are essential. This chapter provides guidelines on specimen collection for each VPD and interpretation of laboratory results.</p>

<p>Each public health professional dealing with vaccine-preventable diseases should identify sources of laboratory support for his or her clinical and public health practice. <span class="bold">Table 1</span> lists appropriate tests for VPDs and provides names and contact information for laboratories and laboratory personnel. In addition to the guidelines presented in this chapter, state health department personnel can provide additional guidance on specimen collection, transport, and other related information.</p>

<h4><a id="t1" name="t1"> </a>Table 1. Contact persons for VPD surveillance laboratory support</h4>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th scope="col">Disease</th>
<th scope="col">Test name</th>
<th scope="col">Lab Name</th>
<th scope="col">Lab contact/Phone</th>
</tr>

</thead>
<tr>
<th scope="row">Diphtheria</th>
<td>
<ul>
<li>Culture</li>

<li>Toxigenicity testing</li>

<li>PCR</li>

<li>Serology (antibodies to diphtheria toxin) <span class="smallFontSize gray4-color">(this test is currently not available at CDC)</span></li>
</ul>
</td>
<td>CDC Pertussis and Diphtheria Laboratory</td>
<td>Dr. M. Lucia Tondella<br />
<a href="mailto:mlt5@cdc.gov">mlt5@cdc.gov</a><br />
404-639-1239<br />
<br />
or<br />
<br />
Ms. Pam Cassiday<br />
<a href="mailto:pxc1@cdc.gov">pxc1@cdc.gov</a><br />
404-639-1231<br />
<br />
FAX: 404-639-4421</td>
</tr>

<tr>
<th scope="row"><em>Haemophilus influenzae</em></th>
<td>
<ul>
<li>Culture</li>

<li>Serotyping</li>

<li>Antigen detection</li>

<li>Subtyping</li>
</ul>
</td>
<td>CDC Meningitis Laboratory</td>
<td>
<p>Dr. Leonard Mayer<br />
<a href="mailto:LWM1@cdc.gov">lwm1@cdc.gov</a><br />
404-639-2841</p>

<p>FAX: 404-639-4421</p>
</td>
</tr>

<tr>
<th scope="row">Hepatitis A</th>
<td>&#160;</td>
<td>Hepatitis Reference Laboratory</td>
<td>
<p>Ruth Jiles<br />
<a href="mailto:rxg0@cdc.gov">rxg0@cdc.gov</a><br />
404-639-2339</p>

<p>FAX: 404-639-1563</p>
</td>
</tr>

<tr>
<th scope="row">Hepatitis B</th>
<td>&#160;</td>
<td>Hepatitis Reference Laboratory</td>
<td>
<p>Ruth Jiles<br />
<a href="mailto:rxg0@cdc.gov">rxg0@cdc.gov</a><br />
404-639-2339</p>

<p>FAX: 404-639-1563</p>
</td>
</tr>

<tr>
<th scope="row">Influenza</th>
<td>
<ul>
<li>Culture/viral isolation</li>

<li>Antigen detection</li>

<li>RT-PCR/<br />
real time RT-PCR</li>

<li>Serology</li>
</ul>
</td>
<td>CDC Influenza Surveillance and Diagnosis Laboratory</td>
<td>
<p>Dr. Michael Shaw<br />
<a href="mailto:mws2@cdc.gov">mws2@cdc.gov</a><br />
404-639-1405</p>

<p>FAX: 404-639-2350</p>
</td>
</tr>

<tr>
<th scope="row">Measles</th>
<td>
<ul>
<li>IgM antibody</li>

<li>IgG antibody</li>

<li>Culture</li>

<li>PCR</li>
</ul>
</td>
<td>CDC MMR &#38; Herpes Virus Laboratory</td>
<td>
<p>Dr. Paul Rota<br />
<a href="mailto:par1@cdc.gov">par1@cdc.gov</a><br />
404-639-4181</p>

<p>FAX: 404-639-4187</p>
</td>
</tr>

<tr>
<th scope="row">Meningococcal disease</th>
<td>
<ul>
<li>Culture</li>

<li>SASG</li>

<li>PCR</li>

<li>Susceptibility testing</li>

<li>Molecular genotyping<br />
(PFGE, MLST, etc.)</li>
</ul>
</td>
<td>CDC Meningitis Laboratory</td>
<td>
<p>Dr. Leonard Mayer<br />
<a href="mailto:LWM1@cdc.gov">lwm1@cdc.gov</a><br />
404-639-2841</p>

<p>FAX: 404-639-4421</p>
</td>
</tr>

<tr>
<th scope="row">Mumps</th>
<td>
<ul>
<li>Culture</li>

<li>IgM antibody</li>

<li>IgG antibody</li>
</ul>
</td>
<td>CDC MMR &#38; Herpes Virus Laboratory</td>
<td>
<p>Dr. Paul Rota<br />
<a href="mailto:par1@cdc.gov">par1@cdc.gov</a><br />
404-639-4181</p>

<p>FAX: 404-639-4187</p>
</td>
</tr>

<tr>
<th scope="row">Pertussis</th>
<td>
<ul>
<li>Culture</li>

<li>PCR</li>
</ul>
</td>
<td>CDC Pertussis and Diphtheria Laboratory</td>
<td>
<p>Dr. M. Lucia Tondella<br />
<a href="mailto:mlt5@cdc.gov">mlt5@cdc.gov</a><br />
404-639-1239<br />
<br />
or<br />
<br />
Dr. Kathy Tatti<br />
<a href="mailto:ket2@cdc.gov">ket2@cdc.gov</a><br />
404-639-3797</p>

<p>FAX: 404-639-4421</p>
</td>
</tr>

<tr>
<th scope="row">Pneumococcal disease</th>
<td>
<ul>
<li>Culture</li>

<li>PCR</li>

<li>Susceptibility testing</li>

<li>Serotyping, (conventional or PCR-based)*</li>

<li>Genotyping</li>

<li>Antibiotic resistance</li>
</ul>
</td>
<td>CDC Streptococcus Laboratory</td>
<td>
<p>Dr. Bernard Beall<br />
BBEALL@cdc.gov<br />
404-639-1237<br />
or<br />
Dr. Gloria Carvalho<br />
MCarvalho@cdc.gov<br />
404-639-3558 (except for antibiotic resistance testing)</p>

<p>FAX: 404-639-2070</p>
</td>
</tr>

<tr>
<th scope="row">Poliomyelitis</th>
<td>
<ul>
<li>Culture</li>

<li>Intratypic differentiation</li>

<li>Serology</li>
</ul>
</td>
<td>CDC Polio/Picornavirus Laboratory</td>
<td>
<p>Dr. Steve Oberste<br />
<a href="mailto:soberste@cdc.gov">soberste@cdc.gov</a><br />
404-639-5497</p>

<p>FAX: 404-639-4011</p>
</td>
</tr>

<tr>
<th scope="row">Rotavirus</th>
<td>
<ul>
<li>Antigen EIA</li>

<li>Intratypic differentiation</li>

<li>Serology</li>

<li>Culture</li>
</ul>
</td>
<td>CDC Gastroenteritis Virus Laboratory</td>
<td>
<p>Dr Jon Gentsch<br />
<a href="mailto:jgentsch@cdc.gov">jgentsch@cdc.gov</a><br />
404-639-2860</p>

<p>FAX: 404-639-3645</p>
</td>
</tr>

<tr>
<th scope="row">Rubella</th>
<td>
<ul>
<li>IgM antibody</li>

<li>IgG antibody</li>

<li>Culture</li>

<li>PCR</li>
</ul>
</td>
<td>CDC MMR &#38; Herpes Virus Laboratory</td>
<td>
<p>Dr. Joe Icenogle<br />
<a href="mailto:jicenogle@cdc.gov">jicenogle@cdc.gov</a><br />
404-639-4557</p>

<p>FAX: 404-639-1516</p>
</td>
</tr>

<tr>
<th scope="row">Congenital rubella syndrome</th>
<td>
<ul>
<li>IgM antibody</li>

<li>IgG antibody</li>

<li>Culture</li>

<li>PCR</li>

<li>Serology</li>
</ul>
</td>
<td>CDC MMR &#38; Herpes Virus Laboratory</td>
<td>
<p>Dr. Joe Icenogle<br />
<a href="mailto:jicenogle@cdc.gov">jicenogle@cdc.gov</a><br />
404-639-4557</p>

<p>FAX: 404-639-1516</p>
</td>
</tr>

<tr>
<th scope="row">Varicella</th>
<td>
<ul>
<li>DFA</li>

<li>Culture</li>

<li>Viral typing/ strain identification</li>
</ul>
</td>
<td>National VZV Laboratory</td>
<td>
<p>Dr. Scott Schmid<br />
<a href="mailto:sschmid@cdc.gov">sschmid@cdc.gov</a><br />
404-639-0066</p>

<p>FAX: 404-639-4056</p>
</td>
</tr>
</table>

<!--END TABLE-->
<p class="bold smallFontSize">*Note for Pneumococcal Serotyping (conventional or PCR-based): Provide typing of isolates of <em>S. pneumoniae</em> only in the setting of an outbreak. PCR-based serotyping can be performed using commercially available supplies.</p>

<hr />
<p><span class="toTop"><span class="icon-angle-up"><!-- --></span><a href="chpt22-lab-support.html#">Top of Page</a></span></p>

<h2><a class="onThisPageAnchor" id="lab-test" name="lab-test" title="General Guidelines for Specimen Collection and Laboratory Testing"> </a>General Guidelines for Specimen Collection and Laboratory Testing</h2>

<p>Specimen collection and shipping are important steps in obtaining laboratory diagnosis or confirmation for VPDs. Guidelines have been published for specimen collection and handling for viral and microbiologic agents.<a href="chpt22-lab-support.html#f1">[1-3]</a> Information is also available on using CDC laboratories as support for reference and disease surveillance;<a href="chpt22-lab-support.html#f4">[4, 5]</a> this includes the form required for submitting specimens to CDC (See Appendix 23, Form # CDC 0.5034) and information on general requirements for shipment of etiologic agents (<a href="https://www.cdc.gov/vaccines/pubs/surv-manual/appx/appendix24-etiologic-agent.pdf">Appendix 24 [4 pages]</a>). Although written to guide specimen submission to CDC, this information may be applicable to the submission of specimens to other laboratories.</p>

<p><span class="toTop"><span class="icon-angle-up"><!-- --></span>&#160;<a href="chpt22-lab-support.html#">Top of Page</a></span></p>

<h2><a class="onThisPageAnchor" id="specimen" name="specimen" title="Disease-specific Guidelines for Specimen Collection and Laboratory Testing"> </a>Disease-specific Guidelines for Specimen Collection and Laboratory Testing</h2>

<p>This chapter provides a quick reference summary of the laboratory information from Chapters 1-17 of this manual. Table 2 lists confirmatory and other useful tests for surveillance of vaccine-preventable diseases.</p>

<h4><a id="t2" name="t2"> </a>Table 2. Confirmatory and other useful tests for the surveillance of vaccine-preventable diseases</h4>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th scope="col">Disease</th>
<th scope="col">Confirmatory tests</th>
<th scope="col">Other useful tests</th>
</tr>

</thead>
<tr>
<th scope="col">Diphtheria</th>
<td>Culture<br />
Toxigenicity testing</td>
<td>PCR<br />
Serology (antibodies to diphtheria toxin)</td>
</tr>

<tr>
<th scope="col"><em>Haemophilus influenzae</em></th>
<td>Culture</td>
<td>Serotyping (identification of capsular type of encapsulated strains)<br />
Antigen detection<br />
Subtyping</td>
</tr>

<tr>
<th scope="col">Hepatitis A</th>
<td>IgM anti-HAV (positive)</td>
<td>Total anti-HAV (marker of immunity)<br />
PCR</td>
</tr>

<tr>
<th scope="col">Hepatitis B</th>
<td>IgM anti-HBc (acute infection)<br />
HBsAg (acute or chronic infection)*</td>
<td>Anti-HBs (marker of immunity)<br />
Total anti-HBc (marker of past or present infection)</td>
</tr>

<tr>
<th scope="col">Influenza</th>
<td>Culture<br />
Antigen detection (EIA, IFA, EM)<br />
Serology<br />
PCR</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="col">Measles</th>
<td>IgM<br />
Paired sera for IgG</td>
<td>Culture (for molecular epi)<br />
PCR</td>
</tr>

<tr>
<th scope="col">Meningococcal disease</th>
<td>Culture</td>
<td>Serogroup-specific PCR<br />
Slide agglutination serogrouping<br />
PCR</td>
</tr>

<tr>
<th scope="col">Mumps</th>
<td>Culture<br />
IgM<br />
IgG</td>
<td>IgG (for immunity testing)</td>
</tr>

<tr>
<th scope="col">Pertussis</th>
<td>Culture<br />
PCR</td>
<td>Serology</td>
</tr>

<tr>
<th scope="col">Pneumococcal disease</th>
<td>Culture<br />
PCR</td>
<td>Antibiotic resistance<br />
- serotyping<br />
- PCR deduction of serotypes<br />
- strain identification (MLST,PFGE)</td>
</tr>

<tr>
<th scope="col">Poliomyelitis</th>
<td>Culture-from stool, pharynx,<br />
or CSF</td>
<td>Intratypic differentiation (wild vs. vaccine type)<br />
Paired serology<br />
CSF analysis</td>
</tr>

<tr>
<th scope="col">Rotavirus</th>
<td>Culture<br />
Paired serology</td>
<td>Nucleic acid electrophoresis<br />
PCR genotyping</td>
</tr>

<tr>
<th scope="col">Rubella</th>
<td>Paired sera for IgG<br />
IgM<br />
Culture</td>
<td>PCR</td>
</tr>

<tr>
<th scope="col">Tetanus</th>
<td>There are no lab findings characteristic of tetanus</td>
<td>Serology (for immunity testing)</td>
</tr>

<tr>
<th scope="col">Varicella</th>
<td>Culture<br />
Serology</td>
<td>Viral typing/strain identification<br />
DFA</td>
</tr>
</table>

<p class="bold smallFontSize">* Confirmation of HBsAg positive results by HBsAg neutralization assay should be performed as specified in test package insert.</p>

<hr />
<p><span class="bold">Table 3</span> summarizes specimen collection procedures for laboratory testing. Because some specimens require different handling procedures, be sure to check with the diagnostic laboratory prior to shipping. When in doubt about what specimens to collect, timing of specimen collection, or where or how to transport specimens, call the state health department and the state laboratory.</p>

<h4><a id="t3" name="t3"> </a>Table 3. Specimen collection for laboratory testing for VPDs</h4>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th scope="col">Disease</th>
<th scope="col">Test name</th>
<th scope="col">Specimens to take</th>
<th scope="col">Timing for specimen collection</th>
<th scope="col">Transport requirements</th>
<th scope="col">Collection requirements</th>
</tr>

</thead>
<tr class="cell-shade-teal">
<th scope="row">Diphtheria</th>
<td>
<p>Culture</p>

<p><span class="bold">Note:</span> ALERT lab that diphtheria is suspected, so that tellurite-containing media will be used.</p>
</td>
<td>Swab of nose, throat, membrane</td>
<td>ASAP, when diphtheria is suspected</td>
<td>&#60; 24 hrs: Amies’ or similar transport medium<br />
≥24 hrs: silica gel sachets</td>
<td>State health departments may call CDC diphtheria lab at 404-639-1231 or 404-639-1239.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Diphtheria</th>
<td>
<p>PCR</p>

<p><span class="bold">Note:</span> ALERT lab that diphtheria is suspected, so that specific PCR assay will be used.</p>
</td>
<td>Swabs (as above), pieces of membrane, biopsy tissue</td>
<td>Take these specimens at same time as those for culture.</td>
<td>Silica gel sachet; or a sterile dry container at 4°C</td>
<td>State health departments may call CDC diphtheria lab at 404-639-1231 or 404-639-1239.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Diphtheria</th>
<td>Toxigenicity testing (Elek test)</td>
<td>Isolate from culture (above)</td>
<td>After <em>C. diphtheriae</em> has been isolated</td>
<td>Transport medium such as Amies medium, or silica gel sachets</td>
<td>State health departments may call CDC diphtheria lab at 404-639-1231 or 404-639-1239.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Diphtheria</th>
<td>
<p>Serology (antibodies to diphtheria toxin)</p>

<p><span class="bold">Note:</span> Collect paired sera, taken 2-3 weeks apart.<br />
This test is currently not available at CDC.</p>
</td>
<td>Serum</td>
<td>Before administration of antitoxin</td>
<td>Frozen (-20°C)</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row"><em>Haemophilus influenzae</em> type b</th>
<td>
<p>Culture</p>

<p><span class="bold">Note:</span> Request that lab conduct serotyping on any <em>H. influenzae</em> isolate from any normally sterile site.</p>
</td>
<td>Blood</td>
<td>ASAP</td>
<td>Blood culture bottles w/broth or lysis-centrifugation tube</td>
<td>Collect 3 separate samples in a 24-hr period.</td>
</tr>

<tr>
<th scope="row"><em>Haemophilus influenzae</em> type b</th>
<td>
<p>Culture</p>

<p><span class="bold">Note:</span> Request that lab conduct serotyping on any <em>H. influenzae</em> isolate from any normally sterile site.</p>
</td>
<td>CSF</td>
<td>ASAP</td>
<td>Sterile, screw-capped tube</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row"><em>Haemophilus influenzae</em> type b</th>
<td>Culture</td>
<td>Other normally sterile site</td>
<td>ASAP</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row"><em>Haemophilus influenzae</em> type b</th>
<td>Serotyping</td>
<td>Isolate from culture (above)</td>
<td>&#160;</td>
<td>&#160;</td>
<td>Highest priority are isolates from persons &#60;15 years.</td>
</tr>

<tr>
<th scope="row"><em>Haemophilus influenzae</em> type b</th>
<td>Antigen detection</td>
<td>Any normally sterile site</td>
<td>ASAP</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Hepatitis A</th>
<td>IgM anti-HAV</td>
<td>Serum</td>
<td>ASAP after symptom onset (detectable up to 6 months)</td>
<td>All sera to be tested for serologic markers of HAV and HBV infection can be kept at ambient temperatures, refrigerated, or frozen for short term (&#60;48 hours). For longer than 48 hours storage, sera should be frozen or refrigerated.</td>
<td>Non-hemolyzed</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Hepatitis A</th>
<td>
<p>Total anti-HAV</p>

<p><span class="bold">Note:</span> Measures both IgM and IgG.</p>
</td>
<td>Serum</td>
<td>No time limit</td>
<td>&#160;</td>
<td>Non-hemolyzed</td>
</tr>

<tr>
<th scope="row">Hepatitis B</th>
<td>IgM anti-HBc</td>
<td>Serum</td>
<td>ASAP after symptom onset (Detectable up to 6 months)</td>
<td>&#160;</td>
<td>Non-hemolyzed</td>
</tr>

<tr>
<th scope="row">Hepatitis B</th>
<td>
<p>HBsAg</p>

<p><span class="bold">Note:</span> HBsAg-positive results should be confirmed by HBsAg neutralization assay as specified in the package insert for each assay.</p>
</td>
<td>Serum</td>
<td>&#160;</td>
<td>&#160;</td>
<td>Non-hemolyzed</td>
</tr>

<tr>
<th scope="row">Hepatitis B</th>
<td>Anti-HBs</td>
<td>Serum</td>
<td>1-2 months after vaccination</td>
<td>&#160;</td>
<td>Non-hemolyzed</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Influenza</th>
<td>Culture/viral isolation</td>
<td>Nasal wash, nasopharyngeal aspirates, nasal/throat swabs, transtracheal aspirate, bronchoalveolar lavage</td>
<td>Within 72 hours of onset of illness</td>
<td>Transport specimens at 4°C if tests are to be performed within 72 hours; otherwise, freeze at -70°C until tests can be performed.</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Influenza</th>
<td>Antigen detection and RT-PCR</td>
<td>Nasal wash, nasopharyngeal aspirate, nasal/throat swabs, gargling fluid, transtracheal aspirates, bronchoalveolar lavage</td>
<td>Within 72 hours of onset of illness</td>
<td>Transport specimens at 4°C if tests are to be performed within 72 hours; otherwise, freeze at -70°C until tests can be performed.</td>
<td><span class="bold">Note:</span> Save an aliquot of the clinical sample for confirmation and isolation. Viral isolates may be further characterized by WHO/CDC.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Influenza</th>
<td>Serology</td>
<td>Paired sera</td>
<td>Acute: within 1 week of onset<br />
Convalescent: 2-3 weeks after acute</td>
<td>Store at 4°C or frozen</td>
<td><span class="bold">Note:</span> Fourfold rise is a positive result. Consider vaccination history.</td>
</tr>

<tr>
<th scope="row">Measles</th>
<td>Culture/PCR</td>
<td>Nasopharyngeal aspirates, throat swabs, urine, heparinized blood</td>
<td>Collect at same time as samples for serology (best within 3 days of rash onset)</td>
<td>&#160;</td>
<td><span class="bold">Note:</span> PCR for molecular typing. Do not collect if after 10 days from rash onset.</td>
</tr>

<tr>
<th scope="row">Measles</th>
<td>IgM antibody</td>
<td>Serum</td>
<td>ASAP, and repeat 72 hours after onset if first negative</td>
<td>&#160;</td>
<td><span class="bold">Note:</span> IgM is detectable for at least 30 days after rash onset.</td>
</tr>

<tr>
<th scope="row">Measles</th>
<td>IgG antibody</td>
<td>Paired sera</td>
<td>Acute: ASAP after rash onset (7 days at the latest)<br />
Convalescent: 14-30 days after acute</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Meningococcal disease</th>
<td>Culture*</td>
<td>Blood</td>
<td>ASAP</td>
<td>TI medium is preferred.<br />
Blood culture bottles w/broth or lysis-centrifugation tube</td>
<td><span class="bold">Note:</span> Request that lab conduct serogrouping on any <em>N. meningitidis</em> isolate from any normally sterile site.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Meningococcal disease</th>
<td>Culture*</td>
<td>CSF</td>
<td>ASAP</td>
<td>TI medium is preferred. Sterile, screw-capped tube</td>
<td><span class="bold">Note:</span> Request that lab conduct serogrouping on any <em>N. meningitidis</em> isolate from any normally sterile site.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Meningococcal disease</th>
<td>Culture*</td>
<td>Other normally sterile site</td>
<td>ASAP</td>
<td>TI medium is preferred.</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Meningococcal disease</th>
<td>Serogrouping</td>
<td>Isolate from culture (above)</td>
<td>&#160;</td>
<td>Slant, frozen, lyophilized or silica gel pack.</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Meningococcal disease</th>
<td>PCR</td>
<td>Any normally sterile site</td>
<td>ASAP</td>
<td>Sent frozen on blue ice packs.</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Mumps</th>
<td>Culture</td>
<td>Buccal /parotid swabs, CSF</td>
<td>&#160;</td>
<td>&#160;</td>
<td>Massage the salivary/parotid gland area for 30 seconds prior to swab collection.</td>
</tr>

<tr>
<th scope="row">Mumps</th>
<td>IgM antibody</td>
<td>Serum</td>
<td>ASAP; antibodies peak about a week after onset</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Mumps</th>
<td>IgG antibody</td>
<td>Paired sera</td>
<td>Acute: within several days of onset<br />
Convalescent: 2 weeks after acute</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Pertussis</th>
<td>
<p>Culture</p>

<p><span class="bold">Note:</span> Inoculate selective primary isolation media such as charcoal horse blood agar or Bordet-Gengou as soon as possible.<br />
A negative culture does NOT rule out pertussis.</p>
</td>
<td>Posterior nasopharyngeal swab or aspirate</td>
<td>Within the first 2 weeks of cough onset</td>
<td>Swabs: half-strength charcoal horse blood agar at 4°C<br />
Aspirates: in catheter trap at 4°C</td>
<td>Use Dacron or calcium alginate (not cotton) swabs with flexible shaft or aspiration by catheter attached to catheter trap.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Pertussis</th>
<td>
<p>PCR</p>

<p><span class="bold">Note:</span> PCR should be validated with culture when possible.</p>
</td>
<td>Nasopharyngeal swab or aspirate</td>
<td>Within the first 2 weeks of cough onset</td>
<td>Short term at 4°C; long term -20°C or below</td>
<td>Use Dacron (not calcium alginate or cotton) swabs with flexible shaft or aspiration by catheter attached to catheter trap.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Pertussis</th>
<td>Serology</td>
<td>Acute and convalescent sera</td>
<td>Acute: within the first 2 weeks of cough onset<br />
Convalescent: 3-9 weeks after acute</td>
<td>-20°C</td>
<td><span class="bold">Note:</span> Results are presumptive and should be validated with culture. Serologic results are currently not accepted as laboratory confirmation for purposes of national surveillance.</td>
</tr>

<tr>
<th scope="row">Pneumococcal disease</th>
<td>Culture</td>
<td>Normally sterile site</td>
<td>As soon as possible after onset of clinical illness but before administration of antibiotics</td>
<td>Blood culture bottles w/broth or lysis-centrifugation tube or, if from another sterile site, a sterile, screw-capped tube</td>
<td>Collect 2 separate blood samples in a 24-hr period. Most other sterile specimens (e.g., CSF) are collected only once.</td>
</tr>

<tr>
<th scope="row">Pneumococcal disease</th>
<td>PCR</td>
<td>Normally sterile site</td>
<td>ASAP, soon after administration of antibiotics is a viable option.</td>
<td>Send specimen frozen on blue ice packs</td>
<td>PCR</td>
</tr>

<tr>
<th scope="row">Pneumococcal disease</th>
<td>PCR deduction of serotype</td>
<td>Culture-negative sterile site specimen</td>
<td>Specimen frozen immediately</td>
<td>&#160;</td>
<td>PCR deduction of serotype</td>
</tr>

<tr>
<th scope="row">Pneumococcal disease</th>
<td>Susceptibility testing</td>
<td>Pure culture</td>
<td>&#160;</td>
<td>Slant, frozen, or silica packet</td>
<td>Susceptibility testing</td>
</tr>

<tr>
<th scope="row">Pneumococcal disease</th>
<td>Serotyping</td>
<td>Pure culture</td>
<td>&#160;</td>
<td>Slant, frozen, or silica packet</td>
<td>Serotyping</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Poliomyelitis</th>
<td>Culture</td>
<td>Stool, pharyngeal swab, CSF</td>
<td>Acute</td>
<td>Sterile, screw-capped container</td>
<td>
<p>No carrier for stool; saline buffer for swabs</p>

<p><span class="bold">Note:</span> Maintain frozen or transport rapidly to lab; avoid desiccation of swab specimens.</p>
</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Poliomyelitis</th>
<td>Intratypic differentiation</td>
<td>Isolate from culture (above)</td>
<td>&#160;</td>
<td>&#160;</td>
<td><span class="bold">Note:</span> Maintain frozen or transport rapidly to lab; avoid desiccation of swab specimens.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Poliomyelitis</th>
<td>Serology</td>
<td>Paired sera</td>
<td>Acute: ASAP<br />
Convalescent: 3 weeks after acute</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Rotavirus gastroenteritis</th>
<td>EIA, PCR genotyping</td>
<td>Stool, sera if stool not available</td>
<td>First to fourth day of illness optimal (stool); third to seventh day (serum)</td>
<td>Sterile, screw-capped container</td>
<td>
<p>Bulk stool, whole serum</p>

<p><span class="bold">Note:</span> Keep frozen or transport rapidly to lab; avoid multiple freeze-thaw cycles.</p>
</td>
</tr>

<tr>
<th scope="row">Rotavirus gastroenteritis</th>
<td>Culture, RNA electrophoresis, EM</td>
<td>Stool</td>
<td>First to fourth day of illness optimal</td>
<td>Sterile, screw-capped container</td>
<td>
<p>Bulk stool, whole serum</p>

<p><span class="bold">Note:</span> Keep frozen or transport rapidly to lab; avoid multiple freeze-thaw cycles.</p>
</td>
</tr>

<tr>
<th scope="row">Rotavirus gastroenteritis</th>
<td>Serology</td>
<td>Paired sera</td>
<td>Acute: ASAP<br />
Convalescent: 3 weeks after acute</td>
<td>Sterile, screw-capped container</td>
<td>Whole serum</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Rotavirus-associated seizures</th>
<td>PCR</td>
<td>CSF</td>
<td>ASAP after symptoms begin</td>
<td>Sterile, screw-capped container</td>
<td>
<p>No carrier</p>

<p><span class="bold">Note:</span> Keep frozen or transport rapidly to lab; avoid multiple freeze-thaw cycles.</p>
</td>
</tr>

<tr>
<th scope="row">Rubella</th>
<td>IgM antibody</td>
<td>Serum</td>
<td>Within 7-10 days of onset</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Rubella</th>
<td>IgG antibody</td>
<td>Paired sera</td>
<td>Acute: within 7-10 days of onset of illness<br />
Convalescent: 2-3 weeks after acute</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Rubella</th>
<td>Culture/PCR</td>
<td>Nasopharyngeal swab/wash, throat, urine.</td>
<td>Within 4 days of onset of rash</td>
<td>Viral transport media</td>
<td><span class="bold">Note:</span> Maintain frozen<br />
(except urine) or transport rapidly to lab; avoid desiccation of swab specimens.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Congenital rubella syndrome (CRS)</th>
<td>IgM antibody</td>
<td>Serum</td>
<td>As soon as possible, within 6 months of birth</td>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Congenital rubella syndrome (CRS)</th>
<td>IgG antibody</td>
<td>Paired sera</td>
<td>&#160;</td>
<td>&#160;</td>
<td><span class="bold">Note:</span> Confirmation is by documenting persistence of serum IgG titer beyond the time expected from passive transfer of maternal IgG antibody.</td>
</tr>

<tr class="cell-shade-teal">
<th scope="row">Congenital rubella syndrome (CRS)</th>
<td>Culture/PCR</td>
<td>Nasopharyngeal swab/wash, urine, blood, cataracts</td>
<td>As soon as possible; every 1-3 months until cultures are repeatedly negative</td>
<td>Viral transport media</td>
<td><span class="bold">Note:</span> Maintain frozen (except urine) or transport rapidly to lab; avoid desiccation of swab specimens.</td>
</tr>

<tr>
<th scope="row">Varicella</th>
<td>Serology</td>
<td>Serum</td>
<td>Immune status: collect anytime except during acute illness<br />
Paired serologic diagnosis: acute within 7-10 days of onset; convalescent 2-3 weeks after acute</td>
<td>&#160;</td>
<td>Single IgG assay is useful to assess immune status.<br />
<br />
Paired serum used to identify recent infection, but is not the method of choice when rapid diagnosis needed.</td>
</tr>

<tr>
<th scope="row">Varicella</th>
<td>Direct immuno-fluorescent antibody (DFA)</td>
<td>Scraping/swab from base of vesicle</td>
<td>Acute illness 2-3 days after rash onset and fresh vesicles</td>
<td>&#160;</td>
<td><span class="bold">Note:</span> Used for rapid diagnosis.</td>
</tr>

<tr>
<th scope="row">Varicella</th>
<td>Culture</td>
<td>Fluid from vesicles, nasal or throat swabs, serum, spinal fluid, urine, bronchial tree washing or inflamed joints</td>
<td>Acute illness 2-3 days after rash onset and fresh vesicles</td>
<td>&#160;</td>
<td><span class="bold">Note:</span> Definitive diagnosis, but not useful for rapid diagnosis.</td>
</tr>

<tr>
<th scope="row">Varicella</th>
<td>Viral typing/strain identification</td>
<td>Viral isolate (from culture)</td>
<td>Within 2-3 days of rash onset</td>
<td>Storage more than a few hours must be kept on dry ice or frozen at -70°C or below</td>
<td><span class="bold">Note:</span> Merck and Co., Inc., offers a free viral identification service using PCR analysis (1-800-672-6372).</td>
</tr>
</table>

<p class="bold smallFontSize">* <em>Neisseria meningitidis</em> culture cannot be performed on specimens sent to CDC, but CDC is available to provide advice and answer questions on culture methods.</p>

<hr />
<h3>A. Diphtheria (see <a href="chpt01-dip.html">Chapter 1</a>)</h3>

<p>Diagnostic tests used to confirm infection include isolation of <em>Corynebacterium diphtheriae</em> on culture and toxigenicity testing. Although no other tests for diagnosing diphtheria are commercially available, CDC can perform a polymerase chain reaction (PCR) test on clinical specimens to confirm infection with a potentially toxigenic strain. PCR can detect nonviable <em>C. diphtheriae</em> organisms from specimens taken after antibiotic therapy has been initiated.</p>

<p>Although PCR for the diphtheria toxin gene and its regulatory element, as performed by the CDC Pertussis and Diphtheria Laboratory, provides supportive evidence for the diagnosis, data are not yet sufficient for PCR to be accepted as a criterion for laboratory confirmation. At present, a case that is PCR positive without the isolation of the organism or histopathologic diagnosis or without epidemiologic linkage to a laboratory-confirmed case should be classified as a probable case.</p>

<h4>Isolation of <em>C. diphtheriae</em> by culture</h4>

<p>Isolation of <em>C. diphtheriae</em> by bacteriological culture is essential for confirming diphtheria. The following should be considered:</p>

<ul>
<li>A clinical specimen for culture should be obtained as soon as possible when diphtheria (involving any site) is suspected, even if treatment with antibiotics has already begun.</li>

<li>Specimens should be taken from the nose and throat, and from the diphtheritic membrane.<br />
If possible, swabs also should be taken from beneath the membrane.</li>

<li>The laboratory should be alerted to the suspicion of diphtheria because isolation of <em>C. diphtheriae</em> requires special culture media containing tellurite.</li>

<li>Isolation of <em>C. diphtheriae</em> from close contacts may confirm the diagnosis of the case, even<br />
if the patient’s culture is negative.</li>
</ul>

<p>All suspected cases and their close contacts should supply specimens from the nose and throat (i.e., both a nasopharyngeal and a pharyngeal swab) for culture.</p>

<h4>Biotype testing</h4>

<p>After <em>C. diphtheriae</em> has been isolated, the biotype (substrain) should be determined. The four biotypes are intermedius, belfanti, mitis, and gravis.</p>

<h4>Toxigenicity testing</h4>

<p>In addition to determining biotype, toxigenicity testing using the Elek test should be performed to determine if the <em>C. diphtheriae</em> isolate produces toxin. These tests are not readily available in many clinical microbiology laboratories; isolates should be sent to a reference laboratory proficient in performing the tests.</p>

<h4>Polymerase chain reaction testing</h4>

<p>Additional clinical specimens for PCR testing at CDC should be collected at the time specimens are collected for culture. Because isolation of <em>C. diphtheriae</em> is not always possible (many patients have already received several days of antibiotics by the time a diphtheria diagnosis is considered), PCR can provide additional supportive evidence for the diagnosis of diphtheria. The PCR assay allows for detection of the regulatory gene for toxin production (<em>dtxR</em>) and the diphtheria toxin gene (<em>tox</em>).<a href="chpt22-lab-support.html#f6">[6]</a> Clinical specimens (swabs, pieces of membrane, biopsy tissue) can be transported to CDC with cold packs in a sterile empty container or in silica gel sachets. For detailed information on specimen collection and shipping and to arrange for PCR testing, the state health department may contact the CDC Pertussis and Diphtheria Laboratory at 404-639-1231 or 404-639-1239.</p>

<h4>Serologic testing</h4>

<p>Measurement of the patient’s serum antibodies to diphtheria toxin before administration of antitoxin may help in assessing the probability of the diagnosis of diphtheria. The state health department or CDC can provide information on laboratories that offer this test (few laboratories have the capability to accurately test antibody levels). If antibody levels are low, diphtheria cannot be accurately ruled out, but if levels are high, <em>C. diphtheriae</em> is less likely to produce serious illness.</p>

<h4>Submission of <em>C. diphtheriae</em> isolates</h4>

<p>All isolates of <em>C. diphtheriae</em> from any body site (respiratory or cutaneous), whether toxigenic or nontoxigenic, should be sent to the CDC Pertussis and Diphtheria Laboratory for reference testing. Clinical specimens from patients with suspected diphtheria to whom diphtheria antitoxin has been released for treatment should also be sent to the CDC Pertussis and Diphtheria Laboratory for culture and PCR testing. To arrange for shipping of specimens, contact your state health department.</p>

<h3><em>B. Haemophilus influenzae</em> type b (Hib) invasive disease (see <a href="chpt02-hib.html">Chapter 2</a>)</h3>

<h4>Culture</h4>

<p>Confirming a case of Hib disease requires culturing and isolating the bacterium from a normally sterile body site. Normally sterile site specimens include cerebrospinal fluid (CSF), blood, joint fluid, pleural effusion, pericardial effusion, peritoneal fluid, subcutaneous tissue fluid, placenta, and amniotic fluid. Most hospital and commercial microbiologic laboratories have the ability to isolate <em>H. influenzae</em> (Hi) from cultured specimens. All Hi isolates should also be tested for antimicrobial susceptibility according to guidelines in M02-A11 Performance Standards for Antimicrobial Disk Susceptibility Tests (January 2012) from the Clinical Laboratory Standards Institute.<a href="chpt22-lab-support.html#f7">[7]</a></p>

<h4>Serotype testing (serotyping)</h4>

<p>Serotyping distinguishes encapsulated strains, including Hib, from unencapsulated strains, which cannot be typed. The six encapsulated types (designated a-f) have distinct capsular polysaccharides that can be differentiated by slide agglutination with type-specific antisera.</p>

<p>To monitor the occurrence of invasive Hib disease, microbiology laboratories should perform serotype testing of all <em>H. influenzae</em> isolates,<a href="chpt22-lab-support.html#f8">[8, 9]</a> particularly those obtained from children younger than 5 years of age. To monitor the disease burden and long-term vaccine effectiveness, Hi isolates from children aged 5-14 years should also be serotyped and reported. Even though Hib disease has declined, laboratories should continue routine serotyping. Contact your state health department if serotyping is not available at your laboratory. State health departments with questions about serotyping should contact the CDC Meningitis and Vaccine Preventable Disease Branch laboratory at 404-639-3158.</p>

<h4>Antigen detection</h4>

<p>Because the type b capsular antigen can be detected in body fluids, including urine, blood, and CSF of patients, clinicians often request a rapid antigen detection test for diagnosis of Hib disease. Antigen detection may be used as an adjunct to culture, particularly in the diagnosis of patients who have received antimicrobial agents before specimens are obtained for culture. Methods for antigen detection include latex agglutination (LA) and counterimmunoelectrophoresis. LA is a rapid and sensitive method used to detect Hib capsular polysaccharide antigen in CSF, serum, urine, pleural fluid, or joint fluid. Counterimmunoelectrophoresis is more specific but less sensitive than LA, but takes longer and is more difficult to perform.</p>

<p>If the Hib antigen is detected in CSF but a positive result is not obtained from culture or sterile site, the patient should be considered as having a probable case of Hib disease and reported as such. Because antigen detection tests can be positive in urine and serum of persons without invasive Hib disease, persons who are identified exclusively by positive antigen tests in urine or serum should not be reported as cases. PCR assays for Hib in clinical specimens are available for research purposes only.<a href="chpt22-lab-support.html#f10">[10-12]</a> Isolation of the bacterium is needed to confirm Hi invasive disease, determine the serotype, and test for antimicrobial susceptibility.</p>

<h4>Subtyping</h4>

<p>Although not widely available, subtyping the Hib bacterium by pulsed-field gel electrophoresis (PFGE),<a href="chpt22-lab-support.html#f13">[13, 14]</a> multilocus sequence typing (MLST), and 16S rRNA gene sequence typing can be performed for epidemiologic purposes. Some subtyping methods, such as outer membrane proteins, lipopolysaccharides, or enzyme electrophoresis, are no longer recommended or performed because they were unreliable or too labor intensive. The state health department may direct questions about subtyping to the CDC Meningitis and Vaccine Preventable Disease Branch laboratory at 404-639-3158.</p>

<h3>C. Hepatitis A (see <a href="chpt03-hepa.html">Chapter 3</a>)</h3>

<p>Diagnostic tests used to confirm hepatitis A virus infection include serologic testing, and occasionally, PCR-based assays to amplify and sequence viral genomes.</p>

<h4>Serologic testing</h4>

<p>The diagnosis of acute hepatitis due to hepatitis A virus (HAV) is confirmed during the acute or early convalescent phase of infection by the presence of IgM anti-HAV in serum.</p>

<p>Serum for IgM anti-HAV testing should be obtained as soon as possible after onset of symptoms because IgM anti-HAV generally disappears within 6 months after onset of symptoms.</p>

<p>IgG anti-HAV appears in the acute or convalescent phase of infection, remains for the lifetime of the person, and confers enduring protection against disease.</p>

<p>The antibody test for total anti-HAV measures both IgG anti-HAV and IgM anti-HAV. The presence of total anti-HAV and absence of IgM anti-HAV indicates immunity consistent with either past infection or vaccination. Commercial diagnostic tests are widely available for the detection of IgM and total (IgM and IgG) anti-HAV in serum.</p>

<h4>CDC laboratory special studies</h4>

<p>Occasionally, molecular virologic methods such as PCR-based assays are used to amplify and sequence viral genomes. These assays may be helpful to investigate common-source outbreaks of hepatitis A. Providers with questions about molecular virologic methods should consult with their state health department or the Division of Viral Hepatitis, Laboratory Branch, CDC.</p>

<h3>D. Hepatitis B (see <a href="chpt04-hepb.html">Chapter 4</a>)</h3>

<p>Diagnostic tests used to confirm hepatitis B virus (HBV) infection include serologic testing, genotyping and subtyping (in outbreak investigations), and occasionally PCR-based assays to amplify/quantify and determine the sequence of viral genomes.</p>

<h4>Serologic testing</h4>

<p>Several well-defined antigen-antibody systems are associated with HBV infection, including HBsAg and anti-HBs; hepatitis B core antigen (HBcAg) and antibody to HBcAg (anti-HBc); and hepatitis B e antigen (HBeAg) and antibody to HBeAg (anti-HBe). Serologic assays are commercially available for all of these except HBcAg because no free HBcAg circulates in blood.</p>

<p>The presence of HBsAg is indicative of ongoing HBV infection and potential infectiousness. In newly infected persons, HBsAg is present in serum 30-60 days after exposure to HBV. Anti-HBc develops in all HBV infections, appearing at onset of symptoms or liver test abnormalities in acute HBV infection, rising rapidly to high levels, and persisting for life. Acute or recently acquired infection can be distinguished by presence of the immunoglobulin M (IgM) class of anti-HBc, which persists for approximately 6 months. IgM anti-HBc may not be present in newly infected children younger than 2 years of age, especially if they acquired their infection through perinatal transmission.</p>

<p>In persons who recover from HBV infection, HBsAg is eliminated from the blood, usually in 2-3 months, and anti-HBs develops during convalescence. The presence of anti-HBs indicates immunity from HBV infection. After recovery from natural infection, most persons will be positive for both anti-HBs and anti-HBc, whereas only anti-HBs develops in persons who are successfully vaccinated against hepatitis B. Persons who do not recover from HBV infection and become chronically infected remain positive for HBsAg (and anti-HBc), although a small proportion (0.3% per year) of these persons may eventually clear HBsAg and develop anti-HBs.</p>

<p>In some cases, anti-HBc is the only serologic marker detected. Isolated anti-HBc can occur after HBV infection in persons who have recovered but whose anti-HBs levels have waned or in persons in whom anti-HBs failed to develop. Certain chronically infected persons may be positive for anti-HBc alone, with HBsAg levels that are below levels detectable by commercially available tests. Infants who are born to HBsAg-positive mothers and who do not become infected may also have detectable anti-HBc for up to 24 months after birth from passively transferred maternal antibody.</p>

<p>The diagnosis of acute hepatitis due to hepatitis B virus infection is serologically confirmed by a positive test for IgM antibody to hepatitis B core antigen (anti-HBc). If testing for IgM antiHBc is not available, the diagnosis of acute hepatitis B can also be confirmed by a positive test for hepatitis B surface antigen (HBsAg) with a negative test for hepatitis A antibody (anti-HAV) (Table 4). Confirmation of HBsAg-positive results by HBsAg neutralization assay should be done as needed according to the manufacturer’s instructions in the package insert. In addition to acute HBV infection, both perinatal HBV infection and chronic HBV infection are reportable vaccine-preventable conditions. Chronic infection with HBV is confirmed by a positive test for HBsAg accompanied by a negative test for IgM anti-HBc or by two positive HBsAg test results that are at least 6 months apart. A diagnosis of perinatal HBV infection is confirmed by a positive test for HBsAg in an infant aged 1-24 months born in the United States or in U.S. territories to an HBsAg-positive mother.</p>

<h4><a id="t4" name="t4"> </a>Table 4. Interpretation of hepatitis B serologic tests.</h4>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th scope="col">Serologic Marker: HBsAg*</th>
<th scope="col">Serologic Marker: Total Anti-HBc †</th>
<th scope="col">Serologic Marker: IgM Anti-HBc §</th>
<th scope="col">Serologic Marker: Anti-HBs ¶</th>
<th scope="col">Interpretation</th>
</tr>

</thead>
<tr>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>Susceptible, never infected</td>
</tr>

<tr>
<td>+</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>Acute infection, early incubation**</td>
</tr>

<tr>
<td>+</td>
<td>+</td>
<td>+</td>
<td>-</td>
<td>Acute infection</td>
</tr>

<tr>
<td>-</td>
<td>+</td>
<td>+</td>
<td>-</td>
<td>Acute resolving infection</td>
</tr>

<tr>
<td>-</td>
<td>+</td>
<td>-</td>
<td>+</td>
<td>Past infection, recovered and immune</td>
</tr>

<tr>
<td>+</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>Chronic infection</td>
</tr>

<tr>
<td>-</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>False positive (i.e., susceptible), past infection, or ‘low level’ chronic infection</td>
</tr>

<tr>
<td>-</td>
<td>-</td>
<td>-</td>
<td>+</td>
<td>Immune if titer is &#62;10 mIU/ml</td>
</tr>
</table>

<p class="smallFontSize bold">* Hepatitis B surface antigen<br />
† Antibody to hepatitis B core antigen<br />
§ Immunoglobulin M<br />
¶ Antibody to hepatitis B surface antigen<br />
** Transient HBsAg positivity (lasting less than 18 days) might be detected in some patients during vaccination.</p>

<hr />
<h4>Genotyping and subtyping</h4>

<p>Genotyping and subtyping of HBsAg has occasionally been used to investigate outbreaks of hepatitis B, but this procedure is not routinely available in commercial laboratories.</p>

<h4>Molecular analysis</h4>

<p>Molecular virologic methods such as PCR-based assays are available from CDC and commercial laboratories for detection and sequencing of HBV DNA. Although results for HBV DNA are not currently included in the definition for acute hepatitis B, they are included for the chronic HBV definition. Testing for HBV DNA is most commonly used for the purpose of evaluating a patient with diagnosed HBV infection who is receiving or being considered for treatment; these tests are not typically used for the initial diagnosis of infection.</p>

<p>PCR-based methods for amplifying and sequencing the HBV genome, done in conjunction with epidemiologic studies, may be helpful for investigating common-source outbreaks of hepatitis B infection. In addition, these assays are essential for detecting the emergence of vaccine-resistant strains. For example, detection of HBV variants or “escape mutants” among vaccinated infants of HBsAg-positive women is important to determine their potential role in vaccine failures.<a href="chpt22-lab-support.html#f15">[15]</a> Healthcare professionals with questions about molecular virologic methods or those who identify HBsAg-positive events among vaccinated persons should consult with their state health department or the Epidemiology Branch, Division of Viral Hepatitis, CDC, 404-718-8500.</p>

<h3>E. Influenza (see <a href="chpt06-influenza.html">Chapter 6</a>)</h3>

<p>Methods available for the diagnosis of influenza include virus isolation (standard methods and rapid culture assays), molecular detection (reverse transcriptase-polymerase chain reaction [RT-PCR]), detection of viral antigens (enzyme immunoassays [EIA], immunofluorescent antibody [IFA], and commercially available rapid diagnostic kits), and less frequently, electron microscopy, and serologic testing.</p>

<h4>Virus isolation</h4>

<p>Virus isolation is the gold standard for influenza diagnosis. The following guidelines should be considered:</p>

<ul>
<li>Appropriate samples include nasal washes, nasopharyngeal aspirates, nasal and throat swabs, transtracheal aspirates, and bronchoalveolar lavage.</li>

<li>Samples should be taken within 72 hours of onset of illness to maximize the probability of isolating virus.</li>

<li>Rapid culture assays that use immunologic methods to detect viral antigens in cell culture are available. These assays can provide results in 18-40 hours, compared with an average of 4.5 days to obtain positive results from standard culture.</li>
</ul>

<h4>Molecular testing methods</h4>

<p>RT-PCR, including real-time RT-PCR, can be used to detect the presence of influenza<br />
virus in a clinical specimen or to characterize an influenza virus grown in tissue culture or embryonated eggs.</p>

<p>RT-PCR testing can be performed under biosafety level 2 conditions, even for viruses such as avian influenza A(H5N1), which require biosafety level 3 with enhancements for viral culture.</p>

<h4>Antigen detection assays</h4>

<p>Several methods exist for the diagnosis of influenza infection directly from clinical material:</p>

<ul>
<li>Cells from the clinical sample can be stained using an immunofluorescent antibody to look for the presence of viral antigen. Nasal washes, nasopharyngeal aspirates, nasal and throat swabs, gargling fluid, transtracheal aspirates, and bronchoalveolar lavage are suitable clinical specimens.</li>

<li>Commercially available kits to test for the presence of viral antigens fall into three groups: the first detects only influenza type A viruses, while the second detects both influenza type A and B viruses but does not differentiate between virus types; and the third detects both influenza type A and B viruses and distinguishes between the two. Results of these rapid antigen detection tests can be available in less than 1 hour.</li>

<li>Other less frequently used methods include immunostaining and visualization of viral antigens by electron microscopy.</li>

<li>When direct antigen detection methods are used for the diagnosis of influenza, it is important to collect and reserve an aliquot of the clinical sample for possible further testing. The medium used to store the specimen for some rapid testing methods is inappropriate for viral culture; in this case, it is necessary to collect two separate samples. These additional or reserved samples may be used to confirm direct test results by culture and to subtype influenza A isolates.</li>
</ul>

<h4>Serologic testing</h4>

<p>Serologic diagnosis of influenza infection requires paired serum specimens. The acute-phase sample should be collected within 1 week of the onset of illness, and the convalescent-phase sample should be collected approximately 2-3 weeks later.</p>

<p>Hemagglutination inhibition (HI) tests are the preferred method of serodiagnosis. A positive result is a fourfold or greater rise in titer between the acute- and convalescent-phase samples when tested at the same time. Serologic test results are usually available in 24 hours.</p>

<p>While serologic testing can be useful in certain situations where viral culture is not possible or in special studies, serologic diagnosis of influenza is not used for national surveillance because of the lack of standardized testing methods and interpretation.</p>

<h3>F. Measles (see <a href="chpt07-measles.html">Chapter 7</a>)</h3>

<h4>Serologic testing</h4>

<p>Serologic testing for antibodies to measles is widely available. Generally, in a previously susceptible person exposed to either vaccine- or wild-type measles virus, the IgM response begins around the time of rash onset and can be detected for 1-2 months. The IgG response starts more slowly, at about 5-10 days after rash onset, but typically persists for a lifetime. The diagnosis of acute measles infection can be made by detecting IgM antibody to measles in a single serum specimen or by detecting a rise in the titer of IgG antibody in two serum specimens obtained approximately 2 weeks apart. Uninfected persons are IgM negative but will either be IgG negative or IgG positive, depending upon their previous infection or vaccination histories.</p>

<h5>Recommendations for serologic testing for measles</h5>

<ul>
<li>An enzyme immunoassay (EIA) test for IgM antibody to measles in a single serum specimen, obtained at the first contact with the suspected measles case-patient, is the recommended method for diagnosing acute measles.</li>

<li>A single-specimen test for IgG is the most commonly used test for immunity to measles because IgG antibody is long-lasting.</li>

<li>Testing for IgG along with IgM is recommended for suspected measles cases.</li>

<li>Paired sera (acute and convalescent) may be tested for a rise in IgG antibody to measles to confirm acute measles infection.</li>

<li>When a patient with suspected measles has been recently vaccinated (6-45 days prior to rash onset), neither IgM nor IgG antibody responses can distinguish measles disease from the response to vaccination. In this instance, a viral specimen should be obtained so CDC can attempt to distinguish between vaccine virus and wild-type virus (Table 5).</li>
</ul>

<h4><a id="t5" name="t5"> </a>Table 5. Interpretation of measles enzyme immunoassay results*</h4>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th scope="col">IgM Result</th>
<th scope="col">IgG Result</th>
<th scope="col">Previous infection history</th>
<th scope="col">Current infection</th>
<th scope="col">Comments</th>
</tr>

</thead>
<tr>
<td>+</td>
<td>- or +</td>
<td>Not vaccinated, no prior history of measles</td>
<td>Recently received first dose of measles vaccine</td>
<td>Seroconversion. IgG response depends on timing of specimen collection.</td>
</tr>

<tr>
<td>+</td>
<td>- or +</td>
<td>Not vaccinated, no prior history of measles</td>
<td>Wild-type measles</td>
<td>Seroconversion. Classic clinical measles. IgG response depends on timing of specimen collection.</td>
</tr>

<tr>
<td>+</td>
<td>- or +</td>
<td>Previously vaccinated, primary vaccine failure</td>
<td>Recently received second dose of measles vaccine</td>
<td>Seroconversion. IgG response depends on timing of specimen collection.</td>
</tr>

<tr>
<td>-</td>
<td>+</td>
<td>Previously vaccinated, IgG+</td>
<td>Recently received second dose of measles vaccine</td>
<td>IgG level may stay the same or may boost.</td>
</tr>

<tr>
<td>+</td>
<td>+</td>
<td>Previously vaccinated, IgG+</td>
<td>Wild-type measles</td>
<td>May have few or no symptoms (e.g., no fever or rash).</td>
</tr>

<tr>
<td>+</td>
<td>+</td>
<td>Recently vaccinated</td>
<td>Exposed to wild-type measles</td>
<td>Cannot distinguish between vaccine or wild-type virus; evaluate on epidemiologic grounds.†</td>
</tr>

<tr>
<td>-</td>
<td>+</td>
<td>Distant history of natural measles</td>
<td>Vaccine</td>
<td>IgG level may stay the same or may boost.</td>
</tr>

<tr>
<td>+<br />
(at least in some patients)</td>
<td>+</td>
<td>Distant history of natural measles</td>
<td>Wild-type measles</td>
<td>May have few or no symptoms.</td>
</tr>
</table>

<p class="bold smallFontSize">* These results are those expected when using the capture IgM and indirect IgG enzyme immunoassays and may not apply to different assays due to different techniques and sensitivities/specificities.<br />
† However, in this circumstance, IgM testing will be helpful. If negative, it could rule out wild-type measles infection.</p>

<hr />
<p><em>Tests for IgM antibody.</em> Although multiple possible methods exist for testing for IgM antibody, EIA is the most consistently accurate test and is therefore the recommended method. There are two formats for IgM tests. The first and most widely available is the indirect format, which requires a specific step to remove IgG antibodies. Problems with removal of IgG antibodies can lead to false-positive<a href="chpt22-lab-support.html#f16">[16]</a> or, less commonly, false-negative results.</p>

<p>The second format, IgM capture, does not require the removal of IgG antibodies. This is the preferred reference test for measles. One direct-capture IgM EIA is commercially available.</p>

<p>EIA tests for measles are often positive on the day of rash onset. However, in the first 72 hours after rash onset, up to 30% of tests for IgM may give false-negative results. Tests that are negative in the first 72 hours after rash onset should be repeated (<a href="chpt22-lab-support.html#t3">Table 3</a>); serum should be obtained for repeat testing 72 hours after rash onset. IgM is detectable for at least 28 days after rash onset and frequently longer.<a href="chpt22-lab-support.html#f17">[17]</a></p>

<p>When a laboratory IgM test result is suspected of being false-positive (<a href="chpt22-lab-support.html#t3">Table 3</a>), additional tests may be performed. False-positive IgM results for measles may be due to the presence of rheumatoid factor in serum specimens. Serum specimens from patients with other rash illness, such as parvovirus B19, rubella, and roseola, have been observed to yield false-positive reactions in some IgM tests for measles. False-positive tests may be suspected when thorough surveillance reveals no source or spread of cases, when the case does not meet the clinical case definition, or when the IgG result is positive within 3 days of rash onset. In these situations, confirmatory tests may be done at the state public health laboratory or at CDC. IgM results by tests other than EIA can be validated with EIA. Indirect EIA tests may be validated with capture EIA.</p>

<p><em>Tests for IgG antibody.</em> Because tests for IgG require two serum specimens and a confirmed diagnosis cannot be made until the second specimen is obtained, IgM tests are generally preferred. However, if the IgM tests remain inconclusive, a second (convalescent-phase) serum specimen, collected 14-30 days after the first (acute-phase) specimen, can be used to test for an increase in the IgG titer. These tests can be performed in the state laboratory or at CDC. A variety of tests for IgG antibodies to measles are available; these include EIA, hemagglutination inhibition, indirect fluorescent antibody tests, and plaque reduction neutralization. Complement fixation, although widely used in the past, is no longer recommended. The “gold standard” test for serologic evidence of recent measles virus infection is plaque reduction neutralization test of IgG in acute- and convalescent-phase paired sera.</p>

<p>Paired IgG testing for laboratory confirmation of measles requires the demonstration of a fourfold rise in titer of antibody against measles. The tests for IgG antibody should be conducted on both acute- and convalescent-phase specimens at the same time. The same type of test should be used on both specimens. The specific criteria for documenting an increase in titer depend on the test. EIA values are not titers, and increases in EIA values do not directly correspond to rises in titer.</p>

<h4>Virus isolation</h4>

<p>Isolation of measles virus in culture or detection of measles virus by RT-PCR in clinical specimens confirms the diagnosis of measles. However, since culture and RT-PCR can take weeks to perform, they are rarely useful in confirming an actual diagnosis of measles. Also, a negative culture or RT-PCR result does not rule out measles because the tests are greatly affected by the timing of specimen collection and the quality and handling of the clinical specimens. If positive, these tests can be useful adjuncts to diagnosing acute measles when serology results are inconclusive. If measles virus is cultured or detected by RT-PCR, the viral genotype can be used for molecular epidemiology and to distinguish between measles disease caused by a wild-type measles virus and a response to measles vaccination, caused by a vaccine strain.</p>

<p>Viral culture and RT-PCR are important for molecular epidemiologic surveillance to help determine:</p>

<ol>
<li>the origin of the virus,</li>

<li>which viral strains are circulating in the United States, and</li>

<li>whether these viral strains have become endemic in the United States.</li>
</ol>

<p>Isolation of measles virus is technically difficult and is generally performed in research laboratories.</p>

<p>Specimens (urine, nasopharyngeal aspirates, heparinized blood, or throat swabs) from clinically suspected cases of measles obtained for virus culture should be shipped to the state public health laboratory or to CDC at the direction of the state health department as soon as measles is confirmed. Specimens should be properly stored while awaiting case confirmation (see <a href="https://www.cdc.gov/vaccines/pubs/surv-manual/appx/appendix07-meas-vi.pdf">Appendix 7[2 pages]</a>). Clinical specimens for virus isolation should be collected at the same time as samples for serologic testing. Because virus is more likely to be isolated when the specimens are collected within 3 days of rash onset, collection of specimens for virus isolation should not be delayed until laboratory confirmation is obtained. Clinical specimens should ideally be obtained within 7 days of rash onset and should not be collected if more than 10 days have passed after rash onset.</p>

<h3><em>G. Neisseria meningitiditis</em>, Meningococcal disease (see <a href="chpt08-mening.html">Chapter 8</a>)</h3>

<p><em>Neisseria meningitidis</em> is a gram-negative, encapsulated, aerobic diplococcus. Thirteen different meningococcal serologic groups have been defined, five of which (A, B, C, Y, and W135) cause the great majority of disease. The distinction between serogroups is based on the immunochemistry of the capsular polysaccharide, but more recently polymerase chain reaction (PCR) of capsule biosynthesis genes has been used for serogroup determination of isolates.<a href="chpt22-lab-support.html#f18">[18]</a> Serogroup A, C, Y, and W135 polysaccharides all elicit a serogroup-specific immune response, which allows for serogroup-specific vaccines. The serogroup B capsular polysaccharide is poorly immunogenic, thus making it challenging to develop a vaccine to protect against this serogroup. Vaccine development efforts for serogroup B are focusing on outer membrane proteins (OMPs) or other surface molecules rather than the capsular polysaccharide.<a href="chpt22-lab-support.html#f19">[19]</a></p>

<h4>Identification of <em>N. meningitidis</em></h4>

<p>The case definition for confirmed meningococcal disease requires isolation of <em>N. meningitidis</em> from a normally sterile site. Typically, the isolate comes from blood or cerebrospinal fluid (CSF), but it can also be from joint, pleural, or pericardial fluid. Aspirates or skin biopsies of purpura or petechiae can yield meningococci in cases of meningococcemia. The typical media used to grow the organism are chocolate agar or Mueller-Hinton medium in an atmosphere containing 5% carbon dioxide.<a href="chpt22-lab-support.html#f20">[20]</a> Gram staining for <em>N. meningitidis</em> is commonly used and continues to be a reliable and rapid method for presumptive identification. Intracellular gram-negative diplococci in CSF can be considered meningococci until proven otherwise.</p>

<p>In addition to bacteriology for definitive detection and identification, latex agglutination can be used for rapid detection of meningococcal capsular polysaccharides in CSF; however, false-negative or false-positive results can occur. Antigen agglutination tests on serum or urine samples are unreliable for the diagnosis of meningococcal disease.<a href="chpt22-lab-support.html#f21">[21]</a></p>

<p>Real-time PCR detects DNA of meningococci in blood, CSF, or other clinical specimens. A major advantage of PCR is that it allows for detection of <em>N. meningitidis</em> from clinical samples in which the organism could not be detected by culture methods, such as when a patient has been treated with antibiotics before a clinical specimen is obtained for culture. Even when the organisms are nonviable following antimicrobial treatment, PCR can still detect <em>N. meningitidis</em> DNA.<a href="chpt22-lab-support.html#f18">[18]</a> Because of the severity of meningococcal disease, it is critical to treat the patient as soon as infection is suspected and not delay to obtain a culture or laboratory results.</p>

<h4>Susceptibility testing</h4>

<p>Routine antimicrobial susceptibility testing of meningococcal isolates is not recommended. <em>N. meningitidis</em> strains with decreased susceptibility to penicillin G have been identified sporadically from several regions of the United States, Europe and Africa.<a href="chpt22-lab-support.html#f22">[22]</a> Most of these isolates with reduced penicillin susceptibility remain moderately susceptible (minimum penicillin inhibitory concentration of between 0.12 µg/mL and 1.0 µg/mL). High-dose penicillin G remains an effective treatment against moderately susceptible meningococci. Surveillance of susceptibility patterns in populations should be conducted to monitor trends in <em>N. meningitidis</em> susceptibility.</p>

<h4>Testing during outbreaks</h4>

<p>Phenotypic and genotypic methods are used to investigate meningococcal diversity. Capsular polysaccharide (serogroup), porin protein PorB (serotype), and porin protein PorA (serosubtype) are all phenotypic characteristics that can be used to distinguish meningococci from one another.<a href="chpt22-lab-support.html#f19">[19]</a> Outbreaks of meningococcal disease are usually caused by the same or closely related strains.<a href="chpt22-lab-support.html#f23">[23]</a> Molecular genotyping techniques such as pulsed-field gel electrophoresis (PFGE), 16S rRNA gene sequencing, or multilocus sequence typing (MLST) are used for subtype characterization of an outbreak clone.<a href="chpt22-lab-support.html#f24">[24, 25]</a> This subtyping helps to better define the extent of the outbreak. It is crucial to have rapid and reliable results in determining the meningococcal serogroup in an outbreak because public health response will differ for vaccine-preventable or nonvaccine-preventable disease. Molecular genotyping provides important tools for understanding the overall epidemiology of meningococcal disease, but different methods may be more useful in certain settings. PFGE or 16S rRNA gene typing seem to be most useful for outbreak and short-time-period epidemiology, whereas MLST has become the “gold standard” for long-term, global tracing of meningococcal population changes.</p>

<h3>H. Mumps (see <a href="chpt09-mumps.html">Chapter 9</a>)</h3>

<p>Acute mumps infection can be confirmed by the presence of serum mumps IgM, a significant rise in IgG antibody titer in acute- and convalescent-phase serum specimens, positive mumps virus culture, or detection of virus by RT-PCR.</p>

<p>Serum should be collected as soon as possible after onset of parotitis for IgM testing or as the acute-phase specimen for determining seroconversion. The convalescent-phase specimen for IgG detection should be obtained about 2 weeks later. IgM antibodies are detectable within 5 days after onset of symptoms, reach a maximum level about a week after onset of symptoms, and remain elevated for several weeks or months.<a href="chpt22-lab-support.html#f26">[26, 27]</a> The timing of the IgM response to mumps infection in vaccinated persons is highly variable and may be delayed. Virus may be isolated from the buccal mucosa from 6 days before until 10 days after salivary enlargement. Urine is less likely than oral specimens to contain sufficient virus for culture or detection; therefore, buccal swabs are preferred.<a href="chpt22-lab-support.html#f28">[28]</a> However, maximal viral shedding occurs 1-3 days prior to onset and through day 5 following onset of symptoms. Virus may be cleared more rapidly from vaccinated persons who become infected, so early collection of viral samples is critical. Prior immunization against mumps or previous natural infection may be documented by the presence of serum IgG mumps-specific antibodies by EIA. The level of IgG, as measured by EIA, that provides immunity has not been established.</p>

<h4>Serologic testing for IgM antibody</h4>

<p>The serologic tests available for laboratory confirmation of mumps acute infections and immunity vary among laboratories. The state health department can provide guidance on available laboratory services and preferred tests.</p>

<p><em>Enzyme immunoassay.</em> EIA is a highly specific test for diagnosing acute mumps infection At the direction of the state health department, healthcare providers and state and local health departments may send serum specimens from persons with suspected mumps cases to the CDC Measles, Mumps, Rubella &#38; Herpes Virus Laboratory Branch for IgM detection by EIA.</p>

<p><em>Immunofluorescence assay (IFA).</em> IFA assays have the advantage of being relatively inexpensive and simple. The reading of IFA IgM tests requires considerable skill and experience since nonspecific staining may cause false-positive readings.</p>

<p><span class="bold">Note:</span> Commercially available IFA antibody assays and EIA kits for detection of mumps IgM are currently not FDA approved. Each laboratory must validate these tests independently.</p>

<h4>Viral cultures</h4>

<p>Mumps virus can be isolated from fluid collected from the parotid duct, other affected salivary gland ducts, throat, CSF and urine, although urine is unlikely to yield virus and therefore not recommended. Parotid duct swabs yield the best sample, particularly when the salivary gland area is massaged approximately 30 seconds prior to collection of the buccal/parotid duct fluid. An effort should be made to obtain the specimen as soon as possible after parotitis or meningitis onset. Because few laboratories perform mumps virus culture, it is rarely used for clinical diagnosis in uncomplicated cases. Successful isolation should always be confirmed by immunofluorescence with a mumps-specific monoclonal antibody or by molecular techniques. Molecular typing of virus isolates provides epidemiologically important information and is now recommended (see below).</p>

<h4>Molecular typing</h4>

<p>Molecular techniques such as RT-PCR can be used to detect mumps RNA for mumps confirmation in appropriately collected specimens. Molecular epidemiologic surveillance makes it possible to build a sequence database that will help track transmission pathways of mumps strains circulating in the United States. In addition, typing methods are available to distinguish wild-type mumps virus from vaccine virus. Specimens for molecular typing should ideally be obtained as soon as possible after the onset of parotitis, ideally from the day of onset to 3 days later (not more than 10 days after parotitis). Specific instructions for specimen collection and shipping may be obtained from CDC by contacting the Measles, Mumps, Rubella &#38; Herpes Virus Branch at 404-639-1156/3512. Specimens for virus isolation and molecular typing should be sent to CDC as directed by the state health department.</p>

<h3>I. Pertussis (see Chapter 10)</h3>

<h4>Culture</h4>

<p>The preferred laboratory test for diagnosis of pertussis is isolation of <em>Bordetella pertussis</em> by bacterial culture.</p>

<p>Isolation of the <em>B. pertussis</em> bacterium is required to test for antimicrobial resistance and for molecular typing by PFGE. Although bacterial culture is specific for the diagnosis, it is relatively insensitive. Under optimal conditions 80% of suspected cases in outbreak investigations can be confirmed by culture; in most clinical situations isolation rates are much lower.<a href="chpt22-lab-support.html#f29">[29]</a> The timing of specimen collection can affect the isolation rate, as can inadequately collected specimens and concurrent use of effective antimicrobial agents. Because patients can remain culture positive even while taking effective antibiotics (e.g., when strains are resistant to the antibiotic), nasopharyngeal swab for culture should be obtained regardless of concurrent use of an antibiotic.</p>

<p>Fastidious growth requirements make <em>B. pertussis</em> difficult to isolate. Isolation of the organism using direct plating is most successful during the catarrhal stage (i.e., first 1-2 weeks of cough). All persons with suspected pertussis disease should have a nasopharyngeal aspirate or swab obtained from the posterior nasopharynx for culture. <em>B. pertussis</em> recovery rates from nasopharyngeal aspirates are similar to or higher than rates of recovery from nasopharyngeal swabs;<a href="chpt22-lab-support.html#f29">[29-32]</a> rates of recovery from throat and anterior nasal swabs are unacceptably low. Therefore, specimens should be obtained from the posterior nasopharynx (Figure 1), not the throat, by using Dacron® or calcium alginate swabs, not cotton. Specimens should be plated directly onto selective culture medium or placed in transport medium. Regan-Lowe agar or freshly prepared Bordet-Gengou medium generally is used for culture; half-strength Regan-Lowe can be used as the transport medium. Success in isolating the organism declines if the patient has received prior antibiotic therapy effective against susceptible <em>B. pertussis</em> (erythromycin or trimethoprim-sulfamethoxazole), if there is a delay in specimen collection beyond the first 2 weeks of illness, or if the patient has been vaccinated. A positive culture for <em>B. pertussis</em> confirms the diagnosis of pertussis. For this reason, access to a microbiology laboratory that is prepared to perform this service for no cost or for limited cost to the patient is a key component of pertussis surveillance.</p>

<h4>Polymerase chain reaction</h4>

<div class="module-typeC span9 pull-right margin-left-ten">
<p class="padding-top-fifteen"><a id="figure1" name="figure1"> </a><img alt="Figure 1: Proper technique for obtaining a nasopharyngeal specimen for isolation of B. pertussis" class="center" src="https://www.cdc.gov/vaccines/pubs/surv-manual/images/chapt10-figure01.gif" /></p>

<p class="bold">Figure 1: Proper technique for obtaining a nasopharyngeal specimen for isolation of <em>B. pertussis</em></p>
</div>

<p>PCR testing of nasopharyngeal swabs or aspirates can be a rapid, sensitive, and specific method for diagnosing pertussis.<a href="chpt22-lab-support.html#f33">[33]</a> However, false-positive results may be obtained because of contamination in the laboratory or during specimen collection.<a href="chpt22-lab-support.html#f33">[33, 34]</a> PCR is currently available in some laboratories; the assay varies among laboratories and is not standardized. Direct comparison with culture is necessary for validation. Even if a laboratory has validated its PCR method, the result should be considered presumptive, and isolation of <em>B. pertussis</em> by culture should always be attempted to ensure that the disease is truly pertussis. <em>B. pertussis</em> isolates can then be evaluated for erythromycin susceptibility and by PFGE, which can help define the molecular epidemiology of strains circulating in the United States. Calcium alginate swabs are not acceptable for collecting specimens for PCR.</p>

<h4>Serologic testing</h4>

<p>Although serologic testing has proved useful in clinical studies, it is not yet standardized. Also, the lack of association between antibody levels and immunity to pertussis makes results of serologic testing difficult to interpret. For these reasons, serologic testing is not widely available. In Massachusetts, it is used for clinical diagnosis and reporting.<a href="chpt22-lab-support.html#f35">[35]</a> Elsewhere, with few exceptions, it is not known if serologic testing has been appropriately validated or standardized. Therefore, serologic testing should not be relied upon to confirm cases for the purpose of national reporting. Cases meeting the clinical case definition that are serologically positive, but not culture positive or PCR positive, should be reported as probable cases.</p>

<h4>Direct fluorescent antibody testing</h4>

<p>DFA testing of nasopharyngeal secretions may be useful as a screening test for pertussis. A positive DFA result may increase the probability that the patient has pertussis, but it has limited specificity (frequent false-positive results) and is not a confirmatory test. A monoclonal DFA test is available but the sensitivity and specificity are variable.</p>

<h4>Elevated white blood cell count</h4>

<p>An elevated white blood cell count with a lymphocytosis (i.e., increase in lymphocyte count) is usually present in cases of pertussis. The absolute lymphocyte count can reach 20,000/mm or higher. However, there may be no lymphocytosis in very young infants, vaccinated children, or adults with mild cases of pertussis. The white blood cell count is not a confirmation test.</p>

<h4>Pulsed-field gel electrophoresis</h4>

<p>Pulsed-field gel electrophoresis (PFGE) is a type of DNA fingerprinting. This technique has been useful tool for distinguishing epidemiologically related strains (e.g., strains from the same household or small community), while showing diversity within larger geographic areas such as cities, counties, and states.<a href="chpt22-lab-support.html#f36">[36, 37]</a></p>

<p>Questions about performing PFGE on <em>B. pertussis</em> isolates, as well as questions about isolating <em>B. pertussis</em>, performing erythromycin susceptibility testing, and performing PCR can be directed to the Pertussis and Diphtheria Laboratory at CDC. Call Dr. M. Lucia Tondella at 404-639-1239, or Ms. Pam Cassiday at 404-639-1231. If needed, <em>B. pertussis</em> isolates can be sent to:</p>

<p>&#160;</p>

<div class="margin-left-twenty">CDC, Pertussis and Diphtheria Laboratory<br />
Attention: Pam Cassiday<br />
DASH Unit 12<br />
1600 Clifton Road NE<br />
Atlanta, GA 30333</div>

<p>&#160;</p>

<h3>J. Pneumococcal infection (see <a href="chpt11-pneumo.html">Chapter 11</a>)</h3>

<h4>Culture</h4>

<p><em>Streptococcus pneumoniae</em> is a gram-positive, lancet-shaped diplococcus that commonly inhabits the throat as normal flora. <em>S. pneumoniae</em> commonly causes lower and upper respiratory diseases, including pneumonia, meningitis and acute otitis media. Diagnosis of invasive pneumococcal infection is confirmed by culture and isolation of <em>S. pneumoniae</em> from a normally sterile body site (e.g., blood, CSF, pleural fluid, or peritoneal fluid). Alternatively, diagnosis can be confirmed from culture-negative specimens from normally sterile sites using real-time PCR.</p>

<h4>Antibiotic resistance</h4>

<p>The Clinical Laboratory Standards Institute (CLSI) recommends that clinical laboratories test all isolates of <em>S. pneumoniae</em> from CSF for resistance to penicillin, cefotaxime or ceftriaxone, meropenem, and vancomycin.<a href="chpt22-lab-support.html#f38">[38]</a> For organisms from other sources, laboratories should consider testing for resistance to erythromycin, penicillin, trimethoprim-sulfamethoxazole, clindamycin, cefepime, cefotaxime or ceftriaxone, a fluoroquinolone, meropenem, tetracycline, and vancomycin. Pneumococci resistant to vancomycin have never been described; a strain with a vancomycin minimum inhibitory concentration of 2 μg/ml or greater or zone diameter less than 17 mm should be submitted to a reference laboratory for confirmatory testing, and if resistant, should be reported to the state health department. Because pneumococci are fastidious organisms, some susceptibility testing methods used for other organisms are not appropriate for pneumococci; see the CLSI document for testing recommendations.<a href="chpt22-lab-support.html#f38">[38]</a></p>

<h4>Serotyping</h4>

<p>Current pneumococcal vaccines are based upon capsular polysaccharides. There are currently 91 known capsular serotypes. Since only subsets of capsular serotypes are included in pneumococcal vaccines, serotyping allows the measurement of vaccine efficacy and can provide data for development of expanded-serotype vaccines.<a href="chpt22-lab-support.html#f39">[39]</a> CDC and its partners perform active, population-based surveillance for invasive pneumococcal serotypes in specific areas that represent about 30 million people in the United States. CDC does not provide serotyping outside of this surveillance except in specific situations, and this must first be cleared with Dr. Bernard Beall or a representative of the CDC Respiratory Diseases Branch Epidemiology section. Since typing sera are expensive and serotyping is technically difficult, detailed protocols are provided for variations of a simple <a href="https://www.cdc.gov/streplab/pcr.html">PCR-based method</a> for serotype deduction and in several publications.<a href="chpt22-lab-support.html#f40">[40-43]</a></p>

<h3>K. Poliomyelitis (see <a href="chpt12-polio.html">Chapter 12</a>)</h3>

<h4>Virus isolation</h4>

<p>The likelihood of poliovirus isolation is highest from stool specimens, intermediate from pharyngeal swabs, and very low from blood or spinal fluid. Isolation of poliovirus from stool specimens contributes to the diagnostic evaluation but does not constitute proof of a causal association between the isolated viruses and paralytic poliomyelitis.<a href="chpt22-lab-support.html#f44">[44]</a> Isolation of virus from CSF is diagnostic but is rarely accomplished. To increase the probability of poliovirus isolation, at least two stool specimens and two throat swabs should be obtained 24 hours apart from patients with suspected poliomyelitis as early in the course of the disease as possible (i.e., immediately after poliomyelitis is considered as a possible differential diagnosis), but ideally within the first 15 days after onset of paralytic disease. Specimens should be sent to the state or other reference laboratories for primary isolation. Laboratories should forward isolates to CDC for intratypic differentiation to determine whether the poliovirus isolate is wild or vaccine-derived.</p>

<p>Isolation of wild poliovirus constitutes a public health emergency, and appropriate control efforts must be immediately initiated (in consultation among healthcare providers, the state and local health departments, and CDC).</p>

<h4>Serologic testing</h4>

<p>Serology may be helpful in supporting or ruling out the diagnosis of paralytic poliomyelitis. An acute-phase serum specimen should be obtained as early in the course of disease as possible, and a convalescent-phase specimen should be obtained at least 3 weeks later. A fourfold rise in titer between the acute- and convalescent-phase specimens suggests poliovirus infection. Nondetectable antibody titers in both specimens may help rule out poliomyelitis but may be falsely negative in immunocompromised persons, who are also at highest risk for paralytic poliomyelitis. In addition, neutralizing antibodies appear early and may be at high levels by the time the patient is hospitalized, so that a fourfold rise may not be demonstrated. Vaccinated persons would also be expected to have measurable titers; therefore, vaccination history is important for interpretation of serologic tests. One of the limitations of serology is the inability to distinguish between antibody induced by vaccine-related poliovirus and antibody induced by wild virus. Serologic assays to detect anti-poliovirus antibodies are available in most commercial and state public health laboratories.</p>

<h3>L. Rotavirus (see <a href="chpt13-rotavirus.html">Chapter 13</a>)</h3>

<p>Laboratory testing is necessary to confirm group A rotavirus infection and to ensure reliable surveillance and clinical therapy. Because rotavirus is shed in such high concentrations in stool, fecal specimens are preferred for diagnosis of rotavirus. Methods available to diagnose rotavirus infection include detection of viral antigens (EIA, immunochromatography, electron microscopy, and immunostaining) and molecular detection by RT-PCR and nucleotide sequencing. <a href="chpt22-lab-support.html#f45">[45]</a> Serologic testing, although less commonly used, can detect a rise in serum IgG and IgA antibodies for recent infections.</p>

<h4>Detection of viral antigens</h4>

<p>The most widely available method of antigen detection in stool is EIA, which detects an antigen common to all group A rotaviruses.<a href="chpt22-lab-support.html#f45">[45]</a> Several inexpensive commercial EIA kits are available and provide rapid and highly sensitive results (90%-100%). Because EIA is rapid, inexpensive and highly sensitive, it is the most appropriate method for clinical diagnosis and surveillance.</p>

<p>Serotyping and subgrouping can be carried out using EIA methods. Monoclonal antibody-based EIA techniques have been invaluable in defining four globally common rotavirus serotypes (G1-G4) that represent more than 90% of the circulating strains and make up four of the five serotypes in the Rotateq® vaccine.<a href="chpt22-lab-support.html#f46">[46]</a> Two subgroups can also be differentiated by EIA techniques based on the reactivity of different monoclonal antibodies with the major capsid antigen that is common to all group A rotaviruses.</p>

<p>Another less frequently used method more appropriate for a research setting is visualization of viral particles by electron microscopy.</p>

<h4>Molecular detection</h4>

<p>Several molecular methods can be used to detect rotavirus infection in a clinical specimen and to characterize the virus, but these are used most commonly in research settings. Molecular methods for detection of viral RNA include RT-PCR, nucleotide sequencing, hybridization and silver staining.<a href="chpt22-lab-support.html#f45">[45, 47]</a></p>

<ul>
<li>In recent years, multiplexed, semi-nested RT-PCR genotyping and nucleotide sequencing have become widely used to identify the most common and several uncommon rotavirus G and P genotypes. Hybridization can be used to confirm the results of RT-PCR genotyping.<a href="chpt22-lab-support.html#f45">[45, 47]</a></li>

<li>Nucleotide sequencing has been used extensively to identify uncommon strains and genetic variants that cannot be identified by RT-PCR genotyping and to confirm the results of genotyping methods.<a href="chpt22-lab-support.html#f45">[45, 47]</a></li>

<li>Nucleic acid hybridization is a less commonly used method to genotype rotaviruses.</li>

<li>Electrophoresis and silver staining of viral RNA extracted from fecal specimens is a commonly used method for detection of rotavirus in research settings.</li>
</ul>

<h4>Virus isolation</h4>

<p>Rotavirus can be isolated directly from fecal specimens by inoculation of cell cultures in the presence of trypsin-containing growth medium. This procedure is more appropriate for research laboratories.</p>

<h4>Serologic testing</h4>

<p>Routine diagnostic testing for rotavirus infection is primarily based on fecal specimen testing, although rotavirus antigen has been identified in serum samples of patients within 3-7 days of disease onset. Rotavirus diagnosis using serum specimens may prove especially valuable when fecal specimens are not available.<a href="chpt22-lab-support.html#f46">[46]</a> Serologic methods most commonly used to detect recent infections are EIA methods that detect a rise in serum IgG and IgA antibodies. In vaccine trials, the immunogenicity of rotavirus vaccines has been assessed by measuring rotavirus-specific IgG, IgA and neutralizing antibodies to vaccine strains.</p>

<h3>M. Rubella (see <a href="chpt14-rubella.html">Chapter 14</a>)</h3>

<p>Diagnostic tests used to confirm acute or recent rubella infection or congenital rubella syndrome (CRS) include serologic testing and virus isolation.</p>

<h4>Serologic testing</h4>

<p>Sera should be collected as early as possible (within 7-10 days) after onset of illness, and again at least 7-14 days (preferably 2-3 weeks) later. IgM antibodies may not be detectable before day 5 after rash onset. In case of a negative rubella IgM and IgG in specimens taken before day 5, serologic testing should be repeated. Virus may be isolated from 1 week before to 2 weeks after rash onset. However, maximum viral shedding occurs up to day 4 after rash onset.</p>

<p>False-positive serum rubella IgM tests have occurred in persons with parvovirus infections or positive heterophile test (indicating infectious mononucleosis) or with a positive rheumatoid factor (indicating rheumatologic disease).<a href="chpt22-lab-support.html#f48">[48, 49]</a> When a false-positive rubella IgM is suspected, a rheumatoid factor, parvovirus IgM, and heterophile test should be done to rule out a false-positive rubella IgM test result.</p>

<p>The serologic tests available for laboratory confirmation of rubella infections and immunity vary among laboratories. The following tests are widely available and may be used for screening for rubella immunity and/or laboratory confirmation of disease. The state health department can provide guidance on available laboratory services and preferred tests.</p>

<ul>
<li><em>Enzyme immunoassay.</em> Most of the diagnostic testing done for rubella antibodies use some variation of the EIA, which is sensitive, widely available, and relatively easy to perform. EIA is the preferred testing method for IgM, using the capture technique; indirect assays are also acceptable.</li>

<li><em>Hemagglutination inhibition (HI) test.</em> HI once was the gold standard and most commonly used technique for confirmation of rubella infections. It allows for either screening or diagnosis (if paired acute- and convalescent-phase sera are tested). A fourfold rise or greater<br />
in HI antibody titer in paired sera is diagnostic of recent infection. The test may be modified to detect rubella-specific IgM antibody, indicative of primary infection.</li>

<li><em>Latex agglutination (LA) test.</em> The 15-minute LA test appears to be sensitive and specific for screening when performed by experienced laboratory personnel.</li>

<li><em>Immunofluorescent antibody (IFA) assay.</em> IFA is a rapid and sensitive assay. Commercial assays for both IgG and IgM are available in the United States. Care must be taken with the IgM assay to avoid false-positive results due to complexes with rheumatoid antibody.</li>
</ul>

<h4>Virus isolation</h4>

<p>Rubella virus can be isolated from nasal, throat, urine, and cataract specimens from persons with rubella or CRS. The best results come from throat swabs. Efforts should be made to obtain clinical specimens for virus isolation from all case-patients (or from at least some patients in each outbreak) at the time of the initial investigation. Virus may be isolated from 1 week before to 2 weeks after rash onset. However, maximum viral shedding occurs up to day 4 after rash onset.</p>

<h4>Molecular typing</h4>

<p>Rubella virus isolates are very important for surveillance. Molecular epidemiologic surveillance provides important information on the origin of the virus, which virus strains are circulating in the United States, and whether these strains have become endemic in the United States.</p>

<p>In obtaining specimens for rubella molecular typing, collect throat swabs within 4 days of rash onset. Specimens for molecular typing from CRS patients should be collected as soon as possible after diagnosis. Appropriate specimens from CRS patients for molecular typing include throat/nasal swabs, urine, and cataracts from surgery. Specimens for virus isolation should be sent to CDC for molecular typing as directed by the state health department.</p>

<h4>Reverse transcription polymerase chain reaction</h4>

<p>In the United Kingdom, RT-PCR has been evaluated extensively for its usefulness in detection of rubella virus in clinical specimens.<a href="chpt22-lab-support.html#f50">[50, 51]</a> Clinical specimens obtained for virus isolation and sent to CDC are routinely screened by RT-PCR.</p>

<h3>N. Congenital Rubella Syndrome (see <a href="chpt15-crs.html">Chapter 15</a>)</h3>

<p>Diagnostic tests used to confirm CRS include serologic assays and isolation of the virus. Laboratory confirmation can be obtained by any of the following methods:</p>

<ul>
<li>Demonstration of rubella-specific IgM antibodies in the infant’s cord blood or serum. In infants with CRS, IgM antibody persists for at least 6-12 months. In some instances, IgM may not be detected until at least 1 month of age; thus, infants with symptoms consistent with CRS who test negative shortly after birth should be retested at 1 month of age.<a href="chpt22-lab-support.html#f52">[52]</a></li>

<li>Documentation of persistence of serum rubella IgG titer beyond the time expected from passive transfer of maternal IgG antibody.</li>

<li>Isolation of rubella virus. (Virus may be shed from the throat and urine for a year or longer, but best results come from specimens collected at or before 5 months of age.)</li>

<li>Detection of rubella virus by RT-PCR.</li>
</ul>

<h3>O. Varicella (see <a href="chpt17-varicella.html">Chapter 17</a>)</h3>

<p>Laboratory testing for varicella is not routinely required but is indicated to confirm the diagnosis in severe or unusual cases or to determine varicella susceptibility. Because varicella is the most common disease confused with smallpox, rapid laboratory confirmation of varicella zoster virus (VZV) diagnosis is required in cases of vesicular/pustular rash illness that fall into the category of “moderate risk” for smallpox according to the CDC algorithm. As disease continues to decline, laboratory confirmation will become standard practice. Diagnostic tests used to confirm recent varicella infection include virus isolation and identification, in addition to serologic tests.</p>

<h4>Rapid varicella zoster virus identification</h4>

<p>Rapid virus identification techniques are indicated for a case with severe or unusual disease to initiate specific antiviral therapy. The direct fluorescent antibody (DFA) test is the method of choice for rapid clinical diagnosis. This test is sensitive, specific, and widely available. Results are available within several hours. Specimens are best collected by unroofing a vesicle, preferably a fresh fluid-filled vesicle, and then rubbing the base of a skin lesion with a polyester swab. Crusts from lesions are also excellent specimens. Other specimen sources such as nasopharyngeal secretions, saliva, blood, urine, bronchial washings, and cerebrospinal fluid are considered less desirable sources than skin lesions since positive test results from such specimens are much less likely. Because viral proteins persist after cessation of viral replication, DFA may be positive when viral cultures are negative.</p>

<h4>PCR</h4>

<p>PCR is a powerful technique that permits the rapid amplification of specific sequences of viral DNA that would otherwise be present in clinical specimens at concentrations well below detectable limits. Carefully designed primers that target selected small stretches of viral DNA can be used to replicate small quantities of viral DNA extracted from clinical samples. If a PCR product of the expected size is produced, it is evidence that the virus was present in the lesion. This technique has been extended for VZV by amplifying pieces of varicella DNA that include a mutation in the base sequence that distinguishes the vaccine strain from wild-type varicella strains. Highly specific cutting enzymes (restriction endonucleases) can be selected that will cut the fragment from either wild-type strains or vaccine strain, but not both. This provides a convenient means for discriminating between them. More recently, it has been possible to apply these methods to real-time PCR machines that permit direct, single-step discrimination of vaccine strain from wild-type strains on the basis, for example, of the difference in temperature at which the strands from vaccine versus wild-type DNA fragments re-anneal on cooling. This type of approach has reduced the time required to identify a vaccine adverse event from 2 days to several hours.</p>

<h4>Virus strain identification</h4>

<p>Strain identification can distinguish wild-type VZV from the vaccine (Oka/Merck) strain using PCR and restriction fragment length polymorphism (RFLP) analysis. Such testing is important in situations when it is necessary to distinguish wild-type from vaccine-type virus in suspected vaccine adverse events. More recently, rapid real-time PCR methods using Light Cycler® or TaqMan® technology have made it possible to discriminate vaccine strain from wild-type VZV in a single tube assay requiring only a few hours. Postvaccination situations for which specimens should be tested include: 1) rash with more than 50 lesions occurring 7 or more days after vaccination, 2) suspected secondary transmission of the vaccine virus, 3) herpes zoster in a vaccinated person, or 4) any serious adverse event. The National VZV Laboratory at CDC has the capacity to distinguish wild-type VZV from Oka strain using both conventional and real-time PCR methods. Call the National VZV laboratory at 404-639-0066, 404-639-2192, or email <u><a href="mailto:dds1@cdc.gov">dds1@cdc.gov</a></u> or <a href="mailto:vzvlab@cdc.gov">vzvlab@cdc.gov</a> for details about collection and submission of specimens for testing.</p>

<h4>Virus culture</h4>

<p>The diagnosis of VZV infection may be confirmed by culture (isolation) of VZV. Although the virus is difficult to culture, virus isolation should be attempted in cases of severe disease, especially in immunocompromised persons, in order to confirm the diagnosis of varicella. Newer, more sensitive and rapid culture techniques can provide results within 2 to 3 days. Infectious VZV is usually recoverable from fluid from varicella lesions for 2 to 3 days and from zoster lesions for 7 days or longer. VZV may be cultured from other sites such as blood and CSF, especially in immunocompromised patients. Viable VZV cannot be recovered from crusted lesions.</p>

<h4>Serologic testing</h4>

<p>Serologic tests are available for IgG (acute and convalescent) and IgM antibodies to VZV for confirmation of disease. Testing using commercial kits for IgM antibody is not recommended since available methods lack sensitivity and specificity; false-positive IgM results are common in the presence of high IgG levels. The National VZV Laboratory at CDC has developed a reliable IgM capture assay. Call 404-639-0066, 404-639-2192, or email <a href="mailto:dds1@cdc.gov">dds1@cdc.gov</a> or <a href="mailto:kjr7@cdc.gov">kjr7@cdc.gov</a> for details about collection and submission of specimens for testing.</p>

<h4>Testing susceptibles</h4>

<p>Single serologic IgG tests may be used to identify the immune status of persons whose history of varicella is negative or uncertain, and who may be candidates for varicella zoster immune globulin (VZIG) or vaccination. Paired acute- and convalescent-phase antibody tests are used in situations of mild or atypical presentation of disease when immediate therapy is not indicated and when, for clinical reasons, a confirmed diagnosis of the acute illness is important, e.g., a suspected second infection due to varicella. Recent evidence suggests that the latex agglutination method may result in false-positive tests that could mistakenly categorize a susceptible person as immune; less sensitive commercial ELISAs are recommended for the purpose of screening.<a href="chpt22-lab-support.html#f53">[53]</a> Routine testing for varicella immunity following vaccination is not recommended.</p>

<p><span class="toTop"><span class="icon-angle-up"><!-- --></span><a href="chpt22-lab-support.html#">Top of Page</a></span></p>

<hr />
<h3>References</h3>

<ol>
<li><a id="f1" name="f1"> </a>CDC. Handbook of specimen collection and handling in microbiology. Atlanta: Centers for Disease Control, 1985.</li>

<li><a id="f2" name="f2"> </a>[CDC] Centers for Disease Control and Prevention. Measles lab manual from the World Health Organization (Isolation and identification of measles virus in cell culture) <span class="smallFontSize gray4-color">[updated 2009 AUG 31; accessed 2012 OCT 10].</span> Available at <a href="../../../measles/lab-tools/who-lab-manual.html">http://www.cdc.gov/measles/lab-tools/WHO-lab-manual.html</a>.</li>

<li><a id="f3" name="f3"> </a>[CDC] Centers for Disease Control and Prevention. Laboratory protocols of rubella virus <span class="smallFontSize gray4-color">[updated 2008 JUL 28; accessed 2012 OCT 10]</span>. Available at <a href="../../../rubella/lab/index.html">http://www.cdc.gov/rubella/lab/index.html</a>.</li>

<li><a id="f4" name="f4"> </a>[CDC] Centers for Disease Control and Prevention. Division of Scientific Resources (DSR) <span class="smallFontSize gray4-color">[updated 2011 APR 8; accessed 2012 OCT 10].</span> Available at <a href="https://www.cdc.gov/ncezid/dsr/">http://www.cdc.gov/ncezid/dsr/</a>.</li>

<li><a id="f5" name="f5"> </a>[CDC] Centers for Disease Control and Prevention. Reference and disease surveillance.</li>

<li><a id="f6" name="f6"> </a>Mothershed EA, et al. Development of a real-time PCR assay for rapid detection of the diphtheria toxin gene. <em>J Clin Microbiol</em> 2002;40:4713-9.</li>

<li><a id="f7" name="f7"> </a>Clinical and Laboratory Standards Institute. M02-A11 Performance standards for antimicrobial disk susceptibility tests; Approved Standards-Eleventh Edition. Wayne, PA: Clinical and Laboratory Standards Institute, 2012.</li>

<li><a id="f8" name="f8"> </a>[CSTE] Council of State and Territorial Epidemiologists. Vaccine preventable disease surveillance and reporting. Position Statement ID-9 1999.</li>

<li><a id="f9" name="f9"> </a>CDC. Serotyping discrepancies in <em>Haemophilus influenzae</em> type b disease-United States, 1998-1999. <em>MMWR</em> 2002;51(32):706-7.</li>

<li><a id="f10" name="f10"> </a>Corless CE, et al. Simultaneous detection of <em>Neisseria meningitidis</em>, <em>Haemophilus influenzae</em>, and <em>Streptococcus pneumoniae</em> in suspected cases of meningitis and septicemia using real-time PCR. <em>J Clin Microbiol</em> 2001;39:1553-8.</li>

<li><a id="f11" name="f11"> </a>Poppert S et al, Rapid diagnosis of bacterial meningitis by real-time PCR and fluorescence in situ hybridization. <em>J Clin Microbiol</em> 2005;43:3390-7.</li>

<li><a id="f12" name="f12"> </a>Tzanakaki G, et al. Simultaneous single-tube PCR assay for the detection of <em>Neisseria meningitidis</em>, <em>Haemophilus influenzae</em> type b and <em>Streptococcus pneumoniae</em>. <em>Clin Microbiol Infect</em> 2005;11:386-90.</li>

<li><a id="f13" name="f13"> </a>Fry AM, et al. <em>Haemophilus influenzae</em> type b disease among Amish children in Pennsylvania: reasons for persistent disease. <em>Pediatrics</em> 2001;108(4):E60.</li>

<li><a id="f14" name="f14"> </a>Lucher LA, et al. Reemergence, in southwestern Alaska, of invasive <em>Haemophilus influenzae</em> type b disease due to strains indistinguishable from those isolated from vaccinated children. <em>J Infect Dis</em> 2002;186:958-65.</li>

<li><a id="f15" name="f15"> </a>Thomas HC and Carman WF. Envelope and precore/core variants of hepatitis B virus. <em>Gastroenterol Clin North Am</em> 1994;23:499-514.</li>

<li><a id="f16" name="f16"> </a>Jenkerson SA, et al. False-positive rubeola IgM tests. <em>N Engl J Med</em> 1995;332:1103-1104.</li>

<li><a id="f17" name="f17"> </a>Helfand RF, et al. Diagnosis of measles with an IgM capture EIA: the optimal timing of specimen collection after rash onset. <em>J Infect Dis</em> 1997;175:195-9.</li>

<li><a id="f18" name="f18"> </a>Mothershed EA, et al. Use of real-time PCR to resolve slide agglutination discrepancies in serogroup identification of <em>Neisseria meningitidis</em>. <em>J Clin Microbiol</em> 2004;42:320-8.</li>

<li><a id="f19" name="f19"> </a>Raghunathan PL, et al. Opportunities for control of meningococcal disease in the United States. <em>Annu Rev Med</em> 2004;55:333-53.</li>

<li><a id="f20" name="f20"> </a>Granoff DM, et al. Meningococcal vaccines. In: Plotkin SA, Orenstein WA, Offit PA. <em>Vaccines</em>. 4th ed. Philadelphia, PA: Saunders; 2003:959-87.</li>

<li><a id="f21" name="f21"> </a>Rosenstein NE, et al. Meningococcal disease. <em>N Engl J Med</em> 2001;344:1378-88.</li>

<li><a id="f22" name="f22"> </a>American Academy of Pediatrics. 2006 Meningococcal infections. In: Pickering LK, et al, editors. <em>Red book: 2006 Report of the Committee on Infectious Diseases</em>. Elk Grove Village, IL: American Academy of Pediatrics; 2006. p. 452-60.</li>

<li><a id="f23" name="f23"> </a>CDC. Prevention and control of meningococcal disease. <em>MMWR</em> 2005;54(No. RR-7):1-17.</li>

<li><a id="f24" name="f24"> </a>Sacchi, CT, et al. Sequence diversity of <em>Neisseria meningitidis</em> 16S rRNA genes and use of 16S rRNA gene sequencing as a molecular subtyping tool. <em>J Clin Microbiol</em></li>

<li><a id="f25" name="f25"> </a>Maiden, MC, et al. Multilocus sequence typing: a portable approach to the identification of clones within populations of pathogenic microorganisms. <em>Proc Natl Acad Sci USA</em> 1998;95:3140-5.</li>

<li><a id="f26" name="f26"> </a>Ukkonen P, et al. Mumps-specific immunoglobulin M and G antibodies in natural mumps infection as measured by enzyme-linked immunosorbent assay. <em>J Med Virol</em> 1981;8:131-42.</li>

<li><a id="f27" name="f27"> </a>Benito RJ, et al. Persistence of specific IgM antibodies after natural mumps infection. <em>J Infect Dis</em> 1987;155:156-7.</li>

<li><a id="f28" name="f28"> </a>Tolpin MD, and Schauf V. Mumps virus 1984. Belsche RA, editor. Textbook of human virology. Littleton, MA: PSG Publishing Company, p. 311-32.</li>

<li><a id="f29" name="f29"> </a>Bejuk D, et al. Culture of <em>Bordetella pertussis</em> from three upper respiratory tract specimens. Pediatr Infect Dis J 1995;14:64-5.</li>

<li><a id="f30" name="f30"> </a>Halperin SA, et al. Evaluation of culture, immunofluorescence, and serology for the diagnosis of pertussis. <em>J Clin Microbiol</em> 1989;27:752-7.</li>

<li><a id="f31" name="f31"> </a>Hallander HO, et al. Comparison of nasopharyngeal aspirates with swabs for culture of <em>Bordetella pertussis</em>. J Clin Microbiol 1993;31:50-2.</li>

<li><a id="f32" name="f32"> </a>Hoppe JE and Weiss A. Recovery of <em>Bordetella pertussis</em> from four kinds of swabs. Eur J Clin Microbiol 1987;6:203-5.</li>

<li><a id="f33" name="f33"> </a>Meade BD and Bollen A. Recommendations for use of the polymerase chain reaction in the diagnosis of <em>Bordetella pertussis</em> infections. <em>J Med Microbiol</em> 1994;41:51-5.</li>

<li><a id="f34" name="f34"> </a>Lievano FA, et al. Issues associated with and recommendations for using PCR to detect outbreaks of pertussis. <em>J Clin Microbiol</em>. 2002;40:2801-5.</li>

<li><a id="f35" name="f35"> </a>Marchant CD, et al. Pertussis in Massachusetts, 1981-1991: incidence, serologic diagnosis, and vaccine effectiveness. <em>J Infect Dis</em> 1994;169:1297-1305.</li>

<li><a id="f36" name="f36"> </a>de Moissac YR, et al. Use of pulsed-field gel electrophoresis for epidemiological study of <em>Bordetella pertussis</em> in a whooping cough outbreak. <em>J Clin Microbiol</em> 1994;32:398-402.</li>

<li><a id="f37" name="f37"> </a>Bisgard KM, et al. Molecular epidemiology of <em>Bordetella pertussis</em> by pulsed-field gel electrophoresis profile: Cincinnati, 1989-1996. <em>J Infect Dis</em> 2001;183:1360-7.</li>

<li><a id="f38" name="f38"> </a>Clinical and Laboratory Standards Institute. M100-S22 Performance standards for antimicrobial susceptibility testing; Twenty-second informational supplement. Approved Standard. Wayne, PA: Clinical and Laboratory Standards Institute, 2012.</li>

<li><a id="f39" name="f39"> </a>Beall B, et al. Active Bacterial Core Surveillance Team. Pre- and postvaccination clonal compositions of invasive pneumococcal serotypes for isolates collected in the United States in 1999, 2001, and 2002. <em>J Clin Microbiol</em> 2006;44:999-1017.</li>

<li><a id="f40" name="f40"> </a>Pai R, et al. Sequential multiplex PCR approach for determining capsular serotypes of <em>Streptococcus pneumoniae</em> isolates. <em>J Clin Microbiol</em> 2006;44:124-31.</li>

<li><a id="f41" name="f41"> </a>Carvalho MG, et al. Evaluation and Improvement of real-time PCR detection assays to <em>lytA, ply,</em> and <em>psaA</em> genes for detection of pneumococcal DNA. <em>J Clin Microbiol</em> 2007. In press.</li>

<li><a id="f42" name="f42"> </a>Dias CA, et al. Sequential multiplex PCR for determining capsular serotypes of pneumococci recovered from Brazilian children. <em>J Med Microbiol</em> 2007;56:1185-8.</li>

<li><a id="f43" name="f43"> </a>Morais M, et al. Sequential multiplex PCR for identifying pneumococcal capsular serotypes from south-Saharan African clinical isolates. <em>J Med Microbiol</em> 2007;56:1181-4.</li>

<li><a id="f44" name="f44"> </a>Strebel PM, et al. Epidemiology of poliomyelitis in the United States one decade after the last reported case of indigenous wild virus-associated disease. <em>Clin Infect Dis</em> 1992;14:568-79.</li>

<li><a id="f45" name="f45"> </a>Gray JD and Desselberger U editors. 2000. <em>Rotavirus, methods &#38; protocols.</em> (Methods in molecular medicine). New York: Humana Press, 2000.</li>

<li><a id="f46" name="f46"> </a>Estes MK and Kapikian AZ.. Rotaviruses. In: Knipe DM, et al, editors, <em>Fields virology,</em> 5th ed., vol. 2. Philadelphia: Lippincott Williams &#38; Wilkins, 2007.p. 1917-74.</li>

<li><a id="f47" name="f47"> </a>Fischer TK and Gentsch JR. Rotavirus typing methods and algorithms. <em>Rev Med Virol</em> 2004;14:71-82.</li>

<li><a id="f48" name="f48"> </a>Kurtz JB and Anderson MJ. Cross-reactions in rubella and parvovirus specific IgM tests. <em>Lancet</em> 1985;2:1356.</li>

<li><a id="f49" name="f49"> </a>Morgan-Capner P. False positive tests for rubella-specific IgM. <em>Pediatr Infect Dis J</em> 1991;10:415-6.</li>

<li><a id="f50" name="f50"> </a>Bosma TJ, et al. PCR for detection of rubella virus RNA in clinical samples. <em>J Clin Microbiol</em> 1995;33:1075-9.</li>

<li><a id="f51" name="f51"> </a>Bosma TJ, et al. Use of PCR for prenatal and postnatal diagnosis of congenital rubella. <em>J Clin Microbiol</em> 1995;33:2881-7.</li>

<li><a id="f52" name="f52"> </a>CDC. Control and prevention of rubella: evaluation and management of suspected outbreaks, rubella in pregnant women, and surveillance for congenital rubella syndrome <em>MMWR</em> 2001;50(No. RR-12):1-23.</li>

<li><a id="f53" name="f53"> </a>Saiman L, et al. Persistence of immunity to varicella-zoster virus after vaccination of healthcare workers. <em>Infect Control Hosp Epidemiol</em> 2001;22:279-83.</li>
</ol>

<p><span class="toTop"><span class="icon-angle-up"><!-- --></span><a href="chpt22-lab-support.html#">Top of Page</a></span></p>
             					            					            						</div>
            					            					            						</div>
            													        						                                                                                        						                                                                								<!-- Related Nav Area -->
								<div class="nav-related hidden-three hidden-four">
									


<div class="module-typeG emailupdates">
<span class="sprite-24-govd-icon"><!-- span --></span> <h4>Get Email Updates</h4>
<p>To receive email updates about this page, enter your email address:</p>
<form action="https://public.govdelivery.com/accounts/USCDC/subscribers/qualify" name="govdelivery">
<fieldset>
<input name="topic_id" type="hidden" value="USCDC_11_40">
<label>
<span class="hidden">Enter Email Address</span>
<input class="email" name="email" type="text" value="">
</label>
<a class="explain" href="../../../emailupdates/index.html">What's this?</a>
<a class="button" href="chpt22-lab-support.html#;">Submit</a>

<noscript>&lt;a class="explain" href="https://www.cdc.gov/emailupdates/"&gt;What's this?&lt;/a&gt; &lt;label&gt;&lt;span class="hidden"&gt;Submit Button&lt;/span&gt; &lt;input class="button submit" name="commit" type="submit" value="Submit" /&gt;&lt;/label&gt;</noscript>
</fieldset>
</form>
</div>

									



	    <div class="module-typeH">
		<h3>Related Links</h3>
		<div>
																	<ul class="list-block">																														<li>					
<a href="../../index.html">Vaccines &amp; Immunizations</a>					</li>										</ul>														</div>
	</div>
	  



								</div>
    							<!-- Persistent Content Area -->
    							<div id="content-secondary">
                                    



            						




    							</div><!-- /end #content-secondary -->
    						</div><!-- /end #contentArea -->
    					</div><!-- /end body -->	
					</div>
					<div class="row">
						<div class="span24 visible-one">
														<div class="pageoptionsContainer formatOptions formatOptionsOff"><span id="po-format-label-bottom" class="select-label">Format:</span><div id="po-format-bottom" class="format pageoptions pull-right"><div class="select-box"><a href="chpt22-lab-support.html#">Select one</a><span class="icon-angle-down pull-right poarrow"></span></div><ul class="drop-down" aria-labelledby="po-format-label-bottom"><li>
        			    	<a class="noDecoration" href="https://www.cdc.gov/vaccines/pubs/surv-manual/chpt22-lab-support.pdf" target="_blank"><span class="sprite-16-pdf"></span> PDF [806 KB]</a>
      </li></ul></div></div>													</div>
					</div>
					<div class="row">
						<div class="span24">
                            <div id="plugin-legend" class="pluginOff">
	<h5>File Formats Help:</h5><span><a href="../../../Other/plugins/index.html">How do I view different file formats (PDF, DOC, PPT, MPEG) on this site?</a></span>
	<ul>
		<li class="plugin-pdf"><a href="../../../Other/plugins/index.html#pdf"><span class="sprite-16-pdf">Adobe PDF file</span></a></li>
		<li class="plugin-ppt"><a href="../../../Other/plugins/index.html#ppt"><span class="sprite-16-ppt">Microsoft PowerPoint file</span></a></li>
		<li class="plugin-word"><a href="../../../Other/plugins/index.html#doc"><span class="sprite-16-word">Microsoft Word file</span></a></li>
		<li class="plugin-excel"><a href="../../../Other/plugins/index.html#xls"><span class="sprite-16-excel">Microsoft Excel file</span></a></li>
		<li class="plugin-wmv"><a href="../../../Other/plugins/index.html#wmv"><span class="sprite-16-wmv">Audio/Video file</span></a></li>
		<li class="plugin-qt"><a href="../../../Other/plugins/index.html#qt"><span class="sprite-16-qt">Apple Quicktime file</span></a></li>
		<li class="plugin-real"><a href="../../../Other/plugins/index.html#ram"><span class="sprite-16-rp">RealPlayer file</span></a></li>
		<li class="plugin-text"><a href="../../../Other/plugins/index.html#text"><span class="sprite-16-txt">Text file</span></a></li>
		<li class="plugin-zip"><a href="../../../Other/plugins/index.html#zip"><span class="sprite-16-zip">Zip Archive file</span></a></li>
		<li class="plugin-sas"><a href="../../../Other/plugins/index.html#sas"><span class="sprite-16-sas">SAS file</span></a></li>
		<li class="plugin-epub"><a href="../../../Other/plugins/index.html#epub"><span class="sprite-16-ebook">ePub file</span></a></li>
		<li class="plugin-ris"><a href="../../../Other/plugins/index.html#ris"><span class="sprite-16-ris">RIS file</span></a></li>
	</ul>
</div>
						</div>
					</div>
					<!-- date stamp -->
					<div id="datestamp" class="row" itemscope="itemscope" itemtype="http://schema.org/WebPage">
						<div class="span24">
    						<ul> 
            					            						<li class="last-reviewed">Page last reviewed: <span itemprop="lastReviewed">April 1, 2014</span></li>
            													            					            						<li class="last-updated">Page last updated: <span itemprop="dateModified">November 9, 2015</span></li>
            					        							<li class="content-source">Content source:
    									



<ul>
<li itemprop="sourceOrganization" itemscope="itemscope" itemtype="http://schema.org/Organization"><a href="../../../ncird/index.html">National Center for Immunization and Respiratory Diseases</a></li>
</ul>


        							</li>
									



    						</ul>
						</div>
					</div>					
				</div><!-- /end #content -->
			</div><!-- /.container for content -->					
			<!-- footer -->
				<footer id="footer" class="cdcSlimFooter">
	
 <!-- include eng footerSubSocialMedia.html -->
<div class="socialMedia">
	<div class="container">
		<nav class="followcdc">
			<ul>
				<li><a href="../../../Other/emailupdates.1.html"><span class="sprite-24-govd"></span><span class="tp-sr-only">Email</span></a></li>
				<li><a href="https://www.facebook.com/CDC" target="_blank"><span class="sprite-24-facebook"></span><span class="tp-sr-only">Recommend</span></a></li>
				<li><a href="https://twitter.com/CDCgov" target="_blank"><span class="sprite-24-twitter"></span><span class="tp-sr-only">Tweet</span></a></li>
				<li><a href="https://www.youtube.com/user/CDCstreamingHealth" target="_blank"><span class="sprite-24-youtube"></span><span class="tp-sr-only">YouTube</span></a></li>
				<li><a href="https://www.instagram.com/CDCgov/" target="_blank"><span class="sprite-24-instagram"></span><span class="tp-sr-only">Instagram</span></a></li>
			</ul>
		</nav>
		<nav class="cdcmedia">
			<ul>
				<li><a href="https://www2c.cdc.gov/podcasts"><span class="sprite-24-listen"></span><span class="tp-sr-only">Listen</span></a></li>
				<li><a href="../../../cdctv.1.html"><span class="sprite-24-watch"></span><span class="tp-sr-only">Watch</span></a></li>
				<li><a href="https://www2c.cdc.gov/podcasts/rss.asp"><span class="sprite-24-rss"></span><span class="tp-sr-only">RSS</span></a></li>
			</ul>
		</nav>
	</div>
</div>
 <!-- /include eng footerSubSocialMedia.html -->

	<div class="container">
		<div class="cdcLinks">
			<div class="container">
				<ul>
					
<!-- include eng footerSubCdcLinks.html-->
<li>
	<nav>
		<h6 class="hidden-four hidden-three">ABOUT <span class="icon-plus visible-one pull-right"></span></h6>
		<ul class="hidden-one">
			<li><a href="../../../about/default.htm">About CDC</a></li>
			<li><a href="https://jobs.cdc.gov">Jobs</a></li>
			<li><a href="../../../funding.1.html">Funding</a></li>
		</ul>
	</nav>
</li>
<li>
	<nav>
		<h6 class="hidden-four hidden-three">LEGAL <span class="icon-plus visible-one pull-right"></span></h6>
		<ul class="hidden-one">
			<li><a href="../../../Other/policies.html">Policies</a></li>
			<li><a href="../../../Other/privacy.html">Privacy</a></li>
			<li><a href="../../../od/foia.1.html">FOIA</a></li>
			<li><a href="../../../eeo/nofearact/index.htm">No Fear Act</a></li>
			<li><a href="https://oig.hhs.gov" class="no-link">OIG</a></li>
		</ul>
	</nav>
</li>
<!--/ include eng footerSubCdcLinks.html-->

				</ul>
			</div>
		</div>
		<div class="cdcContact">
			<div class="container">
				<ul>
					<li class="local-contact">
						
 <!-- include eng footerSubGlobalContact1.html -->
<section>
	<address class="address" itemscope="itemscope" itemtype="https://schema.org/Organization">
		<span class="value" itemprop="address" itemscope="itemscope" itemtype="https://schema.org/PostalAddress">
			<span itemprop="streetAddress">1600 Clifton Road</span>
			<span itemprop="addressLocality">Atlanta</span>,
			<span itemprop="addressRegion">GA</span>
			<span itemprop="postalCode">30329-4027</span>
			<span itemprop="addressCountry">USA</span>
		</span><br />
		<span class="value" itemprop="telephone">800-CDC-INFO (800-232-4636)</span>,
		<span class="value" itemprop="telephone">TTY: 888-232-6348</span><br />
		<span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://wwwn.cdc.gov/dcs/RequestForm.aspx" class=" default no-link">Email CDC-INFO</a></span>
		</span>
	</address>
</section>
 <!-- /include eng footerSubGlobalContact1.html -->

					</li>
					<li class="contact">
						
 <!-- include eng footerSubGlobalContact2.html -->
<section>
	<address class="address" itemscope="itemscope" itemtype="https://schema.org/Organization">
		<span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://www.hhs.gov" class="no-link">U.S. Department of Health & Human Services</a></span>
		</span><br />
        <span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
        	<span itemprop="url"><a href="https://www.hhs.gov/open" class="no-link">HHS/Open</a></span>
        </span><br />
        <span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://www.usa.gov/" class="no-link">USA.gov</a></span>
		</span>
	</address>
</section>
 <!-- /include eng footerSubGlobalContact2.html -->

					</li>
				</ul>
			</div>
		</div>
	</div>
</footer>			<!-- /#footer -->
			<div id="totop" class="off visible-one">Top</div>
        </div>
		<script>page_timing.libs_start = Date.now();</script><script src="../../../TemplatePackage/3.0/js/libs.min.js@3.2.1.7"></script><script>page_timing.libs_end = Date.now();page_timing.core_start = Date.now();</script><script src="../../../TemplatePackage/3.0/js/core.min.js@3.2.1.7"></script><script>page_timing.core_end = Date.now();</script><!-- Start Site Settings Overrides -->
<script>
CDC.Constants.set("flexSliderVideoDomains", ['youtube.com','ytimg.com','vimeocdn.com'], 'es');
</script>
<!-- End Site Settings Overrides --><script>page_timing.app_start = Date.now();</script><script src="../../../TemplatePackage/3.0/js/app.min.js@3.2.1.7"></script><script>page_timing.app_end = Date.now();</script>		
		        			<!-- Begin Survey scripts -->
<script src="../../../JScript/foresee/foresee-trigger.js"></script>
<script src="../../../JScript/foresee-variables.js"></script>
<!-- End Survey scripts -->
			
            <script>if (typeof page_timing !== 'undefined') { page_timing.metrics_start = Date.now(); }</script>
<!-- xxxxxxxxxxxxxxxxxx METRICS code begins here xxxxxxxxxxxxxxxxxxxxxxxxxxx-->
<div id="metrics">

<!-- Begin Google DAP inclusion -->
<script type="text/javascript" id="_fed_an_ua_tag" src="../../../JScript/metrics/google/Universal-Federated-Analytics-Min.js@agency=HHS&amp;subagency=CDC"></script>
<!-- End Google DAP inclusion -->

<!-- Begin Template Package Custom Interactions -->
<script type="text/javascript" src="../../../JScript/metrics/rd_custom_interactions.js"></script>
<!-- End Template Package Custom Interactions -->

<!-- SiteCatalyst code version: H.21. -->
<script src="../../../JScript/metrics/topic_levels.js"></script>
<script src="../../../JScript/metrics/s_code_v21_cdcgov.js"></script>
<script>

	enquire.register(CDC.Global.selectors.vp1, function() {
		s.prop49 = 1;
	});
	enquire.register(CDC.Global.selectors.vp2, function() {
		s.prop49 = 2;
	});
	enquire.register(CDC.Global.selectors.vp3, function() {
		s.prop49 = 3;
	});
	enquire.register(CDC.Global.selectors.vp4, function() {
		s.prop49 = 4;
	});

            
            


s.pageName=document.title
s.channel="Vaccines" 
siteCatalyst.setLevel1("ncird"); <!--formerly NCIRD before 8/15/2012--><!-- OLD LEVEL 1 = "NCIRD" BEFORE 3/5/2013 -->
siteCatalyst.setLevel2("Vaccines"); <!-- OLD LEVEL 2 = "Surveillance" BEFORE 3/5/2013 -->
siteCatalyst.setLevel3("php"); <!-- NEW LEVEL 3 AS OF 3/5/2013 -->
siteCatalyst.setLevel4("surv-manual"); <!-- NEW LEVEL 4 AS OF 3/5/2013 -->


            
                        
            s.prop2=window.location.href;
s.prop26=document.title;
s.prop30=document.title;
s.prop31=window.location.href;
s.prop46=window.location.href;
s.server=window.location.hostname;

// Update the level variables here.
updateVariables(s);

/************* DO NOT ALTER ANYTHING BELOW THIS LINE ! **************/
var s_code=s.t();



if(s_code) {
	document.write(s_code);
}



if (document.location.protocol) {


	var cookieManagment = {
		getUuid : function (strPrefix) {
			strPrefix = strPrefix || "";
			var d = new Date();
			var seconds = Math.round(d.getTime() / 1000);
			return strPrefix + Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1) + "-" + seconds.toString();
		},
		setCookie : function (cname, cvalue, exdays, domain) {
			domain = domain || ".cdc.gov";
			var d = new Date();
			d.setTime(d.getTime() + (exdays*24*60*60*1000));
			var expires = "expires="+ d.toUTCString();
			document.cookie = cname + "=" + cvalue + ";" + expires + ";domain=" + domain + ";path=/";
		},
		getCookie : function (cname) {
			var name = cname + "=";
			var ca = document.cookie.split(';');
			var i = ca.length;
			while (i --) {
				var c = ca[i];
				while (c.charAt(0)==' ') {
					c = c.substring(1);
				}
				if (c.indexOf(name) == 0) {
					return c.substring(name.length,c.length);
				}
			}
			return "";
		},
		init : function () {



			var currCookieVal = cookieManagment.getCookie('cdc_s_vi');

			if (!currCookieVal.length || currCookieVal.indexOf('[CS]v1') != -1) {
				var newCookieVal = cookieManagment.getUuid('[CS]v2') + "-" + cookieManagment.getUuid('');
				cookieManagment.setCookie("cdc_s_vi", newCookieVal, 7);
				return newCookieVal;
			}
			return currCookieVal;
		}
	};


	var cdcSessionID = cookieManagment.init();

	s3 = s_gi("cdcgovvistorid");

	var sKey, override = {
		fun: "cdcgovvistorid",
		trackingServer: "cdc.112.2o7.net"
	};

	s3.visitorID = cdcSessionID;

	for (sKey in s) {
		if (s.hasOwnProperty(sKey)) {
			if (override.hasOwnProperty(sKey)) {
				s3[sKey] = override[sKey];

			} else {

			}
		}
	}
	updateVariables(s3);


	var s_code_uvid = s3.t();
	if(s_code_uvid) {
		document.write(s_code_uvid);
	}
}




</script>
<script type="text/javascript">
if(navigator.appVersion.indexOf('MSIE')>=0) { document.write(unescape('%3C')+'\!-'+'-'); }
</script>
<noscript><a href="http://www.omniture.com"><img src="https://smtrics.cdc.gov/b/ss/cdcgov/1/H.21--NS/01/H.21--NS/0" height="1" width="1" alt="Web Analytics" /></a></noscript><!--/DO NOT REMOVE/-->
<!-- End SiteCatalyst code version: H.21. -->
<!-- Begin Template Package DigitalGov Search Support -->
<script type="text/javascript" src="../../../JScript/rd_search.js"></script>
<!-- End Template Package DigitalGov Search Support -->
</div>
<!-- xxxxxxxxxxxxxxxxxx METRICS code ends here xxxxxxxxxxxxxxxxxxxxxxxxxxx-->
<script>if (typeof page_timing !== 'undefined') { page_timing.metrics_end = Date.now(); }</script>
				


<script>
<!--//
$(document).ready(function () {
  var cdcLeftNav = new CDC.LeftNav("nav-primary","/migration/ncird_migration/vaccines/pubs/surv-manual/index.html", true, false);
  cdcLeftNav.render();
});
//-->
</script>



    	



						    </body>
	</html>
