
<!DOCTYPE html>
<!--[if lt IE 8]>      <html class="no-js nav theme-purple lt-ie10 lt-ie9 lt-ie8  " lang="en-us"> <![endif]-->
<!--[if IE 8]>         <html class="no-js nav theme-purple lt-ie10 lt-ie9  " lang="en-us"> <![endif]-->
<!--[if IE 9]>         <html class="no-js nav theme-purple lt-ie10  " lang="en-us"> <![endif]-->
<!--[if !IE]><!-->     <html class="no-js nav theme-purple  " lang="en-us" > <!--<![endif]-->	
    <head>
        <meta charset="utf-8" />
                		<title>Meningitis | Lab Manual | PCR Detection and Characterization | CDC</title>
        		
        <meta name="description" content="Primary Culture and Presumptive Identification Laboratory Methods." />        <meta name="keywords" content="meningitis, bacterial meningitis, meningitis in lab, meningitis in laboratory, meningitis laboratory methods, epidemiology, biosafety, clinical specimens" />		<meta name="viewport" content="width=device-width, initial-scale=1" />
		        <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../TemplatePackage/3.0/images/cdc-touch-icon-144x144.png" />
        <link rel="apple-touch-icon-precomposed" sizes="114x114" href="../../TemplatePackage/3.0/images/cdc-touch-icon-114x114.png" />
        <link rel="apple-touch-icon-precomposed" sizes="72x72" href="../../TemplatePackage/3.0/images/cdc-touch-icon-72x72.png" />
        <link rel="apple-touch-icon-precomposed" href="../../TemplatePackage/3.0/images/cdc-touch-icon-57x57.png" />
        <link rel="shortcut icon" href="../../TemplatePackage/3.0/images/favicon.ico" />	
		
		<meta name="robots" content="index,follow" />
		<meta property="cdc:template_version" content="3.0" />
		<meta property="cdc:content-id" content="314796"/>
		<meta property="cdc:revision-id" content="4"/>
				<link rel="canonical" href="http://www.cdc.gov/meningitis/lab-manual/chpt10-pcr.html"/>
				<!-- Internal Release Version # 3.7.2 -->
                			<link rel="stylesheet" href="../../TemplatePackage/3.0/css/lib/bootstrap24.css" />
	<link rel="stylesheet" href="../../TemplatePackage/3.0/css/app.css@3.2.1.7.css" />
	<!--[if gt IE 9]><!--><!--<![endif]-->
	<!--[if IE 7]>
		<link rel="stylesheet" href="/TemplatePackage/3.0/css/lib/font-awesome-ie7.css" />
	<![endif]-->
	<!--[if lt IE 9]>
		<link rel="stylesheet" href="/TemplatePackage/3.0/css/ie.css" />
	<![endif]-->
	<link rel="stylesheet" media="print" href="../../TemplatePackage/3.0/css/print.css@3.2.1.7.css" />
	<script src="../../TemplatePackage/3.0/js/libs/modernizr-latest.js"></script>
	<script>
		if(!Date.now){Date.now=function now(){return new Date().getTime()}}
		var page_timing = page_timing || {};
		page_timing.page_start = Date.now();
		function $$(e){if(window.$){e()}else{setTimeout(function(){$$(e)},50)}}
		// (function(){function e(e,t){var n=e.length;while(n--){if(e[n].indexOf(t)>-1){return e[n]}}return false}var t=e(document.getElementsByTagName("html")[0].className.split(" "),"theme-");if(t){var n=document.createElement("link");n.rel="stylesheet";n.type="text/css";n.media="screen";n.href="/TemplatePackage/3.0/css/themes/"+t+".css";document.getElementsByTagName("head")[0].appendChild(n)}})();

		var settings = {}, addthis = null;
	</script>

	<meta name="application-name" content="CDC.gov"/>
	<meta property="cdc:template_version" content="3.2.1.7" />
	<meta name="msapplication-TileColor" content="#0061aa" />
	<meta name="msapplication-square70x70logo" content="/TemplatePackage/3.0/images/Win8_tile_70x70.png" />
	<meta name="msapplication-square150x150logo" content="/TemplatePackage/3.0/images/Win8_tile_150x150.png" />
	<meta name="msapplication-wide310x150logo" content="/TemplatePackage/3.0/images/Win8_tile_310x150.png" />
	<meta name="msapplication-square310x310logo" content="/TemplatePackage/3.0/images/Win8_tile_310x310.png" />
	<meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">		        




         
<!-- OLD TITLE BEFORE 12/5/2014 <title>CDC - Meningitis - Lab Manual - Results Management and Reporting of Data - Chapter 3</title> -->
<!-- OLD TITLE BEFORE "03/15/2012" DIDNOTEXIST -->
<!-- OLD URL BEFORE "03/15/2012" DIDNOTEXIST -->
<!-- OLD CHANNEL NAME BEFORE "03/15/2012" DIDNOTEXIST" -->
<!-- channel="Meningitis" as of "03/15/2012" -->
<!-- Level 1="NCIRD" new level 1 as of "03/15/2012" -->
<!-- Level 2="Meningitis" new level 2 as of "03/15/2012" -->
<!-- Level 3="Lab" new level 3 as of "03/15/2012" -->
         		    </head>
    <body>
				




				<div id="skipmenu">
	<a href="chpt10-pcr.html#searchTarget" class="skippy">Skip directly to search</a>
	<a href="chpt10-pcr.html#azTab" class="skippy">Skip directly to A to Z list</a>
	<a href="chpt10-pcr.html#nav-container" class="skippy">Skip directly to navigation</a>
	<a href="chpt10-pcr.html#share-bar" class="skippy">Skip directly to page options</a>
	<a href="chpt10-pcr.html#contentArea" class="skippy">Skip directly to site content</a>
</div>		    	<div id="wrapper">
    		
<!-- references to this removed intentionally -->
    		<header id="header">
    			<div class="container">
                    <div class="row">
                    	<div id="logo" class="span14">
                    		<!--iPad touch response and 508 voice reader issue currently under investigation -->
<a href="../../index.html" title="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People">
	<img src="../../TemplatePackage/3.0/images/masthead_subpage.svg" alt="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People" class="hidden-one" />
	<img src="../../TemplatePackage/3.0/images/masthead_1-col.svg" alt="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People" class="visible-one" />
</a>

<!-- IOS9 iPad bug work around - make sure screen readers have a readable version of the logo to read
<a class="tp-sr-only" href="https://www.cdc.gov">Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People</a>
 -->
                    	</div>
                    	<!-- search -->
                    	<div id="searchArea" class="span10 hidden-one hidden-two">							
							
<div class="hidden"><a href="chpt10-pcr.html#" name="searchTarget">Start of Search Controls</a></div>
<form class="searchForm" method="get" action="https://search.cdc.gov/search">
	<fieldset class="search-form-wrapper ">
		<legend>Search Form Controls</legend>
		<div class="search-form-checkbox">
			<label class="subset"><input type="checkbox" name="subset" checked="checked" value="">Search The CDC</label>
		</div>
		<div class="search-input-wrapper">
			<div class="search-buttons">
				<button class="btn-clear"><span class="sprite-14-close">Cancel</span></button>
				<button class="btn-submit"><span class="sprite-14-search-black">Submit</span></button>
				<input type="submit" value="Submit" style="display:none; visibility:hidden;" /> 
			</div>
			<div class="search-input">
				<label><input type="text" placeholder="SEARCH" name="query" /><span class="hidden">Search The CDC</span></label>
			</div>
		</div>
		<input type="hidden" name="utf8" value="&#x2713;" />
					<input type="hidden" name="affiliate" value="cdc-main" />
			</fieldset>
</form>							<div class="a2z-button pull-right hidden-one hidden-two"><a href="chpt10-pcr.html#" id="azTab">CDC A-Z Index <span class="icon-angle-down"></span></a></div>
                    	</div>
                    </div>
                    	<div class="hidden-three hidden-four" id="mobile-menu">
		<ul>
			<li id="vp1-menu-button" class="border-bottom-white"><a href="chpt10-pcr.html#" id="menu-nav"><span class="sprite-16-menu"></span>MENU</a></li>
			<li id="vp1-a2z-button" class="border-bottom-white"><a href="chpt10-pcr.html#" id="menu-az">CDC A-Z</a></li>
			<li id="vp1-search-button" class="border-bottom-white"><a href="chpt10-pcr.html#" id="menu-search"><span class="sprite-16-search-white"></span>SEARCH</a></li>
		</ul>
	</div>
<script>if (typeof page_timing !== 'undefined') { page_timing.menu_load = Date.now(); }</script>
                    <div class="a2z-bar hide">
                    	<div>
	<ul>
		<li><a href="../../az/a.html">A</a></li>
		<li><a href="../../az/b.html">B</a></li>
		<li><a href="../../az/c.html">C</a></li>
		<li><a href="../../az/d.html">D</a></li>
		<li><a href="../../az/e.html">E</a></li>
		<li><a href="../../az/f.html">F</a></li>
		<li><a href="../../az/g.html">G</a></li>
		<li><a href="../../az/h.html">H</a></li>
		<li><a href="../../az/i.html">I</a></li>
		<li><a href="../../az/j.html">J</a></li>
		<li><a href="../../az/k.html">K</a></li>
		<li><a href="../../az/l.html">L</a></li>
		<li><a href="../../az/m.html">M</a></li>
		<li><a href="../../az/n.html">N</a></li>
		<li><a href="../../az/o.html">O</a></li>
		<li><a href="../../az/p.html">P</a></li>
		<li><a href="../../az/q.html">Q</a></li>
		<li><a href="../../az/r.html">R</a></li>
		<li><a href="../../az/s.html">S</a></li>
		<li><a href="../../az/t.html">T</a></li>
		<li><a href="../../az/u.html">U</a></li>
		<li><a href="../../az/v.html">V</a></li>
		<li><a href="../../az/w.html">W</a></li>
		<li><a href="../../az/x.html">X</a></li>
		<li><a href="../../az/y.html">Y</a></li>
		<li><a href="../../az/z.html">Z</a></li>
		<li><a href="../../az/0.html">#</a></li>
	</ul>
</div>
<script>if (typeof page_timing !== 'undefined') { page_timing.a2z_load = Date.now(); }</script>                    </div>
                    <div id="searchArea-two-column" class="span24 searchbar hide hidden-three hidden-four">
						  
<div class="hidden"><a href="chpt10-pcr.html#" name="searchTarget">Start of Search Controls</a></div>
<form class="searchForm" method="get" action="https://search.cdc.gov/search">
	<fieldset class="search-form-wrapper ">
		<legend>Search Form Controls</legend>
		<div class="search-form-checkbox">
			<label class="subset"><input type="checkbox" name="subset" checked="checked" value="">Search The CDC</label>
		</div>
		<div class="search-input-wrapper">
			<div class="search-buttons">
				<button class="btn-clear"><span class="sprite-14-close">Cancel</span></button>
				<button class="btn-submit"><span class="sprite-14-search-black">Submit</span></button>
				<input type="submit" value="Submit" style="display:none; visibility:hidden;" /> 
			</div>
			<div class="search-input">
				<label><input type="text" placeholder="SEARCH" name="query" /><span class="hidden">Search The CDC</span></label>
			</div>
		</div>
		<input type="hidden" name="utf8" value="&#x2713;" />
					<input type="hidden" name="affiliate" value="cdc-main" />
			</fieldset>
</form>					</div>
                    <!--
	See the noscript include for modifying the emergency message. DO NOT add it here.
-->
<script>if (typeof page_timing !== 'undefined') { page_timing.emergency_load = Date.now(); }</script>
    				<div class="titlebar">
					 						<h2>


<a href="../index.html">Meningitis</a>

</h2>
					 					</div>
    				<!-- emergency -->
<div id="emergencyContainer">
	<!--
	<div class="alert alert-error en">
		<b>Attention!</b> Uncomment this section and place the emergency message here.
	</div>
	<div class="alert alert-error esp">
		<b>&#161;Atenci&#243;n!</b> Elimine el comentario de esta secci&#243;n y colocar el mensaje de emergencia aqu&#237;.
	</div>	
	-->
</div>

<!-- no script -->
<noscript>
	<div class="alert alert-block"> Note: Javascript is disabled or is not supported by your browser. For this reason, some items on this page will be unavailable. For more information about this message, please visit this page: <a href="../../other/about_cdcgov.html">About CDC.gov</a>.</div>
</noscript>
<script>if (typeof page_timing !== 'undefined') { page_timing.noscript_load = Date.now(); }</script>
     			</div>
    		</header>
    		<div class="container">				
                <div id="content">
					<div class="row">
        				<!-- left nav -->
        				<div id="nav-container" class="hidden-one hidden-two span5">
    						<nav id="left" class="hidden-one hidden-two">
    							



<h3><a href="../index.html">Meningitis Home</a></h3>

<ul id="nav-primary">
<li><a href="../bacterial.html">Bacterial Meningitis</a>
<ul>
<li><a href="index.html">Laboratory Methods for the Diagnosis of Meningitis</a>
<ul>
<li><a href="disclosure.html">Disclosure</a></li>

<li><a href="authors.html">Authors/Acknowledgements</a></li>

<li><a href="chpt01-intro.html">Chapter 1</a></li>

<li><a href="chpt02-epi.html">Chapter 2</a></li>

<li><a href="chpt03-results-mgmt-rpt-data.html">Chapter 3</a></li>

<li><a href="chpt04-biosafety.html">Chapter 4</a></li>

<li><a href="chpt05-collect-transport-specimens.html">Chapter 5</a></li>

<li><a href="chpt06-culture-id.html">Chapter 6</a></li>

<li><a href="chpt07-id-characterization-nm.html">Chapter 7</a></li>

<li><a href="chpt08-id-characterization-streppneumo.html">Chapter 8</a></li>

<li><a href="chpt09-id-characterization-hi.html">Chapter 9</a></li>

<li><a href="chpt10-pcr.html">Chapter 10</a></li>

<li><a href="chpt11-antimicrobial-suscept-testing.html">Chapter 11</a></li>

<li><a href="chpt12-molecular-typing-methods.html">Chapter 12</a></li>

<li><a href="chpt13-quality-control-assurance.html">Chapter 13</a></li>

<li><a href="chpt14-storage-shipping.html">Chapter 14</a></li>

<li><a href="annex-prep-media-reagents.html">Annex</a></li>
</ul>
</li>
</ul>
</li>

<li><a href="../viral.html">Viral Meningitis</a></li>

<li><a href="../fungal.html">Fungal Meningitis</a></li>

<li><a href="../parasitic.html">Parasitic Meningitis</a></li>

<li><a href="../amebic.html">Amebic Meningitis</a></li>

<li><a href="../non-infectious.html">Non-infectious Meningitis</a></li>

<li><a href="../clinical-resources.html">Resources for Healthcare Professionals</a></li>
</ul>


    						</nav>
    						<div class="nav-related">
							




    						



		    <div class="module-typeH">
			<h3>Related Links</h3>
			<div>
															<ul class="list-block">																									<li>					
<a href="../../vaccines/schedules/index.html">Vaccine Schedules</a>					</li>																				<li>					
<a href="../../vaccines/who/teens/index.html">Preteen &amp; Teen Vaccines</a>					</li>																				<li>					
<a href="../../meningococcal/index.html">Meningococcal Disease</a>					</li>																									<li>					
<a href="../../sepsis/index.html">Sepsis</a>					</li>										</ul>													</div>
	</div>
	  

							




							</div>    						    				
        				</div>
        				<div id="featureArea">
    						<div class="span19">
    							



    						</div>
        				</div>						
    					<div class="breadcrumbs hidden-one span19">
    						


<div class="bread-crumb hidden-one"></div>

    					</div>
						<div class="span19 pagetitle">	
    						    							<div class='syndicate'>
    						    						    							<div class='mSyndicate'>
    																					<h1>Chapter 10: PCR  for Detection and Characterization of Bacterial Meningitis Pathogens: <em>Neisseria  meningitidis</em>, <em>Haemophilus influenzae</em>, and <em>Streptococcus  pneumoniae</em></h1>
															    						    							</div>
    						    						    							</div>
    												</div>
						<div id="share-bar" class="span19">		
    						<div class="socialmediabar">
								<div class="row">
            						<div class="hidden-one pull-right">
        								            							            							<div class="pageoptionsContainer formatOptions formatOptionsOff"><span id="po-format-label" class="select-label">Format:</span><div id="po-format" class="format pageoptions pull-right"><div class="select-box"><a href="chpt10-pcr.html#">Select one</a><span class="icon-angle-down pull-right poarrow"></span></div><ul class="drop-down" aria-labelledby="po-format-label"><li>
        			    	<a class="noDecoration" href="https://www.cdc.gov/meningitis/lab-manual/chpt10-pcr.pdf" target="_blank"><span class="sprite-16-pdf"></span> PDF [962 KB]</a>
      </li></ul></div></div>            						</div>	
    								<div class="span8">
    									<div id="socialMediaShareContainer">
	<div class="share_container noLinking">
	 	<a class="share_button_facebook" title="Facebook" href="chpt10-pcr.html#"><span class="tp-sr-only">Recommend on Facebook</span><span id="share_facebook" class="facebookRecommend"> </span></a>
	 	<a class="share_button_twitter" title="Twitter" href="chpt10-pcr.html#"><span class="tp-sr-only">Tweet</span><span id="share_twitter" class="twitterTweet"> </span></a>
		<a class="share_button" href="chpt10-pcr.html#socialMediaShareContainer" ><img src="../../TemplatePackage/3.0/images/social/addthis-32.png" alt="Share" /><span class="shareButtonEn tp-sr-only" aria-label="Share">Share</span><span class="shareButtonEs tp-sr-only" aria-label="Compartir">Compartir</span></a>
	</div>
</div>

<!-- target anchor for return to top feature in VP1 -->
<a id="content-start" tabindex="-1"></a>
    								</div>
								</div>
    						</div>
    					</div>	
    					<!-- body -->
    					<div id="body" class="span19">
    						<div id="contentArea" >								
        						        																						        						                                                                                            					            						<div class='syndicate'>
            					            					            						<div class='mSyndicate'>
            					            					        							
<p><a href="https://www.cdc.gov/meningitis/lab-manual/chpt10-pcr.pdf">Printer friendly version [56 pages]</a></p>

<ol type="I">
<li><span class="bold">Overview of PCR technologies</span>
<p>In developing countries, the most commonly used approaches for detection and characterization of bacterial meningitis pathogens include culture, Gram stain, and latex agglutination. Although culture is considered the gold standard for case confirmation in clinics, the positive rate is relatively low due to suboptimal storage and transportation conditions, culture practice, and/or antibiotic treatment administered before the specimen is collected. While Gram staining is important, inexpensive, and should be performed whenever possible, it merely gives a clue as to the genus and species of the etiological agent. The reading of latex agglutination results is subjective and can be difficult to interpret, especially when a specimen's bacterial load is low. It is also not feasible to do quality control on latex agglutination. Culture should be kept as the gold standard as cultured bacteria are sources of data for antibiotic susceptibility, complete subtyping, the expression of antigens that are to be included in future vaccines, and pathophysiology of isolates. Specimens that do not yield any culture can still be analyzed by molecular methods (see below) that can be applied on DNA extracted from clinical specimens (typically, blood and CSF).</p>

<ol type="A">
<li><span class="bold">Polymerase chain reaction (PCR)</span>
<p>PCR was developed in the mid- to late 1980s (<a href="chpt10-pcr.html#ref36">36</a>, <a href="chpt10-pcr.html#ref42">42</a>) and is considered one of the most important methodological inventions in molecular biology. It is designed to permit selective amplification of a specific target DNA sequence(s) within a heterogeneous collection of DNA sequences (e.g., total genomic DNA). In PCR, the DNA target is exponentially amplified through repeating three major steps: 1) denaturation of double-stranded DNA into single-stranded DNA; 2) annealing of primers to the complementary single-stranded target sequences; and 3) extension of the primers in the 5' to 3' direction by heat-stable DNA polymerase to produce double-stranded DNA molecules. The copy number of DNA molecules is doubled in each extension step, generating millions of copies of the original DNA molecules when PCR is completed. Because the method does not require live or intact cells, PCR is a valuable tool for detecting bacterial pathogenic agents from clinical specimens, where bacteria die or lyse easily due to inappropriate storage conditions or prior antibiotic treatment. PCR is now widely used in the diagnosis and surveillance of bacterial pathogens because of its high sensitivity and specificity and high throughput capabilities. It provides a complementary tool to classic phenotype-based methods such as culture, Gram stain, and latex agglutination and often enhances confirmatory results (<a href="chpt10-pcr.html#ref3">3</a>).</p>

<p>A number of conventional PCR assays have been developed for detection and subtyping of bacterial and viral pathogens. Conventional PCR detects products at the end point of DNA amplification by visualizing amplicons using agarose gel electrophoresis. Gel-based detection requires that tubes containing PCR amplicons be opened and manipulated, thereby greatly increasing the risk of contamination of laboratory space, equipment, and reagents with amplified materials. Conventional PCR is also very time consuming and less sensitive and specific than a type of PCR called real-time PCR, so it is primarily used for typing assays employing purified cultures or clinical specimens that contain the organism in high density. The use of real-time PCR is rapidly expanding because it is easier to perform and, being a closed system, it reduces potential contamination problems inherent to conventional PCR. Many of the same precautions mentioned for conventional PCR assays also apply to real-time PCR assays.</p>
</li>

<li><span class="bold">Real-time PCR technology</span>
<p>Real-time PCR is also known as quantitative real-time polymerase chain reaction (Q-PCR/qPCR) or kinetic polymerase chain reaction, which combines amplification and detection in one step through the use of fluorescent dyes. There are two types of detection systems: non-specific and specific. Non-specific detection systems use a fluorescent dye that intercalates into any double-stranded DNA molecules and emits enhanced fluorescence. This detection system is relatively inexpensive but susceptible to false positivity. Specific detection systems rely on fluorescent resonance energy transfer probes that specifically recognize target sequences, thus making them the systems of choice for the molecular detection assays described here. Specific detection systems are more expensive than non-specific detection systems and require sophisticated probe designs. Three types of probes are currently in use, including: hydrolysis, hybridization, and hairpin probes (<a href="chpt10-pcr.html#ref7">7</a>, <a href="chpt10-pcr.html#ref13">13</a>). A fluorescent signal is only generated if the probe interacts with its specific target and is subsequently hydrolyzed during amplification. The resulting increase in fluorescence is proportional to the amount of amplified PCR product in the reaction.</p>

<p>The first use of dual-labeled hydrolysis probes was reported in 1993 (<a href="chpt10-pcr.html#ref30">30</a>) and has been widely used in many laboratories since that time. A dual-labeled hydrolysis probe is an oligonucleotide (~17-35 bp long) labeled with a reporter fluorophore (usually a short wavelength colored dye) at the 5' end and a quencher fluorophore (usually a long wavelength colored dye) at the 3' end or at an internal thymine or "T" residue. Optimally, the quencher dye should be 7­-15 base pairs from the reporter dye. When the probe is intact and excited by a light source, the fluorescence emission of the reporter (or donor) dye is absorbed by the quencher (or acceptor) dye as a result of the close proximity of the dyes. This process is also known as fluorescence resonance energy transfer (FRET). During PCR amplification, the probe anneals to an internal region of the target DNA template between the forward and reverse primer. When DNA polymerase catalyzes the extension of the primer and reaches the region where the probe is bound, the 5' exonuclease activity of the DNA polymerase cleaves the probe and releases the reporter from the quencher. This allows emission of the fluorescence from the reporter dye to be observed because it is no longer absorbed by the quencher which has diffused away (<a href="chpt10-pcr.html#figure1">Figure 1</a>). The increase in fluorescence is proportional to the amount of amplified PCR product in the reaction and is measured cumulatively over the course of the entire PCR run (<a href="chpt10-pcr.html#ref7">7</a>, <a href="chpt10-pcr.html#ref30">30</a>).</p>
</li>
</ol>

<div class="span8 fixed"><a id="figure1" name="figure1"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 1 is a picture showing the chemistry of a dual-labeled hydrolysis probe." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure01.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 1. Chemistry of a dual-labeled hydrolysis probe...<a href="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure01-view.gif">view larger</a></p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>

<li><span class="bold">Target gene assays for detection and characterization of bacterial meningitis etiologies</span>
<p>Over the past several years, conventional and real-time PCR assays have been developed for detection of bacterial meningitis pathogens. Reliable assays have been extensively evaluated using invasive clinical isolates and/or clinical specimens from around the world (<a href="chpt10-pcr.html#ref8">8</a>, <a href="chpt10-pcr.html#ref12">12</a>, <a href="chpt10-pcr.html#ref15">15</a>, <a href="chpt10-pcr.html#ref17">17</a>, <a href="chpt10-pcr.html#ref29">29</a>, <a href="chpt10-pcr.html#ref35">35,</a> <a href="chpt10-pcr.html#ref52">52</a>, <a href="chpt10-pcr.html#ref60">60</a>). In general, validated assays should have high sensitivity and specificity. They can be used as complementary approaches in bacterial disease diagnosis. The gene targets, DNA sequence of primers and probes, and the final concentration used in the PCR reactions described here to detect and characterize <em>N. meningitidis, H. influenzae</em>, and <em>S. pneumoniae</em> are listed in <a href="chpt10-pcr.html#table2">Tables 2</a>-<a href="chpt10-pcr.html#table5">5</a>.</p>

<p>The PCR strategy typically employed to detect the causative agent in a suspected case of bacterial meningitis is to first run each of the species-specific assays concurrently on the DNA extracted from the clinical specimen or isolate. The appropriate serogroup/serotype specific assays should then be run on any positive specimens. See <a href="chpt10-pcr.html#sectionV">Section V</a> below, "Workflow for detection of bacterial meningitis pathogens by PCR", for more details.</p>

<ol type="A">
<li><span class="bold">Species-specific real-time PCR assays</span>
<p>PCR detection of <em>N. meningitidis, H. influenzae,</em> and <em>S. pneumoniae</em> can be achieved by amplification of several potential gene targets (<a href="chpt10-pcr.html#ref8">8</a>, <a href="chpt10-pcr.html#ref35">35</a>, <a href="chpt10-pcr.html#ref53">53</a>, <a href="chpt10-pcr.html#ref60">60</a>). The following assays have been developed and validated to be used on DNA extracted from clinical specimens (typically, blood and CSF) and bacterial isolates.</p>

<h5><em>N. meningitidis</em></h5>

<p>Two genes can be targeted in <em>N. meningitidis</em> species-specific assays, <em>ctrA</em> and <em>sodC</em>. The capsule transport to cell surface gene, <em>ctrA,</em> is highly conserved among isolates responsible for invasive meningococcal infections and has been used in both real-time and conventional PCR to detect <em>N. meningitidis</em> (<a href="chpt10-pcr.html#ref35">35</a>). It is a gene within the capsule locus (<a href="chpt10-pcr.html#figure2">Figure 2</a>). However, since at least 16% of carried meningococci lack <em>ctrA</em> (<a href="chpt10-pcr.html#ref10">10</a>, <a href="chpt10-pcr.html#ref14">14</a>, <a href="chpt10-pcr.html#ref41">41</a>), a real-time PCR assay to detect all meningococci, regardless of encapsulation status, was recently developed and validated (<a href="chpt10-pcr.html#ref15">15</a>). This assay targets the Cu, Zn superoxide dismutase gene, <em>sodC</em>, which is not genetically linked to the capsule locus. The <em>sodC</em> assay detects encapsulated meningococci, but it is also useful for detecting nongroupable meningococci that do not contain an intact <em>ctrA</em>, as will be recovered during carriage studies. For this reason, it is recommended that <em>sodC</em> be used for detection of <em>N. meningitidis</em>, if possible. <em>sodC</em> and <em>ctrA</em> primers and probes are listed in <a href="chpt10-pcr.html#table2">Table 2</a>.</p>

<h5><em>H. influenzae</em></h5>

<p>The protein D encoding gene, <em>hpd</em>, encodes protein D, a highly conserved, surface-exposed lipoprotein that is present in all encapsulated and non-encapsulated <em>H. influenzae</em> (<a href="chpt10-pcr.html#ref24">24</a>, <a href="chpt10-pcr.html#ref45">45</a>). The conserved nature of this gene and its presence in all strains of <em>H. influenzae</em> characterized to date make it a highly attractive gene target for the development of a <em>H. influenzae</em> species-specific real-time PCR assay. The recently developed and validated <em>hpd</em> real-time PCR assay is capable of detecting all six serotypes (a-f) and nontypeable (HiNT) <em>H. influenzae</em> with high sensitivity and specificity (<a href="chpt10-pcr.html#ref60">60</a>). Real-time PCR assays targeting <em>bexA</em> were developed and distributed because <em>bexA</em> is present in all six serotypes of <em>H. influenzae</em>. However, though sensitive for detection of Hib, it is less sensitive for Hia, Hic, and Hid, and does not detect Hie, Hif, or HiNT and should no longer be used. The primers and probes for the <em>hpd</em> assay are listed in <a href="chpt10-pcr.html#table2">Table 2</a>.</p>

<h5><em>S. pneumoniae</em></h5>

<p>Both conventional and real-time PCR assays have been developed for the detection of <em>S. pneumoniae,</em> and target genes have included the pneumolysin (<em>ply</em>), autolysin (<em>lytA</em>), and pneumococcal surface adhesion (<em>psaA</em>) genes (<a href="chpt10-pcr.html#ref8">8</a>). However, false-positive results with <em>ply</em>-based PCR have been reported when applied to upper respiratory tract specimens. A suggested explanation for these false positives is the detection of non-pneumococcal alpha-hemolytic streptococci (<a href="chpt10-pcr.html#ref37">37</a>, <a href="chpt10-pcr.html#ref47">47</a>), which are normally present in the respiratory flora (e.g., <em>Streptococcus mitis</em> group and <em>Streptococcus oralis</em>) which sometimes contain a <em>ply</em> gene (<a href="chpt10-pcr.html#ref62">62</a>). The PCR detection assay for <em>S. pneumoniae</em> using a specific segment of the autolysin gene (<em>lyt</em>A) is recommended because it is highly conserved within the species and it has been shown that this assay bestseparates <em>S. pneumoniae</em> from the genotypically similar species <em>S. mitis, S. oralis,</em> and <em>S. pseudopneumoniae</em> (<a href="chpt10-pcr.html#ref33">33</a>). The real-time PCR assay <em>lytA</em> primers and probes that have been found to be extremely reliable for detection of <em>S. pneumoniae</em> are listed in <a href="chpt10-pcr.html#table2">Table 2</a>. Due to recombination events that occur between pneumococci and closely related streptococci, there will probably be rare false-positives or false-negatives for virtually any real-time assay for pneumococcal identification.</p>
</li>

<li><span class="bold">Serogroup/serotype-specific real-time PCR assays</span>
<p>The capsule gene loci of both <em>N. meningitidis</em> and <em>H. influenzae</em> have areas that are both unique and conserved within each serogroup (<em>N. meningitidis</em>) or serotype (<em>H. influenzae</em>) thus providing gene targets for the development or real-time PCR assays designed to identify each specific serogroup or serotype.</p>

<h5><em>N. meningitidis</em></h5>

<p><em>N. meningitidis</em> is classified into 12 serogroups on the basis of the chemical composition and linkage type of saccharide subunits of the capsular polysaccharide that are expressed on the bacterial cell surface. Major disease-causing serogroups include A, B, C, Y, and W135, the latter four of which produce sialic acid containing capsular polysaccharides; whereas serogroup A produces a poly-α1-6-linked N-acetylmannosamine 6-phosphate capsule (<a href="chpt10-pcr.html#ref31">31</a>). Outbreaks caused by serogroup X meningococci, which express poly-α1-4-linked N-acetylglucosamine 1-phosphate capsule (<a href="chpt10-pcr.html#ref6">6</a>) have also been reported (<a href="chpt10-pcr.html#ref2">2</a>, <a href="chpt10-pcr.html#ref20">20</a>). Serogroup D is no longer recognized as a serogroup of <em>N. meningitidis</em>.</p>

<p>As illustrated in <a href="chpt10-pcr.html#figure2">Figure 2</a>, the genetic organization of the capsule locus is conserved among the serogroups. The capsule expression genes are located in four operons: one that encodes capsule biosynthesis (called <em>syn</em> or <em>sia</em> genes, depending on which nomenclature system is used) and three that encode the capsule transport to the cell surface proteins (<em>ctr</em>). The gene products of the <em>ctr</em> operon share high similarity with the ATP-dependent transporters of the ABC family (<a href="chpt10-pcr.html#ref19">19</a>) and are highly conserved among the major disease-causing serogroups (<a href="chpt10-pcr.html#ref1">1</a>, <a href="chpt10-pcr.html#ref18">18</a>, <a href="chpt10-pcr.html#ref39">39</a>, <a href="chpt10-pcr.html#ref55">55</a>) and serogroup X (<a href="chpt10-pcr.html#ref56">56</a>). Sensitive real-time PCR assays targeting <em>ctrA</em>, which is the first gene in the capsule transport operon, have been developed for detection of all encapsulated and some non-encapsulated (nongroupable) <em>N. meningitidis</em> (<a href="chpt10-pcr.html#ref12">12</a>, <a href="chpt10-pcr.html#ref35">35</a>), though a more specific and sensitive assay using the <em>sodC</em> gene as a target has been developed. The genetic differences among the capsule biosynthesis operons of meningococcal serogroups have facilitated the development of real-time PCR assays targeting serogroup-specific genes for capsule biosynthesis to determine the capsule genotype of a meningococcal isolate (<a href="chpt10-pcr.html#ref35">35</a>).</p>

<p>The gene <em>sia</em> (for sialic acid biosynthesis (<a href="chpt10-pcr.html#ref16">16</a>, <a href="chpt10-pcr.html#ref22">22</a>), also called <em>syn</em> forcapsule biosynthesis (<a href="chpt10-pcr.html#ref21">21</a>, <a href="chpt10-pcr.html#ref51">51</a>), are used for genotyping for serogroups B (<em>synD</em>), C (<em>synE</em>), Y (<em>synF</em>) and W135 (<em>synG</em>). The <em>sacB</em> gene is targeted for serogroup A and the <em>xcbA</em> gene, which most likely encodes the capsule polymerase, is targeted for serogroup X (<a href="chpt10-pcr.html#ref2">2</a>, <a href="chpt10-pcr.html#ref35">35</a>). These target genes are depicted within the structure of the capsule gene complex for serogroups A, B, C, Y, W135, and X (<a href="chpt10-pcr.html#figure2">Figure 2</a>). The most current adapted primer and probe sequences for the serogrouping real-time PCR assays are listed in <a href="chpt10-pcr.html#table3">Table 3</a>, though, periodically, the primers and probes are adapted as new information regarding probe chemistries and allelic variations become available.</p>

<p>There have been several different systems in place for naming the genes for meningococcal capsule biosynthesis. Some groups have called this operon <em>syn</em> for capsule bio<em>syn</em>thesis (<a href="chpt10-pcr.html#ref21">21</a>, <a href="chpt10-pcr.html#ref51">51</a>); other groups have used the <em>sia</em> nomenclature for <em>sia</em>lic acid biosynthesis (<a href="chpt10-pcr.html#ref16">16</a>, <a href="chpt10-pcr.html#ref22">22</a>); still others have referred to them as <em>neu</em> genes based on homologies to <em>E. coli</em> K1 genes for <em>N</em>-acetyl<em>neu</em>raminic acidbiosynthesis (<a href="chpt10-pcr.html#ref21">21</a>). While the <em>siaD</em> genes of serogroups B, C, W135, and Y were initially thought to be alleles, more extensive sequencing analysis demonstrated that this is not so. The <em>synD</em> and <em>synE</em> genes of serogroups B and C, respectively, are alleles and encode capsular polysaccharide polymerases that catalyze different linkages of sialic acid monomers (α2→8 linkage for serogroup B and α2→9 linkage for serogroup C). However, the Y and W135 genes are over twice the size of the B and C genes and differ in nucleotide sequence (<a href="chpt10-pcr.html#ref11">11</a>, <a href="chpt10-pcr.html#ref50">50</a>), though they are highly similar to each other. In addition, the polymerases for serogroups Y and W135 link heteropolymers of sialic acid plus either glucose or galactose, respectively. Thus, the capsular polysaccharide polymerase genes of serogroups Y and W135 are alleles. To continue to call all of these polymerase genes <em>siaD</em> would be a misnomer. For these reasons and for simplicity, this text will use the <em>synABCD</em>/<em>E</em>/<em>F</em>/<em>G</em> nomenclature listed above and in <a href="chpt10-pcr.html#table1">Table I</a>.</p>

<a id="table1" name="table1"> </a>
<p><span class="bold">Table 1.</span> Serogroup capsule type and gene targets for genotyping real-time PCR assays</p>

<!--START TABLE-->
<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Sero-group</th>
<th scope="col">Capsule type</th>
<th scope="col">Gene Target Name</th>
<th scope="col">Alternate Gene Names</th>
<th scope="col">Ref</th>
</tr>

</thead>
<tr>
<th scope="row">A</th>
<td>(α1→6)-N-acetyl-D-mannosamine-1-phosphate</td>
<td><em>sacB</em></td>
<td>&#160;</td>
<td>(31)</td>
</tr>

<tr>
<th scope="row">B</th>
<td>(α2→8)- N-acetylneuraminic acid</td>
<td><em>synD</em></td>
<td><em>siaD<br />
siaD</em> of B<br />
<em>siaD<sub>B</sub></em></td>
<td>(5, 9, 23, 48, 61)</td>
</tr>

<tr>
<th scope="row">C</th>
<td>(α2→9)- N-acetylneuraminic acid</td>
<td><em>synE</em></td>
<td><em>siaD</em> of C<br />
<em>siaD<sub>C</sub></em></td>
<td>(5, 9, 50, 58, 61)</td>
</tr>

<tr>
<th scope="row">W135</th>
<td>6-D-Gal(α1→4)-N-acetylneuraminic acid(α2→6</td>
<td><em>synG</em></td>
<td><em>siaD</em> of W135<br />
<em>siaD<sub>W</sub></em></td>
<td>(4, 9, 14, 35, 49, 61)</td>
</tr>

<tr>
<th scope="row">X</th>
<td>(α1→4)-<em>N</em>-acetyl-D-glucosamine-1-phosphate</td>
<td><em>xcbB</em></td>
<td>&#160;</td>
<td>(2, 6)</td>
</tr>

<tr>
<th scope="row">Y</th>
<td>6-D-Glc(α1→4)-N-acetylneuraminic acid(α2→6)</td>
<td><em>synF</em></td>
<td><em>siaD</em> of Y<br />
<em>siaD<sub>Y</sub></em></td>
<td>(4, 9, 14, 35, 49, 61)</td>
</tr>
</table>

<p>For serogroups B, C, Y and W135, the first three genes of the <em>syn</em> operon encode functions for the synthesis of capsule polysaccharide precursors (<a href="chpt10-pcr.html#ref32">32</a>, <a href="chpt10-pcr.html#ref48">48</a>, <a href="chpt10-pcr.html#ref51">51</a>). The fourth gene product is a polymerase that catalyzes the formation of polymers with the serogroup-specific linkage. In serogroups B and C, the products of a four-gene operon (<em>synABC</em> plus the polysialyltransferase gene <em>synD</em> [Nmen B] or <em>synE</em> [Nmen C]) are responsible for biosynthesis of the sialic acid (also known as <em>N</em>-acetylneuraminic acid, NeuNAc, or NANA) homopolymer. Expression of the poly-<em>N</em>-acetylmamosamine-1-phosphate capsule polymer of serogroup A requires the <em>sacABCD</em> operon (formerly known as <em>mynABCD</em> (<a href="chpt10-pcr.html#ref49">49</a>), while expression of the poly-<em>N</em>-acetyl-D-glucosamine-1-phosphate capsule of serogroup X requires the <em>xcbABC</em> operon.</p>

<div class="span14 fixed"><a id="figure2" name="figure2"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 2 is an image showing the genetic maps of the capsule gene complex (cps) of N. meningitidis. The ctrABCD operon encodes ATP-dependent export proteins. synABC D/E/F/G, sacABCD, and xcbABC, encode the serogroup-specific enzymes for capsule polymer biosynthesis. oatC (serogroup C) and oatWY (serogroups W135 and Y), are co-transcribed with the syn operons and encode O-acetyltransferases. lipA and lipB code for proteins that were originally proposed to add a phospholipid-anchoring group onto the polysaccharide reducing end before transport. ctrE and ctrF, formerly known as lipA and lipB, respectively, are involved in capsule transport. These gene products were once thought to be involved in post-polymerization modification. Many nongroupable, carried meningococci lack all or part of the capsule locus; those lacking the entire locus have a genetic configuration at this position like those of N. gonorrhoeae and N. lactamica, which are not known to synthesize capsule." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure02.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 2. Genetic maps of the capsule gene complex (<em>cps</em>) of <em>N. meningitidis</em> (adapted from (<a href="chpt10-pcr.html#ref14">14</a>)). The <em>ctrABCD</em> operon encodes ATP-dependent export proteins (grid pattern). <em>synABC</em> (solid gray) <em>D</em>/<em>E</em>/<em>F</em>/<em>G</em> (dotted), <em>sacABCD</em> (horizontal striped), and <em>xcbABC</em> (dashed upward diagonal), encode the serogroup-specific enzymes for capsule polymer biosynthesis. <em>oatC</em> (serogroup C) and <em>oatWY</em> (serogroups W135 and Y), are co-transcribed with the <em>syn</em> operons and encode O-acetyltransferases (9). <em>lipA</em> and <em>lipB</em> code for proteins that were originally proposed to add a phospholipid-anchoring group onto the polysaccharide reducing end before transport (<a href="chpt10-pcr.html#ref19">19</a>). <em>ctrE</em> and <em>ctrF</em> (diagonal brick), formerly known as <em>lipA</em> and <em>lipB</em>, respectively, are involved in capsule transport (<a href="chpt10-pcr.html#ref57">57</a>). These gene products were once thought to be involved in post-polymerization modification. Many nongroupable, carried meningococci lack all or part of the capsule locus (<a href="chpt10-pcr.html#ref10">10</a>, <a href="chpt10-pcr.html#ref14">14</a>); those lacking the entire locus have a genetic configuration at this position like those of <em>N. gonorrhoeae</em> and <em>N. lactamica</em>, which are not known to synthesize capsule....<a href="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure02-view.gif">view larger</a></p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>

<h5><em>Haemophilus influenzae</em></h5>

<p>The capsule locus of all six serotypes of <em>H. influenzae</em> (Hi a, b, c, d, e, and f) consists of three regions encoding functions for capsule polysaccharide synthesis, modification, and translocation (<a href="chpt10-pcr.html#figure3">Figures 3</a> and <a href="chpt10-pcr.html#figure4">4</a>) (<a href="chpt10-pcr.html#ref25">25</a>, <a href="chpt10-pcr.html#ref43">43</a>, <a href="chpt10-pcr.html#ref44">44</a>). <em>bexDCBA</em> in the ATP-driven export region (also known as Region I) code for protein components of an ATP-driven polysaccharide export apparatus. <em>hcsA</em> and <em>hcsB</em> in post polymerization modification region (also known as Region III) share high similarity with <em>lipA</em> and <em>lipB</em> (recently renamed <em>ctrE</em> and <em>ctrF</em>), respectively, which are involved in modification and export of meningococcal capsule polysaccharide (<a href="chpt10-pcr.html#ref43">43</a>). Both regions are common to all serotypes. The same nomenclature is used for genes in the two regions for all six serotypes. The serotype-specific region (previously Region II) contains genes for capsule synthesis and is unique to each serotype. The serotype-specific genes are named <em>acs</em>, <em>bcs</em>, etc. for "a capsule synthesis", "b capsule synthesis", and so on.</p>

<p>The <em>cap</em> locus encodes functions for <em>H. influenzae</em> capsule synthesis. The genetic organization of the <em>cap</em> locus has been well characterized in Hib and Hif, which belong to two phylogenetic divisions that are defined by multilocus enzyme electrophoresis typing. The ATP-driven export region includes most of Hia and Hib strains, and all of Hic, Hid, and Hie strains. Strains from this region have at least one completed <em>cap</em> locus flanked by insertion sequence (IS) element IS<em>1016</em>, except Hie. The majority of Hib strains and some Hia strains from this region have the direct-repeat configuration with the second copy of the <em>bexA</em> gene partially deleted as illustrated in <a href="chpt10-pcr.html#figure4">Figure 4</a>. The truncated <em>cap</em> locus is not required for capsule synthesis (<a href="chpt10-pcr.html#ref26">26</a>-<a href="chpt10-pcr.html#ref28">28</a>). In Hif, Hib, and Hia strains of the serotype-specific region and some Hie strains of the ATP-driven export region, the <em>cap</em> locus is flanked by <em>sodC</em> and HI1637 (<a href="chpt10-pcr.html#ref26">26</a>, <a href="chpt10-pcr.html#ref44">44</a>).</p>

<p>While serotype b causes the vast majority of <em>H. influenzae</em> disease in countries without a Hib vaccination program, serotypes a, c, d, e, and f, and nontypeable <em>H. influenzae</em> (NTHi), also contribute to case numbers (<a href="chpt10-pcr.html#ref43">43</a>, <a href="chpt10-pcr.html#ref59">59</a>). As implementation of Hib vaccine becomes more widespread, it is important to monitor incoming specimens for both b and non-b serotypes of <em>H. influenzae</em>. Therefore, serotype-specific real-time PCR assays have been developed that target the serotype-specific region genes where possible, or the 5' end of <em>bexD</em>, which is less conserved among the serotypes compared with the other genes in the export region. The genes targeted for real-time PCR assays specific to each serotype are as follows: <em>acsB</em> (Hia), <em>bcsB</em> (Hib), <em>ccsD</em> (Hic), <em>dcsE</em> (Hid), <em>ecsH</em> (Hie), and <em>bexD</em> (Hif) (<a href="chpt10-pcr.html#figure3">Figure 3</a>). The primer and probe information for each of these serotyping assays can be found in <a href="chpt10-pcr.html#table4">Table 4</a>. Each of these assays has been shown to be highly specific and sensitive for their respective serotypes.</p>

<div class="span14 fixed"><a id="figure3" name="figure3"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 3 is an image showing the capsule loci for H. influenzae serotypes a, b, c, d, e, and f, including the target genes for serotype-specific real-time PCR assays. The capsule locus of all six serotypes of H. influenzae (Hia-f) consists of three regions encoding functions for capsule polysaccharide synthesis, modification, and translocation. bexDCBA in the ATP-driven export region code for protein components of an ATP-driven polysaccharide export apparatus. hcsA and hcsB are in the post polymerization modification region and may be involved in the modification and export of capsule polysaccharide. The serotype-specific region contains genes for capsule synthesis and is unique to each serotype. The serotype-specific genes are named acs, bcs, etc. for a capsule synthesis, b capsule synthesis, and so on. With the exception of the Hif serotype-specific assay, the target genes for the serotype-specific assays can be found in this region." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure03.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 3. Capsule loci for <em>H. influenzae</em> serotypes a, b, c, d, e, and f, including the target genes for serotype-specific real-time PCR assays. The capsule locus of all six serotypes of <em>H. influenzae</em> (Hia-f) consists of three regions encoding functions for capsule polysaccharide synthesis, modification, and translocation. <em>bexDCBA</em> in the ATP-driven export region (white arrows) code for protein components of an ATP-driven polysaccharide export apparatus. <em>hcsA</em> and <em>hcsB</em> are in the post polymerization modification region (gray arrows) and may be involved in the modification and export of capsule polysaccharide. The serotype-specific region (colored arrows) contains genes for capsule synthesis and is unique to each serotype. The serotype-specific genes are named <em>acs</em>, <em>bcs</em>, etc. for "a capsule synthesis", "b capsule synthesis", and so on. With the exception of the Hif serotype-specific assay, the target genes for the serotype-specific assays can be found in this region and are highlighted by the red boxes...<a href="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure03-view.gif">view larger</a></p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>

<div class="span14 fixed"><a id="figure4" name="figure4"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 4 is an image showing the genetic organization of the cap locus in Hib. Partially duplicated cap locus of Hib Hi 1007 showing the truncated ATP-driven export region with the 1.2-kb deletion between IS1016 and bexA." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure04.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 4. Genetic organization of the <em>cap</em> locus in Hib (adapted from (43)). Partially duplicated <em>cap</em> locus of Hib Hi 1007 showing the truncated ATP-driven export region with the 1.2-kb deletion between IS<em>1016</em> and <em>bexA</em>...<a href="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure04-view.gif">view larger</a></p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>

<li><span class="bold"><em>S. pneumoniae</em> serotyping PCR assays</span>
<p><em>S. pneumoniae</em> can be further classified into at least 93 serotypes based on the immunochemistry of their capsular polysaccharides. The high cost of antisera, subjectivity in interpretation, need for a complete set of control strains, and technical expertise requirements associated with these serologic methods have resulted in the more recent development of PCR-based serotyping systems. PCR-serotyping has the potential to overcome some of the difficulties associated with serologic testing and the development of PCR-based assays for direct detection of serotypes from clinical specimens is a valuable aid in surveillance, particularly in situations where culture is insensitive. PCR assays (both conventional and real-time) for the detection of the more common serotypes are being developed and are discussed in more detail below (<a href="chpt10-pcr.html#ref38">38</a>, <a href="chpt10-pcr.html#ref54">54</a>).</p>

<h5>Multiplex conventional PCR assays for serotyping <em>S. pneumoniae</em></h5>

<p>A multiplex PCR-based serotyping scheme that includes 40 serotype specificities has been developed (<a href="chpt10-pcr.html#ref38">38</a>). This PCR approach has the potential to greatly reduce reliance upon conventional serotyping and provides serotype-determining potential to laboratories that lack type-specific antisera and other reagents needed for conventional serotyping, yet have the equipment necessary for DNA amplification and electrophoresis.</p>

<p>The multiplex approach uses 9 reactions to identify 40 serospecificities (<a href="chpt10-pcr.html#table5">Table 5</a> for primers, <a href="chpt10-pcr.html#table6">Tables 6</a>-<a href="chpt10-pcr.html#table8">8</a> for schemes, and <a href="chpt10-pcr.html#figure5">Figure 5</a> for PCR products) but also provides some flexibility that allows for altering combinations of serotypes included in each sequential reaction. These can be modified based on the most prevalent serotypes in any given geographic region but do require validation to ensure no cross reaction between serotype primer sets. Three such schemes based upon pneumococcal serotype prevalence in the USA, Africa, and Latin America have been designed (<a href="chpt10-pcr.html#table6">Tables 6</a>-<a href="chpt10-pcr.html#table8">8</a>). These schemes will continue to be refined as additional serotypes are added and primer sets updated to improve specificity and sensitivity. The most current methods are described at the <a href="https://www.cdc.gov/streplab/pcr.html">CDC <em>Streptococcus</em> website</a> and should be consulted on a regular basis</p>

<h5>Real-time PCR assays for serotyping <em>S. pneumoniae</em></h5>

<p>A number of real-time PCR assays for serotyping <em>S. pneumoniae</em> have been published in the literature and others are being developed (<a href="chpt10-pcr.html#ref34">34</a>, <a href="chpt10-pcr.html#ref40">40</a>). These real-time assays are recommended for determining serotypes from clinical specimens when DNA may be present in low amounts and insufficient for conventional multiplex PCR serotyping. Please see the <a href="https://www.cdc.gov/streplab/pcr.html"><em>Streptococcus</em> Laboratory website</a> for an example.</p>
</li>

<li><span class="bold">Multiplex real-time PCR for pathogen detection</span>
<p>Real-time PCR allows for development of multiplex assays for detection of several genes in the same reaction by using specific probes with different fluorescent dye labels. Multiplex real-time PCR assays are available for detection of <em>N. meningitidis</em>, <em>H. influenzae</em>, and <em>S. pneumoniae</em> in a single reaction. In addition, assays for serogrouping <em>N. meningitidis</em> and serotyping <em>S. pneumoniae</em> using a multiplex approach have also been developed. However, they are not recommended for routine use. The multiplex approach requires careful optimization. Changes in DNA polymerase or PCR reagent concentrations may lead to loss of sensitivity. The assays need to be optimized and re-evaluated, which is a very time-consuming process and requires a complete set of reference strains.</p>
</li>
</ol>
</li>

<li><span class="bold">Preparation of DNA template for PCR</span>
<ol type="A">
<li>General considerations

<p>Both crude DNA preps (boiled cell suspension) and extracted genomic DNA can be used as template for PCR amplification. DNA extraction from clinical specimens or isolates should be performed in a separate room from the room used for PCR reaction assembly (prior to DNA addition). If separate rooms are not possible, separate laboratory benches should be used. The use of boiled extracts may limit the risk of contamination as it limits manipulation. Moreover, PCR on clinical specimens should be performed in a separate room from where bacteria are cultured with strict organization of the laboratory workflow. Use of a biological safety cabinet is necessary for infectious materials such as clinical isolates, blood, and CSF. Separate pipettes, laboratory coats, and gloves should be used for performing DNA extraction procedures. Signage indicating which spaces and equipment are DNA-free and which are used to prepare or manipulate DNA would be helpful. UV irradiation and decontamination of surfaces and equipment with 10% bleach followed by 70% ethanol should be performed after any manipulation of nucleic acids at the laboratory bench. Always use filter-barrier pipette tips and change and discard gloves frequently.</p>
</li>

<li>Equipment, consumables, and reagents for DNA extraction

<ul>
<li><span class="bold">Equipment</span>
<ul>
<li>Microcentrifuge with refrigerating function</li>

<li>Water bath or dry block heater</li>

<li>Vortexer</li>

<li>Freezer</li>

<li>Refrigerator</li>

<li>pH meter</li>

<li>Balance</li>

<li>Stir plate</li>
</ul>
</li>

<li><strong>Consumables:</strong>
<ul>
<li>10% bleach (10:1, water: concentrated bleach) (make fresh weekly)</li>

<li>70% ethanol</li>

<li>1.5 ml microcentrifuge tubes (sterile, DNAase free, or PCR grade)</li>

<li>1 set of micropipettors (1-10 µl, 2-20 µl, 20-200 µl, and 100-1000 µl)</li>

<li>Pre-sterilized filter tips (10 µl, 200 µl, and 1000 µl)</li>
</ul>
</li>

<li><span class="bold">Reagents:</span>
<ul>
<li>TE Buffer (10 mM Tris HCl, pH 8.0, 1 mM EDTA)</li>

<li>Tris buffer (10 mM Tris HCl, pH 8.0)</li>

<li>Lysozyme</li>

<li>Mutanolysin</li>

<li>Proteinase K (20 mg/ml)</li>

<li>Lysis buffer (4% SDS, 10 mM EDTA pH 8.0)</li>

<li>Digestion buffer</li>

<li>Phenol</li>

<li>Chloroform</li>

<li>Phenol: chloroform (1:1)</li>

<li>Commercial DNA extraction kits are available for culture and blood and body fluids</li>

<li>Commercial PCR Master Mix containing dNTPs and DNA polymerase is available</li>
</ul>
</li>
</ul>
</li>

<li><span class="bold">Preparing reagent solutions</span>
<h5><strong>EDTA, 0.1 M, pH 8.0 (100ml)</strong></h5>

<ol>
<li>Dissolve 3.7 g EDTA in 70 ml distilled deionized H<sub>2</sub>O (ddH<sub>2</sub>O).</li>

<li>Adjust pH to 8.0 with 10 M NaOH.</li>

<li>Add ddH<sub>2</sub>O to 100 ml.</li>

<li>Autoclave at 121°C for 20 minutes.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>Tris-HCl, 0.1 M, pH 8.0</strong></h5>

<ol>
<li>Dissolve 1.2 g Tris base in 80 ml ddH<sub>2</sub>O.</li>

<li>Adjust to pH 8.0 with concentrated HCl.</li>

<li>Mix and add ddH<sub>2</sub>O to 100 ml.</li>

<li>Autoclave at 121°C for 20 minutes.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>Tris-HCl, 0.1 M, pH 7.6</strong></h5>

<ol>
<li>Dissolve 1.2 g Tris base in 80 ml ddH<sub>2</sub>O.</li>

<li>Adjust to pH 7.6 with concentrated HCl.</li>

<li>Mix and add ddH<sub>2</sub>O to 100 ml.</li>

<li>Autoclave at 121°C for 20 minutes.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>20% Sodium dodecyl sulfate (SDS)*</strong></h5>

<ol>
<li>Dissolve 20 g SDS in 100 ml sterile ddH<sub>2</sub>O.</li>

<li>Place the container into a 50-607°C water bath to facilitate dissolving.</li>

<li>Avoid vigorous shaking that generates bubbles.</li>

<li>Store at room temperature.</li>
</ol>

<p>*Eye and respiratory protection should be worn when weighing powdered SDS.</p>

<h5><strong>TE buffer (10 mM Tris HCl, pH 8.0, 1 mM EDTA)</strong></h5>

<ol>
<li>Add 10 ml of 0.1 M Tris-HCl, pH 8.0.</li>

<li>Add 1 ml of 0.1 M EDTA, pH 8.0.</li>

<li>Add sterile ddH<sub>2</sub>O to 100 ml and mix well.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>Tris-HCl buffer (10 mM, pH 8.0)</strong></h5>

<ol>
<li>Add 10 ml of 0.1 M Tris-HCl.</li>

<li>Add 90 ml sterile ddH<sub>2</sub>O and mix well.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>Lysis buffer (4% SDS, 10 mM EDTA pH 8.0)</strong></h5>

<ol>
<li>Add 20 ml of 20% SDS.</li>

<li>Add 10 ml of 0.1 M EDTA pH 8.0.</li>

<li>Add sterile ddH<sub>2</sub>O to 100 ml and mix well.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>Mutanolysin stock solution (2,500 U/ml)</strong></h5>

<ol>
<li>Reconstitute the entire bottle of mutanolysin with sterile ddH<sub>2</sub>O to produce a concentration of 2,500 U/ml.</li>

<li>Aliquot into sterile screw-top microcentrifuge tubes.</li>

<li>Store aliquots at -20°C.</li>
</ol>

<h5><strong>Digestion buffer (0.04 g/ml lysozyme and 75 U/ml mutanolysin in TE buffer)</strong></h5>

<ol>
<li>For 1 ml of TE buffer containing lysozyme and mutanolysin, add 40 mg lyophilized lysozyme to a 1.5 ml microcentrifuge tube.</li>

<li>Add 1 ml TE buffer.</li>

<li>Add 30 µl of a stock solution of mutanolysin at 2500 U/ml.

<ul>
<li>This solution should be prepared ≤ 15 minutes prior to use and should not be reused.</li>
</ul>
</li>
</ol>

<h5><strong>Proteinase K (20 mg/ml)</strong></h5>

<ol>
<li>Dissolve 200 mg of proteinase K powder in 10 ml of sterile ddH<sub>2</sub>O.</li>

<li>Aliquot 1 ml into microcentrifuge tubes.</li>

<li>Store at -20°C.</li>
</ol>

<h5><strong>Hyaluronidase (30 mg/ml)</strong></h5>

<ol>
<li>Dilute 100 mg in 3.3 ml sterile ddH<sub>2</sub>O to make 30 mg/ml solution.</li>

<li>Dispense in 500 µl aliquots; store at -20°C.</li>
</ol>

<h5><strong>Sodium acetate (3.0 M, pH 5.5)</strong></h5>

<ol>
<li>Dissolve 40.8 g sodium acetate in 80 ml sterile ddH<sub>2</sub>O.</li>

<li>Adjust pH to 5.5 with glacial acetic acid.</li>

<li>Add sterile ddH<sub>2</sub>O to final volume of 100 ml.</li>

<li>Autoclave at 121°C for 20 minutes.</li>

<li>Store at room temperature.</li>
</ol>

<h5><strong>Phenol: chloroform (1:1)</strong></h5>

<ol>
<li>Melt solid or liquified phenol in a 68°C water bath. Liquified phenol should be stored at -20°C.</li>

<li>Mix equal volumes of phenol and chloroform.</li>

<li>Add an equal volume of 0.1 M Tris-HCl pH 7.6 to the phenol.</li>

<li>Mix for 15 minutes and place the bottle back to water bath to allow the phases to separate.</li>

<li>Remove the top aqueous layer as much as possible.</li>

<li>Repeat step b-d until the top aqueous layer reaches ~pH 7.6 (pH should be measured with pH paper. Do not use a pH meter.).</li>

<li>After the phenol is equilibrated, add an equal volume of 0.01 M Tris-HCl (pH 7.6).</li>

<li>Store in a dark glass bottle at 4°C up to 6 months.</li>
</ol>
</li>

<li><span class="bold">Fast preparation of DNA template from clinical isolates</span>
<p class="bold"><em>N. meningitidis</em> and <em>H. influenzae</em> (gram-negative)</p>

<ol>
<li>Dispense 1.0 ml of 10 mM Tris (pH 8.0) buffer into 1.5 ml microcentrifuge tubes and label.</li>

<li>Harvest colonies from 18-24 hour pure cultures of <em>H. influenzae</em> or <em>N. meningitidis</em> using a sterile polyester or rayon-tipped swab and swirl the swab in the Tris buffer to make a turbid suspension (equivalent to a McFarland 3.0 standard). Be careful not to pick up pieces of agar on the swab.</li>

<li>Vortex briefly and boil cell suspension at 100°C for 10 minutes.</li>

<li>Proceed immediately with PCR or store at -20°C.</li>
</ol>

<h5><strong>Fast DNA extraction protocol for</strong> <em><strong>S. pneumoniae</strong></em> <strong>(gram-positive)</strong></h5>

<ol>
<li>Dispense 300 µl of 0.85% NaCl into 1.5 ml microcentrifuge tubes and label.</li>

<li>Harvest colonies (use 1 loopful of a 10 µl loop) from 18-24 hour pure cultures of <em>S. pneumoniae</em> using a sterile polyester or rayon-tipped swab and swirl the swab in the 0.85% NaCl to make a turbid suspension (equivalent to McFarland 3.0 standard). Be careful not to pick up pieces of agar on swab.</li>

<li>Vortex briefly and incubate at 70°C for 15 minutes.</li>

<li>Microcentrifuge at 12,000 x <em>g</em> for 2 minutes and remove the supernatant.</li>

<li>Re-suspend in 50 µl TE buffer (10 mM Tris-HCl, 100 µM EDTA, pH 8.0) and add 10 µl mutanolysin (3000 U/ ml)* and 8 µl of hyaluronidase (30 mg/ml)**</li>

<li>Incubate at 37°C for 30 minutes up to 18 hours (overnight).</li>

<li>Heat-inactivate the enzymes in the suspension by boiling at 100°C for 10 minutes.</li>

<li>Microcentrifuge at 12,000 x <em>g</em> for 4 minutes and remove supernatant for use as DNA template.</li>

<li>Proceed immediately with PCR or store at -20°C.</li>
</ol>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty">*Mutanolysin (10,000 U). Dilute in 3.3 ml of TE buffer to make 3000 U/ml stock solution, store at -20°C as 500 µl aliquots.</p>

<p class="margin-left-twenty">**Hyaluronidase (100 mg). Dilute in 3.3 ml of TE buffer to make 30 mg/ml solution, store at -20°C as 500 µl aliquots.</p>
</li>

<li><span class="bold">Extracting genomic DNA from clinical isolates and specimens</span>
<p>Efficient extraction of the DNA template is a necessary step for any real-time PCR assay. The goal of DNA extraction is to lyse the bacterial cells in the specimens to maximize bacterial DNA yield and quality while removing any PCR inhibitors (i.e. salts, proteins), dissolve the DNA in a buffer compatible with the enzymes used in the next step and concentrating the DNA at the same time. When considering a DNA extraction method, it is important to select one that will produce an adequate DNA yield for detection by real-time PCR (dependant on the assay-specific lower limit of detection) without purifying potential PCR inhibitors as well. Things to consider are the type and volume of specimen, nucleic acid sought (DNA or RNA), concentration of the target DNA present in the specimen, impurities present that could act as PCR inhibitors, facilities/equipment available, and safety requirements. Generally, methods with fewer steps decrease chances of contamination and loss of DNA. Commercial methods are available for both cell lysis and purification and include silica membrane, spin column, and magnetic bead technology, in addition to biochemical and physical methods. In general, these methods produce adequate results as long as the protocol provided by the manufacturer is precisely followed.</p>

<ol>
<li><span class="bold">Bacterial cell lysis</span>
<p>The first step in extracting and purifying bacterial DNA is to lyse the bacterial cell walls for maximum DNA yield. There are multiple ways to lyse bacterial cells, either physically or chemically, and this step can be optimized by considering the suspected bacteria and starting specimen material, as well as the materials available to each laboratory. Chemical or enzymatic based lysis methods are typically simpler to perform and can be more cost efficient. Both <em>N. meningitidis</em> and <em>H. influenzae</em> are Gram negative and can be effectively lysed using lysis buffer containing protease such as Proteinase K along with a detergent. Incubation temperature and duration vary between organisms and specimen material. The optimal temperature range for Proteinase K activity is between 55-65°C. At temperatures above 65°C, the enzyme activity decreases. However, specimens incubated at 37°C can be left for longer incubation periods without affecting DNA quality. Specimens should be incubated until cells are completely lysed (when solution clears) and the time will vary between specimens. Once the bacteria are completely lysed one should proceed to the next step.</p>

<p>For optimal yields of <em>S. pneumoniae</em>, which is gram-positive, additional enzyme digestion with lysozyme and mutanolysin will help to degrade the higher content of peptidoglycan in the cell wall before being lysed with buffer. The temperature and length of the enzyme incubation will depend on the concentration and type of enzyme used, as well as the lysis buffer used. High temperature incubation and repeated freeze/thaw cycles are generally used with higher concentrations of cells, such as when extracting from cultures. Physical lysis can be performed using a liquid or pressure cell homogenizer, sonication, or shaking with glass beads, although some of these methods will require additional and sometimes costly equipment and they tend to shear the DNA in to smaller fragments.</p>

<p class="bold">Enzyme lysis for clinical specimens of unknown etiology or known gram-positive cell suspensions (<em>S. pneumoniae</em>)</p>

<ol>
<li>Prepare digestion buffer (0.04 g/ml lysozyme and 75 U/ml mutanolysin in TE buffer).

<ul>
<li>This solution should be prepared ≤ 15 minutes before use and not reused.</li>
</ul>
</li>

<li>Add 100 µl of digestion buffer to each microcentrifuge tube.</li>

<li>Add 200 µl of bacterial cell suspension or clinical specimen to each microcentrifuge tube. Vortex and incubate at 37°C for 1 hour.

<ul>
<li>If specimen volume is less than 200 µl, note the volume in lab records and add TE buffer to a total volume of 200 µl. If the specimen tube appears to be empty, wash the sides of the tube with 200 µl TE buffer.</li>
</ul>
</li>

<li>Add 200 µl of cell lysis buffer to each microcentrifuge tube.</li>

<li>Add 20 µl of Proteinase K (20 mg/ml) and invert each tube until the phases are completely mixed.</li>

<li>Incubate at 37°C for 30 minutes to 1 hour.</li>

<li>Purify DNA before using in real-time PCR reactions.</li>
</ol>

<p class="bold">Enzyme lysis for known gram-negative clinical specimens or cell suspensions (<em>N. meningitidis</em> and <em>H. influenzae</em>)</p>

<ol>
<li>Prepare cell lysis buffer (4% SDS, 1 mM EDTA pH 8.0).</li>

<li>Add 200 µl of lysis buffer to each microcentrifuge tube.</li>

<li>Add 200 µl of bacterial cell suspension or clinical specimen (CSF, serum, or blood) to each microcentrifuge tube.</li>

<li>Add 20 µl of Proteinase K (20 mg/ml) for a final concentration of 1 mg/ml and invert each tube until the phases are completely mixed.</li>

<li>Incubate at 37°C for 30 minutes to 1 hour.</li>

<li>Purify DNA before using in real-time PCR reactions.</li>
</ol>
</li>

<li><span class="bold">Removal of eukaryotes from blood</span>
<p>If DNA is being extracted from clinical specimens such as blood, CSF, or serum, considerations should be taken to remove potential PCR inhibitors from the surrounding material before lysing the cells. If the starting specimen is blood, steps to remove erythrocytes can results in higher DNA yield with fewer PCR inhibitors. Hemoglobin is very inhibitory to DNA polymerases. Erythrocytes can be removed through the use of a hypotonic buffer or by using gradient centrifugation to create a buffy coat in which leukocytes are concentrated. Bacteria in large volumes of clinical specimens can be concentrated by centrifugation or antigen capture to increase yield.</p>

<p class="bold">Preparation of buffy coat</p>

<ol>
<li>Centrifuge the blood specimen at 2,500 x <em>g</em> for 10 minutes at room temperature.</li>

<li>Three layers should be apparent after centrifugation. The top layer should be clear and contains plasma. The light tan middle layer is the buffy coat and contains concentrated leukocytes. The bottom red layer contains erythrocytes.</li>
</ol>
</li>

<li><span class="bold">Purification of DNA</span>
<p>Purification of the extraction product is important to remove any residual material that could potentially inhibit real-time PCR. Purification can be performed by many commercially available extraction kits or with the use of organic solvents, such as the chloroform/phenol method. Some methods may purify RNA along with DNA and as RNA may inhibit some reactions, use of RNAase improves purity of DNA as well.</p>

<p class="bold">Phenol/Chloroform to remove cell debris and proteins</p>

<p>Phenol is a hazardous organic solvent and safety precautions should be taken when working with phenol. Always use suitable chemical protection gloves when handling phenol containing solutions. Specific waste procedures may be required for the disposal of solutions containing phenol.</p>

<ol>
<li>To a lysed specimen, add an equal volume of phenol: chloroform solution (1:1). Mix well by inversion or briefly vortex.</li>

<li>Centrifuge the tube at 16,000 x <em>g</em>for 15 minutes in a microcentrifuge.</li>

<li>Carefully remove the top aqueous layer from the bottom phenol layer and transfer to a new tube, being careful to avoid the interface.</li>

<li>Steps 1-3 can be repeated until an interface is no longer visible.</li>

<li>To remove all traces of phenol, add an equal volume of chloroform to the aqueous layer and centrifuge the tube at 16,000 x <em>g</em>for 15 minutes in a microcentrifuge.</li>

<li>Carefully remove the top aqueous layer from the bottom chloroform layer and transfer to a new tube, being careful to avoid the interface.</li>

<li>Steps 5-6 can be repeated until an interface is no longer visible.</li>

<li>Precipitate the DNA by ethanol or isopropanol.</li>
</ol>

<p class="bold">Precipitation of DNA by ethanol or isopropanol</p>

<ol>
<li>Add a 0.1 (1/10th) volume of 3.0 M sodium acetate (pH 5.5) to the aqueous phase and then 2 volumes of 95% ethanol. Incubate at -20°C overnight or for shorter periods at -80°C (e.g. 20-30 minutes). Proceed with step 3.</li>

<li>If isopropanol is used: Add a 0.1 volume of 3.0 M sodium acetate (pH 5.5) to the aqueous phase and then 0.6 volumes of 100% isopropanol. Incubate at -20°C for 2 hours or for shorter periods at -80°C (e.g., 10-20 minutes).</li>

<li>Centrifuge at 16,000 x g for 30 min at 4°C.</li>

<li>Recover the precipitated DNA by centrifuging the tube at 16,000 x g for 15 minutes at 4°C. Remove the aqueous phase with care.</li>

<li>Add 2 volumes (of original sample) of 75% (v/v) ethanol and leave at room temperature for 5-10 minutes to remove excess salt and traces of phenol and chloroform from the pellet.</li>

<li>Centrifuge at 16,000 x g for 5 minutes. Remove with care as much ethanol as possible from the microcentrifuge tube using a filtered pipette tip to avoid dislodging the pellet.</li>

<li>Dry the DNA pellet in air, in a desiccator, or in a 50°C oven for 5 minutes.</li>

<li>The dried DNA may be dissolved in sterile Tris buffer (10mM Tris-HCl, pH 8.0) and stored at 4°C for further manipulation or at -20°C for long-term storage.</li>
</ol>

<p class="bold">Storage of DNA</p>

<p>Extracted and purified DNA should be stored in a designated elution buffer from a commercial kit or in Tris buffer (10 mM Tris-HCl, pH 8.0). Distilled water can also be used but these specimens may experience degradation from acid hydrolysis. DNA can be kept at 4°C for short periods of time and at -20°C for long-term storage.</p>
</li>
</ol>
</li>

<li><span class="bold">Alternative protocol for genomic DNA extraction: Boom method</span>
<p class="bold">Reagent preparation</p>

<p class="bold">Extraction buffer L6</p>

<ol>
<li>Add 120 g of guanidinium isothiocyanate (GuSCN) to 100 ml of 0.1 M Tris/HCl (pH 6.4) and 22 ml of 0.2 M EDTA (pH 8.0) and 2.6 g of Triton X-100.</li>

<li>Stir overnight in the dark to dissolve.</li>

<li>Store away from light for up to 1 month.

<ul>
<li>GuSCN is toxic and care should be taken when handling this substance.</li>
</ul>
</li>
</ol>

<p class="bold">Extraction buffer L2</p>

<ol>
<li>Add 120 g of guanidinium isothiocyanate (GuSCN) to 100 ml of 0.1 M Tris/HCl (pH 6.4).</li>

<li>Stir overnight in the dark to dissolve.</li>

<li>Store away from light for 1 month.</li>
</ol>

<p class="bold">Size fractionated silica</p>

<ol>
<li>Add 60 g of silicon dioxide to 500 ml of distilled water in a graduated cylinder and leave at room temperature for 24 hours.</li>

<li>Remove and discard 430 ml of supernatant and re-suspend solids in 500 ml of ddH<sub>2</sub>O.</li>

<li>Leave at room temperature for 5 hours and remove and discard 440 ml of supernatant.</li>

<li>Add 600 µl of concentrated HCl (pH 2.0), mix, and aliquot into 1.5 ml volumes.</li>

<li>Sterilize by autoclaving and store away from light for up to 6 months.</li>
</ol>

<p class="bold">DNA isolation</p>

<ol>
<li>Add 100 µl of specimen (bacterial suspension or clinical specimen) to 500 µl of L6 extraction buffer and 10 µl of size fractionated silica in a 1.5 ml microcentrifuge tube. For double volumes of specimen, double the amount of L6 extraction buffer and silica, as well as reagents in the wash steps (L2 extraction buffer, ethanol and acetone).</li>

<li>Vortex the tube for 10 seconds and incubate, with shaking, at room temperature for 15 minutes.</li>

<li>Centrifuge the tube for 15 seconds at 16,100 x g and dispose of the supernatant.</li>

<li>Wash the pellet two times with 500 µl L2 extraction buffer, two times with 500 µl of 70% ethanol and one time with 500 µl of acetone. Centrifuge for 15 seconds at 16,100 x g after each wash and dispose of the supernatant following appropriate procedures for chemical waste.</li>

<li>To remove the acetone, place the tube with the lid open at 56°C in a dry heating block for 5 minutes.</li>

<li>Elute the nucleic acid from the silica by adding 30 µl of distilled water, close the tube, vortex, and incubate at 56°C for 15 minutes.</li>

<li>Centrifuge the tube at 16,100 x g for 2 minutes and collect the supernatant, taking care not to include any silica. The extracted DNA can be stored at 4°C overnight or at -70°C for long-term storage.</li>
</ol>
</li>
</ol>
</li>

<li><span class="bold">Conventional PCR</span>
<ol type="A">
<li><span class="bold">General considerations</span>
<p>PCR is a very sensitive method for amplifying a specific DNA target, but also very susceptible to contamination with extraneous DNA. Extra precautions should be taken to minimize such cross-contamination. It is recommended to physically separate the different steps including PCR reaction assembly, addition of template DNA to the reaction wells, and agarose gel detection of PCR products. If separate rooms are not possible, separate laboratory benches should be used for these steps. Working in an unventilated still air biocontainment cabinet (sometimes called a PCR hood), are also suggested to minimize cross-contamination. Separate micropipettors, laboratory coats, and gloves should be used for reaction assembly. Signage indicating which spaces and equipment are DNA-free and which are used to prepare or manipulate DNA would be helpful. Decontamination of surfaces and equipment with 10% bleach followed by 70% ethanol should be performed after any manipulation of nucleic acids at the laboratory bench. Always use filter-barrier pipette tips and change and discard gloves frequently.</p>
</li>

<li><span class="bold">Equipment, consumables, and reagents</span>
<p class="bold">Equipment:</p>

<ul>
<li>PCR thermocycler</li>

<li>Unventilated biocontainment cabinet or PCR hood</li>

<li>Freezer</li>

<li>Refrigerator</li>

<li>Electrophoresis tank</li>

<li>Power supply</li>

<li>Stir plate</li>

<li>Microwave oven</li>

<li>Gel viewing system</li>

<li>Gel documentation system</li>
</ul>

<p class="bold">Consumables:</p>

<ul>
<li>10% bleach (10:1, water: concentrated bleach) (make fresh weekly)</li>

<li>70% ethanol</li>

<li>1.5 mL microcentrifuge tubes (sterile DNase free or PCR grade)</li>

<li>96 well polypropylene plates, tube strips or individual</li>

<li>1 set of micropipettors (1-10 µl, 2-20 µl, 20-200 µl, and 100-1000 µl)</li>

<li>Pre-sterilized filter tips (10 µl, 200 µl, and 1000 µl)</li>

<li>Optical caps</li>
</ul>

<p class="bold">Optional:</p>

<ul>
<li>Commercial DNA-removing surface decontaminant liquid</li>

<li>Cap installing tool</li>
</ul>

<p class="bold">Reagents:</p>

<ul>
<li>DNA polymerase</li>

<li>dNTPs</li>

<li>Primers Positive and negative control DNA diluted to approximately 5 µg/ml</li>

<li>PCR grade water</li>

<li>TAE or TBE buffer</li>

<li>Agarose powder (molecular biology grade)</li>

<li>DNA ladder</li>

<li>6x DNA loading dye</li>

<li>Ethidium bromide</li>

<li>Bromophenol blue</li>

<li>Xylene cyanol FF</li>

<li>Sucrose</li>
</ul>
</li>

<li><span class="bold">Preparing reagent stock solution and primer working solution</span>
<p class="bold">EDTA, 0.5 M, pH 8.0 (100 ml)</p>

<ol>
<li>Dissolve 18.6 g EDTA in 70 ml ddH<sub>2</sub>O.</li>

<li>Adjust pH to 8.0 with 10 M NaOH (~5 ml).</li>

<li>Add ddH<sub>2</sub>O to 100 ml and mix well on a stir plate.</li>

<li>Store at room temperature.</li>
</ol>

<p class="bold">Ethidium bromide (EtBr), 10 mg/ml</p>

<ol>
<li>Dissolve 0.2 g ethidium bromide in 20 ml ddH<sub>2</sub>O.</li>

<li>Mix well and store at 4°C in the dark in 1 ml aliquots.</li>

<li>Store at room temperature.</li>
</ol>

<p class="bold">TAE (Tris/acetate/EDTA) electrophoresis buffer, 50 X stock solution*</p>

<ol>
<li>To 750 ml of ddH<sub>2</sub>O add:

<ul>
<li>242 g Tris base</li>

<li>57.1 ml of glacial acetic acid</li>

<li>100 ml of 0.5 M EDTA pH 8.0</li>
</ul>
</li>

<li>Add ddH<sub>2</sub>O to 1000 ml and mix well on a stir plate.</li>

<li>Store at room temperature.</li>
</ol>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*TAE stock solution should be diluted to 1X in H<sub>2</sub>O before use.</p>

<p class="bold">TBE (Tris/borate/EDTA) electrophoresis buffer, 10X stock solution*</p>

<ol>
<li>To 900 ml of ddH<sub>2</sub>O add:

<ul>
<li>108 g Tris base (890 mM)</li>

<li>55 g boric acid (890 mM)</li>

<li>40 ml 0.5 M EDTA, pH 8.0 (20 mM)</li>
</ul>
</li>

<li>Add ddH<sub>2</sub>O to 1000 ml and mix well on a stir plate.</li>

<li>Store at room temperature.</li>
</ol>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*TBE stock solution should be diluted to 0.5X in H<sub>2</sub>O before use.</p>

<p class="bold">6X DNA loading dye I</p>

<ol>
<li>0.25% bromophenol blue</li>

<li>0.25% xylene cyanol FF</li>

<li>40% (w/v) sucrose in water</li>

<li>Store at 4°C.</li>
</ol>

<p class="bold">6X DNA loading dye II</p>

<ol>
<li>0.25% bromophenol blue</li>

<li>0.25% xylene cyanol FF</li>

<li>30% glycerol in water</li>

<li>Store at 4°C.</li>
</ol>

<p class="bold">6X DNA loading dye III</p>

<ol>
<li>0.25% bromophenol blue</li>

<li>40% (w/v) sucrose in water</li>

<li>Store at 4°C.</li>
</ol>

<p class="bold">2% agarose gel</p>

<ol>
<li>Add 2 g of electrophoresis-grade agarose to 100 ml of 1X TAE or 0.5X TBE buffer in a 250 ml flask or bottle.</li>

<li>Melt the agarose in a microwave until the agarose is fully melted and the solution is clear. Swirl the flask a few times while microwaving to avoid boiling and spilling over.</li>

<li>Cool to 55-60°C and then add 5 µl EtBr for a final concentration of 0.5 µg/ml.

<ul>
<li>EtBr is a powerful carcinogen and must be handled with care.</li>
</ul>
</li>
</ol>

<p class="bold">Primer working stock solution</p>

<p>The primer working stock solution should be 20 µM.</p>
</li>

<li><span class="bold">Performing multiplex PCR for <em>S. pneumoniae</em> serotype deduction</span>
<p class="bold">PCR protocol</p>

<p>Prior to beginning the PCR, plan the experiment by filling out and printing a plate template worksheet. Also, be sure sufficient quantities of primer working solutions to be used are available.</p>

<ol>
<li>Remove DNA templates and positive control DNAs from -20°C to the DNA addition area to allow them to thaw completely.</li>

<li>In the PCR reaction assembly area, gather reagents needed for the PCR reactions, including PCR master mix, primers, and PCR grade water. If the reagents are stored at -20°C, allow them to thaw completely and vortex or flick each tube before use.</li>

<li>Sequential multiplex PCR reactions, based on one of the described schemes, are prepared in standard 25 µl reaction volumes using primers and concentrations described in <a href="chpt10-pcr.html#table5">Table 5</a>.</li>

<li>Each PCR reaction should contain:

<ul>
<li>200 µM (each) of deoxynucleoside triphosphates (dNTPs)</li>

<li>3.5 mM MgCl2</li>

<li>2 Units Taq DNA polymerase</li>

<li>Forward primer (<a href="chpt10-pcr.html#table5">Table 5</a>)</li>

<li>Reverse primer (<a href="chpt10-pcr.html#table5">Table 5</a>)</li>

<li>2.5 µl of DNA template from isolates (use 5 µl for clinical specimens)</li>

<li>PCR-grade water to 25 µl</li>
</ul>
</li>

<li>A master mix can be prepared, which includes all components listed above except DNA template. When calculating volumes of master mix reagents, remember to add enough master mix reagents for 2 extra reactions than the number of specimens there are to be tested to ensure there will be enough master mix.</li>

<li>Pipette 22.5 µl of this master mix into each appropriate well of 96-well plate, according to your plate template worksheet.</li>

<li>Cover the wells of the plate using cap strips. Spray down the clean workspace with 10% bleach (10:1 water: concentrated bleach), and wipe. Repeat with 70% alcohol. Remove laboratory coat and gloves. Put on a fresh pair of gloves. Carefully transport the 96-well plate to the DNA addition area.</li>

<li>Put on new laboratory coat and keep the same pair of gloves on. Remove the cap strips from the plate. Add 2.5 µl of template DNA to each appropriate well of 96-well plate, according to your plate template worksheet.</li>

<li>At least one negative and one positive control should be set up for each serotype per PCR run.

<ul>
<li>Negative control: add 2.5 µl DNA resolving buffer to a reaction well instead of DNA template.</li>

<li>Positive control: add 2.5 µl of DNA template that is known to contain the amplified sequence to a reaction well.</li>
</ul>
</li>

<li>Cap columns of wells as you go. Use the roller tool to secure caps tightly.</li>

<li>Wipe down the dirty workspace with 10% bleach, then 70% ethanol. Remove laboratory coat and gloves. If possible, quickly spin the plate at 1000 rpm to bring down any droplets. Transport plate directly to and place it in the PCR thermocycler.</li>

<li>The following PCR conditions are used:

<ul>
<li>1 cycle of 94°C for 4 minutes</li>

<li>30 cycles of 94°C for 45 seconds; 54°C for 45 seconds and 72°C for 2 minutes</li>

<li>1 cycle of 72°C for 2 minutes</li>
</ul>
</li>

<li>Current methodologies are detailed at the <a href="https://www.cdc.gov/streplab/pcr.html">CDC website</a></li>
</ol>

<p class="bold">Analysis of PCR products on an agarose gel</p>

<p>PCR products (10 µl) are run on 2% agarose gels to determine band sizes using positive controls. A positive control for each serotype and a 50 bp ladder molecular size marker should be included on each gel.</p>

<ol>
<li>Melt the 2% agarose gel in a microwave oven. Cool the agar to approximately 55°C. Add ethidium bromide or other gel stain. Pour into a gel casting cassette, insert the comb, and allow time for hardening (~30 minutes).</li>

<li>Add 1X TAE or TBE buffer to the electrophoresis tank and properly place the gel cassette containing the solidified agarose gel into the tank.</li>

<li>Briefly spin the PCR plate or tubes at 500 x g to ensure all liquid is at the bottom.</li>

<li>Mix 10 µl of PCR reaction with 2 µl of 6X loading dye.</li>

<li>Pipette the DNA/loading dye mixtures into the wells. Load 5 µl of DNA size markers in one of the wells.</li>

<li>Run the gel at 50-100 volts for 15-20 minutes or until the Bromophenol blue dye band is halfway down the gel. The dye runs at approximately the same rate as a 500 base-pair DNA fragment.</li>

<li>Visualize the gel under a UV light and print out or save the image, if possible.</li>

<li>Each reaction should give two bands, i.e., species-specific positive control (cpsA, although some are cpsA negative) and a serotype-specific band.</li>

<li>Store the remainder of the amplicon at -20°C, if necessary.</li>
</ol>

<p class="bold">Interpretation</p>

<p>Band sizes on agarose gelsmust match those of positive controls before assigning a putative serotype (<a href="chpt10-pcr.html#figure5">Figure 5</a>). PCR reactions are setup sequentially and will include those serotypes most frequently determined for each of the schemes. For example, if a strain is negative for any of the serotypes included in reaction 1 then proceed to reaction 2 and so forth until a serotype is determined. If all reactions are completed for a specific scheme and no serotype bands were identified that matched any of the positive controls then the strain may be a nontypeable or one of the serotypes not yet included in the scheme and would need to be further typed using the Quellung reaction (see <a href="chpt08-id-characterization-streppneumo.html">Chapter 8</a>: Identification and Characterization of <em>Streptococcus pneumoniae</em>). Resolution of individual serotypes within some of the positive reactions could be applicable in some circumstances. For example, to further resolve a PCR positive reaction for 12F/12A/44/46 (reaction 3 in <a href="chpt10-pcr.html#table6">Table 6</a>), perform a Quellung reaction using type-specific antisera to determine whether the strain has a 12F, 12A, 44, or 46 capsular type. It must be mentioned that for practical reasons this is usually not necessary. For example, serotype 12F is commonly detected in disease and carriage specimens, while serotypes 12A, 44, and 46 are extremely rare.</p>

<p class="bold">Helpful tips</p>

<p>The positive pneumococcal control band for <em>cpsA</em> can be negative in 1-2% of PCR-serotypeable isolates. This is most often encountered in serotypes 25 and 38, but has also rarely been found for serotypes 14 and 35A. Although a <em>cpsA</em> negative result is relatively rare, at present a negative <em>cpsA</em> does not necessarily equate to a non-serotypeable isolate or a pneumococcus-negative clinical specimen.</p>

<p class="bold">Quality control</p>

<p>A positive control for each serotype included in the PCR reaction(s) should be run on each gel to ensure that bands are assigned to the correct serotype.</p>

<a id="table2" name="table2"> </a>
<p><span class="bold">Table 2.</span> Primers and probes used for detection of bacterial meningitis pathogens</p>

<table class="opt-in table"><thead class="primary-bg-color">
<tr>
<th id="t2h1">Target</th>
<th id="t2h2">Primer or Probe Name</th>
<th id="t2h3">Real-time Primers and Probes Nucleotide Sequence (5' to 3')</th>
<th id="t2h4">Working Stock Conc (µM)</th>
<th id="t2h5">Final Conc (nM)</th>
<th id="t2h6">Suggested Probe Modifications</th>
</tr>

</thead><tbody>
<tr>
<th class="secondary-bg-color" colspan="6" id="t2r2c1"><em>N. meningitidis</em></th>
</tr>

<tr>
<th headers="t2h1 t2r2c1" id="t2r3c1"><em>ctrA</em></th>
<td headers="t2h2 t2r2c1 t2r3c1">F753</td>
<td headers="t2h3 t2r2c1 t2r3c1">TGTGTTCCGCTATACGCCATT</td>
<td headers="t2h4 t2r2c1 t2r3c1">3.75</td>
<td headers="t2h5 t2r2c1 t2r3c1">300</td>
<td headers="t2h6 t2r2c1 t2r3c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r2c1 t2r3c1">R846</td>
<td headers="t2h3 t2r2c1 t2r3c1">GCCATATTCACACGATATACC</td>
<td headers="t2h4 t2r2c1 t2r3c1">11.25</td>
<td headers="t2h5 t2r2c1 t2r3c1">900</td>
<td headers="t2h6 t2r2c1 t2r3c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r2c1 t2r3c1">Pb820i</td>
<td headers="t2h3 t2r2c1 t2r3c1">AACCTTGAGCAA"T"CCATTTA<br />
TCCTGACGTTCT</td>
<td headers="t2h4 t2r2c1 t2r3c1">1.25</td>
<td headers="t2h5 t2r2c1 t2r3c1">100</td>
<td headers="t2h6 t2r2c1 t2r3c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t2h1 t2r2c1" id="t2r7c1"><em>sodC</em></th>
<td headers="t2h2 t2r2c1 t2r7c1">F351</td>
<td headers="t2h3 t2r2c1 t2r7c1">GCACACTTAGGTGATT<br />
TACCTGCAT</td>
<td headers="t2h4 t2r2c1 t2r7c1">3.75</td>
<td headers="t2h5 t2r2c1 t2r7c1">300</td>
<td headers="t2h6 t2r2c1 t2r7c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r2c1 t2r7c1">R478</td>
<td headers="t2h3 t2r2c1 t2r7c1">CCACCCGTGTGGATCATAATAGA</td>
<td headers="t2h4 t2r2c1 t2r7c1">7.5</td>
<td headers="t2h5 t2r2c1 t2r7c1">600</td>
<td headers="t2h6 t2r2c1 t2r7c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r2c1 t2r7c1">Pb387</td>
<td headers="t2h3 t2r2c1 t2r7c1">CATGATGGCACAGCAACAAA<br />
TCCTGTTT</td>
<td headers="t2h4 t2r2c1 t2r7c1">1.25</td>
<td headers="t2h5 t2r2c1 t2r7c1">100</td>
<td headers="t2h6 t2r2c1 t2r7c1">5' FAM, 3' BHQ1</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th class="secondary-bg-color" colspan="6" id="t2r11c1"><em>H. influenzae</em></th>
</tr>

<tr>
<th headers="t2h1 t2r11c1" id="t2r12c1"><em>hpd</em></th>
<td headers="t2h2 t2r11c1 t2r12c1">hpdF822</td>
<td headers="t2h3 t2r11c1 t2r12c1">GGTTAAATATGCCGATGGTGTTG</td>
<td headers="t2h4 t2r11c1 t2r12c1">1.25</td>
<td headers="t2h5 t2r11c1 t2r12c1">100</td>
<td headers="t2h6 t2r11c1 t2r12c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r11c1 t2r12c1">hpdR952</td>
<td headers="t2h3 t2r11c1 t2r12c1">TGCATCTTTACGCACGGTGTA</td>
<td headers="t2h4 t2r11c1 t2r12c1">3.75</td>
<td headers="t2h5 t2r11c1 t2r12c1">300</td>
<td headers="t2h6 t2r11c1 t2r12c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r11c1 t2r12c1">Pb896i</td>
<td headers="t2h3 t2r11c1 t2r12c1">TTGTGTACACTCCGT"T"GGT<br />
AAAAGAACTTGCAC</td>
<td headers="t2h4 t2r11c1 t2r12c1">1.25</td>
<td headers="t2h5 t2r11c1 t2r12c1">100</td>
<td headers="t2h6 t2r11c1 t2r12c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th class="secondary-bg-color" colspan="6" id="t2r16c1"><em>S. pneumoniae</em></th>
</tr>

<tr>
<th headers="t2h1 t2r16c1" id="t2r17c1"><em>lytA</em></th>
<td headers="t2h2 t2r16c1 t2r17c1">F373</td>
<td headers="t2h3 t2r16c1 t2r17c1">ACGCAATCTAGCAGATGAAGCA</td>
<td headers="t2h4 t2r16c1 t2r17c1">2.5</td>
<td headers="t2h5 t2r16c1 t2r17c1">200</td>
<td headers="t2h6 t2r16c1 t2r17c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r16c1 t2r17c1">R424</td>
<td headers="t2h3 t2r16c1 t2r17c1">TCGTGCGTTTTAATTCCAGCT</td>
<td headers="t2h4 t2r16c1 t2r17c1">2.5</td>
<td headers="t2h5 t2r16c1 t2r17c1">200</td>
<td headers="t2h6 t2r16c1 t2r17c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r16c1 t2r17c1">Pb400i</td>
<td headers="t2h3 t2r16c1 t2r17c1">TGCCGAAAACGC”T”TGATAC<br />
AGGGAG</td>
<td headers="t2h4 t2r16c1 t2r17c1">2.5</td>
<td headers="t2h5 t2r16c1 t2r17c1">200</td>
<td headers="t2h6 t2r16c1 t2r17c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th class="secondary-bg-color" colspan="6" id="t2r21c1">Human DNA</th>
</tr>

<tr>
<th headers="t2h1 t2r21c1" id="t2r22c1">RNAaseP</th>
<td headers="t2h2 t2r21c1 t2r22c1">RNAsePF</td>
<td headers="t2h3 t2r21c1 t2r22c1">CCA AGT GTG AGG GCT GAA AAG</td>
<td headers="t2h4 t2r21c1 t2r22c1">4</td>
<td headers="t2h5 t2r21c1 t2r22c1">400</td>
<td headers="t2h6 t2r21c1 t2r22c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r21c1 t2r22c1">RNAsePR</td>
<td headers="t2h3 t2r21c1 t2r22c1">TGT TGT GGC TGA TGA<br />
ACT ATA AAA GG</td>
<td headers="t2h4 t2r21c1 t2r22c1">4</td>
<td headers="t2h5 t2r21c1 t2r22c1">400</td>
<td headers="t2h6 t2r21c1 t2r22c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t2h2 t2r21c1 t2r22c1">RNAsePPb</td>
<td headers="t2h3 t2r21c1 t2r22c1">CCC CAG TCT CTG TCA GCA<br />
CTC CCT TC</td>
<td headers="t2h4 t2r21c1 t2r22c1">1</td>
<td headers="t2h5 t2r21c1 t2r22c1">100</td>
<td headers="t2h6 t2r21c1 t2r22c1">5' FAM, 3' BHQ1</td>
</tr>

</tbody></table>

<a id="table3" name="table3"> </a>
<p><span class="bold">Table 3.</span> Primers and probes used for detection of the serogroups of <em>N. meningitidis</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th id="t3h1">Target</th>
<th id="t3h2">Primer or Probe Name</th>
<th id="t3h3">Real-time Primers and Probes Nucleotide Sequence (5' to 3')</th>
<th id="t3h4">Working Stock Conc (µM)</th>
<th id="t3h5">Final Conc (nM)</th>
<th id="t3h6">Suggested Probe Modifications</th>
</tr>

</thead><tbody>
<tr>
<th headers="t3h1" id="t3r2c1">Nm A</th>
<td headers="t3h2 t3r2c1">F2531</td>
<td headers="t3h3 t3r2c1">AAAATTCAATGGGTATATCACGAAGA</td>
<td headers="t3h4 t3r2c1">3.75</td>
<td headers="t3h5 t3r2c1">300</td>
<td headers="t3h6 t3r2c1">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r3c1"><em>sacB</em></th>
<td headers="t3h2 t3r3c1">R2624</td>
<td headers="t3h3 t3r3c1">ATATGGTGCAAGCTGGTTTCAATAG</td>
<td headers="t3h4 t3r3c1">11.25</td>
<td headers="t3h5 t3r3c1">900</td>
<td headers="t3h6 t3r3c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t3h2 t3r3c1">Pb2591i</td>
<td headers="t3h3 t3r3c1">CTAAAAG"T"AGGAAGGGCACT<br />
TTGTGGCATAAT</td>
<td headers="t3h4 t3r3c1">1.25</td>
<td headers="t3h5 t3r3c1">100</td>
<td headers="t3h6 t3r3c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r6c1">Nm B</th>
<td headers="t3h2 t3r6c1">F737</td>
<td headers="t3h3 t3r6c1">GCTACCCCATTTCAGATGATTTGT</td>
<td headers="t3h4 t3r6c1">3.75</td>
<td headers="t3h5 t3r6c1">300</td>
<td headers="t3h6 t3r6c1">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r7c1"><em>synD</em></th>
<td headers="t3h2 t3r7c1">R882</td>
<td headers="t3h3 t3r7c1">ACCAGCCGAGGGTTTATTTCTAC</td>
<td headers="t3h4 t3r7c1">3.75</td>
<td headers="t3h5 t3r7c1">300</td>
<td headers="t3h6 t3r7c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t3h2 t3r7c1">Pb839i</td>
<td headers="t3h3 t3r7c1">AAGAGATGGGYAACAAC"T"ATGT<br />
AATGTCTTTATTT</td>
<td headers="t3h4 t3r7c1">1.25</td>
<td headers="t3h5 t3r7c1">100</td>
<td headers="t3h6 t3r7c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r10c1">Nm C</th>
<td headers="t3h2 t3r10c1">F478</td>
<td headers="t3h3 t3r10c1">CCCTGAGTATGCGAAAAAAATT</td>
<td headers="t3h4 t3r10c1">11.25</td>
<td headers="t3h5 t3r10c1">900</td>
<td headers="t3h6 t3r10c1">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r11c1"><em>synE</em></th>
<td headers="t3h2 t3r11c1">R551</td>
<td headers="t3h3 t3r11c1">TGCTAATCCCGCCTGAATG</td>
<td headers="t3h4 t3r11c1">3.75</td>
<td headers="t3h5 t3r11c1">300</td>
<td headers="t3h6 t3r11c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t3h2 t3r11c1">Pb495i</td>
<td headers="t3h3 t3r11c1">TTTCAATGC"T"AATGAATACCAC<br />
CGTTTTTTTGC</td>
<td headers="t3h4 t3r11c1">1.25</td>
<td headers="t3h5 t3r11c1">100</td>
<td headers="t3h6 t3r11c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r14c1">Nm W135</th>
<td headers="t3h2 t3r14c1">F857</td>
<td headers="t3h3 t3r14c1">TATTTATGGAAGGCATGGTGTATG</td>
<td headers="t3h4 t3r14c1">1.25</td>
<td headers="t3h5 t3r14c1">100</td>
<td headers="t3h6 t3r14c1">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r15c1"><em>synG</em></th>
<td headers="t3h2 t3r15c1">R964</td>
<td headers="t3h3 t3r15c1">TTGCCATTCCAGAAATATCACC</td>
<td headers="t3h4 t3r15c1">11.25</td>
<td headers="t3h5 t3r15c1">900</td>
<td headers="t3h6 t3r15c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t3h2 t3r15c1">Pb907i</td>
<td headers="t3h3 t3r15c1">AAATATGGAGCGAATGATTACAGT<br />
AACTATAATGAA</td>
<td headers="t3h4 t3r15c1">2.5</td>
<td headers="t3h5 t3r15c1">200</td>
<td headers="t3h6 t3r15c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r18c1">Nm X</th>
<td headers="t3h2 t3r18c1">F173</td>
<td headers="t3h3 t3r18c1">TGTCCCCAACCGTTTATTGG</td>
<td headers="t3h4 t3r18c1">11.25</td>
<td headers="t3h5 t3r18c1">900</td>
<td headers="t3h6 t3r18c1">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r19c1"><em>xcbB</em></th>
<td headers="t3h2 t3r19c1">R237</td>
<td headers="t3h3 t3r19c1">TGCTGCTATCATAGCCGCC</td>
<td headers="t3h4 t3r19c1">11.25</td>
<td headers="t3h5 t3r19c1">900</td>
<td headers="t3h6 t3r19c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t3h2 t3r19c1">Pb196</td>
<td headers="t3h3 t3r19c1">TGTTTGCCCACATGAATGGCGG</td>
<td headers="t3h4 t3r19c1">1.25</td>
<td headers="t3h5 t3r19c1">100</td>
<td headers="t3h6 t3r19c1">5' FAM, 3' BHQ1</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r22c1">Nm Y</th>
<td headers="t3h2 t3r22c1">F787</td>
<td headers="t3h3 t3r22c1">TCCGAGCAGGAAATTTATGAGAATAC</td>
<td headers="t3h4 t3r22c1">11.25</td>
<td headers="t3h5 t3r22c1">900</td>
<td headers="t3h6 t3r22c1">&#160;</td>
</tr>

<tr>
<th headers="t3h1" id="t3r23c1"><em>synF</em></th>
<td headers="t3h2 t3r23c1">R929</td>
<td headers="t3h3 t3r23c1">TTGCTAAAATCATTCGCTCCATAT</td>
<td headers="t3h4 t3r23c1">7.5</td>
<td headers="t3h5 t3r23c1">600</td>
<td headers="t3h6 t3r23c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t3h2 t3r23c1">Pb1099i</td>
<td headers="t3h3 t3r23c1">TATGGTG"T"ACGATATCCCTATCC<br />
TTGCCTATAAT</td>
<td headers="t3h4 t3r23c1">1.25</td>
<td headers="t3h5 t3r23c1">100</td>
<td headers="t3h6 t3r23c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

</tbody></table>

<a id="table4" name="table4"> </a>
<p><span class="bold">Table 4.</span> Primers and probes used for detection of the serotypes of <em>H. influenzae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th id="t4h1">Target</th>
<th id="t4h2">Primer or Probe Name</th>
<th id="t4h3">Real-time Primers and Probes Nucleotide Sequence (5' to 3')</th>
<th id="t4h4">Working Stock Conc (µM)</th>
<th id="t4h5">Final Conc (nM)</th>
<th id="t4h6">Suggested Probe Modifications</th>
</tr>

</thead><tbody>
<tr>
<th headers="t4h1" id="t4r2c1">Hi a</th>
<td headers="t4h2 t4r2c1">F261</td>
<td headers="t4h3 t4r2c1">GGT CTG CGG TGT CCT GTG T</td>
<td headers="t4h4 t4r2c1">1.25</td>
<td headers="t4h5 t4r2c1">100</td>
<td headers="t4h6 t4r2c1">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r3c1"><em>acsB</em></th>
<td headers="t4h2 t4r3c1">R427</td>
<td headers="t4h3 t4r3c1">CCG GTC ATC TTT TAT GCT CCA A</td>
<td headers="t4h4 t4r3c1">3.75</td>
<td headers="t4h5 t4r3c1">300</td>
<td headers="t4h6 t4r3c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t4h2 t4r3c1">Pb375i</td>
<td headers="t4h3 t4r3c1">TAA TTT TCT TGC "T"CA ATA CCG CCT TCC CA</td>
<td headers="t4h4 t4r3c1">1.25</td>
<td headers="t4h5 t4r3c1">100</td>
<td headers="t4h6 t4r3c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r6c1">Hi b</th>
<td headers="t4h2 t4r6c1">F192</td>
<td headers="t4h3 t4r6c1">TGA TGC ATT GAA AGA AGG TGT AAT TT</td>
<td headers="t4h4 t4r6c1">3.75</td>
<td headers="t4h5 t4r6c1">300</td>
<td headers="t4h6 t4r6c1">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r7c1"><em>bcsB</em></th>
<td headers="t4h2 t4r7c1">R359</td>
<td headers="t4h3 t4r7c1">CCT GCG GTA ATA ACA TGA TCA TAA A</td>
<td headers="t4h4 t4r7c1">7.5</td>
<td headers="t4h5 t4r7c1">600</td>
<td headers="t4h6 t4r7c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t4h2 t4r7c1">Pb244i</td>
<td headers="t4h3 t4r7c1">TGT CGT GCA G”T”A GCA AAC CGT AAC CTT ACT C</td>
<td headers="t4h4 t4r7c1">1.25</td>
<td headers="t4h5 t4r7c1">100</td>
<td headers="t4h6 t4r7c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r10c1">Hi c</th>
<td headers="t4h2 t4r10c1">F7667</td>
<td headers="t4h3 t4r10c1">CAT TGG TGA TGG TTC AGT TAT TGG</td>
<td headers="t4h4 t4r10c1">7.5</td>
<td headers="t4h5 t4r10c1">600</td>
<td headers="t4h6 t4r10c1">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r11c1"><em>ccsD</em></th>
<td headers="t4h2 t4r11c1">R7784</td>
<td headers="t4h3 t4r11c1">TAC AGC ATT CAG CAA TAA TGG G</td>
<td headers="t4h4 t4r11c1">3.75</td>
<td headers="t4h5 t4r11c1">300</td>
<td headers="t4h6 t4r11c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t4h2 t4r11c1">Pb7726i</td>
<td headers="t4h3 t4r11c1">ATT GCA “T”CG CCG CAG GAG TTC CCG</td>
<td headers="t4h4 t4r11c1">2.5</td>
<td headers="t4h5 t4r11c1">200</td>
<td headers="t4h6 t4r11c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r14c1">Hi d</th>
<td headers="t4h2 t4r14c1">F2211</td>
<td headers="t4h3 t4r14c1">CCT AAA ATA CGG ACC TAG TGC AC</td>
<td headers="t4h4 t4r14c1">7.5</td>
<td headers="t4h5 t4r14c1">600</td>
<td headers="t4h6 t4r14c1">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r15c1"><em>dcsE</em></th>
<td headers="t4h2 t4r15c1">R2255</td>
<td headers="t4h3 t4r15c1">CCG ATG AGA CCA AGT ATG GTT A</td>
<td headers="t4h4 t4r15c1">1.25</td>
<td headers="t4h5 t4r15c1">100</td>
<td headers="t4h6 t4r15c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t4h2 t4r15c1">Pb2221i</td>
<td headers="t4h3 t4r15c1">AAC GAG C”T”A GAG CTG GTG CTG AA</td>
<td headers="t4h4 t4r15c1">3.75</td>
<td headers="t4h5 t4r15c1">300</td>
<td headers="t4h6 t4r15c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r18c1">Hi e</th>
<td headers="t4h2 t4r18c1">F1523</td>
<td headers="t4h3 t4r18c1">ACT AAA ATA TGG CCC AAA CCC AC</td>
<td headers="t4h4 t4r18c1">7.5</td>
<td headers="t4h5 t4r18c1">600</td>
<td headers="t4h6 t4r18c1">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r19c1"><em>ecsH</em></th>
<td headers="t4h2 t4r19c1">R1589</td>
<td headers="t4h3 t4r19c1">CCG ATG AGC CCA AGT ATG ATG A</td>
<td headers="t4h4 t4r19c1">7.5</td>
<td headers="t4h5 t4r19c1">600</td>
<td headers="t4h6 t4r19c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t4h2 t4r19c1">Pb1555i</td>
<td headers="t4h3 t4r19c1">AAC GAG CAA AAG CCG G"T"G CGG AT</td>
<td headers="t4h4 t4r19c1">2.5</td>
<td headers="t4h5 t4r19c1">200</td>
<td headers="t4h6 t4r19c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

<tr>
<td colspan="6">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r22c1">Hi f</th>
<td headers="t4h2 t4r22c1">F7164</td>
<td headers="t4h3 t4r22c1">CCC TGA AAA GCG TTG ACT TTG</td>
<td headers="t4h4 t4r22c1">7.5</td>
<td headers="t4h5 t4r22c1">600</td>
<td headers="t4h6 t4r22c1">&#160;</td>
</tr>

<tr>
<th headers="t4h1" id="t4r23c1"><em>bexD</em></th>
<td headers="t4h2 t4r23c1">R7313</td>
<td headers="t4h3 t4r23c1">CCA ACT TCA GGA CCA AGT CAT TC</td>
<td headers="t4h4 t4r23c1">3.75</td>
<td headers="t4h5 t4r23c1">300</td>
<td headers="t4h6 t4r23c1">&#160;</td>
</tr>

<tr>
<th>&#160;</th>
<td headers="t4h2 t4r23c1">Pb7242i</td>
<td headers="t4h3 t4r23c1">TGC TGC TAA C"T"C AGA TGC ATC AGC TCC TT</td>
<td headers="t4h4 t4r23c1">2.5</td>
<td headers="t4h5 t4r23c1">200</td>
<td headers="t4h6 t4r23c1">5' FAM, BHQ1 on "T", 3' SpC6</td>
</tr>

</tbody></table>

<a id="table5" name="table5"> </a>
<p><span class="bold">Table 5.</span> <a href="https://www.cdc.gov/streplab/pcr.html">List of primers used for pneumococcal serotype deduction</a>. Please note that this website should be used as the primary source of primer sequences and protocols due to periodic introduced improvements.</p>

<p>*All serotypes that are co-detected are listed</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Target</th>
<th scope="col">Primer or Probe Name</th>
<th scope="col">Real-time Primers and Probes Nucleotide Sequence (5' to 3')</th>
<th scope="col">Working Stock Conc (µM)</th>
<th scope="col">Final Conc (nM)</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">1-f</th>
<td>CTC TAT AGA ATG GAG TAT ATA AAC TAT GGT TA</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>280</td>
</tr>

<tr>
<th scope="row">1-r</th>
<td>CCA AAG AAA ATA CTA ACA TTA TCA CAA TAT TGG C</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">2-f</th>
<td>TAT CCC AGT TCA ATA TTT CTC CAC TAC ACC</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>290</td>
</tr>

<tr>
<th scope="row">2-r</th>
<td>ACA CAA AAT ATA GGC AGA GAG AGA CTA CT</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">3-f</th>
<td>ATG GTG TGA TTT CTC CTA GAT TGG AAA GTA G</td>
<td><em>gal</em>U</td>
<td>0.3</td>
<td>371</td>
</tr>

<tr>
<th scope="row">3-r</th>
<td>CTT CTC CAA TTG CTT ACC AAG TGC AAT AAC G</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">4-f <em>a</em></th>
<td>CTG TTA CTT GTT CTG GAC TCT CGA TAA TTG G</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>430</td>
</tr>

<tr>
<th scope="row">4-r</th>
<td>GCC CAC TCC TGT TAA AAT CCT ACC CGC ATT G</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5-f</th>
<td>ATA CCT ACA CAA CTT CTG ATT ATG CCT TTG TG</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>362</td>
</tr>

<tr>
<th scope="row">5-r</th>
<td>GCT CGA TAA ACA TAA TCA ATA TTT GAA AAA GTA TG</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">6A/6B/6C/6D -f</th>
<td>AAT TTG TAT TTT ATT CAT GCC TAT ATC TGG</td>
<td><em>wci</em>P</td>
<td>0.3</td>
<td>250</td>
</tr>

<tr>
<th scope="row">6A/6B/6C/6D -r</th>
<td>TTA GCG GAG ATA ATT TAA AAT GAT GAC TA</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">6C/6D -f</th>
<td>CAT TTT AGT GAA GTT GGC GGT GGA GTT</td>
<td><em>wci</em>Nbeta</td>
<td>0.5</td>
<td>727</td>
</tr>

<tr>
<th scope="row">6C/6D -r</th>
<td>AGC TTC GAA GCC CAT ACT CTT CAA TTA</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">7C/(7B/40)-f</th>
<td>CTA TCT CAG TCA TCT ATT GTT AAA GTT TAC GAC GGG A</td>
<td><em>wcw</em>L</td>
<td>0.3</td>
<td>260</td>
</tr>

<tr>
<th scope="row">7C/(7B/40)-r</th>
<td>GAA CAT AGA TGT TGA GAC ATC TTT TGT AAT TTC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">7F/7A-f</th>
<td>TCC AAA CTA TTA CAG TGG GAA TTA CGG</td>
<td><em>wzy</em></td>
<td>0.4</td>
<td>599</td>
</tr>

<tr>
<th scope="row">7F/7A-r</th>
<td>ATA GGA ATT GAG ATT GCC AAA GCG AC</td>
<td>&#160;</td>
<td>0.4</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">8-f</th>
<td>GAA GAA ACG AAA CTG TCA GAG CAT TTA CAT</td>
<td><em>wzy</em></td>
<td>0.2</td>
<td>201</td>
</tr>

<tr>
<th scope="row">8-r</th>
<td>CTA TAG ATA CTA GTA GAG CTG TTC TAG TCT</td>
<td>&#160;</td>
<td>0.2</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">9N/9L-f</th>
<td>GAA CTG AAT AAG TCA GAT TTA ATC AGC</td>
<td><em>wzx</em></td>
<td>0.5</td>
<td>516</td>
</tr>

<tr>
<th scope="row">9N/9L-r</th>
<td>ACC AAG ATC TGA CGG GCT AAT CAA T</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">9V/9A-f</th>
<td>GGG TTC AAA G TC AGA CAG TG A ATC TTA A</td>
<td><em>wzy</em></td>
<td>0.5</td>
<td>816</td>
</tr>

<tr>
<th scope="row">9V/9A-r</th>
<td>CCA TGA ATG A AA TCA ACA TT G TCA GTA GC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10A-f</th>
<td>GGT GTA GAT TTA CCA TTA GTG TCG GCA GAC</td>
<td><em>wcr</em>G</td>
<td>0.5</td>
<td>628</td>
</tr>

<tr>
<th scope="row">10A-r</th>
<td>GAA TTT CTT CTT TAA GAT TCG GAT ATT TCT C</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10F/(10C/33C)- f</th>
<td>GGA GTT TAT CGG TAG TGC TCA TTT TAG CA</td>
<td><em>wzx</em></td>
<td>0.3</td>
<td>248</td>
</tr>

<tr>
<th scope="row">10F/(10C/33C)-r</th>
<td>CTA ACA AAT TCG CAA CAC GAG GCA ACA</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">11A/11D-f</th>
<td>GGA CAT GTT CAG GTG ATT TCC CAA TAT AGT G</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>463</td>
</tr>

<tr>
<th scope="row">11A/11D-r</th>
<td>GAT TAT GAG TGT AAT TTA TTC CAA CTT CTC CC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">12F/(12A/44/46)-f</th>
<td>GCA ACA AAC GGC GTG AAA GTA GTT G</td>
<td><em>wzx</em></td>
<td>0.5</td>
<td>376</td>
</tr>

<tr>
<th scope="row">12F/(12A/44/46)-r</th>
<td>CAA GAT GAA TAT CAC TAC CAA TAA CAA AAC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">13-f</th>
<td>TAC TAA GGT AAT CTC TGG AAA TCG AAA GG</td>
<td><em>wzx</em></td>
<td>0.4</td>
<td>655</td>
</tr>

<tr>
<th scope="row">13-r</th>
<td>CTC ATG CAT TTT ATT AAC CG C TTT TTG TTC</td>
<td>&#160;</td>
<td>0.4</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">14-f</th>
<td>GAA ATG TTA CTT GGC GCA GGT GTC AGA ATT</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>189</td>
</tr>

<tr>
<th scope="row">14-r</th>
<td>GCC AAT ACT TCT TAG TCT CTC AGA TGA AT</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">15A/15F-f</th>
<td>ATT AGT ACA GCT GCT GGA ATA TCT CTT C</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>434</td>
</tr>

<tr>
<th scope="row">15A/15F-r</th>
<td>GAT CTA GTG AAC GTA CTA TTC CAA AC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">15B/15C-f</th>
<td>TTG GAA TTT TTT AAT TAG TGG CTT ACC TA</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>496</td>
</tr>

<tr>
<th scope="row">15B/15C-r</th>
<td>CAT CCG CTT ATT AAT TGA AGT AAT CTG AAC C</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">16F-f,</th>
<td>GAA TTT TTC AGG CGT GGG TGT TAA AAG</td>
<td><em>wzy</em></td>
<td>0.4</td>
<td>717</td>
</tr>

<tr>
<th scope="row">16F-r</th>
<td>CAG CAT ATA GCA CCG CTA AGC AAA TA</td>
<td>&#160;</td>
<td>0.4</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">17F-f</th>
<td>TTC GTG ATG ATA ATT CCA ATG ATC AAA CAA GAG</td>
<td><em>wci</em>P</td>
<td>0.5</td>
<td>693</td>
</tr>

<tr>
<th scope="row">17F-r</th>
<td>GAT GTA ACA AAT TTG TAG CGA CTA AGG TCT GC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">18/(18A/18B/18C/18F)-f</th>
<td>CTT AAT AGC TCT CAT TAT TCT TTT TTT AAG CC</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>573</td>
</tr>

<tr>
<th scope="row">18/(18A/18B/18C/18F)-r</th>
<td>TTA TCT GTA AAC CAT ATC AGC ATC TGA AAC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">19A-f</th>
<td>GAG AGA TTC ATA ATC TTG CAC TTA GCC A</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>566</td>
</tr>

<tr>
<th scope="row">19A-r</th>
<td>CAT AAT AGC TAC AAA TGA CTC ATC GCC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">19F-f</th>
<td>GTT AAG ATT GCT GAT CGA TTA ATT GAT ATC C</td>
<td><em>wzy</em></td>
<td>0.5</td>
<td>304</td>
</tr>

<tr>
<th scope="row">19F-r</th>
<td>GTA ATA TGT CTT TAG GGC GTT TAT GGC GAT AG</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">20-f</th>
<td>GAG CAA GAG TTT TTC ACC TGA CAG CGA GAA G</td>
<td><em>wci</em>L</td>
<td>0.3</td>
<td>514</td>
</tr>

<tr>
<th scope="row">20-r</th>
<td>CTA AAT TCC TGT AAT TTA GCT AAA ACT CTT ATC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">21-f</th>
<td>CTA TGG TTA TTT CAA CTC AAT CGT CAC C</td>
<td><em>wzx</em></td>
<td>0.2</td>
<td>192</td>
</tr>

<tr>
<th scope="row">21-r</th>
<td>GGC AAA CTC AGA CAT AGT ATA GCA TAG</td>
<td>&#160;</td>
<td>0.2</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">22F/22A-f</th>
<td>GAG TAT AGC CAG ATT ATG GCA GTT TTA TTG TC</td>
<td><em>wcw</em>V</td>
<td>0.5</td>
<td>643</td>
</tr>

<tr>
<th scope="row">22F/22A-r</th>
<td>CTC CAG CAC TTG CGC TGG AAA CAA CAG ACA AC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">23A-f</th>
<td>TAT TCT AGC AAG TGA CGA AGA TGC G</td>
<td><em>wzy</em></td>
<td>0.5</td>
<td>722</td>
</tr>

<tr>
<th scope="row">23A-r</th>
<td>CCA ACA TGC TTA AAA ACG CTG CTT TAC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">23B-f</th>
<td>CCA CAA TTA G CG CTA TAT TCA TTC AAT CG</td>
<td><em>wzx</em></td>
<td>0.2</td>
<td>199</td>
</tr>

<tr>
<th scope="row">23B-r</th>
<td>GTC CAC GCT GAA TAA AAT GAA GCT CCG</td>
<td>&#160;</td>
<td>0.2</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">23F-f <em>a</em></th>
<td>GTA ACA GTT GCT GTA GAG GGA ATT GGC TTT TC</td>
<td><em>wzy</em></td>
<td>0.5</td>
<td>384</td>
</tr>

<tr>
<th scope="row">23F-r</th>
<td>CAC AAC ACC TAA CAC TCG ATG GCT ATA TGA TTC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">24A/24B/24F-f</th>
<td>GCT CCC TGC TAT TGT AAT CTT TAA AGA G</td>
<td><em>wzy</em></td>
<td>0.2</td>
<td>99</td>
</tr>

<tr>
<th scope="row">24A/24B/24F-r</th>
<td>GTG TCT TTT ATT GAC TTT ATC ATA GGT CGG</td>
<td>&#160;</td>
<td>0.2</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">31-f</th>
<td>GGA AGT TTT CAA GGA TAT GAT AGT GGT GGT GC</td>
<td><em>wzy</em></td>
<td>0.5</td>
<td>701</td>
</tr>

<tr>
<th scope="row">31-r</th>
<td>CCG AAT AAT ATA TTC AAT ATA TTC CTA CTC</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">33F/33A/37-f</th>
<td>GAA GGC AAT CAA TGT GAT TGT GTC GCG</td>
<td><em>wzy</em></td>
<td>0.3</td>
<td>338</td>
</tr>

<tr>
<th scope="row">33F/33A/37-r</th>
<td>CTT CAA AAT GAA GAT TAT AGT ACC CTT CTA C</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">34-f</th>
<td>GCT TTT GTA AGA GGA GAT TAT TTT CAC CCA AC</td>
<td>wzy</td>
<td>0.3</td>
<td>408</td>
</tr>

<tr>
<th scope="row">34-r</th>
<td>CAA TCC GAC TAA GTC TTC AGT AAA AAA CTT TAC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">35A/35C/42-f</th>
<td>ATT ACG ACT CCT TAT GTG ACG CGC ATA</td>
<td><em>wzx</em></td>
<td>0.3</td>
<td>280</td>
</tr>

<tr>
<th scope="row">35A/35C/42-r</th>
<td>CCA ATC CCA AGA TAT ATG CAA CTA GGT T</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">35B-f</th>
<td>GAT AAG TCT GTT GTG GAG ACT TAA AAA GAA TG</td>
<td><em>wcr</em>H</td>
<td>0.5</td>
<td>677</td>
</tr>

<tr>
<th scope="row">35B-r</th>
<td>CTT TCC AGA TAA TTA CAG GTA TTC CTG AAG CAA G</td>
<td>&#160;</td>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">35F/47F-f</th>
<td>GAA CAT AGT CGC TAT TGT ATT TTA TTT AAA GCA A</td>
<td>wzy</td>
<td>0.3</td>
<td>517</td>
</tr>

<tr>
<th scope="row">35F/47F-r</th>
<td>GAC TAG GAG CAT TAT TCC TAG AGC GAG TAA ACC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">38/25F-f</th>
<td>CGT TCT TTT ATC TCA CTG TAT AGT ATC TTT ATG</td>
<td>wzy</td>
<td>0.3</td>
<td>574</td>
</tr>

<tr>
<th scope="row">38/25F-r</th>
<td>ATG TTT GAA TTA AAG CTA ACG TAA CAA TCC</td>
<td>&#160;</td>
<td>0.3</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">39-f</th>
<td>TCA TTG TAT TAA CCC TAT GCT TTA TTG GTG</td>
<td>wzy</td>
<td>0.2</td>
<td>98</td>
</tr>

<tr>
<th scope="row">39-r</th>
<td>GAG TAT CTC CAT TGT ATT GAA ATC TAC CAA</td>
<td>&#160;</td>
<td>0.2</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row"><em>cps</em>A-f</th>
<td>GCA GTA CAG CAG TTT GTT GGA CTG ACC</td>
<td><em>wzg</em></td>
<td>0.1</td>
<td>160</td>
</tr>

<tr>
<th scope="row"><em>cps</em>A-r</th>
<td>GAA TAT TTT CAT TAT CAG TCC CAG TC</td>
<td>&#160;</td>
<td>0.1</td>
<td>&#160;</td>
</tr>

</tbody></table>

<a id="table6" name="table6"> </a>
<p><span class="bold">Table 6.</span> PCR-serotyping scheme for USA based on current serotype prevalence</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reaction</th>
<th scope="col">Serotypes included</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">1</th>
<td>6A/6B/6C/6D, 3, 19A, 22F/22A, 16F</td>
</tr>

<tr>
<th scope="row">2</th>
<td>8, 33F/33A/37, 15A/15F, 7F/7A, 23A</td>
</tr>

<tr>
<th scope="row">3</th>
<td>19F, 12F/12A/44/46, 11A/11D, 38, 35B</td>
</tr>

<tr>
<th scope="row">4</th>
<td>24A/24B/24F, 7C/7B/40, 4, 18A/18B/18C/18F, 9V/9A</td>
</tr>

<tr>
<th scope="row">5</th>
<td>14, 1, 23F, 15B/15C, 10A</td>
</tr>

<tr>
<th scope="row">6</th>
<td>39, 10F/10C/33C, 5, 35F/47F, 17F</td>
</tr>

<tr>
<th scope="row">7</th>
<td>23B, 35A/35C/42, 34, 9N/9L, 31</td>
</tr>

<tr>
<th scope="row">8*</th>
<td>6A/6B/6C/6D, 6C/6D</td>
</tr>

</tbody></table>

<p>*Only performed if PCR-positive for 6A/6B/6C/6D in reaction 1</p>

<a id="table7" name="table7"> </a>
<p><span class="bold">Table 7.</span> PCR-serotyping scheme for Africa based on current serotype prevalence</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reaction</th>
<th scope="col">Serotypes included</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">1</th>
<td>14, 1, 5, 4, 18A/18B/18C/18F</td>
</tr>

<tr>
<th scope="row">2</th>
<td>6A/6B/6C/6D, 19F, 23F, 38/25F, 9V/9A</td>
</tr>

<tr>
<th scope="row">3</th>
<td>7C/7B/40, 3, 15B/15C, 7F/7A, 17F</td>
</tr>

<tr>
<th scope="row">4</th>
<td>8, 12F/12A/44/46, 9L/9N, 22F/22A, 23A</td>
</tr>

<tr>
<th scope="row">5</th>
<td>24A/24B/24F, 2, 11A/11D, 19A, 16F</td>
</tr>

<tr>
<th scope="row">6</th>
<td>21, 33F/33A/37, 15A/15F, 35F/47F, 13</td>
</tr>

<tr>
<th scope="row">7</th>
<td>39, 23B, 35A/35C/42, 20, 35B</td>
</tr>

<tr>
<th scope="row">8</th>
<td>10F/10C/33C, 34, 10A, 31</td>
</tr>

<tr>
<th scope="row">9*</th>
<td>6A/6B/6C/6D, 6C/6D</td>
</tr>

</tbody></table>

<p>*Only performed if PCR-positive for 6A/6B/6C/6D in reaction 2</p>

<a id="table8" name="table8"> </a>
<p><span class="bold">Table 8.</span> PCR-serotyping scheme for Latin America based on current serotype prevalence</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reaction</th>
<th scope="col">Serotypes included</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">1</th>
<td>14, 6A/6B/6C/6D, 23F, 19A, 9V/9A</td>
</tr>

<tr>
<th scope="row">2</th>
<td>19F, 3, 15B/15C, 18A/18B/18C/18F, 17F</td>
</tr>

<tr>
<th scope="row">3</th>
<td>1, 5, 9L/9N, 7F/7A, 16F</td>
</tr>

<tr>
<th scope="row">4</th>
<td>8, 2, 4, 20, 22F/22A</td>
</tr>

<tr>
<th scope="row">5</th>
<td>7C/7B/40, 12F/12A/44/46, 11A/11D, 10A, 23A</td>
</tr>

<tr>
<th scope="row">6</th>
<td>21, 33F/33A/37, 15A/15F, 35F/47F, 13</td>
</tr>

<tr>
<th scope="row">7</th>
<td>39, 23B, 35A/35C/42, 38/25F/25A, 35B</td>
</tr>

<tr>
<th scope="row">8</th>
<td>24A/24B/24F, 10F/10C/33C, 34, 31</td>
</tr>

<tr>
<th scope="row">9*</th>
<td>6A/6B/6C/6D, 6C/6D</td>
</tr>

</tbody></table>

<p>*Only performed if PCR-positive for 6A/6B/6C/6D in reaction 1</p>

<div class="span8 fixed"><a id="figure5" name="figure5"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 5 is a picture showing identification of hemin (X factor) and NAD (V factor) as growth requirements using paper disks. The top strain is only growing around the disk containing both hemin and NAD, and is presumptively identified as H. influenzae." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure05.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 5. PCR products for USA serotyping scheme detailed in <a href="chpt10-pcr.html#table6">Table 6</a>...<a href="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure05-view.gif">view larger</a></p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>
</ol>
</li>

<li><a id="sectionV" name="sectionV"> </a><span class="bold">Real-time PCR</span>
<ol type="A">
<li><span class="bold">Workstation for real-time PCR reaction set-up</span>
<p>The extremely sensitive lower limit of detection of real-time PCR assays increases the chance of detection of cross-contamination with other DNA. Extra precautions should be taken to minimize such cross-contamination. It is strongly recommended that reaction assembly be performed in one room, which is designated a clean room, while DNA extraction from clinical specimens or isolates and addition of template DNA to the reaction wells should be performed in a separate room, which is designated a dirty room. If separate rooms are not possible, separate laboratory benches should be used for these two steps. Working in an unventilated biocontainment cabinet (sometimes called a PCR hood), is also suggested to minimize cross-contamination. Use of a biological safety cabinet is necessary for infectious materials but does not provide significant protection from amplicon cross-contamination. Separate micropipettors, laboratory coats, and gloves should be used for DNA extraction and reaction assembly. Signage indicating which spaces and equipment are DNA-free and which are used to prepare or manipulate DNA would be helpful. Decontamination of surfaces and equipment with 10% bleach followed by 70% ethanol should be done after any manipulation of nucleic acids at the laboratory bench. Always use filter-barrier pipette tips and change and discard gloves frequently. Even though real-time PCR is more expensive, it may be advisable to use given that it is a closed system and has much less potential for cross-contamination of the workspace.</p>
</li>

<li><span class="bold">Equipment for real-time PCR analysis</span>
<p>The main piece of equipment needed is the real-time PCR machine. There are more than 30 different real-time PCR thermocycler models that are manufactured by 14 companies (<a href="chpt10-pcr.html#ref13">13</a>). When choosing the machine to buy, be sure to consider the fluorescent filters that it will contain; the filters in the machine must be able to detect the wavelengths of light that will be emitted by the fluorophores conjugated to the probes. An accompanying desktop or laptop computer and appropriate software will be necessary to view and analyze the results that are generated by the machine. Given their replacement cost and sensitivity to damage by electrical surges, it is strongly recommended that this machine and computer be plugged directly into a battery backup with surge protection, to protect them from fluctuations in current, to protect your data, and to keep the reactions running in case power is lost.</p>
</li>

<li><span class="bold">Consumables and reagents</span>
<p class="bold">Consumables:</p>

<ul>
<li>10% bleach (10:1, water: concentrated bleach) (make fresh weekly) 70% ethanol</li>

<li>1.5 ml microcentrifuge tubes (sterile, DNase free, or PCR grade)</li>

<li>96 well polypropylene plates, tube strips or individual PCR tubes</li>

<li>1 set of micropipettors (1-10 µl, 2-20 µl, 20-200 µl, and 100-1000 µl)</li>

<li>Pre-sterilized filter tips (10 µl, 200 µl, and 1000 µl) Optical caps</li>
</ul>

<p class="bold">Optional:</p>

<ul>
<li>Optical adhesive film</li>

<li>Commercial DNA-removing surface decontaminant liquid</li>

<li>Cap installing tool</li>
</ul>

<p class="bold">Reagents:</p>

<ul>
<li>TE buffer (10 mM Tris-HCl, pH 8.0, 1 mM EDTA)</li>

<li>Tris buffer (10 mM Tris-HCl, pH 8.0)</li>

<li>Commercial PCR Master Mix (containing dNTPs, DNA polymerase, and reference dye)</li>

<li>Primers and dual-labeled hydrolysis probes (probes contain 5' fluorophore and 3' or internal quencher)</li>

<li>Positive control DNA diluted to about 5 µg/ml</li>

<li>PCR grade water</li>
</ul>
</li>

<li><span class="bold">Preparing primer and probe working stocks</span>
<ul>
<li>The protocols, primers, and probes described in this chapter are adaptations of those previously described (<a href="chpt10-pcr.html#ref8">8</a>, <a href="chpt10-pcr.html#ref35">35</a>, <a href="chpt10-pcr.html#ref60">60</a>).</li>
</ul>

<p>Table 2 lists the recommended nucleotide sequences, working concentrations, and suggested chemical modifications of the primers and probes for <em>N. meningitidis</em>, <em>H. influenzae</em>, and <em>S. pneumoniae</em> species detection targeting the <em>ctrA</em> or <em>sodC</em>, <em>hpd</em>, and <em>lytA</em> genes, respectively. Primers and probes for the detection of <em>N. meningitidis</em> serogroups (<a href="chpt10-pcr.html#table3">Table 3</a>) and <em>H. influenzae</em> serotypes (<a href="chpt10-pcr.html#table4">Table 4</a>) are also given. Additional <em>S. pneumoniae</em> serotype assays are currently in development. A two-tiered approach for detecting and characterizing bacterial meningitis pathogens is recommended with the first tier being species-specific detection of <em>N. meningitidis,</em> <em>H. influenzae,</em> or <em>S. pneumoniae</em> with subsequent serogroup/serotype determination of any positive specimens (<a href="chpt10-pcr.html#figure8">Figure 8</a>).</p>

<p>Primers and probes must be diluted from concentrated stocks into working stocks. It is convenient to dilute working stocks to concentrations that will allow 2 µl of each primer and probe to be added to the master mix per reaction. The optimized working stock concentration of each primer and probe for the assays described here that will allow 2 µl of each to be added to the master mix is presented in the tables above.</p>

<p>To calculate the required dilution from concentrated stock to working stocks, use the formula: (Concentration1)(Volume1)=(Concentration2)(Volume2). Example: A 500 µl working stock of the <em>sacB</em> forward primer is needed. For the purposes of this example, the <em>sacB</em> forward primer has a stock concentration of 220 µM. <a href="chpt10-pcr.html#table3">Table 3</a> shows that the working concentration of the <em>sacB</em> forward primer should be 3.75 µM; therefore the equation becomes:</p>

<blockquote>
<p>(220 µM is the given concentration of the concentrated stock)(x µl of concentrated stock) = (3.75 µM is the desired concentration of the working stock)(500 µl is the desired volume of working stock)</p>
</blockquote>

<p>Or: (220 µM)(x µl) / (3.75 µM)(500 µl)</p>

<p>Solve for x. x = (3.75 µM)(500 µl) / 220 µM = (1875 µM/µl) / 220 µM = 8.5 µl of concentrated stock.</p>

<p>Next, to calculate how much water to add to x µl of concentrated stock calculated above:<br />
500 µl total volume of working stock - x µl of concentrated stock = y µl of water to add</p>

<p>Therefore, using the example above: 500 µl - 8.5 µl = 491.5 µl of water.</p>

<p>Of this working stock of the primer, add 2 µl to the master mix per specimen to be tested.</p>

<p>When diluting primers and probes, it is important to use filter tips and to work in a clean space (i.e., free of template DNA) to avoid cross-contamination of these reagents. It is optimal to use a clean PCR cabinet or hood if one is available. Concentrated stocks of primers and probes should be stored at -20°C. If used on a regular basis, the working stocks can be stored at 4°C. Be sure to store probes in the dark, since they are light-sensitive (ideally, covered in aluminum foil and in a box). Concentrated stocks of probes are especially susceptible to degradation due to their fluorescent tags and should not be freeze-thawed more than 5 times. Primers should not be freeze-thawed more than 20 times (<a href="chpt10-pcr.html#ref46">46</a>). Therefore, it is recommended that the primer and probe stocks be aliquoted upon their arrival, before initially freezing them for longer-term storage.</p>

<p>The length of time for which primers and probes can be stored in this manner will vary. Performing positive controls with each reaction will help you to determine when your primers and probes need to be replaced.</p>
</li>

<li><span class="bold">Performing real-time PCR</span>
<p class="bold">General considerations</p>

<p>Before setting up the reactions, the real-time PCR machine should be turned on. For some machines, it can take about 20 minutes for the lamp to warm up.</p>

<p>The high sensitivity of real-time PCR significantly increases the risk for cross-contamination. Therefore, the following precautions are recommended:</p>

<ul>
<li>Separate rooms are suggested for handling DNA template (dirty room) and for handling other reagents (clean room) to avoid cross-contamination. Post signs to designate these areas.</li>

<li>Always use barrier pipette tips.</li>

<li>Always use clean laboratory coats. Use a different laboratory coat while in each of the separate rooms. Change gloves frequently, and be sure to wear fresh gloves when moving from the clean room to the dirty room.</li>

<li>Use separate pipette sets in each room.</li>
</ul>

<p>Assays are carried out in 25 µl reaction volumes, using a commercial PCR master mix according to the manufacturer's instructions. Per sample to be tested, each reaction mix contains: 2 µl of DNA sample, 2 µl of each primer, 2 µl of probe, 12.5 µl of master mix, and 4.5 µl of sterile, PCR-grade water. Add enough reagents for 1-2 extra reactions than the number of specimens there are to be tested to ensure there will be enough mix.</p>

<p>Positive and negative controls are extremely important to ensure the laboratorian that contamination of reagents and workspace has not generated false positives and that the assay is detecting targets as expected. Non-template controls (NTCs) contain all reagents except for template DNA; instead, 2 µl of sterile, PCR-grade water should be added to the NTC reaction wells. It is advisable to add water used in the clean room to at least two NTC reaction wells and to add water used in the dirty room to at least two NTC reaction wells. If either NTC reaction generates an amplification curve that crosses the threshold, the water from that room should be discarded and replaced, and NTCs should be performed again to determine if other reagents are the source of contamination. An additional control for contamination at the DNA extraction step is the extraction of water, as suggested in the extraction protocol. If the extracted water negative control(s) generate an amplification curve that crosses the threshold, then contamination occurred during the DNA extraction process, and replacement of extraction reagents with new ones and cleaning of workspace and pipettes is recommended. Positive control reactions should be performed using DNA from known positive isolates. Dilute the positive control DNA in order to decrease the likelihood of contamination of the PCR workspace, thereby avoiding false positives. In summary, for each target gene to be detected (i.e., for each mix that is made up), the following controls should be run:</p>

<ul>
<li>No-template negative controls (NTCs), at least in duplicate</li>

<li>Extracted-water negative controls for DNA prep equipment and reagent cross-contamination, in duplicate</li>

<li>Positive control, using prepared DNA from a known isolate and running RNAse P when using clinical specimens</li>
</ul>

<p>A sample spreadsheet depicting the set-up of a 96-well real-time PCR plate is pictured below (<a href="chpt10-pcr.html#figure6">Figure 6</a>):</p>

<div class="span14 fixed"><a id="figure6" name="figure6"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Example PCR template sheet for the N. meningitidis, H. influenzae, and S. pneumoniae species-specific real-time PCR assays. This example template should provide all the information needed for the laboratorian to set up the real-time PCR assay. The template should include: the assay(s) being run, date performed, PCR machine used, file name, cycle conditions, strain DNA to use as positive controls, and any notes to assist the laboratorian." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure06.gif" />
<div class="figcaption">
<p class="cdc-caption-text"><span class="bold">Figure 6.</span> Example PCR template sheet for the <em>N. meningitidis, H. influenzae</em>, and <em>S. pneumoniae</em> species-specific real-time PCR assays. This example template should provide all the information needed for the laboratorian to set up the real-time PCR assay. The template should include: the assay(s) being run, date performed, PCR machine used, file name, cycle conditions, strain DNA to use as positive controls, and any notes to assist the laboratorian. On the layout, each of the wells to be used on the 96-well plate should be clearly labeled with either the specimen name or type of control. In this case, three assays are being run on same plate and each assay is color-coded: <em>sodC</em> (or <em>ctrA</em>) for <em>N. meningitidis</em> detection in pale red; <em>hpd</em> for <em>H. influenzae</em> detection in green, and <em>lytA</em> for <em>S. pneumoniae</em> detection in blue. The non-template controls (ntc) for both the clean and dirty rooms are done in duplicate for each assay type (more can be added if the user deems it necessary). One positive control well should be used for each assay type and is labeled <em>sodC +, hpd +</em>, and <em>lytA +</em>. The water extraction to control for contamination during DNA extraction is labeled Extracted H<sub>2</sub>O. The 17 unknown specimens are labeled unk-1 to unk-17. The panel to the right depicts the amount of reagent needed for 1 reaction and also the amount needed to make a master mix of reagents for each assay being run. In this example, enough Master Mix (Mmix), which is the commercial mix of DNA polymerase, dNTPs, and buffer to be used, H<sub>2</sub>O, forward and reverse primers, and probe should be made for 25 reactions. The number 25 was calculated from the number of wells needed for each assay plus an additional 2 wells to ensure that the user has enough master mix...<a href="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure06-view.gif">view larger</a></p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>

<p class="bold">Real-time PCR protocol</p>

<ul>
<li>Prior to beginning, plan the experiment by filling out and printing a PCR template worksheet (see <a href="chpt10-pcr.html#figure6">Figure 6</a> for an example). Also, be sure sufficient quantities of working stocks of primers and probes to be used are available.</li>
</ul>

<ol>
<li>Turn on the real-time PCR machine and make sure lamp is warming up.</li>

<li>Remove DNA preps and positive control DNA from -20°C to the dirty room/hood to thaw.</li>

<li>In the clean room/hood, gather reagents: commercial PCR master mix, primers, probes, and PCR grade water. If they are used infrequently enough and are therefore stored at -20°C, allow working stocks of primers and probes to thaw completely before use.

<p>Vortex or flick each tube before using. Assemble one master mix per primer and probe set to be used. For each extraction to be tested, the master mix should contain:</p>

<p>12.5 µl of master mix<br />
4.5 µl of sterile, PCR-grade water<br />
2 µl of forward primer<br />
2 µl of reverse primer<br />
2 µl of probe<br />
<span class="bold">23 µl total before adding 2 µl DNA</span></p>

<p>When calculating volumes of master mix reagents, add enough master mix reagents for 2 extra reactions to ensure there will be enough mix.</p>
</li>

<li>Pipette 23 µl of this master mix into each appropriate well of a 96-well plate, according to the plate template worksheet. Add 2 µl of PCR-grade water to the clean NTC wells and then cap only that row. Ensure that the lid is flush with the plate to avoid evaporation from wells during PCR.

<ul>
<li>Incorrectly capped or uncapped wells, if run in the machine, will lead to reaction failures and could contaminate the machine. If this happens, wipe down the interior of the machine with 70% isopropanol.</li>
</ul>
</li>

<li>If available, cover plate with one-time use adhesive film. Wipe down the clean workspace with 10% bleach (10:1 water: concentrated bleach), then 70% ethanol and turn on UV light for 1 hour, if available. Remove laboratory coat and gloves. Put on a fresh pair of gloves. Carefully transport the plate to the dirty room/hood.</li>

<li>Place the plate in the dirty room/hood. Put on a new laboratory coat and keep the same pair of gloves on. If one was used, remove adhesive cover from plate.</li>

<li>According to your template worksheet, add 2 µl to the appropriate well of the following in this order:

<ul>
<li>Template DNA</li>

<li>Extracted water controls</li>

<li>Dirty NTCs</li>

<li>Positive control DNA</li>
</ul>
</li>

<li>Cap columns of wells as you go, capping NTC control wells last. Use the roller tool to secure caps tightly.</li>

<li>Wipe down the workspace with 10% bleach, then 70% ethanol and turn on the UV light for 1 hour, if available. Remove laboratory coat and gloves and discard the gloves.</li>

<li>If possible, spin the plate at 500 x g for a few seconds to bring down any droplets and to mix. Transport the plate directly to and place it in the real-time PCR machine.</li>

<li>Follow the instructions for machine operation that were provided by the manufacturer. Make sure that the machine is set to read the fluorescence of the reference dye contained in the commercial master mix, which is often ROX, in addition to the dye conjugated to each probe used (e.g., FAM, HEX, CY5). The cycle parameters suggested for the primers and probes given in the table are:

<ul>
<li>1 cycle of 50°C for 2 minutes</li>

<li>1 cycle of 95°C for 10 minutes</li>

<li>50 cycles of 95°C for 15 seconds + 60°C for 1 minute</li>
</ul>
</li>

<li>Turn off the machine lamp when the assay is complete.</li>
</ol>
</li>

<li><span class="bold">Data analysis</span>
<div style=" clear: both;"></div>

<div class="span14 fixed"><a id="figure7" name="figure7"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 7 is an image showing the amplification plot generated by a real-time polymerase chain reaction (PCR) assay. This is a graph showing a plot of amplification cycle numbers on the X axis versus fluorescence units (dRn) on the Y axis for each reaction. dRN is the baseline subtracted fluorescent reading normalized to the reference dye. The green line of the fluorescence threshold is highlighted by the black arrow and the exponential phase is highlighted by the red arrow." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure07.gif" />
<div class="figcaption">
<p class="cdc-caption-text"><span class="bold">Figure 7.</span> Amplification plot generated by a real-time PCR assay. This is a graph showing a plot of amplification cycle numbers on the X axis versus fluorescence units (dRn) on the Y axis for each reaction. dRn is the baseline subtracted fluorescent reading normalized to the reference dye. The green line of the fluorescence threshold is highlighted by the black arrow and the exponential phase is highlighted by the red arrow.</p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>

<p>The readout of the data generated by real-time PCR machines will come in two main formats: amplification plots (<a href="chpt10-pcr.html#figure7">Figure 7</a>) and plate sample values. The curves that are generated should be sigmoidal in shape, ideally plateauing as the last cycle is approached, indicating complete use of reactants. The cycle number at which the fluorescence curve for each sample crosses the fluorescence threshold (green line in the above graph, which is generated automatically by the data analysis software) is referred to as the cycle threshold value, or C<sub>t</sub> value. The plate sample values format of data readout is simply a listing of the C<sub>t</sub> value generated by each reaction. The fluorescence threshold should be set higher than the negative controls and negative specimens and should be within the start of the exponential phase (see <a href="chpt10-pcr.html#figure7">Figure 7</a>). In an optimal PCR reaction running at 100% efficiency, exact doubling of the PCR product occurs at every cycle during the exponential phase. The laboratorian should carefully examine each curve and document all C<sub>t</sub> values obtained. If possible, it is recommended that the electronic data files be saved for each run of the real-time PCR machine, including amplification plots for future reference.</p>

<p>As with any experiment, results for the unknowns cannot be evaluated without first assessing the results for the controls. NTCs and negative controls should give "No C<sub>t</sub>" and should produce amplification curves that are straight lines near zero. Positive controls should give C<sub>t</sub> values less than 35, and the amplifications plots of the positive controls and any unknown specimens that generate curves should all be sigmoidal. The commercial master mix used may contain a reference dye, which is often ROX. If the real-time PCR machine is working properly, the amplification plot and plate sample value for ROX in each reaction should resemble those of the NTCs and negative controls. If these criteria are not met, it will be difficult to interpret the results of the unknown samples. If this is the case, refer to the troubleshooting section below.</p>

<p>When determining whether to call an unknown specimen positive or negative for an etiology, the following cutoff values are used in the Meningitis Laboratory at CDC:</p>

<p class="margin-left-twenty">Positive = Ct ≤ 35<br />
Negative = Ct &#62; 40<br />
Equivocal = Ct 36-40</p>

<br />
<ul>
<li>C<sub>t</sub> values ≤ 35 are considered positive and C<sub>t</sub> values &#62; 40 are considered negative. Those with Ct values between 36 and 40 are considered equivocal and should be retested after diluting the template DNA 1:4 and 1:10 in PCR-grade water to reduce any inhibitors that may be interfering with the reaction. If the C<sub>t</sub> value decreases to ≤35, the specimen should be considered positive.</li>

<li>The amplification plots should be analyzed to ensure they are smooth and sigmoidal in shape. If a plot is lacking these characteristics, it should be considered negative or retested.</li>

<li>Other troubleshooting suggestions are described below.</li>

<li>Labs that are starting real-time PCR programs should try to pair with a reference lab to help with QA/QC.</li>
</ul>

<p class="bold">Commonly Used Terms</p>

<ul>
<li><span class="bold">Reporter dye/fluorophore:</span> Used to monitor PCR product accumulation.</li>

<li><span class="bold">Quencher dye:</span> Absorbs the light energy of the excited-state reporter dye.</li>

<li><span class="bold">Reference dye:</span> A dye, commonly ROX, which fluoresces at a constant level during the reaction. It is used to normalize the fluorescent signal of the reporter dye. Some systems do not use a reference dye.</li>

<li><span class="bold">C<sub>t</sub> or cycle threshold:</span> The PCR cycle number at which fluorescence measured by the instrument is at a statistically significant level above background. The C<sub>t</sub> is inversely proportional to the log of the initial copy number.</li>

<li><span class="bold">Exponential phase:</span> Phase at which exact doubling of the PCR product is accumulating at every cycle, assuming 100 % reaction efficiency. Exponential amplification occurs because all of the reagents are fresh and available and the kinetics of the reaction push the reaction to favor doubling of amplicon.</li>

<li><span class="bold">R:</span> Raw fluorescence reading in arbitrary units</li>

<li><span class="bold">Rn:</span> Fluorescent reading normalized to the reference dye</li>

<li><span class="bold">dRn, ΔRn:</span> Baseline subtracted fluorescent reading normalized to the reference dye</li>

<li><span class="bold">NTC (non-template control):</span> A sample type containing all the reaction components except the DNA template. Instead of template DNA, 2 µl of sterile, PCR-grade water is added to the NTC reaction wells. Water added in the clean room is called "clean NTC" and water added in the dirty room is called "dirty NTC".</li>

<li><span class="bold">Amplification plot:</span> Graph showing a plot of amplification cycle numbers on the X axis versus fluorescence units on the Y axis for each reaction</li>
</ul>
</li>
</ol>
</li>

<li><span class="bold">Workflow for detection of bacterial meningitis pathogens by PCR</span>
<div class="span14 fixed"><a id="figure8" name="figure8"> </a>
<div class="module-typeC">
<div class="figure"><img alt="Figure 8 is a workflow for detection and characterization of bacterial meningitis pathogens by polymerase chain reaction (PCR). Once DNA template is prepared from either a clinical specimen, an isolate, or from sampling a bottle of inoculated Tran-Isolate medium, the species-specific real-time PCR assays should be run. Any reactions positive for N. meningitis, H. influenzae or S. pneumoniae should be further characterized using the appropriate serogrouping or serotyping PCR assay." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt10-figure08.gif" />
<div class="figcaption">
<p class="cdc-caption-text"><span class="bold">Figure 8.</span> Workflow for detection and characterization of bacterial meningitis pathogens by PCR. Once DNA template is prepared from either a clinical specimen, an isolate, or from sampling a bottle of inoculated Tran-Isolate medium, the species-specific real-time PCR assays should be run. Any reactions positive for <em>N. meningitis, H. influenzae</em>, or <em>S. pneumoniae</em> should be further characterized using the appropriate serogrouping or serotyping PCR assay.<br />
<sup>1</sup> It is not always practical to test for all serogroups for which assays are available in a laboratory. Testing algorithms may be set up in laboratories with previous knowledge of the predominance or lack of serogroups within that particular geographic region to test for the most common serogroups first. Modifications may be made to the testing algorithm for any laboratory based on information about current strains that are circulating in the region. For example, in Africa, testing to detect serogroups A and W135 (and X in some regions) should be adequate to characterize most specimens. Specimens reacting negatively in the A and W135 assays should then be tested using the other available assays, particularly C, Y, X and B. Nearly all invasive specimens are serogroupable, if they are tested against a comprehensive panel of assays and proper controls are used.<br />
<br />
<sup>2</sup> It is not always practical to test for all serotypes for which assays are available in a laboratory. If a Hib vaccination program does not exist or is fairly new, testing all <em>hpd</em> positive specimens for Hib first may be adequate to characterize most specimens.<br />
<br />
<sup>3</sup> For bacterial isolates confirmed as <em>S. pneumoniae</em> by microbiological methods, conventional multiplex PCR serotyping is used. For clinical specimens that are <em>lyt</em>A-positive by real-time PCR and have Ct values ≤30, conventional multiplex PCR serotyping can be used. If Ct values are &#62;30, then a real-time PCR serotyping approach is recommended for increased sensitivity.</p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>

<li><span class="bold">Troubleshooting</span>
<ul>
<li>Are separate rooms, laboratory coats, gloves, and pipettes being used for master mix preparation versus template addition?</li>

<li>Are sterile barrier-filtered tips being used?</li>

<li>Are bleach and ethanol being used to clean work areas?</li>

<li>Reaction tubes should not be re-opened and should be immediately discarded.</li>

<li>Reaction tubes should be protected during transport from the clean room to the dirty room.</li>

<li>Consumable reagents (tips, plates, caps, adhesive films, etc.) should not be reused.</li>

<li>The amount of positive control DNA should be limited.</li>
</ul>

<p>If there are high or no C<sub>t</sub> counts in known positives, consider:</p>

<ul>
<li>Is specimen handling questioned? Was there sufficient specimen volume?

<ul>
<li>Clinical specimens should ideally be shipped on dry ice and frozen at -70°C.</li>

<li>The sample volume should ideally be at least 200 µl for optimal yields.</li>
</ul>
</li>

<li>Was the DNA extraction process performed correctly?

<ul>
<li>If mutanolysin and lysozyme are not used, DNA from gram-positive bacteria may not be efficiently extracted and thus will not be detectable.</li>

<li>Could any of the extraction reagents, supplies, or workspaces be contaminated?</li>
</ul>
</li>

<li>Are poor DNA integrity and/or inhibitors suspected?

<ul>
<li>Dilute the template DNA 1:4 and 1:10 and repeating the reaction. If C<sub>t</sub> values decrease compared to undiluted, this can be indicative of an inhibitor in the DNA preparation. The specimen should be considered positive if the C<sub>t</sub> count is less than 35 upon repeat.</li>

<li>Buffers containing phosphate such as phosphate buffered saline should be avoided as phosphate is a strong inhibitor of PCR.</li>

<li>If possible, consider sending the specimen or extraction to a reference laboratory.</li>

<li>Perform human RNAaseP assay to determine if extensive DNA degradation had occurred.</li>

<li>A dilution of the DNA preparation should also be assayed for RNAaseP to determine if inhibitors are present.</li>
</ul>
</li>

<li>Did a power failure or electrical current surge affect your results?

<ul>
<li>It is strongly recommended that the real-time PCR machine and associated computer be plugged directly into a surge protector with a battery back-up.</li>
</ul>
</li>
</ul>

<p>If problems persist, contact an experienced reference laboratory for assistance.<br />
Recommended for further reading: (<a href="chpt10-pcr.html#ref7">7</a>, <a href="chpt10-pcr.html#ref46">46</a>).</p>
</li>
</ol>

<h2>References</h2>

<ol style=" list-style-type: decimal;">
<li><a id="ref1" name="ref1"> </a>Bentley, S. D., G. S. Vernikos, L. A. Snyder, C. Churcher, C. Arrowsmith, T. Chillingworth, A. Cronin, P. H. Davis, N. E. Holroyd, K. Jagels, M. Maddison, S. Moule, E. Rabbinowitsch, S. Sharp, L. Unwin, S. Whitehead, M. A. Quail, M. Achtman, B. Barrell, N. J. Saunders, and J. Parkhill. 2007. Meningococcal genetic variation mechanisms viewed through comparative analysis of serogroup C strain FAM18. PLoS Genetics 3:e23.</li>

<li><a id="ref2" name="ref2"> </a>Boisier, P., P. Nicolas, S. Djibo, M. K. Taha, I. Jeanne, H. B. Mainassara, B. Tenebray, K. K. Kairo, D. Giorgini, and S. Chanteau. 2007. Meningococcal meningitis: unprecedented incidence of serogroup X-related cases in 2006 in Niger. Clinical Infectious Diseases 44:657-663.</li>

<li><a id="ref3" name="ref3"> </a>Borel, T., A. M. Rose, M. Guillerm, F. Sidikou, S. Gerstl, A. Djibo, N. Nathan, S. Chanteau, and P. J. Guerin. 2006. High sensitivity and specificity of the Pastorex latex agglutination test for Neisseria meningitidis serogroup A during a clinical trial in Niger. Transactions of the Royal Society of Tropical Medicine and Hygiene 100:964-969.</li>

<li><a id="ref4" name="ref4"> </a>Borrow, R., H. Claus, U. Chaudhry, M. Guiver, E. B. Kaczmarski, M. Frosch, and A. J. Fox. 1998. <em>siaD</em> PCR ELISA for confirmation and identification of serogroup Y and W135 meningococcal infections. FEMS Microbiology Letters 159:209-214.</li>

<li><a id="ref5" name="ref5"> </a>Borrow, R., H. Claus, M. Guiver, L. Smart, D. M. Jones, E. B. Kaczmarski, M. Frosch, and A. J. Fox. 1997. Non-culture diagnosis and serogroup determination of meningococcal B and C infection by a sialyltransferase (<em>siaD</em>) PCR ELISA. Epidemiology and Infection 118:111-117.</li>

<li><a id="ref6" name="ref6"> </a>Bundle, D. R., H. J. Jennings, and C. P. Kenney. 1974. Studies on the group-specific polysaccharide of <em>Neisseria meningitidis</em> serogroup X and an improved procedure for its isolation. Journal of Biological Chemistry 249:4797-4801.</li>

<li><a id="ref7" name="ref7"> </a>Bustin, S. A. 2004. A-Z of quantitative PCR. International University Line, La Jolla, CA.</li>

<li><a id="ref8" name="ref8"> </a>Carvalho, M. G., M. L. Tondella, K. McCaustland, L. Weidlich, L. McGee, L. W. Mayer, A. Steigerwalt, M. Whaley, R. R. Facklam, B. Fields, G. Carlone, E. W. Ades, R. Dagan, and J. S. Sampson. 2007. Evaluation and improvement of real-time PCR assays targeting <em>lytA</em>, <em>ply</em>, and <em>psaA</em> genes for detection of pneumococcal DNA. Journal of Clinical Microbiology 45:2460-2466.</li>

<li><a id="ref9" name="ref9"> </a>Claus, H., R. Borrow, M. Achtman, G. Morelli, C. Kantelberg, E. Longworth, M. Frosch, and U. Vogel. 2004. Genetics of capsule O-acetylation in serogroup C, W-135, and Y meningococci. Molecular Microbiology 51:227-239.</li>

<li><a id="ref10" name="ref10"> </a>Claus, H., M. C. Maiden, R. Maag, M. Frosch, and U. Vogel. 2002. Many carried meningococci lack the genes required for capsule synthesis and transport. Microbiology 148:1813-1819.</li>

<li><a id="ref11" name="ref11"> </a>Claus, H., U. Vogel, M. Muhlenhoff, R. Gerardy-Schahn, and M. Frosch. 1997. Molecular divergence of the sia locus in different serogroups of Neisseria meningitidis expressing polysialic acid capsules. Molecular and General Genetics 257:28-34.</li>

<li><a id="ref12" name="ref12"> </a>Corless, C. E., M. Guiver, R. Borrow, V. Edwards-Jones, A. J. Fox, and E. B. Kaczmarski. 2001. Simultaneous detection of Neisseria meningitidis, Haemophilus influenzae, and Streptococcus pneumoniae in suspected cases of meningitis and septicemia using real-time PCR. Journal of Clinical Microbiology 39:1553-1558.</li>

<li><a id="ref13" name="ref13"> </a>Csako, G. 2006. Present and future of rapid and/or high-throughput methods for nucleic acid testing. Clinica Chimica Acta 363:6-31.</li>

<li><a id="ref14" name="ref14"> </a>Dolan-Livengood, J. M., Y. K. Miller, L. E. Martin, R. Urwin, and D. S. Stephens. 2003. Genetic basis for nongroupable Neisseria meningitidis. Journal of Infectious Diseases 187:1616-1628.</li>

<li><a id="ref15" name="ref15"> </a>Dolan, J. M., X. Wang, J. Theodore, C. Hatcher, M. G. Carvalho, B. Harcourt, F. C. Pimenta, B. Beall, K. Edmond, J. Mendisaihan, D. Altantsetseg, P. Nymadawa, M. Bakir, D. Toprak, A. Soysal, N. Messonnier, and L. W. Mayer. 2008. Presented at the 16th International Pathogenic Neisseria Conference, Rotterdam, The Netherlands.</li>

<li><a id="ref16" name="ref16"> </a>Edwards, U., A. Muller, S. Hammerschmidt, R. Gerardy-Schahn, and M. Frosch. 1994. Molecular analysis of the biosynthesis pathway of the alpha-2,8 polysialic acid capsule by Neisseria meningitidis serogroup B. Molecular Microbiology 14:141-149.</li>

<li><a id="ref17" name="ref17"> </a>Falla, T. J., D. W. Crook, L. N. Brophy, D. Maskell, J. S. Kroll, and E. R. Moxon. 1994. PCR for capsular typing of Haemophilus influenzae. Journal of Clinical Microbiology 32:2382-2386.</li>

<li><a id="ref18" name="ref18"> </a>Frosch, M., U. Edwards, K. Bousset, B. Krausse, and C. Weisgerber. 1991. Evidence for a common molecular origin of the capsule gene loci in gram-negative bacteria expressing group II capsular polysaccharides. Molecular Microbiology 5:1251-1263.</li>

<li><a id="ref19" name="ref19"> </a>Frosch, M., and A. Muller. 1993. Phospholipid substitution of capsular polysaccharides and mechanisms of capsule formation in Neisseria meningitidis. Molecular Microbiology 8:483-493.</li>

<li><a id="ref20" name="ref20"> </a>Gagneux, S. P., A. Hodgson, T. A. Smith, T. Wirth, I. Ehrhard, G. Morelli, B. Genton, F. N. Binka, M. Achtman, and G. Pluschke. 2002. Prospective study of a serogroup X Neisseria meningitidis outbreak in northern Ghana. Journal of Infectious Diseases 185:618-626.</li>

<li><a id="ref21" name="ref21"> </a>Ganguli, S., G. Zapata, T. Wallis, C. Reid, G. Boulnois, W. F. Vann, and I. S. Roberts. 1994. Molecular cloning and analysis of genes for sialic acid synthesis in Neisseria meningitidis group B and purification of the meningococcal CMP-NeuNAc synthetase enzyme. Journal of Bacteriology 176:4583-4589.</li>

<li><a id="ref22" name="ref22"> </a>Hammerschmidt, S., C. Birkholz, U. Zahringer, B. D. Robertson, J. van Putten, O. Ebeling, and M. Frosch. 1994. Contribution of genes from the capsule gene complex (cps) to lipooligosaccharide biosynthesis and serum resistance in Neisseria meningitidis. Molecular Microbiology 11:885-896.</li>

<li><a id="ref23" name="ref23"> </a>Hammerschmidt, S., A. Muller, H. Sillmann, M. Muhlenhoff, R. Borrow, A. Fox, J. van Putten, W. D. Zollinger, R. Gerardy-Schahn, and M. Frosch. 1996. Capsule phase variation in Neisseria meningitidis serogroup B by slipped-strand mispairing in the polysialyltransferase gene (siaD): correlation with bacterial invasion and the outbreak of meningococcal disease. Molecular Microbiology 20:1211-1220.</li>

<li><a id="ref24" name="ref24"> </a>Janson, H., M. Ruan, and A. Forsgren. 1993. Limited diversity of the protein D gene (hpd) among encapsulated and nonencapsulated Haemophilus influenzae strains. Infection and Immunity 61:4546-4552.</li>

<li><a id="ref25" name="ref25"> </a>Kroll, J. S. 1992. The genetics of encapsulation in Haemophilus influenzae. Journal of Infectious Diseases 165 Suppl 1:S93-96.</li>

<li><a id="ref26" name="ref26"> </a>Kroll, J. S., B. M. Loynds, and E. R. Moxon. 1991. The Haemophilus influenzae capsulation gene cluster: a compound transposon. Molecular Microbiology 5:1549-1560.</li>

<li><a id="ref27" name="ref27"> </a>Kroll, J. S., and E. R. Moxon. 1988. Capsulation and gene copy number at the cap locus of Haemophilus influenzae type b. Journal of Bacteriology 170:859-864.</li>

<li><a id="ref28" name="ref28"> </a>Kroll, J. S., S. Zamze, B. Loynds, and E. R. Moxon. 1989. Common organization of chromosomal loci for production of different capsular polysaccharides in Haemophilus influenzae. Journal of Bacteriology 171:3343-3347.</li>

<li><a id="ref29" name="ref29"> </a>LaClaire, L. L., M. L. Tondella, D. S. Beall, C. A. Noble, P. L. Raghunathan, N. E. Rosenstein, and T. Popovic. 2003. Identification of Haemophilus influenzae serotypes by standard slide agglutination serotyping and PCR-based capsule typing. Journal of Clinical Microbiology 41:393-396.</li>

<li><a id="ref30" name="ref30"> </a>Lee, L. G., C. R. Connell, and W. Bloch. 1993. Allelic discrimination by nick-translation PCR with fluorogenic probes. Nucleic Acids Research 21:3761-3766.</li>

<li><a id="ref31" name="ref31"> </a>Liu, T. Y., E. C. Gotschlich, E. K. Jonssen, and J. R. Wysocki. 1971. Studies on the meningococcal polysaccharides. I. Composition and chemical properties of the group A polysaccharide. Journal of Biological Chemistry 246:2849-2858.</li>

<li><a id="ref32" name="ref32"> </a>Masson, L., and B. E. Holbein. 1983. Physiology of sialic acid capsular polysaccharide synthesis in serogroup B Neisseria meningitidis. Journal of Bacteriology 154:728-736.</li>

<li><a id="ref33" name="ref33"> </a>Messmer, T. O., J. S. Sampson, A. Stinson, B. Wong, G. M. Carlone, and R. R. Facklam. 2004. Comparison of four polymerase chain reaction assays for specificity in the identification of Streptococcus pneumoniae. Diagnostic Microbiology and Infectious Disease 49:249-254.</li>

<li><a id="ref34" name="ref34"> </a>Moore, C. E., A. Sengduangphachanh, T. Thaojaikong, J. Sirisouk, D. Foster, R. Phetsouvanh, L. McGee, D. W. Crook, P. N. Newton, and S. J. Peacock. 2010. Enhanced determination of Streptococcus pneumoniae serotypes associated with invasive disease in Laos by using a real-time polymerase chain reaction serotyping assay with cerebrospinal fluid. American Journal of Tropical Medicine and Hygiene 83:451-457.</li>

<li><a id="ref35" name="ref35"> </a>Mothershed, E. A., C. T. Sacchi, A. M. Whitney, G. A. Barnett, G. W. Ajello, S. Schmink, L. W. Mayer, M. Phelan, T. H. Taylor, Jr., S. A. Bernhardt, N. E. Rosenstein, and T. Popovic. 2004. Use of real-time PCR to resolve slide agglutination discrepancies in serogroup identification of Neisseria meningitidis. Journal of Clinical Microbiology 42:320-328.</li>

<li><a id="ref36" name="ref36"> </a>Mullis, K. B., and F. A. Faloona. 1987. Specific synthesis of DNA in vitro via a polymerase-catalyzed chain reaction. Methods In Enzymology 155:335-350.</li>

<li><a id="ref37" name="ref37"> </a>Murdoch, D. R., T. P. Anderson, K. A. Beynon, A. Chua, A. M. Fleming, R. T. Laing, G. I. Town, G. D. Mills, S. T. Chambers, and L. C. Jennings. 2003. Evaluation of a PCR assay for detection of Streptococcus pneumoniae in respiratory and nonrespiratory samples from adults with community-acquired pneumonia. Journal of Clinical Microbiology 41:63-66.</li>

<li><a id="ref38" name="ref38"> </a>Pai, R., R. E. Gertz, and B. Beall. 2006. Sequential multiplex PCR approach for determining capsular serotypes of Streptococcus pneumoniae isolates. Journal of Clinical Microbiology 44:124-131.</li>

<li><a id="ref39" name="ref39"> </a>Parkhill, J., M. Achtman, K. D. James, S. D. Bentley, C. Churcher, S. R. Klee, G. Morelli, D. Basham, D. Brown, T. Chillingworth, R. M. Davies, P. Davis, K. Devlin, T. Feltwell, N. Hamlin, S. Holroyd, K. Jagels, S. Leather, S. Moule, K. Mungall, M. A. Quail, M. A. Rajandream, K. M. Rutherford, M. Simmonds, J. Skelton, S. Whitehead, B. G. Spratt, and B. G. Barrell. 2000. Complete DNA sequence of a serogroup A strain of Neisseria meningitidis Z2491. Nature 404:502-506.</li>

<li><a id="ref40" name="ref40"> </a>Resti, M., M. Moriondo, M. Cortimiglia, G. Indolfi, C. Canessa, L. Becciolini, E. Bartolini, F. M. de Benedictis, M. de Martino, and C. Azzari. 2010. Community-acquired bacteremic pneumococcal pneumonia in children: diagnosis and serotyping by real-time polymerase chain reaction using blood samples. Clinical Infectious Diseases 51:1042-1049.</li>

<li><a id="ref41" name="ref41"> </a>Sadler, F., A. Fox, K. Neal, M. Dawson, K. Cartwright, and R. Borrow. 2003. Genetic analysis of capsular status of meningococcal carrier isolates. Epidemiology and Infection 130:59-70.</li>

<li><a id="ref42" name="ref42"> </a>Saiki, R. K., S. Scharf, F. Faloona, K. B. Mullis, G. T. Horn, H. A. Erlich, and N. Arnheim. 1985. Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia. Science 230:1350-1354.</li>

<li><a id="ref43" name="ref43"> </a>Satola, S. W., P. L. Schirmer, and M. M. Farley. 2003. Complete sequence of the cap locus of Haemophilus influenzae serotype b and nonencapsulated b capsule-negative variants. Infection and Immunity 71:3639-3644.</li>

<li><a id="ref44" name="ref44"> </a>Satola, S. W., P. L. Schirmer, and M. M. Farley. 2003. Genetic analysis of the capsule locus of Haemophilus influenzae serotype f. Infection and Immunity 71:7202-7207.</li>

<li><a id="ref45" name="ref45"> </a>Song, X. M., A. Forsgren, and H. Janson. 1995. The gene encoding protein D (hpd) is highly conserved among Haemophilus influenzae type b and nontypeable strains. Infection and Immunity 63:696-699.</li>

<li><a id="ref46" name="ref46"> </a>Stratagene. 2006. Introduction to Quantitative PCR: Methods and Application Guide.</li>

<li><a id="ref47" name="ref47"> </a>Suzuki, N., M. Yuyama, S. Maeda, H. Ogawa, K. Mashiko, and Y. Kiyoura. 2006. Genotypic identification of presumptive Streptococcus pneumoniae by PCR using four genes highly specific for S. pneumoniae. Journal of Medical Microbiology 55:709-714.</li>

<li><a id="ref48" name="ref48"> </a>Swartley, J. S., J. H. Ahn, L. J. Liu, C. M. Kahler, and D. S. Stephens. 1996. Expression of sialic acid and polysialic acid in serogroup B Neisseria meningitidis: divergent transcription of biosynthesis and transport operons through a common promoter region. Journal of Bacteriology 178:4052-4059.</li>

<li><a id="ref49" name="ref49"> </a>Swartley, J. S., L. J. Liu, Y. K. Miller, L. E. Martin, S. Edupuganti, and D. S. Stephens. 1998. Characterization of the gene cassette required for biosynthesis of the (alpha1--&#62;6)-linked N-acetyl-D-mannosamine-1-phosphate capsule of serogroup A Neisseria meningitidis. Journal of Bacteriology 180:1533-1539.</li>

<li><a id="ref50" name="ref50"> </a>Swartley, J. S., A. A. Marfin, S. Edupuganti, L. J. Liu, P. Cieslak, B. A. Perkins, J. D. Wenger, and D. S. Stephens. 1997. Capsule switching of Neisseria meningitidis. Proceedings of the National Academy of Sciences of the United States of America 94:271-276.</li>

<li><a id="ref51" name="ref51"> </a>Swartley, J. S., and D. S. Stephens. 1994. Identification of a genetic locus involved in the biosynthesis of N-acetyl-D-mannosamine, a precursor of the (alpha 2--&#62;8)-linked polysialic acid capsule of serogroup B Neisseria meningitidis. Journal of Bacteriology 176:1530-1534.</li>

<li><a id="ref52" name="ref52"> </a>Taha, M.-K. 2000. Simultaneous approach for nonculture PCR-based identification and serogroup prediction of Neisseria meningitidis. Journal of Clinical Microbiology 38:855-857.</li>

<li><a id="ref53" name="ref53"> </a>Taha, M. K., J. M. Alonso, M. Cafferkey, D. A. Caugant, S. C. Clarke, M. A. Diggle, A. Fox, M. Frosch, S. J. Gray, M. Guiver, S. Heuberger, J. Kalmusova, K. Kesanopoulos, A. M. Klem, P. Kriz, J. Marsh, P. Mölling, K. Murphy, P. Olcén, O. Sanou, G. Tzanakaki, and U. Vogel. 2005. Interlaboratory comparison of PCR-based identification and genogrouping of Neisseria meningitidis. Journal of Clinical Microbiology 43:144-149.</li>

<li><a id="ref54" name="ref54"> </a>Tarrago, D., A. Fenoll, D. Sanchez-Tatay, L. A. Arroyo, C. Munoz-Almagro, C. Esteva, W. P. Hausdorff, J. Casal, and I. Obando. 2008. Identification of pneumococcal serotypes from culture-negative clinical specimens by novel real-time PCR. Clinical Microbiology and Infection 14:828-834.</li>

<li><a id="ref55" name="ref55"> </a>Tettelin, H., N. J. Saunders, J. Heidelberg, A. C. Jeffries, K. E. Nelson, J. A. Eisen, K. A. Ketchum, D. W. Hood, J. F. Peden, R. J. Dodson, W. C. Nelson, M. L. Gwinn, R. DeBoy, J. D. Peterson, E. K. Hickey, D. H. Haft, S. L. Salzberg, O. White, R. D. Fleischmann, B. A. Dougherty, T. Mason, A. Ciecko, D. S. Parksey, E. Blair, H. Cittone, E. B. Clark, M. D. Cotton, T. R. Utterback, H. Khouri, H. Qin, J. Vamathevan, J. Gill, V. Scarlato, V. Masignani, M. Pizza, G. Grandi, L. Sun, H. O. Smith, C. M. Fraser, E. R. Moxon, R. Rappuoli, and J. C. Venter. 2000. Complete genome sequence of Neisseria meningitidis serogroup B strain MC58. Science 287:1809-1815.</li>

<li><a id="ref56" name="ref56"> </a>Tzeng, Y.-L., C. Noble, and D. S. Stephens. 2003. Genetic basis for biosynthesis of the (alpha 1--&#62;4)-linked N-acetyl-D-glucosamine 1-phosphate capsule of Neisseria meningitidis serogroup X. Infection and Immunity 71:6712-6720.</li>

<li><a id="ref57" name="ref57"> </a>Tzeng, Y. L., A. K. Datta, C. A. Strole, M. A. Lobritz, R. W. Carlson, and D. S. Stephens. 2005. Translocation and surface expression of lipidated serogroup B capsular Polysaccharide in Neisseria meningitidis. Infection and Immunity 73:1491-1505.</li>

<li><a id="ref58" name="ref58"> </a>Vogel, U., H. Claus, and M. Frosch. 2000. Rapid serogroup switching in Neisseria meningitidis. New England Journal of Medicine 342:219-220.</li>

<li><a id="ref59" name="ref59"> </a>Waggoner-Fountain, L. A., J. O. Hendley, E. J. Cody, V. A. Perriello, and L. G. Donowitz. 1995. The emergence of Haemophilus influenzae types e and f as significant pathogens. Clinical Infectious Diseases 21:1322-1324.</li>

<li><a id="ref60" name="ref60"> </a>Wang, X., R. Mair, C. Hatcher, M. J. Theodore, K. Edmond, H. M. Wu, B. H. Harcourt, G. Carvalho Mda, F. Pimenta, P. Nymadawa, D. Altantsetseg, M. Kirsch, S. W. Satola, A. Cohn, N. E. Messonnier, and L. W. Mayer. 2011. Detection of bacterial pathogens in Mongolia meningitis surveillance with a new real-time PCR assay to detect Haemophilus influenzae. International Journal of Medical Microbiology 301:303-309.</li>

<li><a id="ref61" name="ref61"> </a>Weber, M. V., H. Claus, M. C. Maiden, M. Frosch, and U. Vogel. 2006. Genetic mechanisms for loss of encapsulation in polysialyltransferase-gene-positive meningococci isolated from healthy carriers. International Journal of Medical Microbiology 296:475-484.</li>

<li><a id="ref62" name="ref62"> </a>Whatmore, A. M., A. Efstratiou, A. P. Pickerill, K. Broughton, G. Woodard, D. Sturgeon, R. George, and C. G. Dowson. 2000. Genetic relationships between clinical isolates of <em>Streptococcus pneumoniae, Streptococcus oralis,</em> and <em>Streptococcus mitis</em>: Characterization of “atypical” pneumococci and organisms allied to <em>S. mitis</em> harboring <em>S. pneumoniae</em> virulence factor-encoding genes. Infection and Immunity 68:1374–1382.</li>
</ol>

<span class="toTop"><span class="icon-angle-up"><!-- --></span>&#160;<a href="chpt10-pcr.html#">Top of Page</a></span>
<p><a href="index.html">Back to Laboratory Methods Manual</a></p>
             					            					            						</div>
            					            					            						</div>
            													        						                                                                                        						                                                                								<!-- Related Nav Area -->
								<div class="nav-related hidden-three hidden-four">
									




									



		    <div class="module-typeH">
			<h3>Related Links</h3>
			<div>
															<ul class="list-block">																									<li>					
<a href="../../vaccines/schedules/index.html">Vaccine Schedules</a>					</li>																				<li>					
<a href="../../vaccines/who/teens/index.html">Preteen &amp; Teen Vaccines</a>					</li>																				<li>					
<a href="../../meningococcal/index.html">Meningococcal Disease</a>					</li>																									<li>					
<a href="../../sepsis/index.html">Sepsis</a>					</li>										</ul>													</div>
	</div>
	  

								</div>
    							<!-- Persistent Content Area -->
    							<div id="content-secondary">
                                    



            						




    							</div><!-- /end #content-secondary -->
    						</div><!-- /end #contentArea -->
    					</div><!-- /end body -->	
					</div>
					<div class="row">
						<div class="span24 visible-one">
														<div class="pageoptionsContainer formatOptions formatOptionsOff"><span id="po-format-label-bottom" class="select-label">Format:</span><div id="po-format-bottom" class="format pageoptions pull-right"><div class="select-box"><a href="chpt10-pcr.html#">Select one</a><span class="icon-angle-down pull-right poarrow"></span></div><ul class="drop-down" aria-labelledby="po-format-label-bottom"><li>
        			    	<a class="noDecoration" href="https://www.cdc.gov/meningitis/lab-manual/chpt10-pcr.pdf" target="_blank"><span class="sprite-16-pdf"></span> PDF [962 KB]</a>
      </li></ul></div></div>													</div>
					</div>
					<div class="row">
						<div class="span24">
                            <div id="plugin-legend" class="pluginOff">
	<h5>File Formats Help:</h5><span><a href="../../Other/plugins/index.html">How do I view different file formats (PDF, DOC, PPT, MPEG) on this site?</a></span>
	<ul>
		<li class="plugin-pdf"><a href="../../Other/plugins/index.html#pdf"><span class="sprite-16-pdf">Adobe PDF file</span></a></li>
		<li class="plugin-ppt"><a href="../../Other/plugins/index.html#ppt"><span class="sprite-16-ppt">Microsoft PowerPoint file</span></a></li>
		<li class="plugin-word"><a href="../../Other/plugins/index.html#doc"><span class="sprite-16-word">Microsoft Word file</span></a></li>
		<li class="plugin-excel"><a href="../../Other/plugins/index.html#xls"><span class="sprite-16-excel">Microsoft Excel file</span></a></li>
		<li class="plugin-wmv"><a href="../../Other/plugins/index.html#wmv"><span class="sprite-16-wmv">Audio/Video file</span></a></li>
		<li class="plugin-qt"><a href="../../Other/plugins/index.html#qt"><span class="sprite-16-qt">Apple Quicktime file</span></a></li>
		<li class="plugin-real"><a href="../../Other/plugins/index.html#ram"><span class="sprite-16-rp">RealPlayer file</span></a></li>
		<li class="plugin-text"><a href="../../Other/plugins/index.html#text"><span class="sprite-16-txt">Text file</span></a></li>
		<li class="plugin-zip"><a href="../../Other/plugins/index.html#zip"><span class="sprite-16-zip">Zip Archive file</span></a></li>
		<li class="plugin-sas"><a href="../../Other/plugins/index.html#sas"><span class="sprite-16-sas">SAS file</span></a></li>
		<li class="plugin-epub"><a href="../../Other/plugins/index.html#epub"><span class="sprite-16-ebook">ePub file</span></a></li>
		<li class="plugin-ris"><a href="../../Other/plugins/index.html#ris"><span class="sprite-16-ris">RIS file</span></a></li>
	</ul>
</div>
						</div>
					</div>
					<!-- date stamp -->
					<div id="datestamp" class="row" itemscope="itemscope" itemtype="http://schema.org/WebPage">
						<div class="span24">
    						<ul> 
            					            						<li class="last-reviewed">Page last reviewed: <span itemprop="lastReviewed">April 15, 2016</span></li>
            													            					            						<li class="last-updated">Page last updated: <span itemprop="dateModified">March 15, 2012</span></li>
            					        							<li class="content-source">Content source:
    									



<ul>
<li itemprop="sourceOrganization" itemscope="itemscope" itemtype="http://schema.org/Organization"><a href="../../ncird.1.html" itemprop="url">National Center for Immunization and Respiratory Diseases</a></li>
</ul>


        							</li>
									



    						</ul>
						</div>
					</div>					
				</div><!-- /end #content -->
			</div><!-- /.container for content -->					
			<!-- footer -->
				<footer id="footer" class="cdcSlimFooter">
	
 <!-- include eng footerSubSocialMedia.html -->
<div class="socialMedia">
	<div class="container">
		<nav class="followcdc">
			<ul>
				<li><a href="../../Other/emailupdates.1.html"><span class="sprite-24-govd"></span><span class="tp-sr-only">Email</span></a></li>
				<li><a href="https://www.facebook.com/CDC" target="_blank"><span class="sprite-24-facebook"></span><span class="tp-sr-only">Recommend</span></a></li>
				<li><a href="https://twitter.com/CDCgov" target="_blank"><span class="sprite-24-twitter"></span><span class="tp-sr-only">Tweet</span></a></li>
				<li><a href="https://www.youtube.com/user/CDCstreamingHealth" target="_blank"><span class="sprite-24-youtube"></span><span class="tp-sr-only">YouTube</span></a></li>
				<li><a href="https://www.instagram.com/CDCgov/" target="_blank"><span class="sprite-24-instagram"></span><span class="tp-sr-only">Instagram</span></a></li>
			</ul>
		</nav>
		<nav class="cdcmedia">
			<ul>
				<li><a href="https://www2c.cdc.gov/podcasts"><span class="sprite-24-listen"></span><span class="tp-sr-only">Listen</span></a></li>
				<li><a href="../../cdctv.1.html"><span class="sprite-24-watch"></span><span class="tp-sr-only">Watch</span></a></li>
				<li><a href="https://www2c.cdc.gov/podcasts/rss.asp"><span class="sprite-24-rss"></span><span class="tp-sr-only">RSS</span></a></li>
			</ul>
		</nav>
	</div>
</div>
 <!-- /include eng footerSubSocialMedia.html -->

	<div class="container">
		<div class="cdcLinks">
			<div class="container">
				<ul>
					
<!-- include eng footerSubCdcLinks.html-->
<li>
	<nav>
		<h6 class="hidden-four hidden-three">ABOUT <span class="icon-plus visible-one pull-right"></span></h6>
		<ul class="hidden-one">
			<li><a href="../../about/default.htm">About CDC</a></li>
			<li><a href="https://jobs.cdc.gov">Jobs</a></li>
			<li><a href="../../funding.1.html">Funding</a></li>
		</ul>
	</nav>
</li>
<li>
	<nav>
		<h6 class="hidden-four hidden-three">LEGAL <span class="icon-plus visible-one pull-right"></span></h6>
		<ul class="hidden-one">
			<li><a href="../../Other/policies.html">Policies</a></li>
			<li><a href="../../Other/privacy.html">Privacy</a></li>
			<li><a href="../../od/foia.1.html">FOIA</a></li>
			<li><a href="../../eeo/nofearact/index.htm">No Fear Act</a></li>
			<li><a href="https://oig.hhs.gov" class="no-link">OIG</a></li>
		</ul>
	</nav>
</li>
<!--/ include eng footerSubCdcLinks.html-->

				</ul>
			</div>
		</div>
		<div class="cdcContact">
			<div class="container">
				<ul>
					<li class="local-contact">
						
 <!-- include eng footerSubGlobalContact1.html -->
<section>
	<address class="address" itemscope="itemscope" itemtype="https://schema.org/Organization">
		<span class="value" itemprop="address" itemscope="itemscope" itemtype="https://schema.org/PostalAddress">
			<span itemprop="streetAddress">1600 Clifton Road</span>
			<span itemprop="addressLocality">Atlanta</span>,
			<span itemprop="addressRegion">GA</span>
			<span itemprop="postalCode">30329-4027</span>
			<span itemprop="addressCountry">USA</span>
		</span><br />
		<span class="value" itemprop="telephone">800-CDC-INFO (800-232-4636)</span>,
		<span class="value" itemprop="telephone">TTY: 888-232-6348</span><br />
		<span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://wwwn.cdc.gov/dcs/RequestForm.aspx" class=" default no-link">Email CDC-INFO</a></span>
		</span>
	</address>
</section>
 <!-- /include eng footerSubGlobalContact1.html -->

					</li>
					<li class="contact">
						
 <!-- include eng footerSubGlobalContact2.html -->
<section>
	<address class="address" itemscope="itemscope" itemtype="https://schema.org/Organization">
		<span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://www.hhs.gov" class="no-link">U.S. Department of Health & Human Services</a></span>
		</span><br />
        <span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
        	<span itemprop="url"><a href="https://www.hhs.gov/open" class="no-link">HHS/Open</a></span>
        </span><br />
        <span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://www.usa.gov/" class="no-link">USA.gov</a></span>
		</span>
	</address>
</section>
 <!-- /include eng footerSubGlobalContact2.html -->

					</li>
				</ul>
			</div>
		</div>
	</div>
</footer>			<!-- /#footer -->
			<div id="totop" class="off visible-one">Top</div>
        </div>
		<script>page_timing.libs_start = Date.now();</script><script src="../../TemplatePackage/3.0/js/libs.min.js@3.2.1.7"></script><script>page_timing.libs_end = Date.now();page_timing.core_start = Date.now();</script><script src="../../TemplatePackage/3.0/js/core.min.js@3.2.1.7"></script><script>page_timing.core_end = Date.now();</script><!-- Start Site Settings Overrides -->
<script>
</script>
<!-- End Site Settings Overrides --><script>page_timing.app_start = Date.now();</script><script src="../../TemplatePackage/3.0/js/app.min.js@3.2.1.7"></script><script>page_timing.app_end = Date.now();</script>		
		        			<!-- Begin Survey scripts -->
<script src="../../JScript/foresee/foresee-trigger.js"></script>
<script src="../../JScript/foresee-variables.js"></script>
<!-- End Survey scripts -->
			
            <script>if (typeof page_timing !== 'undefined') { page_timing.metrics_start = Date.now(); }</script>
<!-- xxxxxxxxxxxxxxxxxx METRICS code begins here xxxxxxxxxxxxxxxxxxxxxxxxxxx-->
<div id="metrics">

<!-- Begin Google DAP inclusion -->
<script type="text/javascript" id="_fed_an_ua_tag" src="../../JScript/metrics/google/Universal-Federated-Analytics-Min.js@agency=HHS&amp;subagency=CDC"></script>
<!-- End Google DAP inclusion -->

<!-- Begin Template Package Custom Interactions -->
<script type="text/javascript" src="../../JScript/metrics/rd_custom_interactions.js"></script>
<!-- End Template Package Custom Interactions -->

<!-- SiteCatalyst code version: H.21. -->
<script src="../../JScript/metrics/topic_levels.js"></script>
<script src="../../JScript/metrics/s_code_v21_cdcgov.js"></script>
<script>

	enquire.register(CDC.Global.selectors.vp1, function() {
		s.prop49 = 1;
	});
	enquire.register(CDC.Global.selectors.vp2, function() {
		s.prop49 = 2;
	});
	enquire.register(CDC.Global.selectors.vp3, function() {
		s.prop49 = 3;
	});
	enquire.register(CDC.Global.selectors.vp4, function() {
		s.prop49 = 4;
	});

            
            


s.pageName=document.title
s.channel="Meningitis"
siteCatalyst.setLevel1("ncird"); <!--formerly NCIRD before 8/15/2012-->
siteCatalyst.setLevel2("meningitis"); <!--formerly Meningitis before 8/15/2012-->
siteCatalyst.setLevel3("lab"); 
siteCatalyst.setLevel4("dbd"); <!-- "dbd-dvd" BEFORE 12/8/2014, "hcso"” BETWEEN 12/8/2014 and 10/2/2015-->

            
                        
            s.prop2=window.location.href;
s.prop26=document.title;
s.prop30=document.title;
s.prop31=window.location.href;
s.prop46=window.location.href;
s.server=window.location.hostname;

// Update the level variables here.
updateVariables(s);

/************* DO NOT ALTER ANYTHING BELOW THIS LINE ! **************/
var s_code=s.t();



if(s_code) {
	document.write(s_code);
}



if (document.location.protocol) {


	var cookieManagment = {
		getUuid : function (strPrefix) {
			strPrefix = strPrefix || "";
			var d = new Date();
			var seconds = Math.round(d.getTime() / 1000);
			return strPrefix + Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1) + "-" + seconds.toString();
		},
		setCookie : function (cname, cvalue, exdays, domain) {
			domain = domain || ".cdc.gov";
			var d = new Date();
			d.setTime(d.getTime() + (exdays*24*60*60*1000));
			var expires = "expires="+ d.toUTCString();
			document.cookie = cname + "=" + cvalue + ";" + expires + ";domain=" + domain + ";path=/";
		},
		getCookie : function (cname) {
			var name = cname + "=";
			var ca = document.cookie.split(';');
			var i = ca.length;
			while (i --) {
				var c = ca[i];
				while (c.charAt(0)==' ') {
					c = c.substring(1);
				}
				if (c.indexOf(name) == 0) {
					return c.substring(name.length,c.length);
				}
			}
			return "";
		},
		init : function () {



			var currCookieVal = cookieManagment.getCookie('cdc_s_vi');

			if (!currCookieVal.length || currCookieVal.indexOf('[CS]v1') != -1) {
				var newCookieVal = cookieManagment.getUuid('[CS]v2') + "-" + cookieManagment.getUuid('');
				cookieManagment.setCookie("cdc_s_vi", newCookieVal, 7);
				return newCookieVal;
			}
			return currCookieVal;
		}
	};


	var cdcSessionID = cookieManagment.init();

	s3 = s_gi("cdcgovvistorid");

	var sKey, override = {
		fun: "cdcgovvistorid",
		trackingServer: "cdc.112.2o7.net"
	};

	s3.visitorID = cdcSessionID;

	for (sKey in s) {
		if (s.hasOwnProperty(sKey)) {
			if (override.hasOwnProperty(sKey)) {
				s3[sKey] = override[sKey];

			} else {

			}
		}
	}
	updateVariables(s3);


	var s_code_uvid = s3.t();
	if(s_code_uvid) {
		document.write(s_code_uvid);
	}
}




</script>
<script type="text/javascript">
if(navigator.appVersion.indexOf('MSIE')>=0) { document.write(unescape('%3C')+'\!-'+'-'); }
</script>
<noscript><a href="http://www.omniture.com"><img src="https://smtrics.cdc.gov/b/ss/cdcgov/1/H.21--NS/01/H.21--NS/0" height="1" width="1" alt="Web Analytics" /></a></noscript><!--/DO NOT REMOVE/-->
<!-- End SiteCatalyst code version: H.21. -->
<!-- Begin Template Package DigitalGov Search Support -->
<script type="text/javascript" src="../../JScript/rd_search.js"></script>
<!-- End Template Package DigitalGov Search Support -->
</div>
<!-- xxxxxxxxxxxxxxxxxx METRICS code ends here xxxxxxxxxxxxxxxxxxxxxxxxxxx-->
<script>if (typeof page_timing !== 'undefined') { page_timing.metrics_end = Date.now(); }</script>
				


<script>
<!--//
 $(document).ready(function () {
  var cdcLeftNav = new CDC.LeftNav("nav-primary");
  cdcLeftNav.render();
 });
//-->
</script>

    	




						    </body>
	</html>
