
<!DOCTYPE html>
<!--[if lt IE 8]>      <html class="no-js nav theme-purple lt-ie10 lt-ie9 lt-ie8  " lang="en-us"> <![endif]-->
<!--[if IE 8]>         <html class="no-js nav theme-purple lt-ie10 lt-ie9  " lang="en-us"> <![endif]-->
<!--[if IE 9]>         <html class="no-js nav theme-purple lt-ie10  " lang="en-us"> <![endif]-->
<!--[if !IE]><!-->     <html class="no-js nav theme-purple  " lang="en-us" > <!--<![endif]-->	
    <head>
        <meta charset="utf-8" />
                		<title>Meningitis | Lab Manual | Characterization - Molecular Typing | CDC</title>
        		
        <meta name="description" content="Primary Culture and Presumptive Identification Laboratory Methods." />        <meta name="keywords" content="meningitis, bacterial meningitis, meningitis in lab, meningitis in laboratory, meningitis laboratory methods, epidemiology, biosafety, clinical specimens" />		<meta name="viewport" content="width=device-width, initial-scale=1" />
		        <link rel="apple-touch-icon-precomposed" sizes="144x144" href="../../TemplatePackage/3.0/images/cdc-touch-icon-144x144.png" />
        <link rel="apple-touch-icon-precomposed" sizes="114x114" href="../../TemplatePackage/3.0/images/cdc-touch-icon-114x114.png" />
        <link rel="apple-touch-icon-precomposed" sizes="72x72" href="../../TemplatePackage/3.0/images/cdc-touch-icon-72x72.png" />
        <link rel="apple-touch-icon-precomposed" href="../../TemplatePackage/3.0/images/cdc-touch-icon-57x57.png" />
        <link rel="shortcut icon" href="../../TemplatePackage/3.0/images/favicon.ico" />	
		
		<meta name="robots" content="index,follow" />
		<meta property="cdc:template_version" content="3.0" />
		<meta property="cdc:content-id" content="314802"/>
		<meta property="cdc:revision-id" content="6"/>
				<link rel="canonical" href="http://www.cdc.gov/meningitis/lab-manual/chpt12-molecular-typing-methods.html"/>
				<!-- Internal Release Version # 3.7.2 -->
                			<link rel="stylesheet" href="../../TemplatePackage/3.0/css/lib/bootstrap24.css" />
	<link rel="stylesheet" href="../../TemplatePackage/3.0/css/app.css@3.2.1.7.css" />
	<!--[if gt IE 9]><!--><!--<![endif]-->
	<!--[if IE 7]>
		<link rel="stylesheet" href="/TemplatePackage/3.0/css/lib/font-awesome-ie7.css" />
	<![endif]-->
	<!--[if lt IE 9]>
		<link rel="stylesheet" href="/TemplatePackage/3.0/css/ie.css" />
	<![endif]-->
	<link rel="stylesheet" media="print" href="../../TemplatePackage/3.0/css/print.css@3.2.1.7.css" />
	<script src="../../TemplatePackage/3.0/js/libs/modernizr-latest.js"></script>
	<script>
		if(!Date.now){Date.now=function now(){return new Date().getTime()}}
		var page_timing = page_timing || {};
		page_timing.page_start = Date.now();
		function $$(e){if(window.$){e()}else{setTimeout(function(){$$(e)},50)}}
		// (function(){function e(e,t){var n=e.length;while(n--){if(e[n].indexOf(t)>-1){return e[n]}}return false}var t=e(document.getElementsByTagName("html")[0].className.split(" "),"theme-");if(t){var n=document.createElement("link");n.rel="stylesheet";n.type="text/css";n.media="screen";n.href="/TemplatePackage/3.0/css/themes/"+t+".css";document.getElementsByTagName("head")[0].appendChild(n)}})();

		var settings = {}, addthis = null;
	</script>

	<meta name="application-name" content="CDC.gov"/>
	<meta property="cdc:template_version" content="3.2.1.7" />
	<meta name="msapplication-TileColor" content="#0061aa" />
	<meta name="msapplication-square70x70logo" content="/TemplatePackage/3.0/images/Win8_tile_70x70.png" />
	<meta name="msapplication-square150x150logo" content="/TemplatePackage/3.0/images/Win8_tile_150x150.png" />
	<meta name="msapplication-wide310x150logo" content="/TemplatePackage/3.0/images/Win8_tile_310x150.png" />
	<meta name="msapplication-square310x310logo" content="/TemplatePackage/3.0/images/Win8_tile_310x310.png" />
	<meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">		        




         
<!-- OLD TITLE BEFORE 12/5/2014 <title>CDC - Meningitis - Lab Manual - Antimicrobial Susceptibility Testing - Chapter 11</title> -->
<!-- OLD TITLE BEFORE "03/15/2012" DIDNOTEXIST -->
<!-- OLD URL BEFORE "03/15/2012" DIDNOTEXIST -->
<!-- OLD CHANNEL NAME BEFORE "03/15/2012" DIDNOTEXIST" -->
<!-- channel="Meningitis" as of "03/15/2012" -->
<!-- Level 1="NCIRD" new level 1 as of "03/15/2012" -->
<!-- Level 2="Meningitis" new level 2 as of "03/15/2012" -->
<!-- Level 3="Lab" new level 3 as of "03/15/2012" -->
         		    </head>
    <body>
				




				<div id="skipmenu">
	<a href="chpt12-molecular-typing-methods.html#searchTarget" class="skippy">Skip directly to search</a>
	<a href="chpt12-molecular-typing-methods.html#azTab" class="skippy">Skip directly to A to Z list</a>
	<a href="chpt12-molecular-typing-methods.html#nav-container" class="skippy">Skip directly to navigation</a>
	<a href="chpt12-molecular-typing-methods.html#share-bar" class="skippy">Skip directly to page options</a>
	<a href="chpt12-molecular-typing-methods.html#contentArea" class="skippy">Skip directly to site content</a>
</div>		    	<div id="wrapper">
    		
<!-- references to this removed intentionally -->
    		<header id="header">
    			<div class="container">
                    <div class="row">
                    	<div id="logo" class="span14">
                    		<!--iPad touch response and 508 voice reader issue currently under investigation -->
<a href="../../index.html" title="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People">
	<img src="../../TemplatePackage/3.0/images/masthead_subpage.svg" alt="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People" class="hidden-one" />
	<img src="../../TemplatePackage/3.0/images/masthead_1-col.svg" alt="Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People" class="visible-one" />
</a>

<!-- IOS9 iPad bug work around - make sure screen readers have a readable version of the logo to read
<a class="tp-sr-only" href="https://www.cdc.gov">Centers for Disease Control and Prevention. CDC twenty four seven. Saving Lives, Protecting People</a>
 -->
                    	</div>
                    	<!-- search -->
                    	<div id="searchArea" class="span10 hidden-one hidden-two">							
							
<div class="hidden"><a href="chpt12-molecular-typing-methods.html#" name="searchTarget">Start of Search Controls</a></div>
<form class="searchForm" method="get" action="https://search.cdc.gov/search">
	<fieldset class="search-form-wrapper ">
		<legend>Search Form Controls</legend>
		<div class="search-form-checkbox">
			<label class="subset"><input type="checkbox" name="subset" checked="checked" value="">Search The CDC</label>
		</div>
		<div class="search-input-wrapper">
			<div class="search-buttons">
				<button class="btn-clear"><span class="sprite-14-close">Cancel</span></button>
				<button class="btn-submit"><span class="sprite-14-search-black">Submit</span></button>
				<input type="submit" value="Submit" style="display:none; visibility:hidden;" /> 
			</div>
			<div class="search-input">
				<label><input type="text" placeholder="SEARCH" name="query" /><span class="hidden">Search The CDC</span></label>
			</div>
		</div>
		<input type="hidden" name="utf8" value="&#x2713;" />
					<input type="hidden" name="affiliate" value="cdc-main" />
			</fieldset>
</form>							<div class="a2z-button pull-right hidden-one hidden-two"><a href="chpt12-molecular-typing-methods.html#" id="azTab">CDC A-Z Index <span class="icon-angle-down"></span></a></div>
                    	</div>
                    </div>
                    	<div class="hidden-three hidden-four" id="mobile-menu">
		<ul>
			<li id="vp1-menu-button" class="border-bottom-white"><a href="chpt12-molecular-typing-methods.html#" id="menu-nav"><span class="sprite-16-menu"></span>MENU</a></li>
			<li id="vp1-a2z-button" class="border-bottom-white"><a href="chpt12-molecular-typing-methods.html#" id="menu-az">CDC A-Z</a></li>
			<li id="vp1-search-button" class="border-bottom-white"><a href="chpt12-molecular-typing-methods.html#" id="menu-search"><span class="sprite-16-search-white"></span>SEARCH</a></li>
		</ul>
	</div>
<script>if (typeof page_timing !== 'undefined') { page_timing.menu_load = Date.now(); }</script>
                    <div class="a2z-bar hide">
                    	<div>
	<ul>
		<li><a href="../../az/a.html">A</a></li>
		<li><a href="../../az/b.html">B</a></li>
		<li><a href="../../az/c.html">C</a></li>
		<li><a href="../../az/d.html">D</a></li>
		<li><a href="../../az/e.html">E</a></li>
		<li><a href="../../az/f.html">F</a></li>
		<li><a href="../../az/g.html">G</a></li>
		<li><a href="../../az/h.html">H</a></li>
		<li><a href="../../az/i.html">I</a></li>
		<li><a href="../../az/j.html">J</a></li>
		<li><a href="../../az/k.html">K</a></li>
		<li><a href="../../az/l.html">L</a></li>
		<li><a href="../../az/m.html">M</a></li>
		<li><a href="../../az/n.html">N</a></li>
		<li><a href="../../az/o.html">O</a></li>
		<li><a href="../../az/p.html">P</a></li>
		<li><a href="../../az/q.html">Q</a></li>
		<li><a href="../../az/r.html">R</a></li>
		<li><a href="../../az/s.html">S</a></li>
		<li><a href="../../az/t.html">T</a></li>
		<li><a href="../../az/u.html">U</a></li>
		<li><a href="../../az/v.html">V</a></li>
		<li><a href="../../az/w.html">W</a></li>
		<li><a href="../../az/x.html">X</a></li>
		<li><a href="../../az/y.html">Y</a></li>
		<li><a href="../../az/z.html">Z</a></li>
		<li><a href="../../az/0.html">#</a></li>
	</ul>
</div>
<script>if (typeof page_timing !== 'undefined') { page_timing.a2z_load = Date.now(); }</script>                    </div>
                    <div id="searchArea-two-column" class="span24 searchbar hide hidden-three hidden-four">
						  
<div class="hidden"><a href="chpt12-molecular-typing-methods.html#" name="searchTarget">Start of Search Controls</a></div>
<form class="searchForm" method="get" action="https://search.cdc.gov/search">
	<fieldset class="search-form-wrapper ">
		<legend>Search Form Controls</legend>
		<div class="search-form-checkbox">
			<label class="subset"><input type="checkbox" name="subset" checked="checked" value="">Search The CDC</label>
		</div>
		<div class="search-input-wrapper">
			<div class="search-buttons">
				<button class="btn-clear"><span class="sprite-14-close">Cancel</span></button>
				<button class="btn-submit"><span class="sprite-14-search-black">Submit</span></button>
				<input type="submit" value="Submit" style="display:none; visibility:hidden;" /> 
			</div>
			<div class="search-input">
				<label><input type="text" placeholder="SEARCH" name="query" /><span class="hidden">Search The CDC</span></label>
			</div>
		</div>
		<input type="hidden" name="utf8" value="&#x2713;" />
					<input type="hidden" name="affiliate" value="cdc-main" />
			</fieldset>
</form>					</div>
                    <!--
	See the noscript include for modifying the emergency message. DO NOT add it here.
-->
<script>if (typeof page_timing !== 'undefined') { page_timing.emergency_load = Date.now(); }</script>
    				<div class="titlebar">
					 						<h2>


<a href="../index.html">Meningitis</a>

</h2>
					 					</div>
    				<!-- emergency -->
<div id="emergencyContainer">
	<!--
	<div class="alert alert-error en">
		<b>Attention!</b> Uncomment this section and place the emergency message here.
	</div>
	<div class="alert alert-error esp">
		<b>&#161;Atenci&#243;n!</b> Elimine el comentario de esta secci&#243;n y colocar el mensaje de emergencia aqu&#237;.
	</div>	
	-->
</div>

<!-- no script -->
<noscript>
	<div class="alert alert-block"> Note: Javascript is disabled or is not supported by your browser. For this reason, some items on this page will be unavailable. For more information about this message, please visit this page: <a href="../../other/about_cdcgov.html">About CDC.gov</a>.</div>
</noscript>
<script>if (typeof page_timing !== 'undefined') { page_timing.noscript_load = Date.now(); }</script>
     			</div>
    		</header>
    		<div class="container">				
                <div id="content">
					<div class="row">
        				<!-- left nav -->
        				<div id="nav-container" class="hidden-one hidden-two span5">
    						<nav id="left" class="hidden-one hidden-two">
    							



<h3><a href="../index.html">Meningitis Home</a></h3>

<ul id="nav-primary">
<li><a href="../bacterial.html">Bacterial Meningitis</a>
<ul>
<li><a href="index.html">Laboratory Methods for the Diagnosis of Meningitis</a>
<ul>
<li><a href="disclosure.html">Disclosure</a></li>

<li><a href="authors.html">Authors/Acknowledgements</a></li>

<li><a href="chpt01-intro.html">Chapter 1</a></li>

<li><a href="chpt02-epi.html">Chapter 2</a></li>

<li><a href="chpt03-results-mgmt-rpt-data.html">Chapter 3</a></li>

<li><a href="chpt04-biosafety.html">Chapter 4</a></li>

<li><a href="chpt05-collect-transport-specimens.html">Chapter 5</a></li>

<li><a href="chpt06-culture-id.html">Chapter 6</a></li>

<li><a href="chpt07-id-characterization-nm.html">Chapter 7</a></li>

<li><a href="chpt08-id-characterization-streppneumo.html">Chapter 8</a></li>

<li><a href="chpt09-id-characterization-hi.html">Chapter 9</a></li>

<li><a href="chpt10-pcr.html">Chapter 10</a></li>

<li><a href="chpt11-antimicrobial-suscept-testing.html">Chapter 11</a></li>

<li><a href="chpt12-molecular-typing-methods.html">Chapter 12</a></li>

<li><a href="chpt13-quality-control-assurance.html">Chapter 13</a></li>

<li><a href="chpt14-storage-shipping.html">Chapter 14</a></li>

<li><a href="annex-prep-media-reagents.html">Annex</a></li>
</ul>
</li>
</ul>
</li>

<li><a href="../viral.html">Viral Meningitis</a></li>

<li><a href="../fungal.html">Fungal Meningitis</a></li>

<li><a href="../parasitic.html">Parasitic Meningitis</a></li>

<li><a href="../amebic.html">Amebic Meningitis</a></li>

<li><a href="../non-infectious.html">Non-infectious Meningitis</a></li>

<li><a href="../clinical-resources.html">Resources for Healthcare Professionals</a></li>
</ul>


    						</nav>
    						<div class="nav-related">
							




    						



		    <div class="module-typeH">
			<h3>Related Links</h3>
			<div>
															<ul class="list-block">																									<li>					
<a href="../../vaccines/schedules/index.html">Vaccine Schedules</a>					</li>																				<li>					
<a href="../../vaccines/who/teens/index.html">Preteen &amp; Teen Vaccines</a>					</li>																				<li>					
<a href="../../meningococcal/index.html">Meningococcal Disease</a>					</li>																									<li>					
<a href="../../sepsis/index.html">Sepsis</a>					</li>										</ul>													</div>
	</div>
	  

							




							</div>    						    				
        				</div>
        				<div id="featureArea">
    						<div class="span19">
    							



    						</div>
        				</div>						
    					<div class="breadcrumbs hidden-one span19">
    						


<div class="bread-crumb hidden-one"></div>

    					</div>
						<div class="span19 pagetitle">	
    						    							<div class='syndicate'>
    						    						    							<div class='mSyndicate'>
    																					<h1>Chapter 12: Characterization of <em>Neisseria meningitidis</em>, <em>Haemophilus  influenzae</em>, and <em>Streptococcus  pneumoniae </em>by Molecular Typing Methods</h1>
															    						    							</div>
    						    						    							</div>
    												</div>
						<div id="share-bar" class="span19">		
    						<div class="socialmediabar">
								<div class="row">
            						<div class="hidden-one pull-right">
        								            							            							<div class="pageoptionsContainer formatOptions formatOptionsOff"><span id="po-format-label" class="select-label">Format:</span><div id="po-format" class="format pageoptions pull-right"><div class="select-box"><a href="chpt12-molecular-typing-methods.html#">Select one</a><span class="icon-angle-down pull-right poarrow"></span></div><ul class="drop-down" aria-labelledby="po-format-label"><li>
        			    	<a class="noDecoration" href="https://www.cdc.gov/meningitis/lab-manual/chpt12-molecular-typing-methods.pdf" target="_blank"><span class="sprite-16-pdf"></span> PDF [473 KB]</a>
      </li></ul></div></div>            						</div>	
    								<div class="span8">
    									<div id="socialMediaShareContainer">
	<div class="share_container noLinking">
	 	<a class="share_button_facebook" title="Facebook" href="chpt12-molecular-typing-methods.html#"><span class="tp-sr-only">Recommend on Facebook</span><span id="share_facebook" class="facebookRecommend"> </span></a>
	 	<a class="share_button_twitter" title="Twitter" href="chpt12-molecular-typing-methods.html#"><span class="tp-sr-only">Tweet</span><span id="share_twitter" class="twitterTweet"> </span></a>
		<a class="share_button" href="chpt12-molecular-typing-methods.html#socialMediaShareContainer" ><img src="../../TemplatePackage/3.0/images/social/addthis-32.png" alt="Share" /><span class="shareButtonEn tp-sr-only" aria-label="Share">Share</span><span class="shareButtonEs tp-sr-only" aria-label="Compartir">Compartir</span></a>
	</div>
</div>

<!-- target anchor for return to top feature in VP1 -->
<a id="content-start" tabindex="-1"></a>
    								</div>
								</div>
    						</div>
    					</div>	
    					<!-- body -->
    					<div id="body" class="span19">
    						<div id="contentArea" >								
        						        																						        						                                                                                            					            						<div class='syndicate'>
            					            					            						<div class='mSyndicate'>
            					            					        							
<p><a href="https://www.cdc.gov/meningitis/lab-manual/chpt12-molecular-typing-methods.pdf">Printer friendly version [57 pages]</a></p>

<ol type="I">
<li><span class="bold">Introduction</span>
<p>A number of DNA-based molecular typing methods have been used in epidemiological investigations of bacterial pathogens and studying bacterial population genetics and evolution. Some methods are suitable for characterizing strains that cause localized outbreaks; others are particularly useful for determining the long-term relationships of strains and their population structures (<a href="chpt12-molecular-typing-methods.html#ref1">1</a>). For local outbreak investigation, molecular typing methods with a high degree of resolution are necessary to detect subtle differences in what most likely are genetically related strains. These assays are also useful for detecting changes in molecular type during prolonged outbreaks. Highly discriminatory assays typically index genomic variations that accumulate rapidly and can be used to detect microvariations that can be used to identify strains circulating in a specific geographic area. The methods that have been used for this purpose including ribotyping, random amplified polymorphic DNA, fluorescent amplified-fragment length polymorphism, restriction fragment length polymorphism (RFLP), multiple-locus variable-number tandem repeat analysis (MLVA) and pulsed-field gel electrophoresis (PFGE). The high resolution of these assays will sometimes resolve subtle differences that MLST cannot, thus they are particularly useful for studying local epidemiology or tracing the spread of highly related strains during outbreaks/clusters of cases (<a href="chpt12-molecular-typing-methods.html#ref1">1</a>, <a href="chpt12-molecular-typing-methods.html#ref7">7</a>, <a href="chpt12-molecular-typing-methods.html#ref15">15</a>).</p>

<p>PFGE is one of the most widely used molecular typing methods because of the resolving power of the technique. In brief, PFGE uses a restriction enzyme to cut genomic DNA infrequently at a specific sequence to produce a number of fragments that are then size-fractionated on an agarose gel. The resultant banding patterns are analyzed and compared to other isolates. During an outbreak or a cluster of cases, PFGE is used in combination with epidemiologic information and other typing techniques to help identify outbreak isolates and determine the relationships among the isolates. A promising technique with similar discriminatory capabilities that is gaining acceptance is multiple-locus variable-number tandem repeat analysis (MLVA). In MLVA, the variability in the numbers of short tandem repeat sequences that are found in specific areas of the genome are utilized to create DNA fingerprints for epidemiological studies. Once the appropriate loci are established, MLVA yields clustering similar to PFGE (<a href="chpt12-molecular-typing-methods.html#ref13">13</a>) and the use of highly variable-number tandem repeats results in a high degree of differentiation with suitable resolution to discriminate between strains in an outbreak or cluster (<a href="chpt12-molecular-typing-methods.html#ref24">24</a>, <a href="chpt12-molecular-typing-methods.html#ref34">34</a>).</p>

<p>Tracking the population biology of bacterial organisms on a global or even national scale requires a molecular typing method based on slowly accumulating selectively neutral genetic variations that will still distinguish between genotypes yet can identify clonal groups. One of the most common selectively neutral areas of a genome are housekeeping genes encoding proteins involved in the metabolism of the organism. The first molecular typing method to utilize this attribute was multilocus enzyme electrophoresis (MLEE). MLEE analyses the electrophoretic mobilities of metabolic housekeeping enzymes on a starch gel and equates changes in mobility of each enzyme with allelic variants of each loci (<a href="chpt12-molecular-typing-methods.html#ref1">1</a>, <a href="chpt12-molecular-typing-methods.html#ref7">7</a>). In order to obtain high resolution, 20 or more loci are analyzed for each isolate. The genetic variation leading to the changes necessary to cause mobility shifts are thought to be selectively neutral, thus the electrophoretic type of a bacterial clone should be relatively stable over time (<a href="chpt12-molecular-typing-methods.html#ref7">7</a>). MLEE was instrumental as a typing method in early global epidemiology studies and population biology analysis, but the method is technically demanding, and because MLEE is gel-based, results between laboratories are very difficult to compare.</p>

<p>In 1998, Maiden et al developed a DNA-based method of molecular typing <em>N. meningitidis</em> that retained the concepts utilized by MLEE of analyzing housekeeping genes not under any known immunologic selection. MLST defines strains by their allelic profiles determined from the nucleotide sequences of internal fragments of seven housekeeping genes rather than by shifting electrophoretic mobility patterns of the enzymes they encode (<a href="chpt12-molecular-typing-methods.html#ref15">15</a>). This approach has been adapted for use in typing many types of bacteria, including <em>H. influenzae</em>, and <em>S. pneumoniae</em>, and is now the most widely used method for molecular typing. While MLST is more expensive to perform, it has the following advantages over MLEE:</p>

<ul>
<li>Directly measures genetic variations, and therefore resolves more alleles per locus</li>

<li>Is readily scalable and adaptable for a high throughput format</li>

<li>Sequence data is reproducible and can be objectively compared between laboratories</li>

<li>Sequence analysis can be performed remotely and the results returned via the internet</li>

<li>Sequence data can be uploaded to a centralized database and is accessible to all scientists via the internet to produce a powerful tool for global epidemiology</li>

<li>Information can be obtained from PCR amplification from clinical material</li>
</ul>

<p>MLST is not suitable to use to characterize potential differences in strains within an outbreak as its power to resolve small evolutionary differences is too low. Therefore, to provide further discriminatory power, MLST data can be combined with PFGE and sequence data from other more variable loci under positive selective pressure, such as PorA, PorB, FetA, and fHbp in <em>N. meningitidis</em>, and the penicillin binding proteins (PBPs) and PspA in <em>S. pneumoniae</em>. It is this variability that makes these proteins excellent markers for short-term epidemiology investigations. Sequence data for these markers is also valuable for assessing the allelic distribution of these vaccine candidate proteins that can aid in rational vaccine design.</p>

<ol type="A">
<li><span class="bold">Multilocus sequence typing (MLST) and typing of other variable genes of <em>N. meningitidis</em>, <em>H. influenzae</em>, and <em>S. pneumoniae</em></span>
<p>The PCR primers and protocols included here are used at CDC and work well. However, many laboratories use different PCR reaction protocols, chemistries, buffers, primer concentrations and even different equipment for PCR amplification and DNA sequencing. Other protocols work well and it is up to the discretion of each laboratory to discern the optimal reagents and assay conditions for their laboratory, including appropriate validation and quality control.</p>

<p>The primers listed in this manual have been designed and optimized over time as more sequence data has become available. By comparing DNA sequences for a given area of interest from dozens to hundreds of different strains of <em>N. meningitidis, H. influenzae</em>, or <em>S. pneumoniae</em>, small areas of homology can be detected that provide an area from which PCR and DNA sequencing primers can be designed that have the greatest likelihood of working. In some instances, a perfect consensus cannot be found which will react with all strains. In these cases, degenerate primers are designed that have a mix of nucleotides at a certain position. For example, when either a "C" or a "T" nucleotide can occupy a position, the International Union of Pure and Applied Chemistry (IUPAC), designates it as "Y". For a complete list of IUPAC nucleotide abbreviations, see: <a href="http://www.chem.qmul.ac.uk/iupac/">http://www.chem.qmul.ac.uk/iupac/</a>.</p>
</li>

<li><span class="bold">Preparation of DNA template</span>
<p>For each of the assays described below a preparation of DNA must be extracted. A pure culture of each isolate is grown on trypticase soy agar plates supplemented with 5% sheep blood for <em>N. meningitidis</em> and <em>S. pneumoniae</em>, or chocolated agar plates supplemented with hemin and NAD for <em>H. influenzae</em> in a humidified incubator for 18-24 hours at 37°C with 5% CO<sub>2</sub>. Fast DNA extraction protocols for <em>N. meningitidis, H. influenzae</em>, and <em>S. pneumoniae</em> can be found in Appendix 1 at the end of this chapter. Additional methods for DNA extraction that will provide purified DNA can be found in <a href="chpt10-pcr.html">Chapter 10: PCR Methods</a> or commercially available DNA extraction kits can be used.</p>
</li>
</ol>
</li>

<li><span class="bold">Multilocus Sequence Typing (MLST)</span>
<ol type="A">
<li>Overview and MLST schemes for <em>N. meningitidis</em>, <em>H. influenzae</em>, and <em>S. pneumoniae</em>
<p>The housekeeping genes selected for MLST are not closely linked in bacterial genomes and have conserved sequence regions that are sufficient to design PCR primers that will amplify all or nearly all of the isolates. The number of housekeeping genes to be examined reflects an optimal balance between resolution of genotypes and laboratory workload. The internal fragments are of a length to provide sufficient variation for identification of different alleles and for each strand to be accurately sequenced using only a single primer for each direction. Sequences that differ by even a single nucleotide are considered unique and no weight is given to the number of nucleotide changes in a given allele. Each unique allele is assigned a number in order of discovery and each isolate can be characterized by its multilocus genotypic or allelic profile, designated as sequence type (ST), which is the combination of its alleles over the seven genetic loci. STs can be further grouped into clonal complexes (CC), which are defined in the Neisseria MLST profile database as a group of STs that share at least four of the seven loci in common with a central ST (<a href="chpt12-molecular-typing-methods.html#ref30">30</a>).</p>

<p>MLST schemes have been developed for <em>N. meningitidis</em> (<a href="chpt12-molecular-typing-methods.html#ref15">15</a>) (<a href="chpt12-molecular-typing-methods.html#table1">Table 1</a>), <em>H. influenzae</em> (<a href="chpt12-molecular-typing-methods.html#ref20">20</a>) (<a href="chpt12-molecular-typing-methods.html#table2">Table 2</a>), <em>S. pneumoniae</em> (<a href="chpt12-molecular-typing-methods.html#ref7">7</a>) (<a href="chpt12-molecular-typing-methods.html#table3">Table 3</a>) and many others (see <a href="http://www.mlst.net/">http://www.mlst.net/</a>). Each scheme uses defined regions of seven housekeeping genes. A general protocol for performing MLST and analyzing the data is given below.</p>

<a id="table1" name="table1"> </a>
<p><span class="bold">Table 1.</span> <em>N. meningitidis</em> MLST scheme, including gene locus, amplicon length, and trimmed length of sequence used for allelic determination</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Housekeeping genes</th>
<th scope="col">Gene locus</th>
<th scope="col">Trimmed length</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">Putative ABC transporter</th>
<td><em>abcZ</em></td>
<td>433</td>
</tr>

<tr>
<th scope="row">Adenylate kinase</th>
<td><em>adk</em></td>
<td>465</td>
</tr>

<tr>
<th scope="row">Shikimate dehydrogenase</th>
<td><em>aroE</em></td>
<td>490</td>
</tr>

<tr>
<th scope="row">Fumurate dehydrogenase</th>
<td><em>fumC</em></td>
<td>465</td>
</tr>

<tr>
<th scope="row">Glucose-6-phosphate dehydrogenase</th>
<td><em>gdh</em></td>
<td>501</td>
</tr>

<tr>
<th scope="row">Pyruvate dehydrogenase subunit</th>
<td><em>pdhC</em></td>
<td>480</td>
</tr>

<tr>
<th scope="row">Phosphoglucomutase</th>
<td><em>pgm</em></td>
<td>450</td>
</tr>

</tbody></table>

<a id="table2" name="table2"> </a>
<p><span class="bold">Table 2.</span> <em>H. influenzae</em> MLST scheme, including gene locus, amplicon length, and trimmed length of sequence used for allelic determination</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Housekeeping genes</th>
<th scope="col">Gene locus</th>
<th scope="col">Trimmed length</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">Adenylate kinase</th>
<td><em>adk</em></td>
<td>477</td>
</tr>

<tr>
<th scope="row">ATP synthase F1 subunit gamma</th>
<td><em>atpG</em></td>
<td>447</td>
</tr>

<tr>
<th scope="row">Fumarate reductase iron-sulfur protein</th>
<td><em>frdB</em></td>
<td>489</td>
</tr>

<tr>
<th scope="row">Fuculokinase</th>
<td><em>fucK</em></td>
<td>345</td>
</tr>

<tr>
<th scope="row">Malate dehydrogenase</th>
<td><em>mdh</em></td>
<td>405</td>
</tr>

<tr>
<th scope="row">Glucose-6-phosphate isomerase</th>
<td><em>pgi</em></td>
<td>468</td>
</tr>

<tr>
<th scope="row">RecA protein</th>
<td><em>recA</em></td>
<td>426</td>
</tr>

</tbody></table>

<a id="table3" name="table3"> </a>
<p><span class="bold">Table 3.</span> <em>S. pneumoniae</em> MLST scheme, including gene locus, amplicon length, and trimmed length of sequence used for allelic determination</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Housekeeping genes</th>
<th scope="col">Gene locus</th>
<th scope="col">Trimmed length</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">Shikimate dehydrogenase</th>
<td><em>aroE</em></td>
<td>405</td>
</tr>

<tr>
<th scope="row">Glucose-6-phosphate dehydrogenase</th>
<td><em>gdh</em></td>
<td>460</td>
</tr>

<tr>
<th scope="row">Glucose kinase</th>
<td><em>gki</em></td>
<td>483</td>
</tr>

<tr>
<th scope="row">Transketolase</th>
<td><em>recP</em></td>
<td>450</td>
</tr>

<tr>
<th scope="row">Signal peptidase I</th>
<td><em>spi</em></td>
<td>474</td>
</tr>

<tr>
<th scope="row">Xanthine phosphoribosyltransferase</th>
<td><em>xpt</em></td>
<td>486</td>
</tr>

<tr>
<th scope="row">D-alanine-D-alanine ligase</th>
<td><em>ddl</em></td>
<td>441</td>
</tr>

</tbody></table>

<a id="table4" name="table4"> </a>
<p><span class="bold">Table 4.</span> MLST amplification primers for <em>N. meningitidis</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene locus</th>
<th scope="col">Primer name</th>
<th scope="col">Forward primer (5'-3')</th>
<th scope="col">Primer name</th>
<th scope="col">Reverse primer (5'-3')</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>abcZ</em></th>
<td>abcZ-P1C</td>
<td>TGTTCCGCTTCGACTGCCAAC</td>
<td>abcZ-P2C</td>
<td>TCCCCGTCGTAAAAAACAATC</td>
</tr>

<tr>
<th scope="row"><em>adk</em></th>
<td>adk-P1B</td>
<td>CCAAGCCGTGTAGAATCGTAAACC</td>
<td>adk-P2B</td>
<td>TGCCCAATGCGCCCAATAC</td>
</tr>

<tr>
<th scope="row"><em>aroE</em></th>
<td>aroE-P1B</td>
<td>TTTGAAACAGGCGGTTGCGG</td>
<td>aroE-P2B</td>
<td>CAGCGGTAATCCAGTGCGAC</td>
</tr>

<tr>
<th scope="row"><em>fumC</em></th>
<td>fumC-P1B</td>
<td>TCCCCGCCGTAAAAGCCCTG</td>
<td>fumC-P2B</td>
<td>GCCCGTCAGCAAGCCCAAC</td>
</tr>

<tr>
<th scope="row"><em>gdh</em></th>
<td>gdh-P1B</td>
<td>CTGCCCCCGGGGTTTTCATCT</td>
<td>gdh-P2B</td>
<td>TGTTGCGCGTTATTTCAAAGAAGG</td>
</tr>

<tr>
<th scope="row"><em>pdhC</em></th>
<td>pdhC-P2B</td>
<td>CCGGCCGTACGACGCTGAAC</td>
<td>pdhC-P2B</td>
<td>GATGTCGGAATGGGGCAAACA</td>
</tr>

<tr>
<th scope="row"><em>pgm</em></th>
<td>pgm-P1</td>
<td>CTTCAAAGCCTACGACATCCG</td>
<td>pgm-P2</td>
<td>CGGATTGCTTTCGATGACGGC</td>
</tr>

</tbody></table>

<a id="table5" name="table5"> </a>
<p><span class="bold">Table 5.</span> MLST amplification primers for <em>H. influenzae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene locus</th>
<th scope="col">Primer name</th>
<th scope="col">Forward primer (5'-3')</th>
<th scope="col">Primer name</th>
<th scope="col">Reverse primer (5'-3')</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>adk</em></th>
<td>adk-up</td>
<td>GGTGCACCGGGTGCAGGTAA</td>
<td>adk-dn</td>
<td>CCTAAAGATTTTATCTAACTC</td>
</tr>

<tr>
<th scope="row"><em>atpG</em></th>
<td>atpG-up</td>
<td>ATGGCAGGTGCAAAAGAGAT</td>
<td>atpG-dn</td>
<td>TTGTACAACAGGCTTTTGCG</td>
</tr>

<tr>
<th scope="row"><em>frdB</em></th>
<td>frdB-up</td>
<td>CTTATCGTTGGTCTTGCCGT</td>
<td>frdB-dn</td>
<td>TTGGCACTTTCCACTTTTCC</td>
</tr>

<tr>
<th scope="row"><em>fucK</em></th>
<td>fucK-up</td>
<td>ACCACTTTCGGCGTGGATGG</td>
<td>fucK-dn</td>
<td>AAGATTTCCCAGGTGCCAGA</td>
</tr>

<tr>
<th scope="row"><em>mdh</em></th>
<td>mdh-up</td>
<td>TCATTGTATGATATTGCCCC</td>
<td>mdh-dn</td>
<td>ACTTCTGTACCTGCATTTTG</td>
</tr>

<tr>
<th scope="row"><em>pgi</em></th>
<td>pgi-up</td>
<td>GGTGAAAAAATCAATCGTAC</td>
<td>pgi-dn</td>
<td>ATTGAAAGACCAATAGCTGA</td>
</tr>

<tr>
<th scope="row"><em>recA</em></th>
<td>recA-up</td>
<td>ATGGCAACTCAAGAAGAAAA</td>
<td>recA-dn</td>
<td>TTACCAAACATCACGCCTAT</td>
</tr>

</tbody></table>

<a id="table6" name="table6"> </a>
<p><span class="bold">Table 6.</span> MLST amplification primers for <em>S. pneumoniae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene locus</th>
<th scope="col">Primer name</th>
<th scope="col">Forward primer (5'-3')<sup>1,2</sup></th>
<th scope="col">Primer name</th>
<th scope="col">Reverse primer (5'-3')<sup>1,2</sup></th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>aroE3</em></th>
<td>aroE-fwd</td>
<td>TCCTATTAAGCATTCTATTT<br />
CTCCCTTC</td>
<td>aroE-rev</td>
<td>ACAGGAGAGGATTGGCCATCCA<br />
TGCCCACACTG</td>
</tr>

<tr>
<th scope="row"><em>gdh4</em></th>
<td>gdh-up</td>
<td>ATGGACAAACCAGCNAGYTT</td>
<td>gdh-dn</td>
<td>GCTTGAGGTCCCATRCTNCC</td>
</tr>

<tr>
<th scope="row"><em>gki4</em></th>
<td>gki-up</td>
<td>GGCATTGGAATGGGATCACC</td>
<td>gki-dn</td>
<td>TCTCCCGCAGCTGACAC</td>
</tr>

<tr>
<th scope="row"><em>recP3</em></th>
<td>recP-fwd</td>
<td>GAATGTGTGATTCAATAATCACCTC<br />
AAATAGAAGG</td>
<td>recP-rev</td>
<td>TGCTGTTTCGATAGCAGCA<br />
TGGATGGCTTCC</td>
</tr>

<tr>
<th scope="row"><em>spi3</em></th>
<td>spi-fwd</td>
<td>CGCTTAGAAAGGTAAGTTA<br />
TGAATTT</td>
<td>spi-rev</td>
<td>GAAGAGGCTGAGATTGGTG<br />
ATTCTCGGCC</td>
</tr>

<tr>
<th scope="row"><em>xpt3</em></th>
<td>xpt-fwd</td>
<td>TTAACTTTTAGACTTTAGGA<br />
GGTCTTATG</td>
<td>xpt-rev</td>
<td>CGGCTGCTTGCGAGTGTT<br />
TTTCTTGAG</td>
</tr>

<tr>
<th scope="row"><em>ddl3</em></th>
<td>ddl-fwd</td>
<td>TAAAATCACGACTAAGCGTGTTCTGG</td>
<td>ddl-rev</td>
<td>AAGTAGTGGGTACATAGA<br />
CCACTGGG</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty"><sup>1</sup>IUPAC nucleotide designations: R = A or G; Y = C or T; N = A, T, G, or C.<br />
<sup>2</sup>In this instance, R = equal mixture of A and G; W = equal mixture of A and T; etc.<br />
<sup>3</sup> The primers for aroE, recP, spi, xpt, and ddl listed at <a href="http://spneumoniae.mlst.net/misc/info.asp#experimental">http://spneumoniae.mlst.net/misc/info.asp#experimental</a> are too close to the target sequences for accurate sequencing results using a capillary sequencer, thus alternative primers that lie about 40 bases further upstream and downstream of the target sequences are listed above. See also <a href="https://www.cdc.gov/streplab/alt-MLST-primers.html">primers on CDC site</a>.<br />
<sup>4</sup>Primer sequence taken from: <a href="http://spneumoniae.mlst.net/misc/info.asp#experimental">http://spneumoniae.mlst.net/misc/info.asp#experimental</a>.</p>
</li>

<li><span class="bold">Primers used for PCR amplification</span>
<p>The primers used for PCR amplification for each scheme are shown in <a href="chpt12-molecular-typing-methods.html#table4">Table 4</a> (N. meningitidis), <a href="chpt12-molecular-typing-methods.html#table5">Table 5</a> (H. influenzae), and <a href="chpt12-molecular-typing-methods.html#table6">Table 6</a> (S. pneumoniae). In addition, the primers currently in use are listed in the MLST web pages for the schemes, <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> look under "information", and <a href="http://haemophilus.mlst.net/">http://haemophilus.mlst.net/</a>, look under "Organism Specific Information" and go to page 2. Note that for S. pneumoniae, the primers listed are from two different sources. The original MLST primers for S. pneumoniae can be found at <a href="http://spneumoniae.mlst.net/misc/info.asp#experimental">http://spneumoniae.mlst.net/misc/info.asp#experimental</a>. However, it has been found that these primers for aroE, recP, spi, xpt, and ddl are too close to the target sequences for accurate sequencing results using a capillary sequencer, thus <a href="https://www.cdc.gov/streplab/alt-MLST-primers.html">alternative primers</a> that lie about 40 bases further upstream and downstream of the target sequences were designed.</p>
</li>

<li><span class="bold">PCR reaction setup and cycling conditions</span>
<p>The PCR conditions to be used for each scheme are given below (<a href="chpt12-molecular-typing-methods.html#table7">Tables 7</a> and <a href="chpt12-molecular-typing-methods.html#table8">8</a>). PCR set up and cycling conditions for each assay can also be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> look under "information", and <a href="http://haemophilus.mlst.net/">http://haemophilus.mlst.net/</a> or <a href="http://spneumoniae.mlst.net/">http://spneumoniae.mlst.net/</a>, look under "Organism Specific Information".</p>

<p>Reactions are performed in 50 µl volumes for <em>N. meningitidis</em> and <em>H. influenzae</em>, and in 25 µl volumes for <em>S. pneumoniae</em> in either in 8-well tube strips or in 96-well plates. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. When setting up the PCR reactions keep the reagents from warming up to room temperature by keeping them on ice. This is especially important for the <em>Taq</em> DNA polymerase.</p>

<a id="table7" name="table7"> </a>
<p><span class="bold">Table 7.</span> MLST PCR amplification reaction set-up for N. meningitidis and H. influenzae</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)*</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>40.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>5.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 mM forward primer</th>
<td>1.0</td>
<td>0.4 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 mM reverse primer</th>
<td>1.0</td>
<td>0.4 µM final concentration</td>
</tr>

<tr>
<th scope="row">DNA polymerase</th>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td>50.0</td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty">* Some laboratories scale back the reaction volumes to 25 µl. Adjust each component of the reaction accordingly if 25 µl volumes are desired.</p>

<a id="table8" name="table8"> </a>
<p><span class="bold">Table 8.</span> MLST PCR amplification reaction set-up for <em>S. pneumoniae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>18.0 - 19.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>2.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>0.5</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 µM forward primer</th>
<td>0.5</td>
<td>0.2 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 µM reverse primer</th>
<td>0.5</td>
<td>0.2 µM final concentration</td>
</tr>

<tr>
<th scope="row">DNA polymerase</th>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1 – 2.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td>25.0</td>
<td>&#160;</td>
</tr>

</tbody></table>

<p>Once the PCR reactions are set-up, immediately place them in the PCR machine and run. The cycling conditions for MLST differ slightly for each organism (<a href="chpt12-molecular-typing-methods.html#table9">Table 9</a>, <em>N. meningitidis</em>, except for <em>pgm</em>; <a href="chpt12-molecular-typing-methods.html#table10">Table 10</a>, <em>N. meningitidis, pgm</em>; <a href="chpt12-molecular-typing-methods.html#table11">Table 11</a>, <em>H. influenzae</em>; and <a href="chpt12-molecular-typing-methods.html#table12">Table 12</a>, <em>S. pneumoniae</em>). Note that the annealing temperature of the reactions may need to be optimized when adapting the protocols for use in your laboratory.</p>

<a id="table9" name="table9"> </a>
<p><span class="bold">Table 9.</span> PCR cycling conditions for <em>N. meningitidis</em> MLST amplifications, except for <em>pgm</em></p>

<table class="opt-in table table-bordered">
<tr>
<td>1x (94°C, 4 min)</td>
</tr>

<tr>
<td>35x (94°C, 1 min); (55°C, 1 min); (72°C, 1 min)</td>
</tr>

<tr>
<td>1x (72°C, 5 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>
</table>

<a id="table10" name="table10"> </a>
<p><span class="bold">Table 10.</span> PCR cycling conditions for <em>pgm</em> gene for <em>N. meningitidis</em> MLST amplifications</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (95°C, 5 min)</td>
</tr>

<tr>
<td>30x (94°C, 1 min); (65°C, 1 min)*; (72°C, 1 min)<br />
*Decrease the annealing temperature 0.5°C per cycle</td>
</tr>

<tr>
<td>10x (94°C, 1 min); (50°C, 1 min); (72°C, 2 min)</td>
</tr>

<tr>
<td>1x (72°C, 5 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

<tr>
<td>1x (72°C, 2 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>

<a id="table11" name="table11"> </a>
<p><span class="bold">Table 11.</span> PCR cycling conditions for <em>H. influenzae</em> MLST amplifications</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (95°C, 4 min)</td>
</tr>

<tr>
<td>30x (95°C, 30 sec); (55°C, 30 sec)*; (72°C, 1 min)<br />
*Use 50°C for <em>adk</em> and <em>frdB</em> genes</td>
</tr>

<tr>
<td>1x (72°C, 10 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>

<a id="table12" name="table12"> </a>
<p><span class="bold">Table 12.</span> PCR cycling conditions for <em>S. pneumoniae</em> MLST amplifications</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (94°C, 5 min)</td>
</tr>

<tr>
<td>10x (94°C, 15 sec); (54°C, 30 sec); (72°C, 45 sec)</td>
</tr>

<tr>
<td>20x (94°C, 15 sec); (54°C, 30 sec); (72°C, 45 sec)*<br />
*Add 10 sec to the extension per cycle</td>
</tr>

<tr>
<td>1x (72°C, 10 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Analysis of PCR products on an agarose gel</span>
<p>It is useful to check for a successful PCR amplification before moving onto reaction clean-up and sequencing. The protocol for analyzing the amplified PCR products can be found in Appendix 2 at the end of this chapter.</p>
</li>

<li><span class="bold">Nucleotide sequencing of MLST PCR products</span>
<p>To perform DNA nucleotide sequencing, the DNA amplicons must be purified by the method of choice before sequencing reactions can be performed. These include gel filtration columns, a solid phase reversible immobilization magnetic bead-based system, or PEG<sub>8000</sub>/2.5M NaCl precipitation. Commercial kits for gel filtration columns are available from several companies and a protocol for PEG precipitation can be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "PCR protocol".</p>

<p>The PCR primers and protocols included here are those used at CDC and are optimized for these laboratories. Other protocols work well and it is up to the discretion of each laboratory to discern the optimal reagents and assay conditions for their laboratory, including appropriate validation and quality control. For a commonly used protocol, see: <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "sequencing protocol (microtiter plates)".</p>

<p>Sequencing reactions for all assays described here should be carried out on both strands. Sequencing each strand provides greater confidence in the base calls versus having base calls made using a single data point.</p>
</li>

<li>
<p><span class="bold">Sequencing primers for <em>N. meningitidis, H. influenzae</em>, and <em>S. pneumoniae</em> MLST</span></p>

<p>For <em>H. influenzae</em> and <em>S. pneumoniae</em>, the same primers used for PCR amplification are used for sequencing reactions. The sequencing primers used for <em>N. meningitidis</em> MLST are in <a href="chpt12-molecular-typing-methods.html#table13">Table 13</a>.</p>

<a id="table13" name="table13"> </a>
<p><span class="bold">Table 13.</span> MLST sequencing primers for <em>N. meningitidis</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene locus</th>
<th scope="col">Primer name</th>
<th scope="col">Forward primer (5'-3')<sup>1,2</sup></th>
<th scope="col">Primer name</th>
<th scope="col">Reverse primer (5'-3')<sup>1,2</sup></th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>abcZ</em></th>
<td>abcZ-S1A</td>
<td>AATCGTTTATGTACCGCAGR</td>
<td>abcZ-S2</td>
<td>GAGAACGAGCCGGGATAGGA</td>
</tr>

<tr>
<th scope="row"><em>adk</em></th>
<td>adk-S1A</td>
<td>AGGCWGGCACGCCCTTGG</td>
<td>adk-S2</td>
<td>CAATACTTCGGCTTTCACGG</td>
</tr>

<tr>
<th scope="row"><em>aroE</em></th>
<td>aroE-S1A</td>
<td>TCGGTCAAYACGCTGRTK</td>
<td>aroE-S2</td>
<td>ATGATGTTGCCGTACACATA</td>
</tr>

<tr>
<th scope="row"><em>fumC</em></th>
<td>fumC-S1</td>
<td>TCCGGCTTGCCGTTTGTCAG</td>
<td>fumC-S2</td>
<td>TTGTAGGCGGTTTTGGCGAC</td>
</tr>

<tr>
<th scope="row"><em>gdh</em></th>
<td>gdh-S3</td>
<td>CCTTGGCAAAGAAAGCCTGC</td>
<td>gdh-S4C</td>
<td>RCGCACGGATTCATRYGG</td>
</tr>

<tr>
<th scope="row"><em>pdhC</em></th>
<td>pdhC-S1</td>
<td>TCTACTACATCACCCTGATG</td>
<td>pdhC-S2</td>
<td>ATCGGCTTTGATGCCGTATTT</td>
</tr>

<tr>
<th scope="row"><em>pgm</em></th>
<td>pgm-S1</td>
<td>CGGCGATGCCGACCGCTTGG</td>
<td>pgm-S2A</td>
<td>GGTGATGATTTCGGTYGCRCC</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty"><sup>1</sup>IUPAC nucleotide designations: R = A or G; W = T or A; K = T or G; Y = C or T; N = A, T, G, or C.<br />
<sup>2</sup>In this instance, R = equal mixture of A and G; W = equal mixture of A and T; etc.<br />
&#160;</p>
</li>

<li><span class="bold">Sequencing PCR setup and cycling conditions</span>
<p>Reactions are performed in 20 µl volumes either in 8-well tube strips or in 96-well plates. Some laboratories perform sequencing reactions in 10 µl volumes. Adjust each component of the reaction accordingly if 10 µl volumes are desired. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. The setup in <a href="chpt12-molecular-typing-methods.html#table14">Table 14</a> will work for sequencing MLST amplicons from <em>N. meningitidis, H. influenzae</em>, and <em>S. pneumoniae</em>.</p>

<a id="table14" name="table14"> </a>
<p><span class="bold">Table 14.</span> MLST sequencing PCR reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>11.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5X buffer</th>
<td>4.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Terminator nucleotides and polymerase*</th>
<td>1.0</td>
<td>200 mM final concentration</td>
</tr>

<tr>
<th scope="row">3.2 mM primer</th>
<td>1.0</td>
<td>0.16 mM final concentration</td>
</tr>

<tr>
<th scope="row">Purified amplicon DNA</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<p><span class="note">*Note:</span> chemistries can vary, thus optimize the reactions for your particular system. Primer concentrations and PCR set up parameters may vary depending on the type of enzyme, chemistry, and protocols used in individual laboratories. Each laboratory should optimize the protocols.</p>

<p>Once the reactions are setup, follow the cycling conditions in <a href="chpt12-molecular-typing-methods.html#table15">Table 15</a>.</p>

<a id="table15" name="table15"> </a>
<p><span class="bold">Table 15.</span> Cycling conditions for sequencing PCR</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>25x (95°C, 10 sec); (52°C, 5 sec); (60°C, 4 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Purification of the sequencing reaction products</span>
<p>Before the reaction products can be resolved on an automated DNA sequencer, the products must be purified to remove unincorporated fluorescent dyes, buffer, and unused deoxyribonucleotide triphosphates. This can be achieved using a variety of commercially available kits that utilize gel filtration or a solid phase reversible immobilization magnetic bead-based system to purify the products. Conversely, the reaction products can be purified by precipitation with ethanol and 3M sodium acetate, pH 4.6 (see <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> and click on "information" then "sequencing protocol (microtiter plates)". Typically, the reaction products will be purified and dried down in a 96-well plate and reconstituted with formamide, EDTA, or water before electrophoresis. Each laboratory should follow the manufacturer’s protocol for the automatic sequencer system being employed.</p>

<p>See the Analysis of Sequence Data and Allele Determination section below for further information about data analysis.</p>
</li>
</ol>
</li>

<li><span class="bold"><em>porA</em> and <em>porB</em> typing</span>
<ol type="A">
<li><span class="bold">Overview</span>
<p>The antigenic diversity expressed by the five major classes of outer membrane proteins (OMP) on the surface of <em>N. meningitidis</em> have been utilized to develop isolate subtyping and characterization schemes and to develop vaccines. The largest of the OMPs expressed by most meningococcal isolates is the PorA or class 1 protein. The PorA protein is a transmembrane protein predicted to have 8 loops exposed on the surface of the organism. These loops contain two hypervariable regions, VR1 (loop 1) and VR2 (loop 4), and two semivariable regions, SV1 (loop 5) and SV2 (loop 6). Sequence analysis of SV1 and SV2 demonstrated that sequence differences in these regions were too limited to generate subtype differences between strains, thus PorA typing is based on the VR1 and VR2 hypervariable sequences, which have greater resolution (<a href="chpt12-molecular-typing-methods.html#ref16">16</a>, <a href="chpt12-molecular-typing-methods.html#ref19">19</a>).</p>

<p>Additionally, all <em>N. meningitidis</em> express PorB which is expressed as 2 alternate alleles that were previously called class 2 and 3 proteins. These have been named PorB2 and PorB3, respectively, and are mutually exclusive, though hybrids do exist. PorB has four hypervariable loop regions referred to as VR1, on loop I; VR2, on loop V; VR3, on loop VI; and VR4, on loop VII (<a href="chpt12-molecular-typing-methods.html#ref32">32</a>). Historically, serotyping is based on the reactivity of specific antisera to the variable regions of PorB. The antigenically important variable epitopes that are targeted by serological typing reagents reside in the surface-exposed loops of PorA and PorB (<a href="chpt12-molecular-typing-methods.html#ref29">29</a>, <a href="chpt12-molecular-typing-methods.html#ref31">31</a>). In addition, these regions are targeted by the host immune response and are thus under intensive selective pressure. Subtyping isolates based on sequencing of the <em>porA</em> and <em>porB</em> genes and translation of the DNA into their respective amino acids for typing has been helpful in characterizing isolates in clusters or outbreaks and for determining differences between isolates. Note that the <em>porA</em> gene is deleted in some isolates, though this is a rare occurrence (<a href="chpt12-molecular-typing-methods.html#ref32">32</a>).</p>
</li>

<li><span class="bold">Primers used for PCR amplification</span>
<p>The primers used for PCR amplification of the <em>porA</em> and <em>porB</em> genes of <em>N. meningitidis</em> are shown in <a href="chpt12-molecular-typing-methods.html#table16">Table 16</a>. Primer set 1 should be used first when amplifying the <em>porA</em> gene. If PCR amplification fails using primer set 1; alternative primer sets 2 and/or 3 should be used.</p>
</li>

<li><span class="bold">PCR reaction setup and cycling conditions</span>
<p>Reactions are performed in 50 µl volumes either in 8-well tube strips or in 96-well plates (<a href="chpt12-molecular-typing-methods.html#table17">Table 17</a>). If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA.</p>

<a id="table16" name="table16"> </a>
<p><span class="bold">Table 16.</span> PCR primers for <em>porA</em> and <em>porB</em> amplification</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene</th>
<th scope="col">Set</th>
<th scope="col">Forward primer 5'-3'</th>
<th scope="col">Reverse primer 5'-3'</th>
<th scope="col"><sup>1</sup>GenBank Accession #</th>
<th scope="col"><sup>2</sup>Amplicon size (nts)</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>porA</em></th>
<td>1</td>
<td>P14 GGGTGTTTGCCCG<br />
ATGTTTTTAGG</td>
<td>P22 TTAGAATTTGTGGCG<br />
CAAACCGAC</td>
<td>X12899</td>
<td>1236</td>
</tr>

<tr>
<th scope="row"><em>porA</em></th>
<td>23</td>
<td>P21 CTGTACGGCGAAATCA<br />
AAGCCGGCGT</td>
<td>P22 TTAGAATTTGTGGC<br />
GCAAACCGAC</td>
<td>EF564254</td>
<td>1115</td>
</tr>

<tr>
<th scope="row"><em>porA</em></th>
<td>33</td>
<td>U23 GTGTTTGCCCGATGT<br />
TTTTAGGT</td>
<td>L24 TGCTGTCTTTATTGCC<br />
GTTTTTCT</td>
<td>X12899</td>
<td>1368</td>
</tr>

<tr>
<th scope="row"><em>porB</em></th>
<td>1</td>
<td>PB-A1 TAAATGCAAAGCTA<br />
AGCGGCTTG</td>
<td>PB-A2 TTTGTTGATACCAA<br />
TCTTTTCAG</td>
<td>EU301792</td>
<td>1755</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty"><sup>1</sup>The GenBank Accession number given may not be that of the strain actually used to design the PCR primers, but it is a strain that contains the primer sequences.<br />
<sup>2</sup>These are the predicted amplicon sizes for the strain with the given GenBank Accession number. Because of the variable nature of these genes, the amplicon in other strains may differ from the size listed. porA is deleted in some strains, though deletions are rare.<br />
<sup>3</sup>Primers sets 2 and 3 are alternate primer sets that can be used when the porA gene does not amplify with primer set 1.</p>

<a id="table17" name="table17"> </a>
<p><span class="bold">Table 17.</span> <em>porA</em> and <em>porB</em> PCR amplification reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)*</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>40.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>5.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 µM forward primer</th>
<td>1.0</td>
<td>0.4 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 µM reverse primer</th>
<td>1.0</td>
<td>0.4 µM final concentration</td>
</tr>

<tr>
<th scope="row">DNA polymerase</th>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty"><sup>*</sup>Some laboratories scale back the reaction volumes to 25 µl. Adjust each component of the reaction accordingly if 25 µl volumes are desired.</p>

<p>Once the PCR reactions are set-up, immediately place them in the PCR machine and run. The cycling conditions for <em>porA</em> and <em>porB</em> differ slightly for each assay (<a href="chpt12-molecular-typing-methods.html#table18">Table 18</a> for <em>porA</em> and <a href="chpt12-molecular-typing-methods.html#table19">Table 19</a> for <em>porB</em>). Note that the annealing temperature of the reactions may need to be optimized when adapting the protocols for use in your laboratory.</p>

<a id="table18" name="table18"> </a>
<p><span class="bold">Table 18.</span> Cycling conditions for <em>porA</em> amplification</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (95°C, 5 min)</td>
</tr>

<tr>
<td>30x (95°C, 1 min); (60°C, 30 sec); (72°C, 2 min)</td>
</tr>

<tr>
<td>1x (72°C, 5 min)</td>
</tr>

<tr>
<td>4°C, ∞</td>
</tr>

</tbody></table>

<a id="table19" name="table19"> </a>
<p><span class="bold">Table 19.</span> Cycling conditions for <em>porB</em> amplification</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (94°C, 5 min)</td>
</tr>

<tr>
<td>35x (94°C, 1 min); (60°C, 30 sec); (72°C, 1 min)</td>
</tr>

<tr>
<td>1x (72°C, 5 min)</td>
</tr>

<tr>
<td>4°C, ∞</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Analysis of PCR products on an agarose gel</span>
<p>It is useful to check for a successful PCR amplification before moving onto reaction clean-up and sequencing. The protocol analyzing the amplified PCR products can be found in Appendix 2 at the end of this chapter.</p>
</li>

<li><span class="bold">Nucleotide sequencing of</span> <em><span class="bold">porA</span></em> <span class="bold">and</span> <em><span class="bold">porB</span></em> <span class="bold">PCR products</span>
<p>To perform DNA nucleotide sequencing, the DNA amplicons must be purified by the method of choice before sequencing reactions can be performed. These include gel filtration columns, a solid phase reversible immobilization magnetic bead-based system, or PEG8000/2.5M NaCl precipitation. Commercial kits for gel filtration columns are available from several companies and a protocol for PEG precipitation can be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "PCR protocol".</p>

<p>The PCR primers and protocols included here are those used at CDC and are optimized for these laboratories. Other protocols work well and it is up to the discretion of each laboratory to discern the optimal reagents and assay conditions for their laboratory, including appropriate validation and quality control. For a commonly used protocol, see: <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "sequencing protocol (microtiter plates)".</p>

<p>Sequencing reactions for all assays described here should be carried out on both strands. Sequencing each strand provides greater confidence in the base calls versus having base calls made using a single data point.</p>
</li>

<li><span class="bold">Sequencing primers for <em>porA</em> and <em>porB</em> typing</span>
<p>The primers used to sequence the <em>porA</em> (<a href="chpt12-molecular-typing-methods.html#table20">Table 20</a>) and <em>porB</em> (<a href="chpt12-molecular-typing-methods.html#table21">Table 21</a>) amplicons are shown. VR1 of <em>porA</em> can be determined by using primers U86 and R435 and VR2 can be determined by using F435 and R773. To determine the class of the PorB protein, all seven of the primers listed in the chart for <em>porB</em> must be used.</p>

<a id="table20" name="table20"> </a>
<p><span class="bold">Table 20.</span> Sequencing primers for <em>porA</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene</th>
<th scope="col">Forward primer 5'-3'</th>
<th scope="col">Reverse primer 5'-3'</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>porA</em><br />
VR1</th>
<td>U86<br />
GCCCTCGTATTGTCCGCACTG</td>
<td>R435<br />
TTGCTGTCCCAAGGATCAATGGC</td>
</tr>

<tr>
<th scope="row"><em>porA</em><br />
VR2</th>
<td>F435<br />
GCCATTAATCCTTGGGACAGCAA</td>
<td>R773<br />
GGCATAGTTCCCGGCAAAACCGCCAT</td>
</tr>

</tbody></table>

<a id="table21" name="table21"> </a>
<p><span class="bold">Table 21.</span> Sequencing primers for <em>porB</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene</th>
<th scope="col">Forward primer 5'-3'</th>
<th scope="col">Reverse primer 5'-3'</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>porB</em></th>
<td>PB-S1<br />
GCAGCCCTTCCTGTTGCAGC</td>
<td>PB-S2<br />
TTGCAGATTAGAATTTTGTG</td>
</tr>

<tr>
<th scope="row"><em>porB</em></th>
<td>8U<br />
TCCGTACGCTACGATTCTCC</td>
<td>8L<br />
GGAGAATCGTAGCGTACGGA</td>
</tr>

<tr>
<th scope="row"><em>porB</em></th>
<td>244U<br />
CGCCCCGCGTTTCTTACG</td>
<td>244L<br />
CGTAAGAAACGCGGGGCG</td>
</tr>

<tr>
<th scope="row"><em>porB</em></th>
<td>&#160;</td>
<td>PB260<br />
AGTGCGTTTGGAGAAGTCGT</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Sequencing PCR setup and cycling conditions</span>
<p>Reactions are performed in 20 µl volumes either in 8-well tube strips or in 96-well plates. Some laboratories perform sequencing reactions in 10 µl volumes. Adjust each component of the reaction accordingly if 10 µl volumes are desired. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. The sequencing PCR setup and cycling conditions are shown below in <a href="chpt12-molecular-typing-methods.html#table22">Tables 22</a> and <a href="chpt12-molecular-typing-methods.html#table23">23</a>, respectively.</p>

<a id="table22" name="table22"> </a>
<p><span class="bold">Table 22.</span> <em>porA</em> and <em>porB</em> sequencing PCR reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>11.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5X buffer</th>
<td>4.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Terminator nucleotides and polymerase*</th>
<td>1.0</td>
<td>200 mM final concentration</td>
</tr>

<tr>
<th scope="row">3.2 mM primer</th>
<td>1.0</td>
<td>0.16 mM final concentration</td>
</tr>

<tr>
<th scope="row">Purified amplicon DNA</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<p><span class="note">*Note:</span> chemistries can vary, thus optimize the reactions for your particular system. Primer concentrations and PCR set up parameters may vary depending on the type of enzyme, chemistry, and protocols used in individual laboratories. Each laboratory should optimize the protocols.</p>

<p>Once the reactions are setup, follow the cycling conditions in <a href="chpt12-molecular-typing-methods.html#table23">Table 23</a>.</p>

<a id="table23" name="table23"> </a>
<p><span class="bold">Table 23.</span> Cycling conditions for sequencing PCR</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>25x (95°C, 10 sec); (52°C, 5 sec); (60°C, 4 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Purification of the sequencing reaction products</span>
<p>Before the reaction products can be resolved on an automated DNA sequencer, the products must be purified to remove unincorporated fluorescent dyes, buffer, and unused deoxyribonucleotide triphosphates. This can be achieved using a variety of commercially available kits that utilize gel filtration or a solid phase reversible immobilization magnetic bead-based system to purify the products. Conversely, the reaction products can be purified by precipitation with ethanol and 3M sodium acetate, pH 4.6 (see <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> and click on "information" and "sequencing protocol (microtiter plates)". Typically, the reaction products will be purified and dried down in a 96-well plate and reconstituted with formamide, EDTA, or water before electrophoresis. Each laboratory should follow the manufacturer’s protocol for the automatic sequencer system being employed.</p>

<p>See the Analysis of Sequence Data and Allele Determination section below for further information about data analysis.</p>
</li>
</ol>
</li>

<li><span class="bold"><em>fetA</em> typing</span>
<ol type="A">
<li><span class="bold">Overview</span>
<p>FetA (ferric enterobactin transport), formerly FrpB (iron-repressed protein B) is a 76 kDa iron-regulated OMP that is expressed to a high level during iron limitation (<a href="chpt12-molecular-typing-methods.html#ref6">6</a>). Modeling of the structure of FetA predicts 13 surface-exposed loops. Anti-FetA antibodies to the most variable of these loops have bactericidal properties (<a href="chpt12-molecular-typing-methods.html#ref29">29</a>). FetA has been proposed as a potential vaccine candidate. However, this region is highly variable which limits the potential of this antigen to be the sole component of a vaccine. Obtaining the amino acid sequence of this variable region by sequencing the DNA of this region of the <em>fetA</em> gene is useful in characterizing clones emerging or circulating in local populations (<a href="chpt12-molecular-typing-methods.html#ref28">28</a>). Note that the <em>fetA</em> gene is deleted in some isolates, though this is a rare occurrence (<a href="chpt12-molecular-typing-methods.html#ref3">3</a>, <a href="chpt12-molecular-typing-methods.html#ref17">17</a>).</p>
</li>

<li><span class="bold">Primers used for PCR amplification</span>
<p>The primers used for PCR amplification of the fetA gene of N. meningitidis are shown in <a href="chpt12-molecular-typing-methods.html#table24">Table 24</a>.</p>
</li>

<li><span class="bold">PCR reaction setup and cycling conditions</span>
<p>Reactions are performed in 50 µl volumes either in 8-well tube strips or in 96-well plates (Table 25). If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA.</p>

<p>Once the PCR reactions are set-up, immediately place them in the PCR machine and run. The cycling conditions for <em>fetA</em> are shown in <a href="chpt12-molecular-typing-methods.html#table26">Table 26</a>. Note that the annealing temperature of the reactions may need to be optimized when adapting the protocols for use in your laboratory.</p>
</li>

<li><span class="bold">Analysis of PCR products on an agarose gel</span>
<p>It is useful to check for a successful PCR amplification before moving onto reaction clean-up and sequencing. The protocol analyzing the amplified PCR products can be found in Appendix 2 at the end of this chapter.</p>
</li>

<li><span class="bold">Nucleotide sequencing of <em>fetA</em> PCR products</span>
<p>To perform DNA nucleotide sequencing, the DNA amplicons must be purified by the method of choice before sequencing reactions can be performed. These include gel filtration columns, a solid phase reversible immobilization magnetic bead-based system, or PEG8000/2.5M NaCl precipitation. Commercial kits for gel filtration columns are available from several companies and a protocol for PEG precipitation can be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "PCR protocol".</p>

<p>The PCR primers and protocols included here are those used at CDC and are optimized for these laboratories. Other protocols work well and it is up to the discretion of each laboratory to discern the optimal reagents and assay conditions for their laboratory, including appropriate validation and quality control. For a commonly used protocol, see: <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "sequencing protocol (microtiter plates)".</p>

<p>Sequencing reactions for all assays described here should be carried out on both strands. Sequencing each strand provides greater confidence in the base calls versus having base calls made using a single data point.</p>

<a id="table24" name="table24"> </a>
<p><span class="bold">Table 24.</span> PCR primers for <em>fetA</em> amplification</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene</th>
<th scope="col">Forward primer 5'-3'</th>
<th scope="col">Reverse primer 5'-3'</th>
<th scope="col"><sup>1</sup>GenBank Accession #</th>
<th scope="col"><sup>2</sup>Amplicon size (nts)</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>fetA</em></th>
<td>S1 CGGCGCAAGCGTATTCGG</td>
<td>S8 CGCGCCCAATTCGTAACCGTG</td>
<td>AF439258</td>
<td>1189</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty"><sup>1</sup>The GenBank Accession number given may not be that of the strain actually used to design the PCR primers, but it is a strain that contains the primer sequences.<br />
<sup>2</sup>These are the predicted amplicon sizes for the strain with the given GenBank Accession number. Because of the variable nature of these genes, the amplicon in other strains may differ from the size listed. fetA is deleted in some strains, though it is rare (<a href="chpt12-molecular-typing-methods.html#ref3">3</a>, <a href="chpt12-molecular-typing-methods.html#ref17">17</a>).</p>

<a id="table25" name="table25"> </a>
<p><span class="bold">Table 25.</span> <em>fetA</em> PCR amplification reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)*</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>40.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>5.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 µM S1</th>
<td>1.0</td>
<td>0.4 µM final concentration</td>
</tr>

<tr>
<th scope="row">20 µM S8</th>
<td>1.0</td>
<td>0.4 µM final concentration</td>
</tr>

<tr>
<th scope="row">DNA polymerase</th>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">50.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty">* Some laboratories scale back the reaction volumes to 25 µl. Adjust each component of the reaction accordingly if 25 µl volumes are desired.</p>

<a id="table26" name="table26"> </a>
<p><span class="bold">Table 26.</span> Cycling conditions for <em>fetA</em> amplification</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (95°C, 5 min)</td>
</tr>

<tr>
<td>40x (95°C, 1 min); (55°C, 1 min); (72°C, 2.5 min)</td>
</tr>

<tr>
<td>1x (72°C, 7 min)</td>
</tr>

<tr>
<td>4°C, ∞</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Sequencing primers for fetA typing</span>
<p>The primers used to sequence the <em>fetA</em> amplicon are shown in Table 27.</p>

<a id="table27" name="table27"> </a>
<p><span class="bold">Table 27.</span> PCR primers for <em>fetA</em> amplification</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene</th>
<th scope="col">Forward primer 5'-3'</th>
<th scope="col">Reverse primer 5'-3'</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>fetA</em></th>
<td>S12 TTCAACTTCGACAGCCGCCTT</td>
<td>S15 TTGCAGCGCGTCR*TACAGGCG</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Sequencing PCR setup and cycling conditions</span>
<p>Reactions are performed in 20 µl volumes either in 8-well tube strips or in 96-well plates. Some laboratories perform sequencing reactions in 10 µl volumes. Adjust each component of the reaction accordingly if 10 µl volumes are desired. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. The sequencing PCR setup and cycling conditions are shown below in Tables 28 and 29, respectively.</p>

<a id="table28" name="table28"> </a>
<p><span class="bold">Table 28.</span> <em>fetA</em> sequencing PCR reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>11.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5X buffer</th>
<td>4.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Terminator nucleotides and polymerase*</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">3.2 µM primer</th>
<td>1.0</td>
<td>0.16 µM final concentration</td>
</tr>

<tr>
<th scope="row">Purified amplicon DNA</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty">* Note: chemistries can vary, thus optimize the reactions for your particular system. Primer concentrations and PCR set up parameters may vary depending on the type of enzyme, chemistry, and protocols used in individual laboratories. Each laboratory should optimize the protocols.</p>

<p>Once the reactions are setup, follow the cycling conditions in <a href="chpt12-molecular-typing-methods.html#table29">Table 29</a>.</p>

<a id="table29" name="table29"> </a>
<p><span class="bold">Table 29.</span> Cycling conditions for sequencing PCR</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>25x (95°C, 10 sec); (52°C, 5 sec); (60°C, 4 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Purification of the sequencing reaction products</span>
<p>Before the reaction products can be resolved on an automated DNA sequencer, the products must be purified to remove unincorporated fluorescent dyes, buffer, and unused deoxyribonucleotide triphosphates. This can be achieved using a variety of commercially available kits that utilize gel filtration or a solid phase reversible immobilization magnetic bead-based system to purify the products. Conversely, the reaction products can be purified by precipitation with ethanol and 3M sodium acetate, pH 4.6 (see <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> and click on "information" then "sequencing protocol (microtiter plates)". Typically, the reaction products will be purified and dried down in a 96-well plate and reconstituted with formamide, EDTA, or water before electrophoresis. It is recommended that each laboratory follow the manufacturer’s protocol for the automatic sequencer system being employed.</p>

<p>See the Analysis of Sequence Data and Allele Determination section below for further information about data analysis.</p>
</li>
</ol>
</li>

<li><em>fHbp</em> typing

<ol type="A">
<li><span class="bold">Overview</span>
<p>Factor H binding protein (fHbp), also referred to as Genome-derived Neisserial Antigen 1870 (GNA1870) or lipoprotein 2086 (LP2086 protein), is a ~28 KD surface exposed protein that binds to human factor H, a negative regulator of the alternative pathway of complement activation. fHbp is widely distributed in <em>N. meningitidis</em>. Recruitment of factor H to the surface of <em>N. meningitidis</em> facilitates bacterial escape from the host innate immune system and promotes bacterial survival in the host (<a href="chpt12-molecular-typing-methods.html#ref23">23</a>). fHbp also induces bactericidal activity against <em>N.</em> <em>meningitidis</em> strains expressing this protein; thus it is a viable vaccine candidate protein and component vaccines including fHbp are currently under evaluation (<a href="chpt12-molecular-typing-methods.html#ref9">9</a>, <a href="chpt12-molecular-typing-methods.html#ref22">22</a>). Analysis of the amino acid sequences from the mature form of fHbp, which does not include the leader peptide, reveals two distinct groups, subfamily A and B (<a href="chpt12-molecular-typing-methods.html#ref22">22</a>). However, DNA sequence analysis demonstrates three variant groups: group 1 (corresponding to subfamily B) and group 2 and 3 (together corresponding to subfamily A) (<a href="chpt12-molecular-typing-methods.html#ref18">18</a>). The data collected to date indicate some correlation of fHbp alleles with MLST or serogroups (<a href="chpt12-molecular-typing-methods.html#ref18">18</a>). Continuing determination of fHbp sequence diversity provides valuable information for the evaluation of potential vaccine efficacy and coverage as well as characterizing circulating populations of <em>N. meningitidis</em>.</p>
</li>

<li><span class="bold">Primers used for PCR amplification</span>
<p>The primers used for PCR amplification of the <em>fHbp</em> gene of <em>N. meningitidis</em> are shown in <a href="chpt12-molecular-typing-methods.html#table30">Table 30</a>.</p>
</li>

<li><span class="bold">PCR reaction setup and cycling conditions</span>
<p>Reactions are performed in 50 µl volumes either in 8-well tube strips or in 96-well plates (<a href="chpt12-molecular-typing-methods.html#table31">Table 31</a>). If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA.</p>

<p>Once the PCR reactions are set-up, immediately place them in the PCR machine and run. The cycling conditions for <em>fHbp</em> are shown in <a href="chpt12-molecular-typing-methods.html#table32">Table 32</a>. Note that the annealing temperature of the reactions may need to be optimized when adapting the protocols for use in your laboratory.</p>

<a id="table30" name="table30"> </a>
<p><span class="bold">Table 30.</span> PCR primers for <em>fHbp</em> amplification</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Gene</th>
<th scope="col">Forward primer 5'-3'</th>
<th scope="col">Reverse primer 5'-3'</th>
<th scope="col"><sup>1</sup>GenBank Accession #</th>
<th scope="col"><sup>2</sup>Amplicon size (nts)</th>
</tr>

</thead><tbody>
<tr>
<th scope="row"><em>fHbp</em></th>
<td>CDC3UNI<br />
GTCCGAACGGTAAATTATYGTG</td>
<td>CDC5UNI<br />
CTATTCTGVGTATGACTAG</td>
<td>FM999788</td>
<td>895</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty"><sup>1</sup>The GenBank Accession number given may not be that of the strain actually used to design the PCR primers, but it is a strain that contains the primer sequences.<br />
<sup>2</sup>These are the predicted amplicon sizes for the strain with the given GenBank Accession number. Because of the variable nature of these genes, the amplicon in other strains may differ from the size listed.<br />
<sup>3</sup>IUCAC designations: Y = C or T; V = C, G, or T.</p>

<a id="table31" name="table31"> </a>
<p><span class="bold">Table 31.</span> <em>fHbp</em> PCR amplification reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>31.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>5.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM CDC3UNI</th>
<td>4.0</td>
<td>0.8 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM CDC5UNI</th>
<td>6.0</td>
<td>1.2 µM final concentration</td>
</tr>

<tr>
<th scope="row">DNA polymerase</th>
<td>0.5</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>2.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">50.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty">* Some laboratories scale back the reaction volumes to 25 µl. Adjust each component of the reaction accordingly if 25 µl volumes are desired.</p>

<a id="table32" name="table32"> </a>
<p><span class="bold">Table 32.</span> Cycling conditions for <em>fHbp</em> amplification</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (94°C, 5 min)</td>
</tr>

<tr>
<td>30x (95°C, 15 sec); (50°C, 15 sec); (72°C, 1.5 min)</td>
</tr>

<tr>
<td>1x (72°C, 5 min)</td>
</tr>

<tr>
<td>4°C, ∞</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Analysis of PCR products on an agarose gel</span>
<p>It is useful to check for a successful PCR amplification before moving onto reaction clean-up and sequencing. The protocol analyzing the amplified PCR products can be found in Appendix 2 at the end of this chapter.</p>
</li>

<li><span class="bold">Nucleotide sequencing of <em>fHbp</em> PCR products</span>
<p>To perform DNA nucleotide sequencing, the DNA amplicons must be purified by the method of choice before sequencing reactions can be performed. These include gel filtration columns, a solid phase reversible immobilization magnetic bead-based system, or PEG8000/2.5M NaCl precipitation. Commercial kits for gel filtration columns are available from several companies and a protocol for PEG precipitation can be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "PCR protocol".</p>

<p>The PCR primers and protocols included here are those used at CDC and are optimized for these laboratories. Other protocols work well and it is up to the discretion of each laboratory to discern the optimal reagents and assay conditions for their laboratory, including appropriate validation and quality control. For a commonly used protocol, see: <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "sequencing protocol (microtiter plates)".</p>

<p>Sequencing reactions for all assays described here should be carried out on both strands. Sequencing each strand provides greater confidence in the base calls versus having base calls made using a single data point.</p>
</li>

<li><span class="bold">Sequencing primers for <em>fHbp</em> typing</span>
<p>The primers used to sequence the <em>fHbp</em> amplicon are the same as those used for amplification, but should be used at a concentration of 3.2 µM.</p>
</li>

<li><span class="bold">Sequencing PCR setup and cycling conditions</span>
<p>Reactions are performed in 20 µl volumes either in 8-well tube strips or in 96-well plates. However, some laboratories perform sequencing reactions in 10 µl volumes. Adjust each component of the reaction accordingly if 10 µl volumes are desired. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. The sequencing PCR setup and cycling conditions are shown below in <a href="chpt12-molecular-typing-methods.html#table33">Tables 33</a> and <a href="chpt12-molecular-typing-methods.html#table34">34</a>, respectively.</p>

<a id="table33" name="table33"> </a>
<p><span class="bold">Table 33.</span> <em>fHbp</em> sequencing PCR reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>11.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5X buffer</th>
<td>4.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Terminator nucleotides and polymerase*</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">3.2 µM primer</th>
<td>1.0</td>
<td>0.16 µM final concentration</td>
</tr>

<tr>
<th scope="row">Purified amplicon DNA</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnotes</h3>

<p class="margin-left-twenty">* Note: chemistries can vary, thus optimize the reactions for your particular system. Primer concentrations and PCR set up parameters may vary depending on the type of enzyme, chemistry, and protocols used in individual laboratories. Each laboratory should optimize the protocols.</p>

<p>Once the reactions are setup, follow the cycling conditions in <a href="chpt12-molecular-typing-methods.html#table34">Table 34</a>.</p>

<a id="table34" name="table34"> </a>
<p><span class="bold">Table 34.</span> Cycling conditions for <em>fHbp</em> amplification</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>25x (95°C, 10 sec); (52°C, 5 sec); (60°C, 4 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Purification of the sequencing reaction products</span>
<p>Before the reaction products can be resolved on an automated DNA sequencer, the products must be purified to remove unincorporated fluorescent dyes, buffer, and unused deoxyribonucleotide triphosphates. This can be achieved using a variety of commercially available kits that utilize gel filtration or a solid phase reversible immobilization magnetic bead-based system to purify the products. Conversely, the reaction products can be purified by precipitation with ethanol and 3M sodium acetate, pH 4.6 (see <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> and click on "information" and "sequencing protocol (microtiter plates)". Typically, the reaction products will be purified and dried down in a 96-well plate and reconstituted with formamide, EDTA, or water before electrophoresis. It is recommended that each laboratory follow the manufacturer’s protocol for the automatic sequencer system being employed.</p>

<p>See the Analysis of Sequence Data and Allele Determination section below for further information about data analysis.</p>
</li>
</ol>
</li>

<li><span class="bold">Penicillin-binding proteins (PBPs)</span>
<p>Alterations in PBPs are the major mechanism of resistance to penicillins and cephalosporins in <em>S. pneumoniae</em>. These bacteria possess several high-molecular-weight PBPs, and most decreased susceptibility and resistance is associated with alterations in PBP1a, PBP2b, and PBP2x. Remarkably, the vast majority of naturally occurring isolates displaying penicillin MICs <u>&#62;</u> 0.25 ug/ml are characterized by carrying mosaic alleles of these genes that contain segments of closely related non-pneumococcal species. Sequence differences in the genes encoding these PBPs have been exploited to subtype β-lactam-resistant <em>S. pneumoniae</em> isolates. These methods include restriction fragment length polymorphisms (RFLPs) analysis of PCR products amplified from <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x, or comparisons of sequences of the amplified products to determine amino acid changes in these genes (<a href="chpt12-molecular-typing-methods.html#ref5">5</a>, <a href="chpt12-molecular-typing-methods.html#ref21">21</a>, <a href="chpt12-molecular-typing-methods.html#ref26">26</a>)</p>

<ol type="A">
<li><span class="bold">Primers used for PCR amplification</span>
<p>PCR reactions for <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x are carried out in 3 separate reactions using the primers listed in <a href="chpt12-molecular-typing-methods.html#table35">Table 35</a>.</p>

<a id="table35" name="table35"> </a>
<p><span class="bold">Table 35.</span> Primers for amplification and sequencing of <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x</p>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th id="t1c1">Primer</th>
<th id="t1c2">Sequence (5'-3')</th>
<th id="t1c3">Gene</th>
<th id="t1c4">Product size</th>
<th id="t1c5">Reference</th>
</tr>

</thead><tbody>
<tr>
<th id="t1c1r2">pn1af</th>
<td headers="t1c1r2 t1c2">GGC ATT CGA TTT GAT TCG CTT CTA TCA T</td>
<td headers="t1c1r2 t1c1r3 t1c3" rowspan="2"><em>pbp</em>1a</td>
<td headers="t1c1r2 t1c1r3 t1c4" rowspan="2">AF439258AF439258</td>
<td headers="t1c1r2 t1c1r3 t1c5" rowspan="2">(8)</td>
</tr>

<tr>
<th id="t1c1r3">pn1ar</th>
<td headers="t1c1r3 t1c2">CTG AGA AGA TGT CTT CTC AGG CTT TTG</td>
</tr>

<tr>
<th id="t1c1r4">1a-S1</th>
<td headers="t1c1r4 t1c2">AAG CTC AAA AAC ATC TGT GGG</td>
<td headers="t1c1r4 t1c1r5 t1c1r6 t1c3" rowspan="3"><em>pbp</em>1a</td>
<td headers="t1c1r4 t1c1r5 t1c1r6 t1c4" rowspan="3">Sequencing</td>
<td headers="t1c1r4 t1c1r5 t1c1r6 t1c5" rowspan="3">(2)</td>
</tr>

<tr>
<th id="t1c1r5">1a-S2</th>
<td headers="t1c1r5 t1c2">TAC TCC ACT CTA CAA CTG GG</td>
</tr>

<tr>
<th id="t1c1r6">1a-S3</th>
<td headers="t1c1r6 t1c2">CCA ACA AAC ATT TCA TCT GGA GC</td>
</tr>

<tr>
<th id="t1c1r7">pbp2bf</th>
<td headers="t1c1r7 t1c2">GAT CCT CTA AAT GAT TCT CAG GTG GCT GT</td>
<td headers="t1c1r7 t1c1r8 t1c3" rowspan="2"><em>pbp</em>2b</td>
<td headers="t1c1r7 t1c1r8 t1c4" rowspan="2">1.5kb</td>
<td headers="t1c1r7 t1c1r8 t1c5" rowspan="2">(8)</td>
</tr>

<tr>
<th id="t1c1r8">pbp2bR</th>
<td headers="t1c1r8 t1c2">GTC AAT TAG CTT AGC AAT AGG TGT TGG AT</td>
</tr>

<tr>
<th id="t1c1r9">2b-S1</th>
<td headers="t1c1r9 t1c2">TTG CTG AAA AGT TAT TTC AAT TC</td>
<td headers="t1c1r9 t1c1r10 t1c3" rowspan="2"><em>pbp</em>2b</td>
<td headers="t1c1r9 t1c1r10 t1c4" rowspan="2">Sequencing</td>
<td headers="t1c1r9 t1c1r10 t1c5" rowspan="2">(2)</td>
</tr>

<tr>
<th id="t1c1r10">2b-S2</th>
<td headers="t1c1r10 t1c2">ATT GTC TTC CAA GGT TCA GCT</td>
</tr>

<tr>
<th id="t1c1r11">pbp2xf</th>
<td headers="t1c1r11 t1c2">CGT GGG ACT ATT TAT GAC CGA AAT GGA G</td>
<td headers="t1c1r11 t1c1r12 t1c3" rowspan="2"><em>pbp</em>2x</td>
<td headers="t1c1r11 t1c1r12 t1c4" rowspan="2">AF439258AF439258</td>
<td headers="t1c1r11 t1c1r12 t1c5" rowspan="2">11891189</td>
</tr>

<tr>
<th id="t1c1r12">pbp2xr2</th>
<td headers="t1c1r12 t1c2">GGC GAA TTC CAG CAC TGA TGG AAA TAA</td>
</tr>

<tr>
<th id="t1c1r13">2x-S1</th>
<td headers="t1c1r13 t1c2">GGA ACA GAA CAA GTT TCC CAA C</td>
<td headers="t1c1r13 t1c1r14 t1c1r15 t1c3" rowspan="3"><em>pbp</em>2x</td>
<td headers="t1c1r13 t1c1r14 t1c1r15 t1c4" rowspan="3">Sequencing</td>
<td headers="t1c1r13 t1c1r14 t1c1r15 t1c5" rowspan="3">(2)</td>
</tr>

<tr>
<th id="t1c1r14">2x-S2</th>
<td headers="t1c1r14 t1c2">GAT GCC ACG ATT CGA GAT TGG G</td>
</tr>

<tr>
<th id="t1c1r15">2x-S3</th>
<td headers="t1c1r15 t1c2">TTT ACA GCT ATT GCT ATT GAT GG</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">CR reaction setup and cycling conditions</span>
<p>PCR reactions are set up as shown in <a href="chpt12-molecular-typing-methods.html#table36">Table 36</a>.</p>

<a id="table36" name="table36"> </a>
<p><span class="bold">Table 36.</span> <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x PCR amplification reaction set-up for <em>S. pneumoniae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>22.45</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>0.75</td>
<td>250 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM forward primer</th>
<td>0.9</td>
<td>0.3 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM reverse primer</th>
<td>0.9</td>
<td>0.3 µM final concentration</td>
</tr>

<tr>
<th scope="row"><em>Taq</em> DNA polymerase</th>
<td>1.0</td>
<td>1 unit Taq polymerase</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<p>Once the PCR reactions are set-up, immediately place them in the PCR machine and run. The cycling conditions are shown in <a href="chpt12-molecular-typing-methods.html#table37">Table 37</a>. Note that the annealing temperature of the reactions may need to be optimized when adapting the protocols for use in your laboratory.</p>

<a id="table37" name="table37"> </a>
<p><span class="bold">Table 37.</span> Cycling conditions for <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x PCR amplification</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>1x (93°C, 5 min)</td>
</tr>

<tr>
<td>30x (93°C, 1 min); (52°C, 1 min); (72°C, 2 min)</td>
</tr>

<tr>
<td>1x (72°C, 2 min)</td>
</tr>

<tr>
<td>4°C, ∞</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Analysis of PCR products on an agarose gel</span>
<p>To check for successful PCR amplification, run 5 ml of the end-products on a 1% agarose gel (see protocol in Appendix 2 at the end of this chapter). Store the remainder at -20°C or proceed to RFLP analyses and/or PCR cleanup and DNA sequencing.</p>
</li>

<li><span class="bold">RFLP analyses</span>
<p>PBP-gene amplicons are subjected to <em>Hae</em>III plus <em>Rsa</em>I digestion by the addition of 3U of the respective enzymes to 5 µl of unpurified PCR product, followed by 1-4 hr of incubation at 37°C (<a href="chpt12-molecular-typing-methods.html#ref8">8</a>). Fragments are then separated on a 2% agarose gel (see Appendix 2 at the end of the chapter, but add 2 g of agarose to 100 ml sterile distilled H<sub>2</sub>O instead of 1 g) and should include a DNA ladder (1kb) on each gel to allow for comparison of fingerprints. After electrophoresis, the gel is visualized under UV and the image captured using a camera.</p>
</li>

<li><span class="bold">DNA sequencing</span>
<p>Sequencing of PCR products can also be setup to determine specific DNA sequences and amino acid changes that may be present in <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2xgenes. To perform DNA nucleotide sequencing, the DNA amplicons must be purified by the method of choice before sequencing reactions can be performed. These include gel filtration columns, a solid phase reversible immobilization magnetic bead-based system, or PEG<sub>8000</sub>/2.5M NaCl precipitation. Commercial kits for gel filtration columns are available from several companies and a protocol for PEG precipitation can be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "PCR protocol". Various chemistries and platforms are available for sequencing and many laboratories outsource their sequencing to commercial companies.</p>
</li>

<li>
<p><span class="bold">Sequencing primers for <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x</span></p>

<p>The primers used to sequence the <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x amplicons are the same as those used for amplification of the genes as well as the additional primers listed in <a href="chpt12-molecular-typing-methods.html#table35">Table 35</a>, but should be used at a final concentration of 0.3 µM.</p>
</li>

<li><span class="bold">Sequencing PCR setup and cycling conditions</span>
<p>Reactions are performed in 20 µl volumes either in 8-well tube strips or in 96-well plates. However, some laboratories perform sequencing reactions in 10 ml volumes. Adjust each component of the reaction accordingly if 10 µl volumes are desired. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. The sequencing PCR setup and cycling conditions are shown below in <a href="chpt12-molecular-typing-methods.html#table38">Tables 38</a> and <a href="chpt12-molecular-typing-methods.html#table39">39</a>, respectively.</p>

<a id="table38" name="table38"> </a>
<p><span class="bold">Table 38.</span> <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x sequencing PCR reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>&#160;</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5X buffer</th>
<td>4.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Terminator nucleotides and polymerase*</th>
<td>1.0</td>
<td>200 mM final concentration</td>
</tr>

<tr>
<th scope="row">3.2 mM primer</th>
<td>1.0</td>
<td>0.3 mM final concentration</td>
</tr>

<tr>
<th scope="row">Purified amplicon DNA</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*Note: chemistries can vary, thus optimize the reactions for your particular system. Primer concentrations and PCR set up parameters may vary depending on the type of enzyme, chemistry, and protocols used in individual laboratories. Each laboratory should optimize the protocols.</p>

<p>Once the reactions are setup, follow the cycling conditions in <a href="chpt12-molecular-typing-methods.html#table39">Table 39</a>.</p>

<a id="table39" name="table39"> </a>
<p><span class="bold">Table 39.</span> Cycling conditions for sequencing PCR</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>25x (95°C, 10 sec); (52°C, 5 sec); (60°C, 4 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Interpretation</span>
<p>After PCR and RFLP analysis the patterns can be visually compared or compared by using an RFLP analyses program to determine clusters of related <em>pbp</em> patterns. For sequence analysis of the <em>pbp</em> genes, the nucleotide and derived amino acid sequence data for strains are compared to the corresponding sequence data for the β-lactam susceptible laboratory isolate R6 (sequence available at GenBank accession numbers: <em>pbp1a</em> M90527; <em>pbp2b</em> X16022; <em>pbp2x</em> X16367) using DNA sequence alignment software. Mutations at positions in or close to each of the three (SXXK, SXN, and KXG) conserved motifs for PBP2b, PBP2x and PBP1a can then be identified and compared.</p>
</li>
</ol>
</li>

<li><span class="bold">Pneumococcal surface protein A (PspA)</span>
<p>PspA is a surface protein and virulence factor found on all isolates of <em>S. pneumoniae</em> and is highly immunogenic. The <em>pspA</em> gene is variable at the nucleotide level, and the amino acid similarity of the surface-exposed N-terminal region of PspA proteins can be as little as 40%. Based on nucleotide and amino acid identity, <em>pspA</em> genes and encoded PspA proteins are classified together into six clades, subdivided into three families: family 1 (clades 1 and 2), family 2 (clades 3, 4, and 5), and the rarely observed family 3 (clade 6) (11). The extent of cross-reactivity of PspA clades follows roughly the degree of amino acid sequence homology and is maximal within strains of the same PspA family. The family can be recognized serologically, but the clade must be identified by the sequence. Studies have demonstrated that PspA family and clade distribution are independent of serotype, age, and clinical origin of the isolates, but are highly associated with genotype as determined by PFGE and/or MLST (33).</p>

<p>A PCR assay has been developed using primers specific for families 1, 2, and 3. Most isolates are either in family 1 or 2, thus two PCR reactions using primers family1- and 2-specific should be performed on each isolate. If both of these are negative, a PCR reaction using the primers specific for the rarely seen family 3 should be performed. Clade determination requires obtaining the DNA sequence of the allele, thus another PCR amplification reaction is required using primers that will amplify all three families.</p>

<ol type="A">
<li><span class="bold">Primers used for PCR amplification</span>
<p>Primers specific for family 1 are LSM12 and SKH63 and for family 2 are LSM12 and SKH52 (<a href="chpt12-molecular-typing-methods.html#ref33">33</a>). A test for PspA family 3 can also be run using primers SKH41 and SKH42 (<a href="chpt12-molecular-typing-methods.html#ref10">10</a>) (<a href="chpt12-molecular-typing-methods.html#table40">Table 40</a>). Isolates that are negative in all PCR reactions are classified as nontypeable (<a href="chpt12-molecular-typing-methods.html#ref10">10</a>).</p>

<p>If the isolate is positive for one of the families, the clade in which the isolate belongs can be identified only through obtaining the DNA sequence of the allele. To do this, perform another PCR amplification using primers LSM12 and SKH2, which will amplify all 3 families. This amplicon will then need to be sequenced (see below).</p>

<a id="table40" name="table40"> </a>
<p><span class="bold">Table 40.</span> Primers for amplification of <em>psp</em>A families<sup>1</sup></p>

<table class="opt-in table table-bordered"><thead class="primary-bg-color">
<tr>
<th id="t2c1">Primer</th>
<th id="t2c2">Sequence (5'-3')</th>
<th id="t2c3">Reference</th>
</tr>

</thead><tbody>
<tr>
<th id="t2c1r2">LSM12</th>
<td headers="t2c1r2 t2c2">CCGGATCCAGCGTCGCTATCTTAGGGGCTGGTT</td>
<td headers="t2c1r2 t2c1r3 t2c1r4 t2c1r5" rowspan="4">(33)</td>
</tr>

<tr>
<th id="t2c1r3">SKH63</th>
<td headers="t2c1r3 t2c2">TTTCTGGCTCATYAACTGCTTTC</td>
</tr>

<tr>
<th id="t2c1r4">SKH52</th>
<td headers="t2c1r4 t2c2">TGGGGGTGGAGTTTCTTCTTCATCT</td>
</tr>

<tr>
<th id="t2c1r5">SKH2</th>
<td headers="t2c1r5 t2c2">CCACATACCGTTTTCTTGTTTCCAGCC</td>
</tr>

<tr>
<th id="t2c1r6">SKH41</th>
<td headers="t2c1r6 t2c2">CGCACAGACTTAACAGATGAAC</td>
<td headers="t2c1r6 t2c1r7 t2c3" rowspan="2">(10)</td>
</tr>

<tr>
<th id="t2c1r7">SKH42</th>
<td headers="t2c1r7 t2c2">CTTGTCCATCAACTTCATCC</td>
</tr>

</tbody></table>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty"><sup>1</sup>IUPAC designations: Y = C.</p>
</li>

<li><span class="bold">PCR reaction setup and cycling conditions</span>
<p>PCR reactions are set up as shown in <a href="chpt12-molecular-typing-methods.html#table41">Table 41</a>.</p>

<a id="table41" name="table41"> </a>
<p><span class="bold">Table 41.</span> <em>pspA</em> PCR amplification reaction set-up for <em>S. pneumoniae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>21.45</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>0.75</td>
<td>250 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM forward primer</th>
<td>0.9</td>
<td>0.3 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM reverse primer</th>
<td>0.9</td>
<td>0.3 µM final concentration</td>
</tr>

<tr>
<th scope="row"><em>Taq</em> DNA polymerase</th>
<td>2.0</td>
<td>2 units Taq polymerase</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">30.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<p>Once the PCR reactions are set-up, immediately place them in the PCR machine and run. The cycling conditions are shown in Table 42. Note that the annealing temperature of the reactions may need to be optimized when adapting the protocols for use in your laboratory.</p>

<a id="table42" name="table42"> </a>
<p><span class="bold">Table 42.</span> <em>pspA</em> PCR amplification reaction set-up for <em>S. pneumoniae</em></p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>21.45</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10X buffer</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">10 mM dNTPs</th>
<td>0.75</td>
<td>250 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM forward primer</th>
<td>0.9</td>
<td>0.3 µM final concentration</td>
</tr>

<tr>
<th scope="row">10 µM reverse primer</th>
<td>0.9</td>
<td>0.3 µM final concentration</td>
</tr>

<tr>
<th scope="row"><em>Taq</em> DNA polymerase</th>
<td>2.0</td>
<td>2 units Taq polymerase</td>
</tr>

<tr>
<th scope="row">Template DNA</th>
<td>1.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">30.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">Analysis of PCR products on an agarose gel</span>
<p>To check for successful PCR amplification and to determine to which family the allele belongs, run 5 µl of the end-products on a 1% agarose gel (see protocol in Appendix 2 at the end of the chapter). The remainder should be stored at -20°C or PCR cleanup and DNA sequencing can be performed immediately after successful PCR amplification has been confirmed.</p>
</li>

<li><span class="bold">Interpretation</span>
<p>The family 1-, 2- and 3-specific primers will produce PCR products that are approximately 1,000 bp for family 1, 1,200 bp for family 2 and 770 bp for family 3 and can be used to assign strains into <em>pspA</em> families.</p>
</li>

<li><span class="bold">DNA Sequencing to determine clade</span>
<p>The amplicon derived from the PCR amplification using primers LSM12 and SKH2 can be sequenced to determine the clade in which the isolate belongs. To perform DNA nucleotide sequencing, the DNA amplicons must be purified either by gel filtration columns, a solid phase reversible immobilization magnetic bead-based system, or PEG<sub>8000</sub>/2.5M NaCl precipitation before sequencing reactions can be performed. Commercial kits for gel filtration columns are available from several companies and a protocol for PEG precipitation can be found at <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> under "information" then "PCR protocol". Various chemistries and platforms are available for sequencing and many laboratories outsource their sequencing to commercial companies.</p>
</li>

<li><span class="bold">Sequencing primers</span>
<p>Sequence using the LSM12 and SKH2 primers, but use them at a final concentration of 0.3 µM.</p>
</li>

<li><span class="bold">Sequencing PCR setup and cycling conditions</span>
<p>Reactions are performed in 20 µl volumes either in 8-well tube strips or in 96-well plates. However, some laboratories perform sequencing reactions in 10 µl volumes. Adjust each component of the reaction accordingly if 10 µl volumes are desired. If doing many reactions, it is useful to prepare a master mix of reagents equal to the number of reactions plus one for each locus with all components except for the DNA. The sequencing PCR setup and cycling conditions are shown below in <a href="chpt12-molecular-typing-methods.html#table43">Tables 43</a> and <a href="chpt12-molecular-typing-methods.html#table44">44</a>, respectively.</p>

<a id="table43" name="table43"> </a>
<p><span class="bold">Table 43.</span> Clade sequencing PCR reaction set-up</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Volume (µl)</th>
<th scope="col">Comments</th>
</tr>

</thead><tbody>
<tr>
<th scope="row">PCR grade water</th>
<td>11.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">5X buffer</th>
<td>4.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Terminator nucleotides and polymerase*</th>
<td>1.0</td>
<td>200 µM final concentration</td>
</tr>

<tr>
<th scope="row">100 nM primer</th>
<td>1.0</td>
<td>5 nM final concentration</td>
</tr>

<tr>
<th scope="row">Purified amplicon DNA</th>
<td>3.0</td>
<td>&#160;</td>
</tr>

<tr>
<th scope="row">Final volume</th>
<td><span class="bold">20.0</span></td>
<td>&#160;</td>
</tr>

</tbody></table>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*Note: chemistries can vary, thus optimize the reactions for your particular system. Primer concentrations and PCR set up parameters may vary depending on the type of enzyme, chemistry, and protocols used in individual laboratories. Each laboratory should optimize the protocols.</p>

<p>Once the reactions are setup, follow the cycling conditions in <a href="chpt12-molecular-typing-methods.html#table44">Table 44</a>.</p>

<a id="table44" name="table44"> </a>
<p><span class="bold">Table 44.</span> Cycling conditions for sequencing PCR</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<td>25x (95°C, 10 sec); (52°C, 5 sec); (60°C, 4 min)</td>
</tr>

<tr>
<td>1x (4°C, ∞)</td>
</tr>

</tbody></table>
</li>

<li><span class="bold">. Interpretation</span>
<p>The DNA sequences generated are used to determine the amino acid sequence searched against the sequence database by using BLAST software (<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">www.ncbi.nlm.nih.gov/BLAST</a>). To further classify the strains, the diversity based on <em>pspA</em> clades is determined by comparing the amino acid sequences found for strains tested with those amino acid sequences of the clade-defining region of 24 invasive reference sequences retrieved from GenBank [Accession numbers AF071802 to AF071809 (Clade 1), AF07810 to AF071814, M74122 (Clade 2), AF071816 to AF071818 (Clades 3), AF071821, AF071824, AF071826, U89711 (Clade 4), AF071820 (Clade 5), and AF071823 (Clade 6)] (33). PspA proteins in the same clade share similar sequences and any variation of sequences within clades is restricted to single-amino-acid substitutions. Clade type can also be established by determining clusters using the DNA sequences to generate a dendrogram with software using Pearson’s coefficient and the hierarchical unweighted pair group method (UPGMA). Proteins within the same clade are greater than 90% identical in sequence.</p>
</li>

<li><span class="bold">Quality control</span></li>
</ol>
</li>

<li><span class="bold">Analysis of sequence data and allele determination</span>
<p>After the sequence reaction products have been resolved and the data obtained from the sequencer, the trace files must be analyzed for incorrect base calls and the complementary strands must be aligned. For MLST, the alleles must be aligned and trimmed so that they correspond exactly to the regions that are used to define the alleles. For the OMPs, the sequence is translated so that the allele types or clade determination based on the amino acid sequence can be made. In addition the nucleotide changes in the domains typically responsible for penicillin resistance in the <em>pbp</em>1a, <em>pbp</em>2b and <em>pbp</em>2x genes in <em>S. pneumoniae</em> need to be determined. Various computer packages are suitable for assembling, aligning and editing the trace files from the automated DNA sequencer to create consensus sequence files suitable for these analyses.</p>

<ol type="A">
<li>
<p><span class="bold">MLST sequence analysis</span></p>

<p>Once consensus sequences are available, they can be uploaded to websites for allelic analysis and sequence type designations. For <em>N. meningitidis</em> MLST go to <a href="http://pubmlst.org/neisseria/">http://pubmlst.org/neisseria/</a> and look under "Access main databases", for <em>H. influenzae</em> MLST go to <a href="http://haemophilus.mlst.net/">http://haemophilus.mlst.net/</a> and use "locus query" to obtain allele types and "profile query" to obtain a sequence type, and for <em>S. pneumoniae</em> MLST go to <a href="http://spneumoniae.mlst.net/">http://spneumoniae.mlst.net/</a> and use "locus query" to obtain allele types and "profile query" to obtain sequence types. Consensus sequences of each gene fragment are compared with those in the databases. The software checks that the sequences are the correct length and that they do not contain any undetermined characters. Options are available to identify the allele at a single locus, to enter an allele profile, to find isolates in the database that match or nearly match an allele profile, or to browse the database. Consensus sequences not represented in the database can be submitted as a new allele. The database curator evaluates the trace files of the sequence before assigning a number to the new allele and including it in the database. In addition, the databases have links to various programs for further data analyses.</p>

<p>Freeware programs that provide the capability of assembly, aligning, editing, producing a consensus sequence, and also provide MLST types and clonal complex types in one package are the Sequence Type Analysis and Retrieval System (STARS) and the Meningococcus Genome Informatics Platform (MGIP). In addition, both systems are expandable to include other species and loci. However, STARS is Linux-based, requires technical expertise to program for expansion and no longer has technical support. MGIP is a web-based interface requiring only a computer and an internet connection that allows the user to upload to the website sequence trace files and a worksheet template (<a href="http://mgip.biology.gatech.edu/home.php">http://mgip.biology.gatech.edu/home.php</a>). Analyzed sequence data, including ST and CC designations, will be returned to the user within minutes of submission (<a href="chpt12-molecular-typing-methods.html#ref12">12</a>). Furthermore, MGIP will also perform OMP analysis and is expandable to include any loci the user wishes to add. At this writing, MGIP is only available for <em>N. meningitidis</em> loci, but is being expanded to include <em>S. pneumoniae</em> and <em>H. influenzae</em>.</p>
</li>

<li>
<p><span class="bold">Analyzing a set of sequence data using MGIP</span></p>

<p>MGIP does not require a user to register to use the website. However, if accessing the website functions as a public user any data uploaded is available for anyone to see and download. It is thus advisable to register, which only requires a username, password, first and last name, email address, and institution.</p>

<p>MGIP requires two files to be uploaded in order to analyze data: 1) A zip file or compressed archive of the sequencing trace files; and 2) A spreadsheet template file in the CSV (comma separated values) format that identifies the name of the reaction, the type of assay (MLST, <em>porA</em>, etc), and primer in each well of the sequencing plate. Note that multiple typing schemes can be analyzed on a single plate.</p>

<ol>
<li><span class="bold">Compressing (or zipping) the trace files</span>
<p>Uploading is done by the set, which is all of the trace file data from a 96-well plate. Do not rename the trace files. MGIP depends on the coordinates in each filename to map each trace file back to the spreadsheet. Thus, if you must rename the trace filenames, preserve the coordinates (e.g., instead of the filename 1433_B01.ab1, you can rename the file to 999_aroE_B01.ab1). In order to upload a data set:</p>

<ol style=" list-style: lower-alpha;">
<li>Put all of the sequencing files associated with the data set to be analyzed into a single folder.</li>

<li>Zip the folder. In Windows, right click the folder, select "Send To" and then "Compressed "zipped" folder." (<a href="chpt12-molecular-typing-methods.html#figure1">Figure 1</a>). If the "send to" is not on the menu, then select "add to zip file" or "create archive." Any of these options will create a zip file.</li>
</ol>

<p>There are ways to create zip files in most operating systems such as MacOS or Linux, which are very similar to the method described above. The resulting file must be a zip of the directory of trace files and can have any name.</p>

<div class="span14 fixed"><a id="figure1" name="figure1"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 1 is an image showing how to create a zip file of the sequencing data to be analyzed." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt12-figure01.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 1. Creating a zip file of the sequencing data to be analyzed.</p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>

<li><span class="bold">Creating the template spreadsheet</span>
<p>Each cell in the spreadsheet corresponds to the appropriate well from the sequencing plate. For example, a trace file from a well might be automatically named 1433_<span class="bold">B01</span>.ab1, which corresponds to the first column, second row of your wells. Hypothetically, if the well B01 contains the locus <em>pgm_</em> from the strain M2341 and it was the forward primer, then the correct way to label the cell is M2341.MLST.pgm_.1. <em>The correct way to label each cell in the spreadsheet is</em><span class="bold">strain.sequencingTypingMethod.locus.primer</span>. Any blank well should either be blank or have the word BLANK in the cell. Exact locus names and sequence typing method names can be found on the MGIP upload page at <a href="http://mgip.biology.gatech.edu/uploadTraces.php">http://mgip.biology.gatech.edu/uploadTraces.php</a>.</p>

<p>To generate an entire spreadsheet (which you may have to modify), you can use the form found on the upload page under the Automatic Spreadsheet Generation section. After viewing the resulting table from the Automatic Spreadsheet Generator, you can click the download button to save the spreadsheet. The spreadsheet must be in CSV format, which is available as a "save as..." option in Excel and in other spreadsheet programs. Therefore you can convert any spreadsheet you have to a CSV format by opening it in a spreadsheet program and saving it as a CSV.</p>
</li>

<li><span class="bold">Uploading the files</span>
<p>Upload the zipped sequence files and template spreadsheet at: <a href="http://mgip.biology.gatech.edu/uploadTraces.php">http://mgip.biology.gatech.edu/uploadTraces.php</a>. After uploading, click "Analyze Trace Files".</p>
</li>
</ol>
</li>

<li><span class="bold">Viewing results</span>
<ol>
<li><span class="bold">By set</span>
<p>Once the files are analyzed, the alignments and results can be viewed by clicking on the name of the set of sequences. Or starting from the main page, click "view results" and click on the name of the set of sequences.</p>

<p>From the first box, choose the data set. The sets are named after your original zip file (<a href="chpt12-molecular-typing-methods.html#figure2">Figure 2</a>).</p>

<div class="span8 fixed"><a id="figure2" name="figure2"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 2 is an image showing the selecting of a set of sequence data to view." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt12-figure02.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 2. Selecting a set of sequence data to view.</p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>

<p>An alignment's blast results, fasta format sequence, or the actual trace files that were uploaded can be viewed (<a href="chpt12-molecular-typing-methods.html#figure3">Figure 3</a>). Find the alignment's strain/locus and click the options link to view the aforementioned choices. An editing function is available to adjust incorrect base calls, if necessary. In addition, files that were unable to be fully analyzed and need the attention of the user are marked with a red flag.</p>

<div class="span12 fixed"><a id="figure3" name="figure3"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 3 is an image showing allele calls and options for viewing analyzed data." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt12-figure03.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 3. Allele calls and options for viewing analyzed data.</p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>

<li><span class="bold">By strain</span>
<p>The user can view their strains in a table alongside their alleles. Click the "strain table" link from the main menu. Strains are listed on the left, alleles in the middle, and lastly, the derived Sequence Type is on the right. If enough information is present, then the database will indicate the ST of the isolate. If insufficient data is present to designate a ST, then a list of all possible STs is given (<a href="chpt12-molecular-typing-methods.html#figure4">Figure 4</a>). In the case of a combination of alleles that are not present in the database, a designation of "novel strain" will be displayed.</p>

<div class="span12 fixed"><a id="figure4" name="figure4"> </a>
<div class="module-typeN">
<div class="figure"><img alt="Figure 4 is an image showing sequence types (STs) by strain." class="center" src="https://www.cdc.gov/meningitis/lab-manual/images/chapt12-figure04.gif" />
<div class="figcaption">
<p class="cdc-caption-text">Figure 4. Viewing STs by strain.</p>
</div>
</div>
</div>
</div>

<div style=" clear: both;"></div>
</li>
</ol>
</li>

<li><span class="bold">OMP sequence analysis</span>
<p>See above for assembling, editing, and aligning sequences to produce a consensus sequence. Once the consensus sequences are created, they can be directly uploaded to the <em>Neisseria</em>.org website for typing:</p>

<ol>
<li><span class="bold">PorA</span><br />
<em>Neisseria meningitidis</em> PorA variable region database: <a href="http://pubmlst.org/neisseria/PorA/">http://pubmlst.org/neisseria/PorA/</a>, click on "Single sequence query" or "Batch sequence query" under the heading "Identify PorA variable regions to determine the VR1 and VR2 types.</li>

<li><span class="bold">PorB</span>
<p><em>Neisseria meningitidis</em> PorB typing database: <a href="http://pubmlst.org/neisseria/porB/">http://pubmlst.org/neisseria/porB/</a> click on "<em>porB</em> (NEIS2020) [whole coding region]" or "<em>’porB</em> [partial coding sequence]" under the heading "Identify <em>porB</em> alleles" to type the class and loop regions.</p>
</li>

<li>
<p><span class="bold">FetA</span> <em>Neisseria meningitidis</em> FetA variable region database: <a href="http://pubmlst.org/neisseria/FetA/">http://pubmlst.org/neisseria/FetA/</a>, click on "Single sequence query" or "Batch sequence query" under the heading "Identify FetA variable regions" to determine the FetA type.</p>
</li>

<li>
<p><span class="bold">fHbp</span> Factor H-binding protein database: <a href="http://pubmlst.org/neisseria/fHbp/">http://pubmlst.org/neisseria/fHbp/</a>, click on "Single sequence query" or "Batch sequence query" under the heading "Identify fHbp alleles and peptides" to determine the fHpB type.</p>

<p>Consensus sequences not represented in the database can be submitted as a new allele. The database curator evaluates the traces of the sequence before assigning a number to the new allele and including it in the database. MGIP (see above MLST Analysis section) can also be used to obtain allele types for each of these OMPs.</p>
</li>
</ol>
</li>
</ol>
</li>

<li><span class="bold">Pulsed-field gel electrophoresis</span>
<ol type="A">
<li><span class="bold">Overview</span>
<p>Pulsed-field gel electrophoresis (PFGE) is an agarose gel-based typing method that assesses strain inter-relatedness by comparison of complete genomes and has been applied to at least 40 pathogens (<a href="chpt12-molecular-typing-methods.html#ref25">25</a>). PFGE involves digesting genomic DNA with a restriction enzyme that cleaves chromosomal DNA infrequently to produce a small number (11-41) of fragments of different sizes. The resulting fragments are usually large and unable to be separated efficiently by conventional uni-directional electrical field gel electrophoresis. Two of the most commonly used methods to size-fractionate these large digestion products for PFGE are the contour-clamped homogenous electric field (CHEF) and field inversion gel electrophoresis (FIGE). CHEF uses a hexagonal array of 24 electrodes that produce an alternating 120° angle uniform electrical field. FIGE is based on a conventional electrophoresis in which the electric field is periodically inverted by 180° (<a href="chpt12-molecular-typing-methods.html#ref25">25</a>). In both CHEF and FIGE, the electric field used for PFGE is periodically alternated or pulsed to facilitate the migration of the DNA fragments through the gel. Larger fragments migrate through the gel slower than smaller fragments creating a size-based banding pattern that has larger fragments near the top of the gel and smaller fragments near the bottom of the gel (<a href="chpt12-molecular-typing-methods.html#ref4">4</a>, <a href="chpt12-molecular-typing-methods.html#ref14">14</a>, <a href="chpt12-molecular-typing-methods.html#ref25">25</a>). The resulting banding pattern can be analyzed by visual inspection or by using a computer program to determine differences in the banding patterns between isolates.</p>

<p>PFGE is a highly discriminatory subtyping tool and is particularly suitable for outbreak or cluster investigations. PFGE is used in combination with epidemiologic information to help identify outbreak isolates and to determine the relationships among isolates associated with the outbreak or cluster. The etiological agents in outbreaks are often clonal and produce indistinguishable PFGE patterns, but point mutations and insertions or deletions can occur during an outbreak that lead to a PFGE pattern difference of two or three fragments in isolates linked to the outbreak. Guidelines have been developed to interpret the minor variations in PFGE patterns from closely related strains, but these guidelines do not take into account the total genetic variation observed within the circulating population (<a href="chpt12-molecular-typing-methods.html#ref27">27</a>). Isolates that differ by two or three fragments are still considered epidemiologically linked subtypes of the same strain.</p>
</li>

<li><span class="bold">Preparation for PFGE</span><br />
<p class="bold">Time required for procedure</p>

<p>PFGE requires approximately 28-30 hours once overnight cultures are available, thus time management is important in planning the procedure.</p>

<h5><span class="bold">Equipment</span></h5>

<ul>
<li>Electrophoresis equipment</li>

<li>CHEF system with pump and cooling unit module</li>

<li>Documentation system equipped with a camera that can provide computer compatible images</li>

<li>37°C incubator</li>

<li>37°C water bath</li>

<li>56°C water bath</li>

<li>50°C water bath</li>

<li>Orbital/shaker water bath</li>

<li>Turbidity meter, spectrophotometer, or McFarland standards</li>

<li>Scales/balances to measure solid reagents</li>

<li>Microwave to melt agarose</li>
</ul>

<h5><span class="bold">Select reagents</span></h5>

<ul>
<li>Appropriate agar plates for growing up cultures (see below)</li>

<li>Rapid resolution agarose (rapid resolution of DNA and PCR products between 1 kb and 50 kb by electrophoresis) for making plugs</li>

<li>A serine protease/endopeptidase such as Proteinase K (liquid or powder)</li>

<li>10X 1 M Tris/borate/EDTA, pH 8.0 (TBE)</li>

<li>1 M Tris-HCl, pH 8.0</li>

<li>0.5 M EDTA, pH 8.0</li>

<li>N-Lauroyl sarcosine sodium salt</li>

<li>Ethidium bromide, 10 mg/ml</li>

<li>Restriction enzyme and enzyme-specific buffer</li>

<li>Sterile distilled deionized H<sub>2</sub>O (dd H<sub>2</sub>O)</li>
</ul>

<h5><span class="bold">Supplies</span></h5>

<ul>
<li>PFGE plug molds (reusable or disposable)</li>

<li>Gel comb and holder</li>

<li>Casting frame, platform, and leveling table</li>

<li>Sterile clear polystyrene 12 X 75 mm tubes with caps</li>

<li>Sterile 1.5 ml <span class="bold">microcentrifuge</span> tubes</li>

<li>Sterile 2 ml round bottom tubes</li>

<li>Scalpels or single edge razor blade</li>

<li>Glass slides</li>

<li>Sterile disposable Petri dishes or large glass slides</li>

<li>Flat spatula</li>

<li>Container to stain gel</li>

<li>Sterile screw cap flasks or bottles</li>

<li>Sterile graduated cylinders</li>

<li>Sterile 50 ml screw cap centrifuge tubes</li>

<li>Sterile pipettes</li>

<li>PPE (gloves, eye, and respiratory protection)</li>

<li>Heat-resistant gloves</li>
</ul>
</li>

<li><span class="bold">Reagent and solutions</span>
<p class="bold">Tris-HCl, 1.0 M, pH 8.0 (1 L)</p>

<ol>
<li>Dissolve 121 g Tris base in 800 ml ddH<sub>2</sub>O.</li>

<li>Adjust to pH 8.0 with concentrated HCl.</li>

<li>Mix and add sterile distilled H<sub>2</sub>O to 1 L.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<p class="bold">EDTA, 0.5 M, pH 8.0 (1 L)</p>

<ol>
<li>Dissolve 186 g EDTA in 700 ml ddH<sub>2</sub>O.</li>

<li>Adjust pH to 8.0 with 10 M NaOH (~50 ml).</li>

<li>Add sterile distilled H<sub>2</sub>O to 1 L.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<p class="bold">10% Sodium lauroyl sarcosine (Sarcosine)</p>

<ol>
<li>Add 10 g of N-Lauroyl sarcosine salt to 100 ml ddH<sub>2</sub>O.*</li>

<li>Filter through a .22 micron membrane.</li>
</ol>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*Eye and respiratory protection should be worn when weighing powdered Sarcosine.</p>

<p class="bold">Proteinase K (20 mg/ml)</p>

<ol>
<li>Add 100 mg of Proteinase K powder to 5 ml ddH<sub>2</sub>O.</li>

<li>Filter sterilize, aliquot, and store at -20°C.</li>

<li>Alternatively, 5 ml of a 20 mg/ml solution is available commercially.</li>
</ol>

<p class="bold">Ethidium bromide (EtBr), 10 mg/ml</p>

<ol>
<li>Dissolve 0.2 g ethidium bromide in 20 ml ddH<sub>2</sub>O.</li>

<li>Mix well and store at 4°C in the dark in 1 ml aliquots.</li>

<li>EtBr is a powerful mutagen and should be handled with care.</li>
</ol>

<p class="bold">Cell suspension buffer (100 mM Tris and 100 mM EDTA, pH 8.0)</p>

<ol>
<li>10 ml of 1 M Tris, pH 8.0 (sterile solution, available commercially).</li>

<li>20 ml of 0.5 M EDTA, pH 8.0 (sterile solution, available commercially).</li>

<li>Dilute to 100 ml with ddH<sub>2</sub>O, not tap water.</li>

<li>Can be stored at room temperature (20-25°C) for several months.</li>
</ol>

<p class="bold">Plug wash TE buffer (10 mM Tris; 1 mM EDTA, pH 8.0)</p>

<ol>
<li>10 ml of 1 M Tris, pH 8.0.</li>

<li>2 ml of 0.5M EDTA, pH 8.0.</li>

<li>Dilute to 1000 ml with ddH<sub>2</sub>O, not tap water.</li>

<li>Can be stored at room temperature (20-25°C) for several months.</li>
</ol>

<p class="bold">TBE (Tris/borate/EDTA) electrophoresis buffer, 10X stock solution*</p>

<ol>
<li>To 800 ml of ddH<sub>2</sub>O add:<br />
108 g Tris base (890 mM).<br />
55 g boric acid (890 mM).<br />
40 ml 0.5 M EDTA, pH 8.0 (20mM).</li>

<li>40 ml 0.5 M EDTA, pH 8.0 (20mM).</li>

<li>Add ddH<sub>2</sub>O to 1 L.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*To make 0.5X TBE working solution, add 100 ml 10X TBE to 1.9 L distilled H<sub>2</sub>O.</p>

<p class="bold">Agarose(1.0% rapid resolution agarose)</p>

<ol>
<li>Add 250 mg of rapid resolution agarose to 23.5 ml of 0.5X TBE in a 250 ml flask and microwave for 1 min, swirl, and then microwave in 15 sec increments with swirling until the agarose is fully melted and the solution is clear.</li>

<li>Place in 56°C water bath to keep agarose from hardening.</li>

<li>Agar can be stored at room temperature for several months and can be re-melted and used again.</li>
</ol>

<p class="bold">Cell lysis buffer 50 mM Tris: 50 mM EDTA, pH 8.0 and 1% Sarcosine)</p>

<ul>
<li>Add 25 ml of 1 M Tris, pH 8.0.</li>

<li>Add 50 ml of 0.5 M EDTA, pH 8.0.</li>

<li>Add 50 ml of 10% sodium lauroyl sarcosine (Sarcosine), membrane-filtered.</li>

<li>Dilute to 500 ml with ddH<sub>2</sub>O, not tap water.</li>
</ul>
</li>

<li><span class="bold">Performing PFGE</span>
<p class="bold">Growth of bacteria</p>

A pure culture of each isolate is grown on trypticase soy agar plates supplemented with 5% sheep blood for <em>N. meningitidis</em> and <em>S. pneumoniae</em>, or chocolate agar plates supplemented with hemin and NAD for <em>H. influenzae</em> in a humidified incubator for 18-24 hours at 37°C with 5% CO<sub>2</sub>. If the isolate does not look pure or if the growth is not sufficient, subculture in a humidified incubator for 18-24 hours at 37°C with 5% CO<sub>2</sub>.

<p class="bold">Preparation of gel plugs</p>

<p>Fill water baths to the correct water level with deionized water. Turn on shaking bath at 54°C and non-shaking baths at 56°C and 37°C. Prepare or re-melt plug agarose (see above) and hold at 56°C in the water bath until used.</p>

<ol>
<li>Label and set up a 12 x 75 mm plastic capped tube containing 2 ml of cell suspension buffer (CSB) for each isolate and for a control strain and a size marker.</li>

<li>Using a 1 µl disposable loop, gently harvest enough growth to make a suspension of cells reading 0.48-0.52 using a turbidity meter or spectrophotometer at O.D.<sub>280</sub>. If a turbidity meter or spectrophotometer is not available, this amount of growth is approximately a 0.5 McFarland standard.

<ul>
<li>To make a uniform suspension, rub the growth onto the side-wall of the tube just above the level of the CSB until it washes into the liquid. Before making the turbidity reading, gently mix the tube by finger-tapping until the suspension appears homogeneous within the tube.</li>

<li>The bacterial concentration needs to be adjusted precisely. Achieving uniform concentrations is critical for reproducibility, resolution of the bands of similar size, and for comparison between different strains.</li>
</ul>
</li>

<li>Add 400 µl of each cell suspension to a labeled 1.5 ml microcentrifuge tube. Then add 20 µl of Proteinase K (20 mg/ml) to the side-wall of each tube above the suspension to avoid premature lysis of the cells.</li>

<li>Add 400 µl of melted plug agarose and mix gently by pipetting the mixture up and down several times. Add 400 µl of the mix to fill a well of a 10-well reusable or disposable plug mold. Repeat for each suspension, filling all of the designated wells. Let the plugs harden for 5 min at 4°C or 15 min at room temperature (25°C). Extra plugs can be made from the left-over cell suspensions, if desired.

<ul>
<li>Avoid making bubbles and do not vortex. To prevent hardening of the agarose, keep the flask in a beaker of water at 56°C until all of the plugs are made.</li>
</ul>
</li>

<li>Add 5 ml of cell lysis buffer (CLB) and 133 µl Proteinase K (20 mg/ml) to labeled 50 ml centrifuge tubes.</li>

<li>Open the plug molds and push a plug into each of the designated tubes of CLB and close the caps tightly.</li>

<li>Incubate for 1.5 to 2 hours in a 54°C shaker water bath set at 75 strokes per min.</li>
</ol>

<p class="bold">Washing the plugs</p>

<ol>
<li>Pre-warm 500 ml of sterile reagent grade water and a liter of plug wash TE buffer in a 50°C water bath.</li>

<li>Pour off CLB and add 15 ml sterile distilled water to each plug.</li>

<li>Incubate for 15 min in a 50°C shaker water bath set at 150 strokes per min.</li>

<li>Replace water with 15 ml of plug wash TE buffer and incubate for 20 min in a 50°C shaker water bath set at 150 strokes per min. Decant buffer and repeat 4X.</li>

<li>Store plugs in tubes with 2-5 ml of plug wash TE buffer at 4°C until ready for restriction digestion. Plugs are usable for up to 4 months but it is preferable to use them as soon as possible after the washing step.</li>
</ol>

<p class="bold">Restriction digestion</p>

<p>The restriction enzymes to use are dependent on the type of the bacteria being tested. For <em>N. meningitidis</em> use <em>Nhe</em>I, and for <em>H. influenzae</em> and <em>S. pneumoniae</em> use <em>Sma</em>I. Note that for further resolution, isolates can be cut with an additional enzyme, but do not cut with two enzymes in the same reaction. This is useful if other molecular characterization such as MLST will not be performed. <em>Spe</em>I can also be used for <em>N. meningitidis</em> and <em>Xma</em>I for <em>H. influenzae</em> and <em>S. pneumoniae</em>.</p>

<ol>
<li>Sterilize a single-edge razor blade and a clean 3 x 2 inch glass slide with 70% alcohol.</li>

<li>Add 180 µl of sterile reagent grade water and 20 µl of the 10X restriction bufferfor the enzyme to be used to a 1.5 ml microtube for each isolate.</li>

<li>Using a narrow spatula, remove a plug from the Plug Wash TE buffer and place it on the glass slide.</li>

<li>Using the razor blade cut off and discard any uneven edges that may prevent the plug from fitting in the gel well.</li>

<li>Cut two 1-2 mm thick slices from the plugs, including the standard plug and place them into one of the 1.5 ml microcentrifuge tubes and incubate at 37°C for 15 minutes (30°C for <em>Sma</em>I).

<ul>
<li>The unused portions of the plugs can be returned to their storage tubes and refrigerated.</li>

<li>The overall goal is to cut plug slices that are sufficient in size to easily manipulate, fit in the well, and that contain enough DNA to create a clear, easily readable banding pattern. The size of the band may need to be optimized for the bacteria being tested.</li>
</ul>
</li>

<li>Carefully aspirate the liquid from the tubes with a pipette, taking care not to damage or remove the plug slice.</li>

<li>Replace the liquid with 170 µl of sterile distilled water, 20 µl of 10X restriction buffer, and 50 units of the appropriate enzyme.</li>

<li>This can be prepared as a "master-mix" or each reagent can be added separately with gentle mixing.</li>

<li>Incubate plug slices in a 37°C (30°C for <em>Sma</em>I) water bath for 1.5-3 hours.

<ul>
<li>Note that SmaI loses 50% activity after one hour at 37°C.</li>
</ul>
</li>

<li>Carefully aspirate the liquid from the tubes and add 150 µl of 0.5X TBE buffer.</li>
</ol>

<p class="bold">Gel preparation and loading</p>

<ol>
<li>Make agarose (1.0% rapid resolution agarose) and place in a 56°C water bath until use.

<ul>
<li>Remove 2 ml to a sterile tube and hold at 56°C to use later in the procedure.</li>
</ul>
</li>

<li>Assemble the gel-casting mold and make sure it is level on the leveling stand. Adjust the height of the comb teeth so that, when upright, the teeth touch the gel platform.</li>

<li>Once the comb is in place, add molten agarose to the apparatus until the agarose is nearly to the top of the teeth of the comb. Take care that the agarose is well-mixed to ensure a uniform gel.</li>

<li>Once the gel has hardened, carefully remove the comb.</li>

<li>Lay the comb flat and using a narrow spatula remove the plug slices from each tube and place each one at the bottom of its designated comb tooth and allow plug slices to air-dry at ambient temperature for 15 minutes.

<ul>
<li>Run one plug slice per isolate and save extra plug slices.</li>
</ul>
</li>

<li>Position the comb in the upper pair of slots of the gel casting stand and slowly pour the molten agarose from the flask into the mold and let the gel solidify for 25-30 min.</li>

<li>Remove the comb and seal the wells with the 2 ml of molten agarose set aside.</li>

<li>Transfer the gel on its platform into the chamber, be sure it is positioned properly in its frame and is immersed in the 0.5X TBE buffer. Typically 2 L of buffer is required for the gel to be submersed with 1 cm of buffer over the gel. Close the cover of the chamber and begin the run after setting the following parameters on the power supply:</li>
</ol>

<a id="table45" name="table45"> </a>
<p><span class="bold">Table 45.</span> Electrophoresis parameters for PFGE</p>

<table class="opt-in table table-bordered"><tbody>
<tr>
<th scope="row">Initial switch time</th>
<td>2.2 seconds</td>
</tr>

<tr>
<th scope="row">Final switch time</th>
<td>35 seconds</td>
</tr>

<tr>
<th scope="row">Run time</th>
<td>18 hours</td>
</tr>

<tr>
<th scope="row">Angle</th>
<td>120 degrees</td>
</tr>

<tr>
<th scope="row">Gradient</th>
<td>6.0 volts/cm</td>
</tr>

<tr>
<th scope="row">Temperature</th>
<td>14°C</td>
</tr>

<tr>
<th scope="row">Ramping factor</th>
<td>Linear</td>
</tr>

</tbody></table>

<p class="bold">Staining the gel and documenting the image</p>

<ol>
<li>Turn off the equipment at the end of the run.</li>

<li>Set up a tray containing EtBr staining solution (400 ml of deionized water with 40 µl of [10 mg/ml] EtBr). Note that EtBr is a carcinogen and gloves and eye protection should be used.

<ul>
<li>It is important that the gel box and cooling unit be cleaned after each run. To do this drain the buffer and pour 1 L of distilled H<sub>2</sub>O into the gel box and circulate for 2-3 min. Drain the distilled H<sub>2</sub>O and dry the apparatus.</li>
</ul>
</li>

<li>Remove the gel from the chamber and platform and immerse it in staining solution.</li>

<li>Cover the tray to shield it from light and place it on a rotator or oscillating platform shaker and rotate slowly for 30 min.</li>

<li>Pour off the EtBr staining solution according to safety regulations and destain the gel with 500 ml cold deionized water for 30 min on the rotator. Remove water and repeat 2X.</li>

<li>Transfer the gel to an imaging system to photograph the image under UV light and save it on the computer as a tagged image file format (TIFF) file for further analysis. TIFF is the preferred format because of its greater resolution but the files can be saved as .jpeg if file size is a problem. If an imaging system is unavailable, a photo of the agarose gel can be taken.</li>
</ol>

<p class="bold">Analysis of image and interpretation</p>

<p>Visual analysis can be used to directly compare the band patterns of a limited number of isolates on a gel. However, several computer software programs have been developed to analyze gel images, compare multiple gel images, match banding patterns, construct dendrograms, and store gel data to allow for more accurate and sophisticated data analyses. The banding patterns are analyzed with the Dice coefficient, an optimization of 1.0%, and a position tolerance of 1.5% for the band migration distance. A PFGE-based clonal group is defined as a group of isolates with genetically related PFGE patterns. In general, the PFGE patterns of strains categorized within a clonal group have six or fewer differences from each other and ≥80% genetic relatedness on the dendrogram. When comparing isolates associated with an outbreak, it is helpful to include isolates that do not have an epidemiological link to the outbreak to determine if the outbreak clone is currently circulating within the population or if it has been recently introduced. After results are obtained using the software, it is recommended that the results be visually compared to the bands on the gel to be sure the results make sense.</p>

<p>For <em>S. pneumoniae</em>, comparisons of PFGE profiles to those of major global clones should be made by using the profiles described by the Pneumococcal Molecular Epidemiology Network (PMEN) or reference isolates can be requested from this group to include in PFGE runs. (<a href="http://www.sph.emory.edu/PMEN/pmen_clone_collection.html">http://www.sph.emory.edu/PMEN/pmen_clone_collection.html</a>).</p>
</li>

<li><span class="bold">Technical considerations</span>
<p class="bold">Use of a standard</p>

<p>A standard or molecular weight size marker is used to provide an accurate fragment size estimate and assists with normalization and correction of gel patterns due to variations in electrophoresis. It should be included in each gel run to allow for inter- and intra-gel comparison of isolates. A common commercial marker is the lambda bacteriophage, which consists of concatamers of the bacteriophage lambda DNA. This marker is available in both high molecular weight and low molecular weight varieties and some laboratories use both on a gel. However, inconsistency in the DNA concentration and quality of the commercially available lambda ladders has been observed between vendors and in lot-to-lot variation of the ladder. Alternatively, a bacterial strain standard can be used that produces bands of known molecular weight. The bacterial standard does not necessarily have to be of the same strain being tested. The caveat with using a bacterial strain is that it must be prepared in the lab and mutations or genomic rearrangements can occur to change a cleavage site, thus changing the size of two or more of the bands. Regardless of the type of marker used, 3 lanes of the gel spaced evenly apart, but not on the edge of the gel, should be used for markers.</p>

<p>Care needs to be taken in the interpretation of results, especially in inter-laboratory studies, as small differences in electrophoresis conditions can alter the distance migrated by each band, complicating the comparison between isolates in different gels.</p>

<p class="bold">Quality control</p>

<p>Quality control (QC) is a measure of precision and a way to ensure that test results are correct, consistent and reproducible. Thus, QC for reagents used in a test is critical to the overall result and interpretation of the test. QC measures are listed below:</p>

<ol>
<li>Use clean glassware that is free of detergents to prepare reagents.</li>

<li>Use sterile distilled water, not tap water, to prepare reagents.</li>

<li>Perform regular maintenance on equipment; keep equipment clean; pipettors calibrated; and keep a maintenance/calibration log.</li>

<li>New plugs to be used as standards should be run as an unknown to verify that the new standard produces the same banding pattern and intensity as the old standard.</li>

<li>Test new lots of restriction enzyme to verify that the new enzyme is working properly and produces a consistent pattern.</li>

<li>Sterilize the following reagents by filtration or by autoclaving before use: 10% Sarcosine, 1 M Tris-HCl, pH 8.0; 0.5 M EDTA, pH 8.0; and 10X Tris/Borate/EDTA buffer (TBE).</li>
</ol>

<p class="bold">Troubleshooting</p>

<p>Unexpected results can be attributed to equipment failure, incorrect calculations, and improperly made reagents. When errors arise in an assay, it is helpful to review the equipment, reagents, and steps used in the procedure to look for errors. Listed below are some common problems encountered when performing PFGE with suggestions on how to correct them.</p>

<ol>
<li>No power to equipment:

<ul>
<li>Check that equipment is plugged in and that the fuse in the back of the power supply is functional.</li>
</ul>
</li>

<li>Gel lanes are curved or slanted:

<ul>
<li>Examine and replace any damaged electrodes.</li>

<li>Check chamber for foreign objects and remove them if present.</li>

<li>Be sure agarose plugs are firm enough for loading as fragmented plugs will not run correctly.</li>

<li>Verify that the electrophoresis chamber is level. The gel must be poured level.</li>

<li>Verify that the gel is securely loaded into the gel chamber.</li>
</ul>
</li>

<li>Bands have poor resolution:

<ul>
<li>Fresh buffer should always be used and the buffer tank and circulation lines should be cleaned and flushed after each use.</li>

<li>The level of the buffer used to run the gel could be incorrect or have changed during the run creating an electrical field that is not appropriate; therefore check buffer levels carefully and add or drain buffer as needed.</li>

<li>High buffer concentration elevates the temperature of the run buffer. If this is suspected, remake the running buffer</li>

<li>Check chiller to make sure it is working properly to cool the run buffer.</li>

<li>Remove any kinks in the tubing. This may reduce the flow rate of the pump which would increase the temperature of the buffer.</li>

<li>The percentage of agarose used may have been too low.</li>
</ul>
</li>

<li>Faint bands or no bands on gel:

<ul>
<li>Low amounts of DNA or degraded/sheared DNA in the plugs will not produce good results. If this problem is suspected, remake the plugs.</li>

<li>The bacteria in the plug may not have been completely lysed. Verify that a sufficient amount of Proteinase K was used and that the cell lysis buffer was made properly and used at the correct temperature.</li>

<li>Verify that the gel was stained with EtBr made at the correct concentration.</li>
</ul>
</li>

<li>The gel has unspecific signal in areas where no signal is expected (background):

<ul>
<li>Plugs may not have been washed thoroughly. Rewash the plug and repeat restriction digest.</li>

<li>The DNA concentration in the plug was too high. Use a thinner plug slice or remake the plug with the proper cell concentration.</li>
</ul>
</li>

<li>Faint bands that appear on the gel between normal bands:

<ul>
<li>The agarose plug may not have been properly digested with restriction endonuclease. Digest the plug again making sure that the correct concentration of enzyme is used for the proper length of time, that the proper enzyme buffer was used, and that the plug slice is completely submerged during digestion.</li>
</ul>
</li>

<li>Specks appear in stained gel.

<ul>
<li>Verify that the gel does not contain undissolved agarose. If so, remake the gel with agarose completely dissolved and thoroughly mixed.</li>

<li>Clean the surface of imager and lens to remove particles, if present.</li>

<li>Wear non-powdered gloves during the procedure as powdered gloves leave powder that gets on the gel and fluoresces when exposed to UV light.</li>

<li>The EtBr may be incompletely dissolved.</li>
</ul>
</li>
</ol>
</li>
</ol>
</li>
</ol>

<h2>Appendix 1</h2>

<p class="bold">Fast DNA extraction protocol for <em>N. meningitidis</em> and <em>H. influenzae</em> (gram-negative)</p>

<ol style=" list-style: decimal;">
<li>Dispense 1.0 ml of 10 mM Tris (pH 8.0) buffer into vials and label.</li>

<li>Harvest colonies from 18-24 hour pure cultures of <em>H. influenzae</em> and/or <em>N. meningitidis</em> using a sterile polyester or rayon-tipped swab and swirl the swab in the Tris buffer to make a turbid suspension (equivalent to McFarland 3.0 standard). Be careful not to pick up pieces of agar on swab.</li>

<li>Vortex briefly and boil cell suspension at 100°C for 10 minutes.</li>

<li>Proceed immediately with PCR or store at -20°C.</li>
</ol>

<p>The procedure above is not vigorous enough to completely lyse the more robust cell wall of <em>S. pneumoniae</em>, a gram positive organism. Use the protocol below to prepare DNA from isolates of <em>S. pneumoniae</em>. Also use the protocol below if the identity of the bacterial isolate is unknown.</p>

<p class="bold">Fast DNA extraction protocol for <em>S. pneumoniae</em> (gram-positive)</p>

<ol style=" list-style: decimal;">
<li>Dispense 300 µl of 0.85% NaCl into vials and label.</li>

<li>Harvest colonies from 18-24 hour pure cultures of <em>S. pneumoniae</em> using a sterile polyester or rayon-tipped swab and swirl the swab in the 0.85% NaCl to make a turbid suspension (equivalent to McFarland 3.0 standard). Be careful not to pick up pieces of agar on the swab.</li>

<li>Vortex briefly and incubate at 70ºC for 15 minutes.</li>

<li>Microcentrifuge at 12,000 x <em>g</em> for 2 minutes and remove the supernatant.</li>

<li>Re-suspend in 50 ml TE buffer (10 mM Tris-HCl, 100 mM EDTA, pH 8.0) and add 10 ml mutanolysin (3000 U/ ml)* and 8 ml of hyaluronidase (30 mg/ml)**</li>

<li>Incubate at 37oC for 30 minutes up to 18 hours (overnight).</li>

<li>Heat-inactivate the enzymes in the suspension by boiling at 100ºC for 10 minutes.</li>

<li>Microcentrifuge at 12,000 x <em>g</em> for 4 minutes and remove supernatant for use as DNA template.</li>

<li>Proceed immediately with PCR or store at -20°C.</li>
</ol>

<hr />
<h3>Footnote</h3>

<p class="margin-left-twenty">*Mutanolysin (10,000 U). Dilute in 3.3 ml of TE buffer to make 3000 U/ml stock solution, store at -20°C as 500 µl aliquots.<br />
**Hyaluronidase (100 mg). Dilute in 3.3 ml of TE buffer to make 30 mg/ml solution, store at -20ºC as 500 µl aliquots.</p>

<p>Methods for DNA extraction that will provide purified DNA can be found in <a href="chpt10-pcr.html">Chapter 10: PCR Methods</a>.</p>

<h2>Appendix 2</h2>

<p class="bold">Analysis of PCR products on an agarose gel</p>

<p>To check for a successful PCR amplification, run an aliquot of the end-products on a 1% agarose gel.</p>

<ol style=" list-style: decimal;">
<li>Briefly spin the PCR plate or tubes at 500 x <em>g</em> to ensure all liquid is at the bottom of the wells.</li>

<li>Mix 5 ml of PCR reaction with 1 ml of 6X loading dye (see Table 46 for protocol). It is important to include DNA size markers in one of the wells.</li>

<li>Make a 1% agarose gel. Add 1 g of electrophoresis grade agarose to 100 ml of 1X Tris/Borate/EDTA (TBE) buffer (see below) in a 250 ml flask and melt the agarose in a microwave. Microwave for 1 min, swirl, and then microwave in 15 sec increments with swirling until the agarose is fully melted and the solution is clear. Be cautious as the molten agarose will be extremely hot. Once the agar has cooled to approximately 55°C add 1-2 µl of EtBr (see below) and swirl. Pour into a gel casting box, insert the comb, and allow time for hardening. Remove the comb after the gel has hardened.

<ol>
<li>EtBr is a powerful carcinogen and must be handled with care.</li>
</ol>
</li>

<li>Add 1X TBE buffer to the gel box until the buffer is just over the surface of the gel and pipette the DNA/loading dye mixtures into the wells.</li>

<li>Electrophorese the gel at 50-100 volts for 15-20 minutes or until the Bromophenol Blue dye band is halfway down the gel. The dye runs at approximately the same rate as a 500 base-pair DNA fragment.</li>

<li>Visualize the gel under a UV light and print out or save the image, if possible.</li>

<li>Each reaction should give a single band. If multiple bands are consistently present, annealing temperature optimization may be required.</li>

<li>Store the remainder of the amplicon at -20°C unless proceeding directly to DNA purification.</li>
</ol>

<a id="table46" name="table46"> </a>
<p><span class="bold">Table 46.</span> Protocol for making 6X loading dye</p>

<table class="opt-in table table-striped"><thead class="primary-bg-color">
<tr>
<th scope="col">Reagent</th>
<th scope="col">Needed for 10mls</th>
<th scope="col">Notes</th>
</tr>

</thead><tbody>
<tr>
<td>2.5% Ficoll 400</td>
<td>0.25 ml</td>
<td>&#160;</td>
</tr>

<tr>
<td>11 mM EDTA, pH 8.0</td>
<td>1.1 ml 0.1 M EDTA</td>
<td>See recipe below</td>
</tr>

<tr>
<td>3.3 mM Tris-HCl, pH 8.0</td>
<td>0.3 ml 0.1 M Tris-HCl</td>
<td>See recipe below</td>
</tr>

<tr>
<td>0.017% SDS</td>
<td>0.17 ml 1% SDS</td>
<td>1 g SDS* to 100 ml sterile distilled H<sub>2</sub>O for 1% SDS solution</td>
</tr>

<tr>
<td>0.015% Bromophenol Blue</td>
<td>0.15 ml 1% Bromophenol Blue</td>
<td>0.1 g Bromophenol Blue in 10 ml sterile distilled H<sub>2</sub>O for 1% solution</td>
</tr>

<tr>
<td>sterile distilled H<sub>2</sub>O</td>
<td>8.03 ml sterile distilled H<sub>2</sub>O</td>
<td>Make 0.5-1.0 ml aliquots of the batch of 6X loading dye</td>
</tr>

</tbody></table>

<h3>Footnote</h3>

<p class="margin-left-twenty">*Eye and respiratory protection should be worn when weighing out powdered SDS.</p>

<p class="bold">Stock solutions for reagents:</p>

<p class="bold">EDTA, 0.5 M, pH 8.0 (100 ml)</p>

<ol style=" list-style: decimal;">
<li>Dissolve 18.6 g EDTA in 70 ml ddH<sub>2</sub>O.</li>

<li>Adjust pH to 8.0 with 10 M NaOH (~5 ml).</li>

<li>Add ddH<sub>2</sub>O to 100 ml.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<p class="bold">EDTA, 0.1 M, pH 8.0 (100ml)</p>

<ol style=" list-style: decimal;">
<li>Dissolve 3.7 g EDTA in 70 ml ddH<sub>2</sub>O.</li>

<li>Adjust pH to 8.0 with 10 M NaOH.</li>

<li>Add ddH<sub>2</sub>O to 100 ml.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<p class="bold">Ethidium bromide (EtBr), 10 mg/ml</p>

<ol style=" list-style: decimal;">
<li>Dissolve 0.2 g ethidium bromide in 20 ml ddH<sub>2</sub>O.</li>

<li>Mix well and store at 4°C in the dark in 1 ml aliquots.</li>

<li>EtBr is a powerful mutagen and should be handled with care.</li>
</ol>

<p class="bold">Tris-HCl, 0.1 M, pH 8.0</p>

<ol style=" list-style: decimal;">
<li>Dissolve 1.2 g Tris base in 80 ml ddH<sub>2</sub>O.</li>

<li>Adjust to pH 8.0 with concentrated HCl.</li>

<li>Mix and add sterile distilled H<sub>2</sub>O to 100 ml.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<p class="bold">TBE (Tris/borate/EDTA) electrophoresis buffer, 10X stock solution*</p>

<ol style=" list-style: decimal;">
<li>To 900 ml of ddH<sub>2</sub>O add:

<ul>
<li>108 g Tris base (890 mM).</li>

<li>55 g boric acid (890 mM).</li>

<li>40 ml 0.5 M EDTA, pH 8.0 (20mM).</li>
</ul>
</li>

<li>Add ddH<sub>2</sub>O to a total volume of 1000 ml.</li>

<li>Autoclave or filter sterilize.</li>
</ol>

<p>*10X TBE will have to be diluted 1:10 to 1X in ddH<sub>2</sub>O before use.</p>

<h2>References</h2>

<ol style=" list-style: decimal;">
<li><a id="ref1" name="ref1"> </a>Brehony, C., K. A. Jolley, and M. C. J. Maiden. 2007. Multilocus sequence typing for global surveillance of meningococcal disease. FEMS Microbiology Reviews <span class="bold">31:</span>15-26.</li>

<li><a id="ref2" name="ref2"> </a>Cafini, F., R. del Campo, L. Alou, D. Sevillano, M. I. Morosini, F. Baquero, and J. Prieto. 2006. Alterations of the penicillin-binding proteins and murM alleles of clinical <em>Streptococcus pneumoniae</em> isolates with high-level resistance to amoxicillin in Spain. The Journal of Antimicrobial Chemotherapy <span class="bold">57:</span>224-229.</li>

<li><a id="ref3" name="ref3"> </a>Claus, H., J. Elias, C. Meinhardt, M. Frosch, and U. Vogel. 2007. Deletion of the meningococcal <em>fetA</em> gene used for antigen sequence typing of invasive and commensal isolates from Germany: Frequencies and mechanisms. Journal of Clinical Microbiology <span class="bold">45:</span>2960-2964.</li>

<li><a id="ref4" name="ref4"> </a>Diggle, M. A., and S. C. Clarke. 2006. Molecular methods for the detection and characterization of <em>Neisseria meningitidis</em>. Expert Review of Molecular Diagnostics <span class="bold">6:</span>79-87.</li>

<li><a id="ref5" name="ref5"> </a>Dowson, C. G., A. Hutchison, and B. G. Spratt. 1989. Extensive re-modelling of the transpeptidase domain of penicillin-binding protein 2B of a penicillin-resistant South African isolate of <em>Streptococcus pneumoniae</em>. Molecular Microbiology <span class="bold">3:</span>95-102.</li>

<li><a id="ref6" name="ref6"> </a>Dyer, D. W., West, E.P., McKenna, W., Thompson, S.A., and Sparling, P.F. . 1998. A pleiotrophic iron-uptake mutant of <em>Neisseria meningitidis</em> lacks a 70-kilodalton iron-regulated protein. Infection and Immunity <span class="bold">56:</span>977-983.</li>

<li><a id="ref7" name="ref7"> </a>Enright, M. C., and B. G. Spratt. 1999. Multilocus sequence typing. Trends in Microbiology <span class="bold">7:</span>482-487.</li>

<li><a id="ref8" name="ref8"> </a>Gherardi, G., C. G. Whitney, R. R. Facklam, and B. Beall. 2000. Major related sets of antibiotic-resistant Pneumococci in the United States as determined by pulsed-field gel electrophoresis and <em>pbp1a-pbp2b-pbp2x-dhf</em> restriction profiles. Journal of Infectious Diseases <span class="bold">181:</span>216-229.</li>

<li><a id="ref9" name="ref9"> </a>Giuliani, M. M., J. Adu-Bobie, M. Comanducci, B. Arico, S. Savino, L. Santini, B. Brunelli, S. Bambini, A. Biolchi, B. Capecchi, E. Cartocci, L. Ciucchi, F. Di Marcello, F. Ferlicca, B. Galli, E. Luzzi, V. Masignani, D. Serruto, D. Veggi, M. Contorni, M. Morandi, A. Bartalesi, V. Cinotti, D. Mannucci, F. Titta, E. Ovidi, J. A. Welsch, D. Granoff, R. Rappuoli, and M. Pizza. 2006. A universal vaccine for serogroup B meningococcus. Proceedings of the National Academy of Sciences of the United States of America <span class="bold">103:</span>10834-10839.</li>

<li><a id="ref10" name="ref10"> </a>Hollingshead, S. K., L. Baril, S. Ferro, J. King, P. Coan, and D. E. Briles. 2006. Pneumococcal surface protein A (PspA) family distribution among clinical isolates from adults over 50 years of age collected in seven countries. Journal of Medical Microbiology <span class="bold">55:</span>215-221.</li>

<li><a id="ref11" name="ref11"> </a>Hollingshead, S. K., R. Becker, and D. E. Briles. 2000. Diversity of PspA: mosaic genes and evidence for past recombination in <em>Streptococcus pneumoniae</em>. Infection and Immunity <span class="bold">68:</span>5889-5900.</li>

<li><a id="ref12" name="ref12"> </a>Katz, L. S., C. R. Bolen, B. H. Harcourt, S. Schmink, X. Wang, A. Kislyuk, R. T. Taylor, L. W. Mayer, and I. K. Jordan. 2009. Meningococcus genome informatics platform: a system for analyzing multilocus sequence typing data. Nucleic Acids Research <span class="bold">37:</span>W606-11.</li>

<li><a id="ref13" name="ref13"> </a>Liao, J.-C., C.-C. Li, and C.-S. Chiou. 2006. Use of a multilocus variable-number tandem repeat analysis method for molecular subtyping and phylogenetic analysis of <em>Neisseria meningitidis</em> isolates. BioMed Central Microbiology <span class="bold">6:</span>44.</li>

<li><a id="ref14" name="ref14"> </a>Lukinmaa, S., U.-M. Nakari, M. Eklund, and A. Siitonen. 2004. Application of molecular genetic methods in diagnostics and epidemiology of food-borne bacterial pathogens. Acta Pathologica, Microbiologica et Immunologica Scandinavica (APMIS) <span class="bold">112:</span>908-929.</li>

<li><a id="ref15" name="ref15"> </a>Maiden, M. C., J. A. Bygraves, E. Feil, G. Morelli, J. E. Russell, R. Urwin, Q. Zhang, J. Zhou, K. Zurth, D. A. Caugant, I. M. Feavers, M. Achtman, and B. G. Spratt. 1998. Multilocus sequence typing: a portable approach to the identification of clones within populations of pathogenic microorganisms. Proceedings of the National Academy of Sciences of the United States of America <span class="bold">95:</span>3140-3145.</li>

<li><a id="ref16" name="ref16"> </a>Maiden, M. C., J. Suker, A. J. McKenna, J. A. Bygraves, and I. M. Feavers. 1991. Comparison of the class 1 outer membrane proteins of eight serological reference strains of <em>Neisseria meningitidis</em>. Molecular Microbiology <span class="bold">5:</span>727-736.</li>

<li><a id="ref17" name="ref17"> </a>Marsh, J. W., M. M. O'Leary, K. A. Shutt, and L. H. Harrison. 2007. Deletion of <em>fetA</em> Gene Sequences in Serogroup B and C <em>Neisseria meningitidis</em> Isolates. Journal of Clinical Microbiology <span class="bold">45:</span>1333-1335.</li>

<li><a id="ref18" name="ref18"> </a>Masignani, V., M. Comanducci, M. M. Giuliani, S. Bambini, J. Adu-Bobie, B. Arico, B. Brunelli, A. Pieri, L. Santini, S. Savino, D. Serruto, D. Litt, S. Kroll, J. A. Welsch, D. M. Granoff, R. Rappuoli, and M. Pizza. 2003. Vaccination against <em>Neisseria meningitidis</em> Using Three Variants of the Lipoprotein GNA1870. Journal of Experimental Medicine <span class="bold">197:</span>789-799.</li>

<li><a id="ref19" name="ref19"> </a>McGuinness, B. T., P. R. Lambden, and J. E. Heckels. 1993. Class 1 outer membrane protein of <em>Neisseria meningitidis</em>: epitope analysis of the antigenic diversity between strains, implications for subtype definition and molecular epidemiology. Molecular Microbiology <span class="bold">7:</span>505-514.</li>

<li><a id="ref20" name="ref20"> </a>Meats, E., E. J. Feil, S. Stringer, A. J. Cody, R. Goldstein, J. S. Kroll, T. Popovic, and B. G. Spratt. 2003. Characterization of encapsulated and noncapsulated <em>Haemophilus influenzae</em> and determination of phylogenetic relationships by multilocus sequence typing. Journal of Clinical Microbiology <span class="bold">41:</span>1623-1636.</li>

<li><a id="ref21" name="ref21"> </a>Muñoz, R., T. J. Coffey, M. Daniels, C. G. Dowson, G. Laible, J. Casal, R. Hakenbeck, M. Jacobs, J. M. Musser, and B. G. Spratt. 1991. Intercontinental spread of a multiresistant clone of serotype 23F <em>Streptococcus pneumoniae</em>. Journal of Infectious Diseases <span class="bold">164:</span>302-306.</li>

<li><a id="ref22" name="ref22"> </a>Murphy, E., L. Andrew, K. L. Lee, D. A. Dilts, L. Nunez, P. S. Fink, K. Ambrose, R. Borrow, J. Findlow, M. K. Taha, A. E. Deghmane, P. Kriz, M. Musilek, J. Kalmusova, D. A. Caugant, T. Alvestad, L. W. Mayer, C. T. Sacchi, X. Wang, D. Martin, A. von Gottberg, M. du Plessis, K. P. Klugman, A. S. Anderson, K. U. Jansen, G. W. Zlotnick, and S. K. Hoiseth. 2009. Sequence diversity of the factor H binding protein vaccine candidate in epidemiologically relevant strains of serogroup B <em>Neisseria meningitidis</em>. Journal of Infectious Diseases <span class="bold">200:</span>379-389.</li>

<li><a id="ref23" name="ref23"> </a>Schneider, M. C., R. M. Exley, H. Chan, I. Feavers, Y.-H. Kang, R. B. Sim, and C. M. Tang. 2006. Functional significance of factor H binding to <em>Neisseria meningitidis</em>. Journal of Immunology <span class="bold">176:</span>7566-7575.</li>

<li><a id="ref24" name="ref24"> </a>Schouls, L. M., A. van der Ende, M. Damen, and I. van de Pol. 2006. Multiple-Locus Variable-Number Tandem Repeat analysis of <em>Neisseria meningitidis</em> yields groupings similar to those obtained by Multilocus Sequence Typing. Journal of Clinical Microbiology <span class="bold">44:</span>1509-1518.</li>

<li><a id="ref25" name="ref25"> </a>Singh, A., R. V. Goering, S. Simjee, S. L. Foley, and M. J. Zervos. 2006. Application of molecular techniques to the study of hospital infection. Clinical Microbiology Reviews <span class="bold">19:</span>512-530.</li>

<li><a id="ref26" name="ref26"> </a>Smith, A. M., and K. P. Klugman. 1998. Alterations in PBP 1A essential-for high-level penicillin resistance in <em>Streptococcus pneumoniae</em>. Antimicrobial Agents and Chemotherapy <span class="bold">42:</span>1329-1333.</li>

<li><a id="ref27" name="ref27"> </a>Tenover, F. C., R. D. Arbeit, R. V. Goering, P. A. Mickelsen, B. E. Murray, D. H. Persing, and B. Swaminathan. 1995. Interpreting chromosomal DNA restriction patterns produced by pulsed-field gel electrophoresis: criteria for bacterial strain typing. Journal of Clinical Microbiology <span class="bold">33:</span>2233-2239.</li>

<li><a id="ref28" name="ref28"> </a>Thompson, E. A., I. M. Feavers, and M. C. Maiden. 2003. Antigenic diversity of meningococcal enterobactin receptor FetA, a vaccine component. Microbiology <span class="bold">149:</span>1849-1858.</li>

<li><a id="ref29" name="ref29"> </a>Urwin, R. 2001. Nucleotide sequencing of antigen genes of <em>Neisseria meningitidis</em>, p. 157-167. <em>In</em> M. C. J. Maiden and A. J. Pollard (ed.), Methods in Molecular Medicine., vol. Meningococcal Disease: Methods and Protocols. Humana Press Inc., Totowa, New Jersey.</li>

<li><a id="ref30" name="ref30"> </a>Urwin, R., and M. C. Maiden. 2003. Multi-locus sequence typing: a tool for global epidemiology. Trends in Microbiology <span class="bold">11:</span>479-487.</li>

<li><a id="ref31" name="ref31"> </a>Urwin, R., J. E. Russell, E. A. Thompson, E. C. Holmes, I. M. Feavers, and M. C. Maiden. 2004. Distribution of surface protein variants among hyperinvasive meningococci: implications for vaccine design. Infection and Immunity <span class="bold">72:</span>5955-5962.</li>

<li><a id="ref32" name="ref32"> </a>van der Ende, A., C. T. Hopman, and J. Dankert. 1999. Deletion of <em>porA</em> by recombination between clusters of repetitive extragenic palindromic sequences in <em>Neisseria meningitidis</em>. Infection and Immunity <span class="bold">67:</span>2928-2934.</li>

<li><a id="ref33" name="ref33"> </a>Vela Coral, M. C., N. Fonseca, E. Castañeda, J. L. Di Fabio, S. K. Hollingshead, and D. E. Briles. 2001. Pneumococcal surface protein A of invasive <em>Streptococcus pneumoniae</em> isolates from Colombian children. Emerging Infectious Diseases <span class="bold">7:</span>832-836.</li>

<li><a id="ref34" name="ref34"> </a>Yazdankhah, S. P., B. A. Lindstedt, and D. A. Caugant. 2005. Use of variable-number tandem repeats to examine genetic diversity of <em>Neisseria meningitidis</em>. Journal of Clinical Microbiology <span class="bold">43:</span>1699-1705.</li>
</ol>

<span class="toTop"><span class="icon-angle-up"><!-- --></span>&#160;<a href="chpt12-molecular-typing-methods.html#">Top of Page</a></span>
<p><a href="index.html">Back to Laboratory Methods Manual</a></p>
             					            					            						</div>
            					            					            						</div>
            													        						                                                                                        						                                                                								<!-- Related Nav Area -->
								<div class="nav-related hidden-three hidden-four">
									




									



		    <div class="module-typeH">
			<h3>Related Links</h3>
			<div>
															<ul class="list-block">																									<li>					
<a href="../../vaccines/schedules/index.html">Vaccine Schedules</a>					</li>																				<li>					
<a href="../../vaccines/who/teens/index.html">Preteen &amp; Teen Vaccines</a>					</li>																				<li>					
<a href="../../meningococcal/index.html">Meningococcal Disease</a>					</li>																									<li>					
<a href="../../sepsis/index.html">Sepsis</a>					</li>										</ul>													</div>
	</div>
	  

								</div>
    							<!-- Persistent Content Area -->
    							<div id="content-secondary">
                                    



            						




    							</div><!-- /end #content-secondary -->
    						</div><!-- /end #contentArea -->
    					</div><!-- /end body -->	
					</div>
					<div class="row">
						<div class="span24 visible-one">
														<div class="pageoptionsContainer formatOptions formatOptionsOff"><span id="po-format-label-bottom" class="select-label">Format:</span><div id="po-format-bottom" class="format pageoptions pull-right"><div class="select-box"><a href="chpt12-molecular-typing-methods.html#">Select one</a><span class="icon-angle-down pull-right poarrow"></span></div><ul class="drop-down" aria-labelledby="po-format-label-bottom"><li>
        			    	<a class="noDecoration" href="https://www.cdc.gov/meningitis/lab-manual/chpt12-molecular-typing-methods.pdf" target="_blank"><span class="sprite-16-pdf"></span> PDF [473 KB]</a>
      </li></ul></div></div>													</div>
					</div>
					<div class="row">
						<div class="span24">
                            <div id="plugin-legend" class="pluginOff">
	<h5>File Formats Help:</h5><span><a href="../../Other/plugins/index.html">How do I view different file formats (PDF, DOC, PPT, MPEG) on this site?</a></span>
	<ul>
		<li class="plugin-pdf"><a href="../../Other/plugins/index.html#pdf"><span class="sprite-16-pdf">Adobe PDF file</span></a></li>
		<li class="plugin-ppt"><a href="../../Other/plugins/index.html#ppt"><span class="sprite-16-ppt">Microsoft PowerPoint file</span></a></li>
		<li class="plugin-word"><a href="../../Other/plugins/index.html#doc"><span class="sprite-16-word">Microsoft Word file</span></a></li>
		<li class="plugin-excel"><a href="../../Other/plugins/index.html#xls"><span class="sprite-16-excel">Microsoft Excel file</span></a></li>
		<li class="plugin-wmv"><a href="../../Other/plugins/index.html#wmv"><span class="sprite-16-wmv">Audio/Video file</span></a></li>
		<li class="plugin-qt"><a href="../../Other/plugins/index.html#qt"><span class="sprite-16-qt">Apple Quicktime file</span></a></li>
		<li class="plugin-real"><a href="../../Other/plugins/index.html#ram"><span class="sprite-16-rp">RealPlayer file</span></a></li>
		<li class="plugin-text"><a href="../../Other/plugins/index.html#text"><span class="sprite-16-txt">Text file</span></a></li>
		<li class="plugin-zip"><a href="../../Other/plugins/index.html#zip"><span class="sprite-16-zip">Zip Archive file</span></a></li>
		<li class="plugin-sas"><a href="../../Other/plugins/index.html#sas"><span class="sprite-16-sas">SAS file</span></a></li>
		<li class="plugin-epub"><a href="../../Other/plugins/index.html#epub"><span class="sprite-16-ebook">ePub file</span></a></li>
		<li class="plugin-ris"><a href="../../Other/plugins/index.html#ris"><span class="sprite-16-ris">RIS file</span></a></li>
	</ul>
</div>
						</div>
					</div>
					<!-- date stamp -->
					<div id="datestamp" class="row" itemscope="itemscope" itemtype="http://schema.org/WebPage">
						<div class="span24">
    						<ul> 
            					            						<li class="last-reviewed">Page last reviewed: <span itemprop="lastReviewed">April 15, 2016</span></li>
            													            					            						<li class="last-updated">Page last updated: <span itemprop="dateModified">March 15, 2012</span></li>
            					        							<li class="content-source">Content source:
    									



<ul>
<li itemprop="sourceOrganization" itemscope="itemscope" itemtype="http://schema.org/Organization"><a href="../../ncird.1.html" itemprop="url">National Center for Immunization and Respiratory Diseases</a></li>
</ul>


        							</li>
									



    						</ul>
						</div>
					</div>					
				</div><!-- /end #content -->
			</div><!-- /.container for content -->					
			<!-- footer -->
				<footer id="footer" class="cdcSlimFooter">
	
 <!-- include eng footerSubSocialMedia.html -->
<div class="socialMedia">
	<div class="container">
		<nav class="followcdc">
			<ul>
				<li><a href="../../Other/emailupdates.1.html"><span class="sprite-24-govd"></span><span class="tp-sr-only">Email</span></a></li>
				<li><a href="https://www.facebook.com/CDC" target="_blank"><span class="sprite-24-facebook"></span><span class="tp-sr-only">Recommend</span></a></li>
				<li><a href="https://twitter.com/CDCgov" target="_blank"><span class="sprite-24-twitter"></span><span class="tp-sr-only">Tweet</span></a></li>
				<li><a href="https://www.youtube.com/user/CDCstreamingHealth" target="_blank"><span class="sprite-24-youtube"></span><span class="tp-sr-only">YouTube</span></a></li>
				<li><a href="https://www.instagram.com/CDCgov/" target="_blank"><span class="sprite-24-instagram"></span><span class="tp-sr-only">Instagram</span></a></li>
			</ul>
		</nav>
		<nav class="cdcmedia">
			<ul>
				<li><a href="https://www2c.cdc.gov/podcasts"><span class="sprite-24-listen"></span><span class="tp-sr-only">Listen</span></a></li>
				<li><a href="../../cdctv.1.html"><span class="sprite-24-watch"></span><span class="tp-sr-only">Watch</span></a></li>
				<li><a href="https://www2c.cdc.gov/podcasts/rss.asp"><span class="sprite-24-rss"></span><span class="tp-sr-only">RSS</span></a></li>
			</ul>
		</nav>
	</div>
</div>
 <!-- /include eng footerSubSocialMedia.html -->

	<div class="container">
		<div class="cdcLinks">
			<div class="container">
				<ul>
					
<!-- include eng footerSubCdcLinks.html-->
<li>
	<nav>
		<h6 class="hidden-four hidden-three">ABOUT <span class="icon-plus visible-one pull-right"></span></h6>
		<ul class="hidden-one">
			<li><a href="../../about/default.htm">About CDC</a></li>
			<li><a href="https://jobs.cdc.gov">Jobs</a></li>
			<li><a href="../../funding.1.html">Funding</a></li>
		</ul>
	</nav>
</li>
<li>
	<nav>
		<h6 class="hidden-four hidden-three">LEGAL <span class="icon-plus visible-one pull-right"></span></h6>
		<ul class="hidden-one">
			<li><a href="../../Other/policies.html">Policies</a></li>
			<li><a href="../../Other/privacy.html">Privacy</a></li>
			<li><a href="../../od/foia.1.html">FOIA</a></li>
			<li><a href="../../eeo/nofearact/index.htm">No Fear Act</a></li>
			<li><a href="https://oig.hhs.gov" class="no-link">OIG</a></li>
		</ul>
	</nav>
</li>
<!--/ include eng footerSubCdcLinks.html-->

				</ul>
			</div>
		</div>
		<div class="cdcContact">
			<div class="container">
				<ul>
					<li class="local-contact">
						
 <!-- include eng footerSubGlobalContact1.html -->
<section>
	<address class="address" itemscope="itemscope" itemtype="https://schema.org/Organization">
		<span class="value" itemprop="address" itemscope="itemscope" itemtype="https://schema.org/PostalAddress">
			<span itemprop="streetAddress">1600 Clifton Road</span>
			<span itemprop="addressLocality">Atlanta</span>,
			<span itemprop="addressRegion">GA</span>
			<span itemprop="postalCode">30329-4027</span>
			<span itemprop="addressCountry">USA</span>
		</span><br />
		<span class="value" itemprop="telephone">800-CDC-INFO (800-232-4636)</span>,
		<span class="value" itemprop="telephone">TTY: 888-232-6348</span><br />
		<span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://wwwn.cdc.gov/dcs/RequestForm.aspx" class=" default no-link">Email CDC-INFO</a></span>
		</span>
	</address>
</section>
 <!-- /include eng footerSubGlobalContact1.html -->

					</li>
					<li class="contact">
						
 <!-- include eng footerSubGlobalContact2.html -->
<section>
	<address class="address" itemscope="itemscope" itemtype="https://schema.org/Organization">
		<span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://www.hhs.gov" class="no-link">U.S. Department of Health & Human Services</a></span>
		</span><br />
        <span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
        	<span itemprop="url"><a href="https://www.hhs.gov/open" class="no-link">HHS/Open</a></span>
        </span><br />
        <span class="value" itemprop="contactPoint" itemscope="itemscope" itemtype="https://schema.org/ContactPoint">
			<span itemprop="url"><a href="https://www.usa.gov/" class="no-link">USA.gov</a></span>
		</span>
	</address>
</section>
 <!-- /include eng footerSubGlobalContact2.html -->

					</li>
				</ul>
			</div>
		</div>
	</div>
</footer>			<!-- /#footer -->
			<div id="totop" class="off visible-one">Top</div>
        </div>
		<script>page_timing.libs_start = Date.now();</script><script src="../../TemplatePackage/3.0/js/libs.min.js@3.2.1.7"></script><script>page_timing.libs_end = Date.now();page_timing.core_start = Date.now();</script><script src="../../TemplatePackage/3.0/js/core.min.js@3.2.1.7"></script><script>page_timing.core_end = Date.now();</script><!-- Start Site Settings Overrides -->
<script>
</script>
<!-- End Site Settings Overrides --><script>page_timing.app_start = Date.now();</script><script src="../../TemplatePackage/3.0/js/app.min.js@3.2.1.7"></script><script>page_timing.app_end = Date.now();</script>		
		        			<!-- Begin Survey scripts -->
<script src="../../JScript/foresee/foresee-trigger.js"></script>
<script src="../../JScript/foresee-variables.js"></script>
<!-- End Survey scripts -->
			
            <script>if (typeof page_timing !== 'undefined') { page_timing.metrics_start = Date.now(); }</script>
<!-- xxxxxxxxxxxxxxxxxx METRICS code begins here xxxxxxxxxxxxxxxxxxxxxxxxxxx-->
<div id="metrics">

<!-- Begin Google DAP inclusion -->
<script type="text/javascript" id="_fed_an_ua_tag" src="../../JScript/metrics/google/Universal-Federated-Analytics-Min.js@agency=HHS&amp;subagency=CDC"></script>
<!-- End Google DAP inclusion -->

<!-- Begin Template Package Custom Interactions -->
<script type="text/javascript" src="../../JScript/metrics/rd_custom_interactions.js"></script>
<!-- End Template Package Custom Interactions -->

<!-- SiteCatalyst code version: H.21. -->
<script src="../../JScript/metrics/topic_levels.js"></script>
<script src="../../JScript/metrics/s_code_v21_cdcgov.js"></script>
<script>

	enquire.register(CDC.Global.selectors.vp1, function() {
		s.prop49 = 1;
	});
	enquire.register(CDC.Global.selectors.vp2, function() {
		s.prop49 = 2;
	});
	enquire.register(CDC.Global.selectors.vp3, function() {
		s.prop49 = 3;
	});
	enquire.register(CDC.Global.selectors.vp4, function() {
		s.prop49 = 4;
	});

            
            


s.pageName=document.title
s.channel="Meningitis"
siteCatalyst.setLevel1("ncird"); <!--formerly NCIRD before 8/15/2012-->
siteCatalyst.setLevel2("meningitis"); <!--formerly Meningitis before 8/15/2012-->
siteCatalyst.setLevel3("lab"); 
siteCatalyst.setLevel4("dbd"); <!-- "dbd-dvd" BEFORE 12/8/2014, "hcso"” BETWEEN 12/8/2014 and 10/2/2015-->

            
                        
            s.prop2=window.location.href;
s.prop26=document.title;
s.prop30=document.title;
s.prop31=window.location.href;
s.prop46=window.location.href;
s.server=window.location.hostname;

// Update the level variables here.
updateVariables(s);

/************* DO NOT ALTER ANYTHING BELOW THIS LINE ! **************/
var s_code=s.t();



if(s_code) {
	document.write(s_code);
}



if (document.location.protocol) {


	var cookieManagment = {
		getUuid : function (strPrefix) {
			strPrefix = strPrefix || "";
			var d = new Date();
			var seconds = Math.round(d.getTime() / 1000);
			return strPrefix + Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1) + "-" + seconds.toString();
		},
		setCookie : function (cname, cvalue, exdays, domain) {
			domain = domain || ".cdc.gov";
			var d = new Date();
			d.setTime(d.getTime() + (exdays*24*60*60*1000));
			var expires = "expires="+ d.toUTCString();
			document.cookie = cname + "=" + cvalue + ";" + expires + ";domain=" + domain + ";path=/";
		},
		getCookie : function (cname) {
			var name = cname + "=";
			var ca = document.cookie.split(';');
			var i = ca.length;
			while (i --) {
				var c = ca[i];
				while (c.charAt(0)==' ') {
					c = c.substring(1);
				}
				if (c.indexOf(name) == 0) {
					return c.substring(name.length,c.length);
				}
			}
			return "";
		},
		init : function () {



			var currCookieVal = cookieManagment.getCookie('cdc_s_vi');

			if (!currCookieVal.length || currCookieVal.indexOf('[CS]v1') != -1) {
				var newCookieVal = cookieManagment.getUuid('[CS]v2') + "-" + cookieManagment.getUuid('');
				cookieManagment.setCookie("cdc_s_vi", newCookieVal, 7);
				return newCookieVal;
			}
			return currCookieVal;
		}
	};


	var cdcSessionID = cookieManagment.init();

	s3 = s_gi("cdcgovvistorid");

	var sKey, override = {
		fun: "cdcgovvistorid",
		trackingServer: "cdc.112.2o7.net"
	};

	s3.visitorID = cdcSessionID;

	for (sKey in s) {
		if (s.hasOwnProperty(sKey)) {
			if (override.hasOwnProperty(sKey)) {
				s3[sKey] = override[sKey];

			} else {

			}
		}
	}
	updateVariables(s3);


	var s_code_uvid = s3.t();
	if(s_code_uvid) {
		document.write(s_code_uvid);
	}
}




</script>
<script type="text/javascript">
if(navigator.appVersion.indexOf('MSIE')>=0) { document.write(unescape('%3C')+'\!-'+'-'); }
</script>
<noscript><a href="http://www.omniture.com"><img src="https://smtrics.cdc.gov/b/ss/cdcgov/1/H.21--NS/01/H.21--NS/0" height="1" width="1" alt="Web Analytics" /></a></noscript><!--/DO NOT REMOVE/-->
<!-- End SiteCatalyst code version: H.21. -->
<!-- Begin Template Package DigitalGov Search Support -->
<script type="text/javascript" src="../../JScript/rd_search.js"></script>
<!-- End Template Package DigitalGov Search Support -->
</div>
<!-- xxxxxxxxxxxxxxxxxx METRICS code ends here xxxxxxxxxxxxxxxxxxxxxxxxxxx-->
<script>if (typeof page_timing !== 'undefined') { page_timing.metrics_end = Date.now(); }</script>
				


<script>
<!--//
 $(document).ready(function () {
  var cdcLeftNav = new CDC.LeftNav("nav-primary");
  cdcLeftNav.render();
 });
//-->
</script>

    	




						    </body>
	</html>
